// @clerk/clerk-expo/dist/cache/dummy-data/client-resource.js
"use strict";
var __defProp = Object.defineProperty;
var __getOwnPropDesc = Object.getOwnPropertyDescriptor;
var __getOwnPropNames = Object.getOwnPropertyNames;
var __hasOwnProp = Object.prototype.hasOwnProperty;
var __export = (target, all) => {
  for (var name in all)
    __defProp(target, name, { get: all[name], enumerable: true });
};
var __copyProps = (to, from, except, desc) => {
  if (from && typeof from === "object" || typeof from === "function") {
    for (let key of __getOwnPropNames(from))
      if (!__hasOwnProp.call(to, key) && key !== except)
        __defProp(to, key, { get: () => from[key], enumerable: !(desc = __getOwnPropDesc(from, key)) || desc.enumerable });
  }
  return to;
};
var __toCommonJS = (mod) => __copyProps(__defProp({}, "__esModule", { value: true }), mod);
var client_resource_exports = {};
__export(client_resource_exports, {
  DUMMY_CLERK_CLIENT_RESOURCE: () => DUMMY_CLERK_CLIENT_RESOURCE
});
module.exports = __toCommonJS(client_resource_exports);
const DUMMY_CLERK_CLIENT_RESOURCE = {
  object: "client",
  status: null,
  id: "client_DUMMY_ID",
  sessions: [],
  sign_up: {
    object: "sign_up",
    id: "",
    status: null,
    required_fields: [],
    optional_fields: [],
    missing_fields: [],
    unverified_fields: [],
    verifications: {
      email_address: {
        object: "verification",
        id: "",
        status: null,
        strategy: null,
        nonce: null,
        message: null,
        external_verification_redirect_url: null,
        attempts: null,
        expire_at: null,
        error: { code: "", message: "", meta: {} },
        verified_at_client: null,
        next_action: "",
        supported_strategies: []
      },
      phone_number: {
        object: "verification",
        id: "",
        status: null,
        strategy: null,
        nonce: null,
        message: null,
        external_verification_redirect_url: null,
        attempts: null,
        expire_at: null,
        error: { code: "", message: "", meta: {} },
        verified_at_client: null,
        next_action: "",
        supported_strategies: []
      },
      web3_wallet: {
        object: "verification",
        id: "",
        status: null,
        strategy: null,
        nonce: null,
        message: null,
        external_verification_redirect_url: null,
        attempts: null,
        expire_at: null,
        error: { code: "", message: "", meta: {} },
        verified_at_client: null,
        next_action: "",
        supported_strategies: []
      },
      external_account: {
        object: "verification",
        id: "",
        status: null,
        strategy: null,
        nonce: null,
        message: null,
        external_verification_redirect_url: null,
        attempts: null,
        expire_at: null,
        error: { code: "", message: "", meta: {} },
        verified_at_client: null
      }
    },
    username: null,
    first_name: null,
    last_name: null,
    email_address: null,
    phone_number: null,
    has_password: false,
    unsafe_metadata: {},
    created_session_id: null,
    created_user_id: null,
    abandon_at: null,
    web3_wallet: null,
    legal_accepted_at: null
  },
  sign_in: {
    object: "sign_in",
    id: "",
    status: null,
    supported_identifiers: [],
    supported_first_factors: [],
    supported_second_factors: null,
    first_factor_verification: {
      object: "verification",
      id: "",
      status: null,
      strategy: null,
      nonce: null,
      message: null,
      external_verification_redirect_url: null,
      attempts: null,
      expire_at: null,
      error: { code: "", message: "", meta: {} },
      verified_at_client: null
    },
    second_factor_verification: {
      object: "verification",
      id: "",
      status: null,
      strategy: null,
      nonce: null,
      message: null,
      external_verification_redirect_url: null,
      attempts: null,
      expire_at: null,
      error: { code: "", message: "", meta: {} },
      verified_at_client: null
    },
    identifier: null,
    created_session_id: null,
    user_data: {}
  },
  last_active_session_id: null,
  cookie_expires_at: null,
  last_authentication_strategy: null,
  created_at: (/* @__PURE__ */ new Date()).getTime(),
  updated_at: (/* @__PURE__ */ new Date()).getTime()
};
// Annotate the CommonJS export names for ESM import in node:
0 && (module.exports = {
  DUMMY_CLERK_CLIENT_RESOURCE
});
