// @clerk/clerk-expo/dist/cache/index.js
"use strict";
var __defProp = Object.defineProperty;
var __getOwnPropDesc = Object.getOwnPropertyDescriptor;
var __getOwnPropNames = Object.getOwnPropertyNames;
var __hasOwnProp = Object.prototype.hasOwnProperty;
var __export = (target, all) => {
  for (var name in all)
    __defProp(target, name, { get: all[name], enumerable: true });
};
var __copyProps = (to, from, except, desc) => {
  if (from && typeof from === "object" || typeof from === "function") {
    for (let key of __getOwnPropNames(from))
      if (!__hasOwnProp.call(to, key) && key !== except)
        __defProp(to, key, { get: () => from[key], enumerable: !(desc = __getOwnPropDesc(from, key)) || desc.enumerable });
  }
  return to;
};
var __toCommonJS = (mod) => __copyProps(__defProp({}, "__esModule", { value: true }), mod);
var cache_exports = {};
__export(cache_exports, {
  ClientResourceCache: () => import_ResourceCache.ClientResourceCache,
  DUMMY_CLERK_CLIENT_RESOURCE: () => import_dummy_data.DUMMY_CLERK_CLIENT_RESOURCE,
  DUMMY_CLERK_ENVIRONMENT_RESOURCE: () => import_dummy_data.DUMMY_CLERK_ENVIRONMENT_RESOURCE,
  EnvironmentResourceCache: () => import_ResourceCache.EnvironmentResourceCache,
  MemoryTokenCache: () => import_MemoryTokenCache.MemoryTokenCache,
  SessionJWTCache: () => import_ResourceCache.SessionJWTCache,
  TokenCache: () => import_types.TokenCache
});
module.exports = __toCommonJS(cache_exports);
var import_types = require("./types");
var import_MemoryTokenCache = require("./MemoryTokenCache");
var import_ResourceCache = require("./ResourceCache");
var import_dummy_data = require("./dummy-data");
// Annotate the CommonJS export names for ESM import in node:
0 && (module.exports = {
  ClientResourceCache,
  DUMMY_CLERK_CLIENT_RESOURCE,
  DUMMY_CLERK_ENVIRONMENT_RESOURCE,
  EnvironmentResourceCache,
  MemoryTokenCache,
  SessionJWTCache,
  TokenCache
});
