// @clerk/localizations/dist/es-MX.js
"use strict";
var __defProp = Object.defineProperty;
var __getOwnPropDesc = Object.getOwnPropertyDescriptor;
var __getOwnPropNames = Object.getOwnPropertyNames;
var __hasOwnProp = Object.prototype.hasOwnProperty;
var __export = (target, all) => {
  for (var name in all)
    __defProp(target, name, { get: all[name], enumerable: true });
};
var __copyProps = (to, from, except, desc) => {
  if (from && typeof from === "object" || typeof from === "function") {
    for (let key of __getOwnPropNames(from))
      if (!__hasOwnProp.call(to, key) && key !== except)
        __defProp(to, key, { get: () => from[key], enumerable: !(desc = __getOwnPropDesc(from, key)) || desc.enumerable });
  }
  return to;
};
var __toCommonJS = (mod) => __copyProps(__defProp({}, "__esModule", { value: true }), mod);

// src/es-MX.ts
var es_MX_exports = {};
__export(es_MX_exports, {
  esMX: () => esMX
});
module.exports = __toCommonJS(es_MX_exports);
var esMX = {
  locale: "es-MX",
  apiKeys: {
    action__add: void 0,
    action__search: void 0,
    createdAndExpirationStatus__expiresOn: void 0,
    createdAndExpirationStatus__never: void 0,
    detailsTitle__emptyRow: void 0,
    formButtonPrimary__add: void 0,
    formFieldCaption__expiration__expiresOn: void 0,
    formFieldCaption__expiration__never: void 0,
    formFieldOption__expiration__180d: void 0,
    formFieldOption__expiration__1d: void 0,
    formFieldOption__expiration__1y: void 0,
    formFieldOption__expiration__30d: void 0,
    formFieldOption__expiration__60d: void 0,
    formFieldOption__expiration__7d: void 0,
    formFieldOption__expiration__90d: void 0,
    formFieldOption__expiration__never: void 0,
    formHint: void 0,
    formTitle: void 0,
    lastUsed__days: void 0,
    lastUsed__hours: void 0,
    lastUsed__minutes: void 0,
    lastUsed__months: void 0,
    lastUsed__seconds: void 0,
    lastUsed__years: void 0,
    menuAction__revoke: void 0,
    revokeConfirmation: {
      confirmationText: void 0,
      formButtonPrimary__revoke: void 0,
      formHint: void 0,
      formTitle: void 0
    }
  },
  backButton: "Atr\xE1s",
  badge__activePlan: void 0,
  badge__canceledEndsAt: void 0,
  badge__currentPlan: "Plan actual",
  badge__default: "Por defecto",
  badge__endsAt: "Termina {{ date | shortDate('en-US') }}",
  badge__expired: "Caducado",
  badge__freeTrial: void 0,
  badge__otherImpersonatorDevice: "Otro dispositivo de imitaci\xF3n",
  badge__pastDueAt: void 0,
  badge__pastDuePlan: void 0,
  badge__primary: "Primario",
  badge__renewsAt: void 0,
  badge__requiresAction: "Requiere acci\xF3n",
  badge__startsAt: "Empieza {{ date | shortDate('en-US') }}",
  badge__thisDevice: "Este dispositivo",
  badge__trialEndsAt: void 0,
  badge__unverified: "No confirmado",
  badge__upcomingPlan: "Pr\xF3ximo plan",
  badge__userDevice: "Dispositivo de usuario",
  badge__you: "Usted",
  billing: {
    addPaymentMethod__label: void 0,
    alwaysFree: void 0,
    annually: void 0,
    availableFeatures: void 0,
    billedAnnually: "Facturado anualmente",
    billedMonthlyOnly: void 0,
    cancelFreeTrial: "Cancelar prueba gratuita",
    cancelFreeTrialAccessUntil: "Tu prueba gratuita permanecer\xE1 activa hasta el {{ date | longDate('es-MX') }}. Despu\xE9s de eso, perder\xE1s el acceso a las funciones de prueba y no se te cobrar\xE1.",
    cancelFreeTrialTitle: "\xBFCancelar prueba gratuita del plan {{plan}}?",
    cancelSubscription: "Cancelar suscripci\xF3n",
    cancelSubscriptionAccessUntil: void 0,
    cancelSubscriptionNoCharge: void 0,
    cancelSubscriptionPastDue: void 0,
    cancelSubscriptionTitle: void 0,
    cannotSubscribeMonthly: void 0,
    cannotSubscribeUnrecoverable: void 0,
    checkout: {
      description__paymentSuccessful: void 0,
      description__subscriptionSuccessful: void 0,
      downgradeNotice: void 0,
      emailForm: {
        subtitle: void 0,
        title: void 0
      },
      lineItems: {
        title__freeTrialEndsAt: void 0,
        title__paymentMethod: "M\xE9todo de pago",
        title__statementId: "N\xFAmero de factura",
        title__subscriptionBegins: "La suscripci\xF3n empieza",
        title__totalPaid: "Total pagado"
      },
      pastDueNotice: void 0,
      perMonth: void 0,
      title: void 0,
      title__paymentSuccessful: "\xA1Pago exitoso!",
      title__subscriptionSuccessful: "\xA1\xC9xito!",
      title__trialSuccess: void 0,
      totalDueAfterTrial: void 0
    },
    credit: void 0,
    creditRemainder: void 0,
    defaultFreePlanActive: void 0,
    free: "Gratis",
    getStarted: "Empezar",
    keepFreeTrial: void 0,
    keepSubscription: "Mantener suscripci\xF3n",
    manage: "Gestionar",
    manageSubscription: "Gestionar suscripci\xF3n",
    month: "Mes",
    monthly: void 0,
    pastDue: void 0,
    pay: void 0,
    paymentMethod: {
      applePayDescription: {
        annual: void 0,
        monthly: void 0
      },
      dev: {
        anyNumbers: void 0,
        cardNumber: void 0,
        cvcZip: void 0,
        developmentMode: void 0,
        expirationDate: void 0,
        testCardInfo: void 0
      }
    },
    paymentMethods__label: void 0,
    popular: void 0,
    pricingTable: {
      billingCycle: void 0,
      included: void 0
    },
    reSubscribe: "Re-suscribirse",
    seeAllFeatures: void 0,
    startFreeTrial: void 0,
    startFreeTrial__days: void 0,
    subscribe: void 0,
    subscriptionDetails: {
      beginsOn: void 0,
      currentBillingCycle: void 0,
      endsOn: void 0,
      firstPaymentAmount: void 0,
      firstPaymentOn: void 0,
      nextPaymentAmount: void 0,
      nextPaymentOn: void 0,
      pastDueAt: void 0,
      renewsAt: void 0,
      subscribedOn: void 0,
      title: void 0,
      trialEndsOn: void 0,
      trialStartedOn: void 0
    },
    subtotal: void 0,
    switchPlan: "Cambiar a este plan",
    switchToAnnual: void 0,
    switchToAnnualWithAnnualPrice: void 0,
    switchToMonthly: void 0,
    switchToMonthlyWithPrice: void 0,
    totalDue: void 0,
    totalDueToday: void 0,
    viewFeatures: void 0,
    viewPayment: void 0,
    year: void 0
  },
  createOrganization: {
    formButtonSubmit: "Crear organizaci\xF3n",
    invitePage: {
      formButtonReset: "Saltar"
    },
    title: "Crear organizaci\xF3n"
  },
  dates: {
    lastDay: "Ayer a las {{ date | timeString('es-ES') }}",
    next6Days: "{{ date | weekday('es-ES','long') }} a las {{ date | timeString('es-ES') }}",
    nextDay: "Ma\xF1ana a las {{ date | timeString('es-ES') }}",
    numeric: "{{ date | numeric('es-ES') }}",
    previous6Days: "\xDAltimo {{ date | weekday('es-ES','long') }} en {{ date | timeString('es-ES') }}",
    sameDay: "Hoy a las {{ date | timeString('es-ES') }}"
  },
  dividerText: "o",
  footerActionLink__alternativePhoneCodeProvider: void 0,
  footerActionLink__useAnotherMethod: "Usar otro m\xE9todo",
  footerPageLink__help: "Ayuda",
  footerPageLink__privacy: "Privacidad",
  footerPageLink__terms: "T\xE9rminos",
  formButtonPrimary: "Continuar",
  formButtonPrimary__verify: "Verificar",
  formFieldAction__forgotPassword: "Has olvidado tu contrase\xF1a?",
  formFieldError__matchingPasswords: "Las contrase\xF1as coinciden.",
  formFieldError__notMatchingPasswords: "Las no contrase\xF1as coinciden.",
  formFieldError__verificationLinkExpired: "El link de verificaci\xF3n expiro. Por favor vuelve a solicitarlo.",
  formFieldHintText__optional: "Opcional",
  formFieldHintText__slug: "Un slug es una identificaci\xF3n legible por humanos que debe ser \xFAnica. Se utiliza a menudo en URL.",
  formFieldInputPlaceholder__apiKeyDescription: void 0,
  formFieldInputPlaceholder__apiKeyExpirationDate: void 0,
  formFieldInputPlaceholder__apiKeyName: void 0,
  formFieldInputPlaceholder__backupCode: "Ingresa el c\xF3digo de respaldo",
  formFieldInputPlaceholder__confirmDeletionUserAccount: "Eliminar cuenta",
  formFieldInputPlaceholder__emailAddress: "Ingresa tu correo electr\xF3nico",
  formFieldInputPlaceholder__emailAddress_username: "Ingresa el correo electr\xF3nico o nombre de usuario",
  formFieldInputPlaceholder__emailAddresses: "Ingrese o pegue una o m\xE1s direcciones de correo electr\xF3nico, separadas por espacios o comas",
  formFieldInputPlaceholder__firstName: "Nombre",
  formFieldInputPlaceholder__lastName: "Apellido",
  formFieldInputPlaceholder__organizationDomain: "ejemplo.com",
  formFieldInputPlaceholder__organizationDomainEmailAddress: "nombre@example.com",
  formFieldInputPlaceholder__organizationName: "Nombre de la organizaci\xF3n",
  formFieldInputPlaceholder__organizationSlug: "mi-organizaci\xF3n",
  formFieldInputPlaceholder__password: "Ingresa tu contrase\xF1a",
  formFieldInputPlaceholder__phoneNumber: "Ingresa tu n\xFAmero de tel\xE9fono",
  formFieldInputPlaceholder__username: void 0,
  formFieldInput__emailAddress_format: void 0,
  formFieldLabel__apiKeyDescription: void 0,
  formFieldLabel__apiKeyExpiration: void 0,
  formFieldLabel__apiKeyName: void 0,
  formFieldLabel__automaticInvitations: "Activar invitaciones automaticas para este dominio",
  formFieldLabel__backupCode: "C\xF3digo de respaldo",
  formFieldLabel__confirmDeletion: "Confirmaci\xF3n",
  formFieldLabel__confirmPassword: "Confirme la contrase\xF1a",
  formFieldLabel__currentPassword: "Contrase\xF1a actual",
  formFieldLabel__emailAddress: "Correo electr\xF3nico",
  formFieldLabel__emailAddress_username: "Correo electr\xF3nico o nombre de usuario",
  formFieldLabel__emailAddresses: "Direcciones de correo",
  formFieldLabel__firstName: "Nombre",
  formFieldLabel__lastName: "Apellido",
  formFieldLabel__newPassword: "Nueva contrase\xF1a",
  formFieldLabel__organizationDomain: "Dominio",
  formFieldLabel__organizationDomainDeletePending: "Eliminar invitaciones y sugerencias pendientes",
  formFieldLabel__organizationDomainEmailAddress: "Verificaci\xF3n de correo",
  formFieldLabel__organizationDomainEmailAddressDescription: "Entrar una direcci\xF3n de correo electr\xF3nico bajo este dominio para recibir un c\xF3digo y verificarlo.",
  formFieldLabel__organizationName: "Nombre de la Organizaci\xF3n",
  formFieldLabel__organizationSlug: "Apodo",
  formFieldLabel__passkeyName: "Nombre de llave de acceso",
  formFieldLabel__password: "Contrase\xF1a",
  formFieldLabel__phoneNumber: "N\xFAmero de tel\xE9fono",
  formFieldLabel__role: "Rol",
  formFieldLabel__signOutOfOtherSessions: "Cerrar sesi\xF3n en todos los dem\xE1s dispositivos",
  formFieldLabel__username: "Nombre de usuario",
  impersonationFab: {
    action__signOut: "Cerrar",
    title: "Registrado como {{identifier}}"
  },
  lastAuthenticationStrategy: "\xDAltimo uso",
  maintenanceMode: "Actualmente estamos en mantenimiento, pero no te preocupes, no deber\xEDa llevar m\xE1s de unos minutos.",
  membershipRole__admin: "Administrador",
  membershipRole__basicMember: "Miembro",
  membershipRole__guestMember: "Invitado",
  organizationList: {
    action__createOrganization: "Crear organizaci\xF3n",
    action__invitationAccept: "Unirse",
    action__suggestionsAccept: "Pedir unirse",
    createOrganization: "Crear Organizaci\xF3n",
    invitationAcceptedLabel: "Unido",
    subtitle: "para continuar con {{applicationName}}",
    suggestionsAcceptedLabel: "Aprobaci\xF3n pendiente",
    title: "Elige una cuenta",
    titleWithoutPersonal: "Elige una organizaci\xF3n"
  },
  organizationProfile: {
    apiKeysPage: {
      title: void 0
    },
    badge__automaticInvitation: "Invitaciones automaticas",
    badge__automaticSuggestion: "Sugerencias automaticas",
    badge__manualInvitation: "Sin inscripci\xF3nes automaticas",
    badge__unverified: "No verificado",
    billingPage: {
      paymentHistorySection: {
        empty: void 0,
        notFound: void 0,
        tableHeader__amount: void 0,
        tableHeader__date: void 0,
        tableHeader__status: void 0
      },
      paymentMethodsSection: {
        actionLabel__default: void 0,
        actionLabel__remove: void 0,
        add: void 0,
        addSubtitle: void 0,
        cancelButton: void 0,
        formButtonPrimary__add: void 0,
        formButtonPrimary__pay: void 0,
        payWithTestCardButton: void 0,
        removeMethod: {
          messageLine1: void 0,
          messageLine2: void 0,
          successMessage: void 0,
          title: void 0
        },
        title: void 0
      },
      start: {
        headerTitle__payments: void 0,
        headerTitle__plans: void 0,
        headerTitle__statements: void 0,
        headerTitle__subscriptions: void 0
      },
      statementsSection: {
        empty: void 0,
        itemCaption__paidForPlan: void 0,
        itemCaption__proratedCredit: void 0,
        itemCaption__subscribedAndPaidForPlan: void 0,
        notFound: void 0,
        tableHeader__amount: void 0,
        tableHeader__date: void 0,
        title: void 0,
        totalPaid: void 0
      },
      subscriptionsListSection: {
        actionLabel__manageSubscription: void 0,
        actionLabel__newSubscription: void 0,
        actionLabel__switchPlan: void 0,
        tableHeader__edit: void 0,
        tableHeader__plan: void 0,
        tableHeader__startDate: void 0,
        title: void 0
      },
      subscriptionsSection: {
        actionLabel__default: void 0
      },
      switchPlansSection: {
        title: void 0
      },
      title: void 0
    },
    createDomainPage: {
      subtitle: "A\xF1ade el dominio para verificar. Los usuarios con direcciones de correo electr\xF3nico en este dominio pueden unirse a la organizaci\xF3n aqu\xED o pedir unirse.",
      title: "Anadir dominio"
    },
    invitePage: {
      detailsTitle__inviteFailed: "No se pudieron enviar las invitaciones. Solucione lo siguiente y vuelva a intentarlo:",
      formButtonPrimary__continue: "Enviar invitaciones",
      selectDropdown__role: "Seleccionar rol",
      subtitle: "Ingresa o pega una o m\xE1s direcciones de correo electr\xF3nico, separadas por espacios o comas",
      successMessage: "Invitaciones enviadas con \xE9xito",
      title: "Invitar miembros"
    },
    membersPage: {
      action__invite: "Invitar",
      action__search: "Buscar",
      activeMembersTab: {
        menuAction__remove: "Eliminar miembro",
        tableHeader__actions: "Acciones",
        tableHeader__joined: "Se uni\xF3",
        tableHeader__role: "Rol",
        tableHeader__user: "Usuario"
      },
      detailsTitle__emptyRow: "No hay miembros para mostrar",
      invitationsTab: {
        autoInvitations: {
          headerSubtitle: "Invita usuarios conectando un correo electr\xF3nico del dominio con su organizaci\xF3n. Cualquiera que se registre con un correo electr\xF3nico del dominio existente podr\xE1 unirse a la organizaci\xF3n en cualquier momento.",
          headerTitle: "Invitaciones autom\xE1ticas",
          primaryButton: "Gestionar dominios verificados"
        },
        table__emptyRow: "No hay invitaciones para mostrar"
      },
      invitedMembersTab: {
        menuAction__revoke: "Revocar invitaci\xF3n",
        tableHeader__invited: "Invitado"
      },
      requestsTab: {
        autoSuggestions: {
          headerSubtitle: "Los usuarios que inicien sesi\xF3n con un correo electr\xF3nico del dominio existente podr\xE1n ver una sugerencia para solicitar unirse a su organizaci\xF3n.",
          headerTitle: "Sugerencias autom\xE1ticas",
          primaryButton: "Gestionar dominios verificados"
        },
        menuAction__approve: "Aprobado",
        menuAction__reject: "Rechazado",
        tableHeader__requested: "Acceso solicitado",
        table__emptyRow: "No hay solicitudes para mostrar"
      },
      start: {
        headerTitle__invitations: "Invitaciones",
        headerTitle__members: "Miembros",
        headerTitle__requests: "Solicitudes"
      }
    },
    navbar: {
      apiKeys: "Claves API",
      billing: "Facturaci\xF3n",
      description: "Gestiona tu organizaci\xF3n.",
      general: "General",
      members: "Miembros",
      title: "Organizaci\xF3n"
    },
    plansPage: {
      alerts: {
        noPermissionsToManageBilling: void 0
      },
      title: void 0
    },
    profilePage: {
      dangerSection: {
        deleteOrganization: {
          actionDescription: 'Escribe "{{organizationName}}" a continuaci\xF3n para continuar.',
          messageLine1: "\xBFEst\xE1s seguro que quieres eliminar esta organizaci\xF3n?",
          messageLine2: "Esta acci\xF3n es permanente e irreversible.",
          successMessage: "Haz eliminado la organizaci\xF3n.",
          title: "Eliminar la organizaci\xF3n"
        },
        leaveOrganization: {
          actionDescription: 'Escribe "{{organizationName}}" a continuaci\xF3n para continuar.',
          messageLine1: "\xBFEst\xE1s seguro de que deseas abandonar esta organizaci\xF3n? Perder\xE1s el acceso a esta organizaci\xF3n y sus aplicaciones.",
          messageLine2: "Esta acci\xF3n es permanente e irreversible.",
          successMessage: "Has dejado la organizaci\xF3n.",
          title: "Abandonar la organizaci\xF3n"
        },
        title: "Peligro"
      },
      domainSection: {
        menuAction__manage: "Gestionar",
        menuAction__remove: "Eliminar",
        menuAction__verify: "Verificar",
        primaryButton: "A\xF1adir dominio",
        subtitle: "Permite a los usuarios conectarse autom\xE1ticamente o solicitar unirse a la organizaci\xF3n basado en un correo electr\xF3nico del dominio verificado.",
        title: "Dominios verificados"
      },
      successMessage: "La organizaci\xF3n ha sido actualizada.",
      title: "Perfil de la organizaci\xF3n"
    },
    removeDomainPage: {
      messageLine1: "Se eliminar\xE1 el de correo electr\xF3nico del dominio {{domain}}.",
      messageLine2: "Los usuarios no podr\xE1n unirse a la organizaci\xF3n de manera autom\xE1tica una vez que se haya eliminado.",
      successMessage: "{{domain}} se ha eliminado.",
      title: "Eliminar dominio"
    },
    start: {
      headerTitle__general: "General",
      headerTitle__members: "Miembros",
      profileSection: {
        primaryButton: "Actualizar perfil",
        title: "Perfil de la organizaci\xF3n",
        uploadAction__title: "Logo"
      }
    },
    verifiedDomainPage: {
      dangerTab: {
        calloutInfoLabel: "Eliminar este dominio afectar\xE1 a los usuarios invitados.",
        removeDomainActionLabel__remove: "Eliminar dominio",
        removeDomainSubtitle: "Eliminar este dominio de los dominios verificados",
        removeDomainTitle: "Eliminar dominio"
      },
      enrollmentTab: {
        automaticInvitationOption__description: "Los usuarios son invitados autom\xE1ticamente a la organizaci\xF3n cuando se registran y pueden unirse en cualquier momento.",
        automaticInvitationOption__label: "Invitaciones autom\xE1ticas",
        automaticSuggestionOption__description: "Los usuarios reciben una sugerencia para solicitar unirse, pero deben ser aprobados por un administrador antes de poder unirse a la organizaci\xF3n.",
        automaticSuggestionOption__label: "Sugerencias autom\xE1ticas",
        calloutInfoLabel: "Cambiar el modo de inscripci\xF3n solo afectar\xE1 a los nuevos usuarios.",
        calloutInvitationCountLabel: "Invitaciones pendientes enviadas a usuarios: {{count}}",
        calloutSuggestionCountLabel: "Sugerencias pendientes enviadas a usuarios: {{count}}",
        manualInvitationOption__description: "Los usuarios solo pueden ser invitados manualmente a la organizaci\xF3n.",
        manualInvitationOption__label: "Sin inscripci\xF3n autom\xE1tica",
        subtitle: "Seleccione c\xF3mo los usuarios de este dominio pueden unirse a la organizaci\xF3n."
      },
      start: {
        headerTitle__danger: "Peligro",
        headerTitle__enrollment: "Opciones de inscripci\xF3n"
      },
      subtitle: "El dominio {{domain}} ahora est\xE1 verificado. Contin\xFAa seleccionando el modo de inscripci\xF3n.",
      title: "Actualizar {{domain}}"
    },
    verifyDomainPage: {
      formSubtitle: "Introduce el c\xF3digo de verificaci\xF3n enviado a tu direcci\xF3n de correo electr\xF3nico",
      formTitle: "C\xF3digo de verificaci\xF3n",
      resendButton: "No recibiste un c\xF3digo? Reenviar",
      subtitle: "El dominio {{domainName}} necesita ser verificado v\xEDa correo electr\xF3nico.",
      subtitleVerificationCodeScreen: "Se envi\xF3 un c\xF3digo de verificaci\xF3n a {{emailAddress}}. Introduce el c\xF3digo para continuar.",
      title: "Verificar dominio"
    }
  },
  organizationSwitcher: {
    action__closeOrganizationSwitcher: void 0,
    action__createOrganization: "Crear Organizaci\xF3n",
    action__invitationAccept: "Unirse",
    action__manageOrganization: "Administrar Organizaci\xF3n",
    action__openOrganizationSwitcher: void 0,
    action__suggestionsAccept: "Solicitar unirse",
    notSelected: "Ninguna organizaci\xF3n seleccionada",
    personalWorkspace: "Espacio personal",
    suggestionsAcceptedLabel: "Pendiente de aprobaci\xF3n"
  },
  paginationButton__next: "Siguiente",
  paginationButton__previous: "Anterior",
  paginationRowText__displaying: "Mostrando",
  paginationRowText__of: "de",
  reverification: {
    alternativeMethods: {
      actionLink: "Solicitar ayuda",
      actionText: "\xBFNo posees ninguno de estos?",
      blockButton__backupCode: "Utiliza un c\xF3digo de respaldo",
      blockButton__emailCode: "Env\xEDa el c\xF3digo a {{identifier}}",
      blockButton__passkey: "Utiliza tu llave de acceso",
      blockButton__password: "Contin\xFAa con tu contrase\xF1a",
      blockButton__phoneCode: "Enviar un mensaje de texto a {{identifier}}",
      blockButton__totp: "aplicaci\xF3n de autenticaci\xF3n",
      getHelp: {
        blockButton__emailSupport: "Soporte por correo electr\xF3nico",
        content: "Si tienes problemas para iniciar sesi\xF3n",
        title: "Obtener ayuda"
      },
      subtitle: "\xBFTienes problemas? Puedes utilizar cualquiera de estos m\xE9todos para iniciar sesi\xF3n",
      title: "Utiliza otro m\xE9todo"
    },
    backupCodeMfa: {
      subtitle: "Tu c\xF3digo de respaldo es el que obtuviste cuando configuraste la autenticaci\xF3n en dos pasos",
      title: "Ingresa tu c\xF3digo de respaldo"
    },
    emailCode: {
      formTitle: "C\xF3digo de verificaci\xF3n",
      resendButton: "\xBFNo recibiste un c\xF3digo? Reenviar",
      subtitle: "para continuar con {{applicationName}}",
      title: "Revisa tu correo electr\xF3nico"
    },
    noAvailableMethods: {
      message: "No se puede proceder con el inicio de sesi\xF3n. No hay ning\xFAn factor de autenticaci\xF3n disponible",
      subtitle: "Ocurri\xF3 un error",
      title: "No se puede iniciar sesi\xF3n"
    },
    passkey: {
      blockButton__passkey: "Utiliza tu llave de acceso",
      subtitle: "Utilizar tu llave de acceso confirma que eres t\xFA. Tu dispositivo puede solicitar tu huella dactilar, rostro o pantalla de bloqueo.",
      title: "Utiliza tu llave de acceso"
    },
    password: {
      actionLink: "Utiliza otro m\xE9todo",
      subtitle: "Ingresa tu contrase\xF1a actual para continuar",
      title: "Verificaci\xF3n requerida"
    },
    phoneCode: {
      formTitle: "C\xF3digo de verificaci\xF3n",
      resendButton: "\xBFNo recibiste un c\xF3digo? Reenviar",
      subtitle: "Ingresa el c\xF3digo enviado a tu tel\xE9fono para continuar",
      title: "Verificaci\xF3n requerida"
    },
    phoneCodeMfa: {
      formTitle: "C\xF3digo de verificaci\xF3n",
      resendButton: "\xBFNo recibiste un c\xF3digo? Reenviar",
      subtitle: "Ingresa el c\xF3digo enviado a tu tel\xE9fono para continuar",
      title: "Verificaci\xF3n requerida"
    },
    totpMfa: {
      formTitle: "C\xF3digo de verificaci\xF3n",
      subtitle: "Ingresa el c\xF3digo generado por la aplicaci\xF3n de autenticaci\xF3n para continuar",
      title: "Verificaci\xF3n requerida"
    }
  },
  signIn: {
    accountSwitcher: {
      action__addAccount: "Agregar cuenta",
      action__signOutAll: "Cerrar sesi\xF3n de todas las cuentas",
      subtitle: "Seleccione la cuenta con la que desea continuar.",
      title: "Elija una cuenta"
    },
    alternativeMethods: {
      actionLink: "Obtener ayuda",
      actionText: "\xBFNo tienes ninguno de estos?",
      blockButton__backupCode: "Usa un c\xF3digo de respaldo",
      blockButton__emailCode: "Enviar c\xF3digo a{{identifier}}",
      blockButton__emailLink: "Enviar enlace a{{identifier}}",
      blockButton__passkey: "Inicia sesi\xF3n con tu llave de acceso",
      blockButton__password: "Inicia sesi\xF3n con tu contrase\xF1a",
      blockButton__phoneCode: "Enviar c\xF3digo a{{identifier}}",
      blockButton__totp: "Usa tu aplicaci\xF3n de autenticaci\xF3n",
      getHelp: {
        blockButton__emailSupport: "Soporte de correo electr\xF3nico",
        content: "Si tienes problemas para ingresar a tu cuenta, env\xEDenos un correo electr\xF3nico y trabajaremos contigo para restablecer el acceso lo antes posible.",
        title: "Obtener ayuda"
      },
      subtitle: "Si est\xE1s experimentando problemas, puedes utilizar uno de estos m\xE9todos para ingresar.",
      title: "Utiliza otro m\xE9todo"
    },
    alternativePhoneCodeProvider: {
      formTitle: void 0,
      resendButton: void 0,
      subtitle: void 0,
      title: void 0
    },
    backupCodeMfa: {
      subtitle: "para continuar a {{applicationName}}",
      title: "Introduce un c\xF3digo de seguridad"
    },
    emailCode: {
      formTitle: "C\xF3digo de verificaci\xF3n",
      resendButton: "Re-enviar c\xF3digo",
      subtitle: "para continuar a {{applicationName}}",
      title: "Revise su correo electr\xF3nico"
    },
    emailLink: {
      clientMismatch: {
        subtitle: "Para continuar, abre el enlace de verificaci\xF3n en tu dispositivo o navegador donde intentaste iniciar sesi\xF3n",
        title: "El enlace de verificaci\xF3n es inv\xE1lido para este dispositivo"
      },
      expired: {
        subtitle: "Regresa a la pesta\xF1a original para continuar.",
        title: "El enlace de verificaci\xF3n ha expirado"
      },
      failed: {
        subtitle: "Regresa a la pesta\xF1a original para continuar.",
        title: "El enlace de verificaci\xF3n es inv\xE1lido"
      },
      formSubtitle: "Utiliza el enlace de verificaci\xF3n enviado a tu correo electr\xF3nico",
      formTitle: "Enlace de verificaci\xF3n",
      loading: {
        subtitle: "Ser\xE1s redireccionado pronto",
        title: "Iniciando sesi\xF3n..."
      },
      resendButton: "\xBFNo recibiste un enlace? Reenviar",
      subtitle: "para continuar con {{applicationName}}",
      title: "Revisa tu correo electr\xF3nico",
      unusedTab: {
        title: "Puedes cerrar esta pesta\xF1a"
      },
      verified: {
        subtitle: "Ser\xE1s redireccionado pronto",
        title: "Sesi\xF3n iniciada correctamente"
      },
      verifiedSwitchTab: {
        subtitle: "Regresa a la pesta\xF1a original para continuar",
        subtitleNewTab: "Regresa a la pesta\xF1a reci\xE9n abierta para continuar",
        titleNewTab: "Sesi\xF3n iniciada en otra pesta\xF1a"
      }
    },
    enterpriseConnections: {
      subtitle: void 0,
      title: void 0
    },
    forgotPassword: {
      formTitle: "C\xF3digo para restablecer contrase\xF1a",
      resendButton: "\xBFNo recibiste un c\xF3digo? Reenviar",
      subtitle: "para restablecer tu contrase\xF1a",
      subtitle_email: "Primero, ingresa el c\xF3digo enviado a tu correo electr\xF3nico",
      subtitle_phone: "Primero, ingresa el c\xF3digo enviado a tu tel\xE9fono",
      title: "Restablecer contrase\xF1a"
    },
    forgotPasswordAlternativeMethods: {
      blockButton__resetPassword: "Restablece tu contrase\xF1a",
      label__alternativeMethods: "O, inicia sesi\xF3n con otro m\xE9todo",
      title: "\xBFOlvidaste la contrase\xF1a?"
    },
    noAvailableMethods: {
      message: "No se puede continuar con el inicio de sesi\xF3n. No hay ning\xFAn factor de autenticaci\xF3n disponible.",
      subtitle: "Ocurri\xF3 un error",
      title: "No puedo iniciar sesi\xF3n"
    },
    passkey: {
      subtitle: "Usando tu llave de acceso confirmas que eres t\xFA. Tu dispositivo puede pedirte la huella dactilar, el rostro o el bloqueo de pantalla.",
      title: "Usa tu llave de acceso"
    },
    password: {
      actionLink: "Utiliza otro m\xE9todo",
      subtitle: "para continuar con {{applicationName}}",
      title: "Introduzca su contrase\xF1a"
    },
    passwordPwned: {
      title: "Contrase\xF1a en peligro"
    },
    phoneCode: {
      formTitle: "C\xF3digo de verificaci\xF3n",
      resendButton: "Reenviar c\xF3digo",
      subtitle: "para continuar con {{applicationName}}",
      title: "Revisa tu tel\xE9fono"
    },
    phoneCodeMfa: {
      formTitle: "C\xF3digo de verificaci\xF3n",
      resendButton: "Reenviar c\xF3digo",
      subtitle: "para continuar con {{applicationName}}",
      title: "Revisa tu tel\xE9fono"
    },
    resetPassword: {
      formButtonPrimary: "Restablecer contrase\xF1a",
      requiredMessage: "Por razones de seguridad, es necesario restablecer su contrase\xF1a.",
      successMessage: "Tu contrase\xF1a se ha cambiado correctamente. Te estamos redirigiendo, por favor espera un momento.",
      title: "Establecer nueva contrase\xF1a"
    },
    resetPasswordMfa: {
      detailsLabel: "Es necesario verificar su identidad para restablecer su contrase\xF1a."
    },
    start: {
      actionLink: "Registrarse",
      actionLink__join_waitlist: "Unirse a la lista de espera",
      actionLink__use_email: "Utilizar correo electr\xF3nico",
      actionLink__use_email_username: "Utilizar correo electr\xF3nico o nombre de usuario",
      actionLink__use_passkey: "Utilizar llave de acceso",
      actionLink__use_phone: "Utilizar tel\xE9fono",
      actionLink__use_username: "Utilizar nombre de usuario",
      actionText: "\xBFNo tiene cuenta?",
      actionText__join_waitlist: void 0,
      alternativePhoneCodeProvider: {
        actionLink: void 0,
        label: void 0,
        subtitle: void 0,
        title: void 0
      },
      subtitle: "para continuar con {{applicationName}}",
      subtitleCombined: void 0,
      title: "Iniciar sesi\xF3n",
      titleCombined: "Para continuar con {{applicationName}}"
    },
    totpMfa: {
      formTitle: "C\xF3digo de verificaci\xF3n",
      subtitle: "Para continuar, por favor introduce el c\xF3digo generado por tu aplicaci\xF3n de autenticaci\xF3n",
      title: "Verificaci\xF3n de dos pasos"
    }
  },
  signInEnterPasswordTitle: "Ingresa tu contrase\xF1a",
  signUp: {
    alternativePhoneCodeProvider: {
      resendButton: void 0,
      subtitle: void 0,
      title: void 0
    },
    continue: {
      actionLink: "Entrar",
      actionText: "\xBFTienes una cuenta?",
      subtitle: "Por favor completa los datos para continuar",
      title: "Completa los datos faltantes"
    },
    emailCode: {
      formSubtitle: "Ingresa el c\xF3digo de verificaci\xF3n enviado a tu correo electr\xF3nico",
      formTitle: "C\xF3digo de verificaci\xF3n",
      resendButton: "Reenviar c\xF3digo",
      subtitle: "Ingresa el c\xF3digo de verificaci\xF3n enviado a tu correo electr\xF3nico",
      title: "Verifica tu correo electr\xF3nico"
    },
    emailLink: {
      clientMismatch: {
        subtitle: "Para continuar, abre el enlace de verificaci\xF3n en el dispositivo o navegador desde el que est\xE1s intentando registrarte",
        title: "El enlace de verificaci\xF3n es inv\xE1lido para este dispositivo"
      },
      formSubtitle: "Utiliza el enlace de verificaci\xF3n enviado a tu direcci\xF3n de correo electr\xF3nico",
      formTitle: "Enlace de verificaci\xF3n",
      loading: {
        title: "Registrando..."
      },
      resendButton: "\xBFNo recibiste el enlace? Reenviar",
      subtitle: "para continuar a {{applicationName}}",
      title: "Verifica tu correo electr\xF3nico",
      verified: {
        title: "Registrado con \xE9xito"
      },
      verifiedSwitchTab: {
        subtitle: "Regresa a la pesta\xF1a reci\xE9n abierta para continuar",
        subtitleNewTab: "Regresa a la pesta\xF1a anterior para continuar",
        title: "Correo electr\xF3nico verificado con \xE9xito"
      }
    },
    enterpriseConnections: {
      subtitle: void 0,
      title: void 0
    },
    legalConsent: {
      checkbox: {
        label__onlyPrivacyPolicy: void 0,
        label__onlyTermsOfService: void 0,
        label__termsOfServiceAndPrivacyPolicy: void 0
      },
      continue: {
        subtitle: void 0,
        title: void 0
      }
    },
    phoneCode: {
      formSubtitle: "Introduzca el c\xF3digo de verificaci\xF3n enviado a su n\xFAmero de tel\xE9fono.",
      formTitle: "C\xF3digo de verificaci\xF3n",
      resendButton: "Reenviar c\xF3digo",
      subtitle: "para continuar a {{applicationName}}",
      title: "Verifica tu tel\xE9fono"
    },
    restrictedAccess: {
      actionLink: "Iniciar sesi\xF3n",
      actionText: "\xBFTienes cuenta?",
      blockButton__emailSupport: "Soporte por correo electr\xF3nico",
      blockButton__joinWaitlist: "Unirse a la lista de espera",
      subtitle: "Los registros est\xE1n desactivados actualmente. Si crees que tienes acceso, contacta soporte",
      subtitleWaitlist: "Los registros est\xE1n desactivados actualmente. Para ser el primero en saber cuando estemos listos, \xFAnete a la lista de espera",
      title: "Acceso restringido"
    },
    start: {
      actionLink: "Inicar sesi\xF3n",
      actionLink__use_email: "Utilizar el correo electr\xF3nico",
      actionLink__use_phone: "Utilizar el n\xFAmero telef\xF3nico",
      actionText: "\xBFTienes una cuenta?",
      alternativePhoneCodeProvider: {
        actionLink: void 0,
        label: void 0,
        subtitle: void 0,
        title: void 0
      },
      subtitle: "para continuar con {{applicationName}}",
      subtitleCombined: "para continuar con {{applicationName}}",
      title: "Crea tu cuenta",
      titleCombined: "Crea tu cuenta"
    }
  },
  socialButtonsBlockButton: "Continuar con {{provider|titleize}}",
  socialButtonsBlockButtonManyInView: "{{provider|titleize}}",
  taskChooseOrganization: {
    chooseOrganization: {
      action__createOrganization: void 0,
      action__invitationAccept: void 0,
      action__suggestionsAccept: void 0,
      subtitle: void 0,
      suggestionsAcceptedLabel: void 0,
      title: void 0
    },
    createOrganization: {
      formButtonReset: void 0,
      formButtonSubmit: void 0,
      formFieldInputPlaceholder__name: void 0,
      formFieldInputPlaceholder__slug: void 0,
      formFieldLabel__name: void 0,
      formFieldLabel__slug: void 0,
      subtitle: void 0,
      title: void 0
    },
    signOut: {
      actionLink: void 0,
      actionText: void 0
    }
  },
  unstable__errors: {
    already_a_member_in_organization: "{{email}} ya es miembro de la organizaci\xF3n.",
    captcha_invalid: "El registro fall\xF3 debido a fallos en la validaci\xF3n de seguridad. Por favor, recarga la p\xE1gina o cont\xE1ctanos para obtener m\xE1s asistencia.",
    captcha_unavailable: "El registro fall\xF3 debido a fallos en la validaci\xF3n de robot. Por favor, recarga la p\xE1gina o cont\xE1ctanos para obtener m\xE1s asistencia.",
    form_code_incorrect: "C\xF3digo incorrecto.",
    form_identifier_exists__email_address: "El correo electr\xF3nico ya existe",
    form_identifier_exists__phone_number: "El n\xFAmero telef\xF3nico ya existe.",
    form_identifier_exists__username: "El nombre de usuario ya existe.",
    form_identifier_not_found: "No se encontr\xF3 una cuenta con esos detalles.",
    form_param_format_invalid: "Formato inv\xE1lido.",
    form_param_format_invalid__email_address: "El correo electr\xF3nico debe ser v\xE1lido.",
    form_param_format_invalid__phone_number: "El n\xFAmero telef\xF3nico debe ser en un formato v\xE1lido internacional.",
    form_param_max_length_exceeded__first_name: "El nombre debe tener menos de 256 caracteres.",
    form_param_max_length_exceeded__last_name: "El apellido debe tener menos de 256 caracteres.",
    form_param_max_length_exceeded__name: "El nombre debe tener menos de 256 caracteres.",
    form_param_nil: "Este campo es rquerido y no puede estar vac\xEDo.",
    form_param_type_invalid: void 0,
    form_param_type_invalid__email_address: void 0,
    form_param_type_invalid__phone_number: void 0,
    form_param_value_invalid: "Valor inv\xE1lido",
    form_password_incorrect: "Contrase\xF1a incorrecta.",
    form_password_length_too_short: "La contrase\xF1a es muy corta.",
    form_password_not_strong_enough: "La contrase\xF1a no es suficientemente segura.",
    form_password_pwned: "Esta contrase\xF1a se encontr\xF3 como parte de una brecha y no se puede utilizar, intenta con otra contrase\xF1a.",
    form_password_pwned__sign_in: "Esta contrase\xF1a se encontr\xF3 como parte de una brecha y no se puede utilizar, por favor restablece tu contrase\xF1a.",
    form_password_size_in_bytes_exceeded: "La contrase\xF1a excede el n\xFAmero m\xE1ximo de bytes permitidos. Por favor, elimine algunos caracteres especiales o reduzca la longitud de la contrase\xF1a.",
    form_password_validation_failed: "Contrase\xF1a incorrecta",
    form_username_invalid_character: "El nombre de usuario contiene caracteres inv\xE1lidos.",
    form_username_invalid_length: "La longitud del nombre de usuario es demasiado corta.",
    identification_deletion_failed: "No se puede eliminar la \xFAltima identificaci\xF3n.",
    not_allowed_access: "No tienes permiso para acceder a esta p\xE1gina.",
    organization_domain_blocked: "Este correo electr\xF3nico del dominio est\xE1 bloqueado, intenta con otro.",
    organization_domain_common: "Este correo electr\xF3nico del dominio es muy com\xFAn, intenta con otro.",
    organization_domain_exists_for_enterprise_connection: "Este dominio ya esta en uso para t\xFA organizaci\xF3n",
    organization_membership_quota_exceeded: "Alcanzaste el limite de miembros en la organizaci\xF3n, incluyendo las invitaciones enviadas.",
    organization_minimum_permissions_needed: "Debe existir al menos un miembro en la organizaci\xF3n.",
    organization_not_found_or_unauthorized: void 0,
    organization_not_found_or_unauthorized_with_create_organization_disabled: void 0,
    passkey_already_exists: "Ya se ha registrado una llave de acceso en este dispositivo.",
    passkey_not_supported: "Las llaves de acceso no son compatibles con este dispositivo.",
    passkey_pa_not_supported: "El registro requiere un autenticador de plataforma, pero el dispositivo no es compatible.",
    passkey_registration_cancelled: "El registro de la llave de acceso se ha cancelado o ha expirado.",
    passkey_retrieval_cancelled: "La verificaci\xF3n de la llave de acceso se ha cancelado o ha expirado.",
    passwordComplexity: {
      maximumLength: "menos de {{length}} caracteres",
      minimumLength: "{{length}} o m\xE1s caracteres",
      requireLowercase: "al menos una letra min\xFAscula",
      requireNumbers: "al menos un n\xFAmero",
      requireSpecialCharacter: "al menos un car\xE1cter especial",
      requireUppercase: "al menos una letra may\xFAscula",
      sentencePrefix: "Tu contrase\xF1a debe contener"
    },
    phone_number_exists: "Este n\xFAmero de telef\xF3nico ya est\xE1 en uso.",
    session_exists: "Ya has iniciado sesi\xF3n",
    web3_missing_identifier: "No se puede encontrar la extension de la billetera Web3. Instala una para continuar",
    zxcvbn: {
      couldBeStronger: "Tu contrase\xF1a funciona, pero puede ser m\xE1s segura. Prueba a\xF1adiendo m\xE1s caracteres.",
      goodPassword: "Tu contrase\xF1a cumple con todos los requisitos necesarios.",
      notEnough: "Tu contrase\xF1a no es lo suficientemente segura.",
      suggestions: {
        allUppercase: "Escribe algunas letras en may\xFAsculas, pero no todas.",
        anotherWord: "A\xF1ade palabras menos comunes.",
        associatedYears: "Evita a\xF1os asociados contigo.",
        capitalization: "Escribe algunas letras en may\xFAsculas adem\xE1s de la primera.",
        dates: "Evita fechas asociadas contigo.",
        l33t: "Evita sustituciones predecibles como '@' por 'a'",
        longerKeyboardPattern: "Usa patrones de teclado m\xE1s largos y cambia la direcci\xF3n de escritura varias veces.",
        noNeed: "Puedes crear contrase\xF1as seguras sin usar s\xEDmbolos, n\xFAmeros o may\xFAsculas.",
        pwned: "Si utilizas esta contrase\xF1a en otro lugar, deber\xEDas cambiarla.",
        recentYears: "Evita a\xF1os recientes.",
        repeated: "Evita palabras y letras repetidas.",
        reverseWords: "Evita palabras comunes escritas al rev\xE9s",
        sequences: "Evita secuencias de letras comunes.",
        useWords: "Utiliza varias palabras, pero evita frases comunes."
      },
      warnings: {
        common: "Es una contrase\xF1a utilizada com\xFAnmente.",
        commonNames: "Nombres y apellidos comunes son f\xE1ciles de adivinar.",
        dates: "Las fechas son f\xE1ciles de adivinar.",
        extendedRepeat: 'Patrones repetidos como "abcabcabc" son f\xE1ciles de adivinar.',
        keyPattern: "Patrones cortos son f\xE1ciles de adivinar.",
        namesByThemselves: "Nombres y apellidos son f\xE1ciles de adivinar.",
        pwned: "Tu contrase\xF1a fue expuesta por una violaci\xF3n de datos en Internet.",
        recentYears: "Los a\xF1os recientes son f\xE1ciles de adivinar.",
        sequences: 'Patrones comunes como "abc" son f\xE1ciles de adivinar',
        similarToCommon: "Es similar a una contrase\xF1a utilizada habitualmente.",
        simpleRepeat: 'Caracteres repetidos como "aaa" son f\xE1ciles de adivinar',
        straightRow: "Teclas consecutivas en tu teclado son f\xE1ciles de adivinar.",
        topHundred: "Es una contrase\xF1a utilizada con mucha frecuencia.",
        topTen: "Es de las contrase\xF1as m\xE1s utilizadas.",
        userInputs: "No deber\xEDa haber datos personales o relacionados con esta p\xE1gina.",
        wordByItself: "Palabras \xFAnicas son f\xE1ciles de adivinar."
      }
    }
  },
  userButton: {
    action__addAccount: "A\xF1adir cuenta",
    action__closeUserMenu: void 0,
    action__manageAccount: "Administrar cuenta",
    action__openUserMenu: void 0,
    action__signOut: "Cerrar sesi\xF3n",
    action__signOutAll: "Salir de todas las cuentas"
  },
  userProfile: {
    apiKeysPage: {
      title: void 0
    },
    backupCodePage: {
      actionLabel__copied: "Copiado!",
      actionLabel__copy: "Copiar todo",
      actionLabel__download: "Descargar .txt",
      actionLabel__print: "Imprimir",
      infoText1: "Se habilitar\xE1n c\xF3digos de respaldo para esta cuenta.",
      infoText2: "Mant\xE9n los c\xF3digos de respaldo en secreto y gu\xE1rdalos de forma segura. Puedes regenerar c\xF3digos de respaldo si sospechas que se han visto comprometidos.",
      subtitle__codelist: "Gu\xE1rdalos de forma segura y mantenlos en secreto.",
      successMessage: "Los c\xF3digos de respaldo ahora est\xE1n habilitados. Puedes utilizar uno de estos para iniciar sesi\xF3n en tu cuenta, si pierdes el acceso a tu dispositivo de autenticaci\xF3n. Cada c\xF3digo solo se puede utilizar una vez.",
      successSubtitle: "Puedes utilizar uno de estos para iniciar sesi\xF3n en tu cuenta, si pierdes el acceso a tu dispositivo de autenticaci\xF3n.",
      title: "Agregar verificaci\xF3n de c\xF3digo de respaldo",
      title__codelist: "C\xF3digos de respaldo"
    },
    billingPage: {
      paymentHistorySection: {
        empty: void 0,
        notFound: void 0,
        tableHeader__amount: void 0,
        tableHeader__date: void 0,
        tableHeader__status: void 0
      },
      paymentMethodsSection: {
        actionLabel__default: "Predeterminado",
        actionLabel__remove: "Eliminar",
        add: "Agregar un nuevo m\xE9todo de pago",
        addSubtitle: "Agregar un nuevo m\xE9todo de pago a tu cuenta.",
        cancelButton: "Cancelar",
        formButtonPrimary__add: "Agregar M\xE9todo de Pago",
        formButtonPrimary__pay: "Pagar {{amount}}",
        payWithTestCardButton: void 0,
        removeMethod: {
          messageLine1: "{{identifier}} ser\xE1 eliminado de esta cuenta.",
          messageLine2: "No podr\xE1s utilizar este m\xE9todo de pago para ninguna suscripci\xF3n actual y estas ya no funcionar\xE1n.",
          successMessage: "{{paymentMethod}} ha sido eliminada de tu cuenta.",
          title: "Eliminar m\xE9todo de pago"
        },
        title: "Opciones disponibles"
      },
      start: {
        headerTitle__payments: void 0,
        headerTitle__plans: "Planes",
        headerTitle__statements: "Facturas",
        headerTitle__subscriptions: "Suscripciones"
      },
      statementsSection: {
        empty: void 0,
        itemCaption__paidForPlan: void 0,
        itemCaption__proratedCredit: void 0,
        itemCaption__subscribedAndPaidForPlan: void 0,
        notFound: void 0,
        tableHeader__amount: void 0,
        tableHeader__date: void 0,
        title: void 0,
        totalPaid: void 0
      },
      subscriptionsListSection: {
        actionLabel__manageSubscription: void 0,
        actionLabel__newSubscription: void 0,
        actionLabel__switchPlan: void 0,
        tableHeader__edit: void 0,
        tableHeader__plan: void 0,
        tableHeader__startDate: void 0,
        title: void 0
      },
      subscriptionsSection: {
        actionLabel__default: void 0
      },
      switchPlansSection: {
        title: void 0
      },
      title: "Facturaci\xF3n y Pagos"
    },
    connectedAccountPage: {
      formHint: "Selecciona un proveedor para conectar su cuenta.",
      formHint__noAccounts: "No hay proveedores de cuentas externas disponibles.",
      removeResource: {
        messageLine1: "{{identifier}} ser\xE1 eliminado de esta cuenta.",
        messageLine2: "Ya no podr\xE1s utilizar esta cuenta activa y las funciones dependientes ya no funcionar\xE1n.",
        successMessage: "{{connectedAccount}} ha sido eliminado de tu cuenta.",
        title: "Eliminar cuenta conectada"
      },
      socialButtonsBlockButton: "{{provider|titleize}}",
      successMessage: "El proveedor ha sido agregado a tu cuenta",
      title: "Agregar cuenta conectada"
    },
    deletePage: {
      actionDescription: 'Escribe "Eliminar cuenta" a continuaci\xF3n para continuar',
      confirm: "Eliminar cuenta",
      messageLine1: "\xBFEst\xE1 seguro de que desea eliminar su cuenta? Es posible que se conserven algunos datos asociados. Para solicitar la eliminaci\xF3n completa de datos, comun\xEDquese con el soporte.",
      messageLine2: "Esta acci\xF3n es permanente e irreversible.",
      title: "Eliminar cuenta"
    },
    emailAddressPage: {
      emailCode: {
        formHint: "A esta direcci\xF3n de correo electr\xF3nico se le enviar\xE1 un correo electr\xF3nico con un c\xF3digo de verificaci\xF3n.",
        formSubtitle: "Introduce el c\xF3digo de verificaci\xF3n enviado a {{identifier}}",
        formTitle: "C\xF3digo de verificaci\xF3n",
        resendButton: "Reenviar c\xF3digo",
        successMessage: "El correo electr\xF3nico {{identifier}} se ha agregado a tu cuenta."
      },
      emailLink: {
        formHint: "Se enviar\xE1 un correo electr\xF3nico con un enlace de verificaci\xF3n a esta direcci\xF3n de correo electr\xF3nico.",
        formSubtitle: "Haz clic en el enlace de verificaci\xF3n en el correo electr\xF3nico enviado a {{identifier}}",
        formTitle: "Enlace de verificaci\xF3n",
        resendButton: "Reenviar enlace",
        successMessage: "El correo electr\xF3nico {{identifier}} se ha agregado a su cuenta."
      },
      enterpriseSSOLink: {
        formButton: "Click para iniciar sesi\xF3n",
        formSubtitle: "Completar el inicio de sesi\xF3n con {{identifier}}"
      },
      formHint: "Deber\xE1s verificar el correo electr\xF3nico antes de ser a\xF1adido a tu cuenta",
      removeResource: {
        messageLine1: "{{identifier}} ser\xE1 eliminado de esta cuenta.",
        messageLine2: "Ya no podr\xE1s iniciar sesi\xF3n con esta direcci\xF3n de correo electr\xF3nico.",
        successMessage: "{{emailAddress}} ha sido eliminado de tu cuenta.",
        title: "Eliminar direcci\xF3n de correo electr\xF3nico"
      },
      title: "Agregar direcci\xF3n de correo electr\xF3nico",
      verifyTitle: "Verificar direcci\xF3n de correo electr\xF3nico"
    },
    formButtonPrimary__add: "Agregar",
    formButtonPrimary__continue: "Continuar",
    formButtonPrimary__finish: "Terminar",
    formButtonPrimary__remove: "Eliminar",
    formButtonPrimary__save: "Guardar",
    formButtonReset: "Cancelar",
    mfaPage: {
      formHint: "Seleccione un m\xE9todo para agregar.",
      title: "Agregar verificaci\xF3n en dos pasos"
    },
    mfaPhoneCodePage: {
      backButton: "Usar n\xFAmero existente",
      primaryButton__addPhoneNumber: "Agregar n\xFAmero de tel\xE9fono",
      removeResource: {
        messageLine1: "{{identifier}} dejar\xE1 de recibir c\xF3digos de verificaci\xF3n al iniciar sesi\xF3n.",
        messageLine2: "Es posible que tu cuenta no sea tan segura. \xBFEst\xE1s seguro de que quieres continuar?",
        successMessage: "Se elimin\xF3 la verificaci\xF3n de dos pasos por SMS para {{mfaPhoneCode}}",
        title: "Eliminar la verificaci\xF3n en dos pasos"
      },
      subtitle__availablePhoneNumbers: "Selecciona un n\xFAmero de tel\xE9fono para registrar la verificaci\xF3n en dos pasos por SMS.",
      subtitle__unavailablePhoneNumbers: "No hay n\xFAmeros de tel\xE9fono disponibles para registrar la verificaci\xF3n en dos pasos por SMS.",
      successMessage1: "Al iniciar sesi\xF3n, se te solicitar\xE1 un c\xF3digo de verificaci\xF3n enviado a este n\xFAmero de tel\xE9fono como un paso adicional.",
      successMessage2: "Guarda estos c\xF3digos de respaldo y almac\xE9nalos en un lugar seguro. Si pierdes el acceso a tu dispositivo de autenticaci\xF3n, puedes utilizar los c\xF3digos de respaldo para iniciar sesi\xF3n.",
      successTitle: "Verificaci\xF3n por SMS habilitada",
      title: "Agregar verificaci\xF3n por SMS"
    },
    mfaTOTPPage: {
      authenticatorApp: {
        buttonAbleToScan__nonPrimary: "Escanea el c\xF3digo QR en su lugar",
        buttonUnableToScan__nonPrimary: "\xBFNo puedes escanear el c\xF3digo QR?",
        infoText__ableToScan: "Configura un nuevo m\xE9todo de inicio de sesi\xF3n en tu aplicaci\xF3n de autenticaci\xF3n y escanea el siguiente c\xF3digo QR para vincularlo a tu cuenta.",
        infoText__unableToScan: "Configura un nuevo m\xE9todo de inicio de sesi\xF3n en tu autenticador e ingresa la clave que se proporciona a continuaci\xF3n.",
        inputLabel__unableToScan1: "Aseg\xFArate de que las contrase\xF1as de Un-Solo-Uso o Por-Tiempo est\xE9n habilitadas, luego finaliza la vinculaci\xF3n con tu cuenta.",
        inputLabel__unableToScan2: "Alternativamente, si tu autenticador admite URIs TOTP, tambi\xE9n puedes copiar la URI completa."
      },
      removeResource: {
        messageLine1: "El c\xF3digo de verificaci\xF3n de este autenticador ya no ser\xE1 necesario al iniciar sesi\xF3n.",
        messageLine2: "Es posible que tu cuenta no sea tan segura. \xBFEst\xE1s seguro de que quieres continuar?",
        successMessage: "Se elimin\xF3 la verificaci\xF3n en dos pasos a trav\xE9s de la aplicaci\xF3n de autenticaci\xF3n.",
        title: "Eliminar la verificaci\xF3n en dos pasos"
      },
      successMessage: "La verificaci\xF3n en dos pasos ahora est\xE1 habilitada. Al iniciar sesi\xF3n, deber\xE1s ingresar un c\xF3digo de verificaci\xF3n de este autenticador como un paso adicional.",
      title: "Agregar aplicaci\xF3n de autenticaci\xF3n",
      verifySubtitle: "Ingresa el c\xF3digo de verificaci\xF3n generado por tu autenticador",
      verifyTitle: "C\xF3digo de verificaci\xF3n"
    },
    mobileButton__menu: "Men\xFA",
    navbar: {
      account: "Perfil",
      apiKeys: "Claves API",
      billing: "Facturaci\xF3n",
      description: "Administra la informaci\xF3n de tu cuenta.",
      security: "Seguridad",
      title: "Cuenta"
    },
    passkeyScreen: {
      removeResource: {
        messageLine1: "{{name}} ser\xE1 eliminada de esta cuenta.",
        title: "Eliminar llave de acceso"
      },
      subtitle__rename: "Puedes cambiar el nombre de la llave de acceso para que sea m\xE1s f\xE1cil de encontrar.",
      title__rename: "Renombrar llave de acceso"
    },
    passwordPage: {
      checkboxInfoText__signOutOfOtherSessions: "Se recomienda cerrar la sesi\xF3n de todos los otros dispositivos que hayan utilizado su antigua contrase\xF1a.",
      readonly: "Tu contrase\xF1a no se puede editar actualmente porque solo se puede acceder a trav\xE9s de la conexi\xF3n de empresa.",
      successMessage__set: "Se ha establecido tu contrase\xF1a.",
      successMessage__signOutOfOtherSessions: "Se cerr\xF3 la sesi\xF3n de todos los dem\xE1s dispositivos.",
      successMessage__update: "Tu contrase\xF1a ha sido actualizada.",
      title__set: "Configurar contrase\xF1a",
      title__update: "Cambiar contrase\xF1a"
    },
    phoneNumberPage: {
      infoText: "Se enviar\xE1 un mensaje de texto con un enlace de verificaci\xF3n a este n\xFAmero de tel\xE9fono.",
      removeResource: {
        messageLine1: "{{identifier}} ser\xE1 eliminado de esta cuenta.",
        messageLine2: "Ya no podr\xE1s iniciar sesi\xF3n con este n\xFAmero de tel\xE9fono.",
        successMessage: "{{phoneNumber}} ha sido eliminado de tu cuenta.",
        title: "Eliminar n\xFAmero de tel\xE9fono"
      },
      successMessage: "{{identifier}} ha sido a\xF1adido a tu cuenta.",
      title: "Agregar el n\xFAmero de tel\xE9fono",
      verifySubtitle: "Ingresa el c\xF3digo de verificaci\xF3n enviado a {{identifier}}",
      verifyTitle: "Verificar n\xFAmero de tel\xE9fono"
    },
    plansPage: {
      title: void 0
    },
    profilePage: {
      fileDropAreaHint: "Carga una imagen JPG, PNG, GIF o WEBP de menos de 10 MB",
      imageFormDestructiveActionSubtitle: "Eliminar la imagen",
      imageFormSubtitle: "Cargar imagen",
      imageFormTitle: "Imagen de perfil",
      readonly: "Tu informaci\xF3n de perfil ha sido proporcionada por la conexi\xF3n de empresa y no se puede editar.",
      successMessage: "Tu perfil ha sido actualizado.",
      title: "Actualizar perfil"
    },
    start: {
      activeDevicesSection: {
        destructiveAction: "Cerrar sesi\xF3n en el dispositivo",
        title: "Dispositivos activos"
      },
      connectedAccountsSection: {
        actionLabel__connectionFailed: "Int\xE9ntelo nuevamente",
        actionLabel__reauthorize: "Autorizar ahora",
        destructiveActionTitle: "Quitar",
        primaryButton: "Conectar cuenta",
        subtitle__disconnected: "Esta cuenta ha sido desconectada.",
        subtitle__reauthorize: "Los permisos requeridos han sido actualizados, y podr\xEDas experimentar limitaciones. Por favor, autoriza de nuevo esta aplicaci\xF3n para evitar cualquier problema",
        title: "Cuentas conectadas"
      },
      dangerSection: {
        deleteAccountButton: "Eliminar cuenta",
        title: "Eliminar cuenta"
      },
      emailAddressesSection: {
        destructiveAction: "Eliminar correo electr\xF3nico",
        detailsAction__nonPrimary: "Establecer como primario",
        detailsAction__primary: "Completar verificaci\xF3n",
        detailsAction__unverified: "Completar verificaci\xF3n",
        primaryButton: "Agregar correo electr\xF3nico",
        title: "Correos electr\xF3nicos"
      },
      enterpriseAccountsSection: {
        title: "Cuentas de empresa"
      },
      headerTitle__account: "Detalles de la cuenta",
      headerTitle__security: "Seguridad",
      mfaSection: {
        backupCodes: {
          actionLabel__regenerate: "Regenerar",
          headerTitle: "C\xF3digos de respaldo",
          subtitle__regenerate: "Obt\xE9n un nuevo conjunto de c\xF3digos de respaldo seguros. Los c\xF3digos de respaldo anteriores se eliminar\xE1n y no podr\xE1n ser utilizados.",
          title__regenerate: "Regenerar c\xF3digos de respaldo"
        },
        phoneCode: {
          actionLabel__setDefault: "Establecer como predeterminado",
          destructiveActionLabel: "Eliminar n\xFAmero telef\xF3nico"
        },
        primaryButton: "A\xF1adir verificaci\xF3n de dos pasos",
        title: "Verificaci\xF3n de dos pasos",
        totp: {
          destructiveActionTitle: "Eliminar",
          headerTitle: "Aplicaci\xF3n de autenticaci\xF3n"
        }
      },
      passkeysSection: {
        menuAction__destructive: "Eliminar",
        menuAction__rename: "Renombrar",
        primaryButton: "A\xF1adir llave de acceso",
        title: "Llaves de acceso"
      },
      passwordSection: {
        primaryButton__setPassword: "Establecer contrase\xF1a ",
        primaryButton__updatePassword: "Cambiar contrase\xF1a",
        title: "Contrase\xF1a"
      },
      phoneNumbersSection: {
        destructiveAction: "Eliminar n\xFAmero telef\xF3nico",
        detailsAction__nonPrimary: "Establecer como primario",
        detailsAction__primary: "Completar la verificaci\xF3n",
        detailsAction__unverified: "Completar la verificaci\xF3n",
        primaryButton: "Agregar un n\xFAmero telef\xF3nico",
        title: "N\xFAmeros telef\xF3nicos"
      },
      profileSection: {
        primaryButton: "Actualizar perfil",
        title: "Perfil"
      },
      usernameSection: {
        primaryButton__setUsername: "Definir nombre de usuario",
        primaryButton__updateUsername: "Cambiar nombre de usuario",
        title: "Nombre de usuario"
      },
      web3WalletsSection: {
        destructiveAction: "Quitar billetera",
        detailsAction__nonPrimary: "Establecer como primaria",
        primaryButton: "Conectar billetera",
        title: "Billeteras Web3"
      }
    },
    usernamePage: {
      successMessage: "Tu nombre de usuario ha sido actualizado.",
      title__set: "Actualizar nombre de usuario",
      title__update: "Actualizar nombre de usuario"
    },
    web3WalletPage: {
      removeResource: {
        messageLine1: "{{identifier}} ser\xE1 eliminado de esta cuenta.",
        messageLine2: "Ya no podr\xE1s iniciar sesi\xF3n con esta billetera web3.",
        successMessage: "{{web3Wallet}} ha sido eliminado de tu cuenta.",
        title: "Eliminar la billetera web3"
      },
      subtitle__availableWallets: "Selecciona una billetera web3 para conectarse a tu cuenta.",
      subtitle__unavailableWallets: "No hay billeteras web3 disponibles.",
      successMessage: "La billetera ha sido agregada a tu cuenta.",
      title: "A\xF1adir billetera web3",
      web3WalletButtonsBlockButton: "{{provider|titleize}}"
    }
  },
  waitlist: {
    start: {
      actionLink: "Iniciar sesi\xF3n",
      actionText: "\xBFTienes una cuenta?",
      formButton: "\xDAnete a la lista de espera",
      subtitle: "Ingresa tu correo electr\xF3nico y te avisaremos cuando t\xFA lugar est\xE9 listo",
      title: "\xDAnete a la lista de espera"
    },
    success: {
      message: "Ser\xE1s redirigido pronto...",
      subtitle: "Nos pondremos en contacto contigo cuando tu lugar est\xE9 listo",
      title: "\xA1Gracias por unirte a la lista de espera!"
    }
  }
};
// Annotate the CommonJS export names for ESM import in node:
0 && (module.exports = {
  esMX
});
