// @clerk/localizations/dist/fr-FR.js
"use strict";
var __defProp = Object.defineProperty;
var __getOwnPropDesc = Object.getOwnPropertyDescriptor;
var __getOwnPropNames = Object.getOwnPropertyNames;
var __hasOwnProp = Object.prototype.hasOwnProperty;
var __export = (target, all) => {
  for (var name in all)
    __defProp(target, name, { get: all[name], enumerable: true });
};
var __copyProps = (to, from, except, desc) => {
  if (from && typeof from === "object" || typeof from === "function") {
    for (let key of __getOwnPropNames(from))
      if (!__hasOwnProp.call(to, key) && key !== except)
        __defProp(to, key, { get: () => from[key], enumerable: !(desc = __getOwnPropDesc(from, key)) || desc.enumerable });
  }
  return to;
};
var __toCommonJS = (mod) => __copyProps(__defProp({}, "__esModule", { value: true }), mod);

// src/fr-FR.ts
var fr_FR_exports = {};
__export(fr_FR_exports, {
  frFR: () => frFR
});
module.exports = __toCommonJS(fr_FR_exports);
var frFR = {
  locale: "fr-FR",
  apiKeys: {
    action__add: "Ajouter une nouvelle cl\xE9",
    action__search: "Rechercher des cl\xE9s",
    createdAndExpirationStatus__expiresOn: "Cr\xE9\xE9e le {{ createdDate | shortDate('fr-FR') }} \u2022 Expire le {{ expiresDate | longDate('fr-FR') }}",
    createdAndExpirationStatus__never: "Cr\xE9\xE9e le {{ createdDate | shortDate('fr-FR') }} \u2022 N\u2019expire jamais",
    detailsTitle__emptyRow: "Aucune cl\xE9 API trouv\xE9e",
    formButtonPrimary__add: "Cr\xE9er une cl\xE9",
    formFieldCaption__expiration__expiresOn: "Expire le {{ date }}",
    formFieldCaption__expiration__never: "Cette cl\xE9 n\u2019expirera jamais",
    formFieldOption__expiration__180d: "180 jours",
    formFieldOption__expiration__1d: "1 jour",
    formFieldOption__expiration__1y: "1 an",
    formFieldOption__expiration__30d: "30 jours",
    formFieldOption__expiration__60d: "60 jours",
    formFieldOption__expiration__7d: "7 jours",
    formFieldOption__expiration__90d: "90 jours",
    formFieldOption__expiration__never: "Jamais",
    formHint: "Indiquez un nom pour g\xE9n\xE9rer une nouvelle cl\xE9. Vous pourrez la r\xE9voquer \xE0 tout moment.",
    formTitle: "Ajouter une nouvelle cl\xE9 API",
    lastUsed__days: "il y a {{days}} j",
    lastUsed__hours: "il y a {{hours}} h",
    lastUsed__minutes: "il y a {{minutes}} min",
    lastUsed__months: "il y a {{months}} mois",
    lastUsed__seconds: "il y a {{seconds}} s",
    lastUsed__years: "il y a {{years}} an(s)",
    menuAction__revoke: "R\xE9voquer la cl\xE9",
    revokeConfirmation: {
      confirmationText: "R\xE9voquer",
      formButtonPrimary__revoke: "R\xE9voquer la cl\xE9",
      formHint: "\xCAtes-vous s\xFBr de vouloir supprimer cette cl\xE9 secr\xE8te ?",
      formTitle: "R\xE9voquer la cl\xE9 secr\xE8te \xAB {{apiKeyName}} \xBB ?"
    }
  },
  backButton: "Retour",
  badge__activePlan: "Actif",
  badge__canceledEndsAt: "Annul\xE9 \u2022 Termine le {{ date | shortDate('fr-FR') }}",
  badge__currentPlan: "Plan actuel",
  badge__default: "D\xE9faut",
  badge__endsAt: "Termine le {{ date | shortDate('fr-FR') }}",
  badge__expired: "Expir\xE9",
  badge__freeTrial: "Essai gratuit",
  badge__otherImpersonatorDevice: "Autre dispositif d'imitation",
  badge__pastDueAt: "En retard depuis le {{ date | shortDate('fr-FR') }}",
  badge__pastDuePlan: "Paiement en retard",
  badge__primary: "Principal",
  badge__renewsAt: "Renouvelle le {{ date | shortDate('fr-FR') }}",
  badge__requiresAction: "N\xE9cessite une action",
  badge__startsAt: "D\xE9bute le {{ date | shortDate('fr-FR') }}",
  badge__thisDevice: "Cet appareil",
  badge__trialEndsAt: "L\u2019essai se termine le {{ date | shortDate('fr-FR') }}",
  badge__unverified: "Non v\xE9rifi\xE9",
  badge__upcomingPlan: "\xC0 venir",
  badge__userDevice: "Appareil utilisateur",
  badge__you: "Vous",
  billing: {
    addPaymentMethod__label: "Ajouter une m\xE9thode de paiement",
    alwaysFree: "Toujours gratuit",
    annually: "Annuel",
    availableFeatures: "Fonctionnalit\xE9s disponibles",
    billedAnnually: "Factur\xE9 annuellement",
    billedMonthlyOnly: "Seulement factur\xE9 mensuellement",
    cancelFreeTrial: "Annuler l\u2019essai gratuit",
    cancelFreeTrialAccessUntil: "Votre essai restera actif jusqu\u2019au {{ date | longDate('fr-FR') }}. Apr\xE8s cette date, vous perdrez l\u2019acc\xE8s aux fonctionnalit\xE9s d\u2019essai. Aucun frais ne vous sera factur\xE9.",
    cancelFreeTrialTitle: "Annuler l\u2019essai gratuit du plan {{plan}} ?",
    cancelSubscription: "Annuler la souscription",
    cancelSubscriptionAccessUntil: "Souscription annul\xE9e. Vous pouvez continuer \xE0 utiliser les fonctionnalit\xE9s de '{{plan}}' jusqu'au {{ date | longDate('fr-FR') }}, apr\xE8s quoi vous n'aurez plus acc\xE8s.",
    cancelSubscriptionNoCharge: "Pas de charge",
    cancelSubscriptionPastDue: "Votre abonnement prendra fin imm\xE9diatement et vous perdrez l\u2019acc\xE8s \xE0 toutes les fonctionnalit\xE9s du plan. Le montant d\xFB sera demand\xE9 lors de votre prochaine souscription.",
    cancelSubscriptionTitle: "Annuler la souscription",
    cannotSubscribeMonthly: "Ne peut pas souscrire mensuellement. Pour souscrire \xE0 ce plan, il faut payer annuellement.",
    cannotSubscribeUnrecoverable: "Vous ne pouvez pas souscrire \xE0 ce plan. Votre abonnement actuel est plus cher que celui-ci.",
    checkout: {
      description__paymentSuccessful: "Votre paiement a \xE9t\xE9 effectu\xE9 avec succ\xE8s.",
      description__subscriptionSuccessful: "Votre nouvel abonnement est pr\xEAt.",
      downgradeNotice: "Vous conserverez votre abonnement actuel et ses fonctionnalit\xE9s jusqu'\xE0 la fin du cycle de facturation, puis vous passerez \xE0 cet abonnement.",
      emailForm: {
        subtitle: "Avant de pouvoir finaliser votre achat, vous devez ajouter une adresse e-mail o\xF9 les re\xE7us seront envoy\xE9s.",
        title: "Ajouter une adresse e-mail"
      },
      lineItems: {
        title__freeTrialEndsAt: "L\u2019essai se termine le",
        title__paymentMethod: "M\xE9thode de paiement",
        title__statementId: "Identifiant du relev\xE9",
        title__subscriptionBegins: "L'abonnement commence",
        title__totalPaid: "Total pay\xE9"
      },
      pastDueNotice: "Votre abonnement pr\xE9c\xE9dent \xE9tait en retard de paiement.",
      perMonth: "par mois",
      title: "Paiement",
      title__paymentSuccessful: "Le paiement a r\xE9ussi !",
      title__subscriptionSuccessful: "Succ\xE8s !",
      title__trialSuccess: "Essai d\xE9marr\xE9 avec succ\xE8s !",
      totalDueAfterTrial: "Total d\xFB apr\xE8s la fin de l\u2019essai dans {{days}} jours"
    },
    credit: "Cr\xE9dit",
    creditRemainder: "Cr\xE9dit restant",
    defaultFreePlanActive: "Vous \xEAtes actuellement sur le plan gratuit",
    free: "Gratuit",
    getStarted: "Commencer",
    keepFreeTrial: "Conserver l\u2019essai gratuit",
    keepSubscription: "Conserver l'abonnement",
    manage: "G\xE9rer",
    manageSubscription: "G\xE9rer l'abonnement",
    month: "Mois",
    monthly: "Mensuel",
    pastDue: "En retard",
    pay: "Payer {{amount}}",
    paymentMethod: {
      applePayDescription: {
        annual: "Paiement annuel",
        monthly: "Paiement mensuel"
      },
      dev: {
        anyNumbers: "N\u2019importe quels chiffres",
        cardNumber: "Num\xE9ro de carte",
        cvcZip: "CVC, ZIP",
        developmentMode: "Mode de d\xE9veloppement",
        expirationDate: "Date d'expiration",
        testCardInfo: "Informations de carte de test"
      }
    },
    paymentMethods__label: "M\xE9thodes de paiement",
    popular: "Populaire",
    pricingTable: {
      billingCycle: "Cycle de facturation",
      included: "Inclus"
    },
    reSubscribe: "Se r\xE9abonner",
    seeAllFeatures: "Voir toutes les fonctionnalit\xE9s",
    startFreeTrial: "Commencer l\u2019essai gratuit",
    startFreeTrial__days: "Commencer l\u2019essai gratuit de {{days}} jours",
    subscribe: "S'abonner",
    subscriptionDetails: {
      beginsOn: "Commence le",
      currentBillingCycle: "Cycle de facturation en cours",
      endsOn: "Se termine le",
      firstPaymentAmount: "Montant du premier paiement",
      firstPaymentOn: "Premier paiement le",
      nextPaymentAmount: "Montant du prochain paiement",
      nextPaymentOn: "Prochain paiement le",
      pastDueAt: "En retard le",
      renewsAt: "Renouvel\xE9 le",
      subscribedOn: "Souscrit le",
      title: "Abonnement",
      trialEndsOn: "L\u2019essai se termine le",
      trialStartedOn: "Essai commenc\xE9 le"
    },
    subtotal: "Total",
    switchPlan: "Changer de plan",
    switchToAnnual: "Passer \xE0 l'annuel",
    switchToAnnualWithAnnualPrice: "Passer \xE0 l\u2019annuel {{currency}}{{price}} / an",
    switchToMonthly: "Passer au mensuel",
    switchToMonthlyWithPrice: "Passer au mensuel {{currency}}{{price}} / mois",
    totalDue: "Total d\xFB",
    totalDueToday: "Total d\xFB aujourd'hui",
    viewFeatures: "Voir les fonctionnalit\xE9s",
    viewPayment: "Voir le paiement",
    year: "An"
  },
  createOrganization: {
    formButtonSubmit: "Cr\xE9er l\u2019organisation",
    invitePage: {
      formButtonReset: "Passer"
    },
    title: "Cr\xE9er une organisation"
  },
  dates: {
    lastDay: "Hier \xE0 {{ date | timeString('fr-FR') }}",
    next6Days: "{{ date | weekday('fr-FR','long') }} \xE0 {{ date | timeString('fr-FR') }}",
    nextDay: "Demain \xE0 {{ date | timeString('fr-FR') }}",
    numeric: "{{ date | numeric('fr-FR') }}",
    previous6Days: "{{ date | weekday('fr-FR','long') }} dernier \xE0 {{ date | timeString('fr-FR') }}",
    sameDay: "Aujourd'hui \xE0 {{ date | timeString('fr-FR') }}"
  },
  dividerText: "ou",
  footerActionLink__alternativePhoneCodeProvider: "Envoyer le code par SMS \xE0 la place",
  footerActionLink__useAnotherMethod: "Utiliser une autre m\xE9thode",
  footerPageLink__help: "Aide",
  footerPageLink__privacy: "Vie priv\xE9e",
  footerPageLink__terms: "Conditions",
  formButtonPrimary: "Continuer",
  formButtonPrimary__verify: "V\xE9rifier",
  formFieldAction__forgotPassword: "Mot de passe oubli\xE9 ?",
  formFieldError__matchingPasswords: "Les mots de passe correspondent.",
  formFieldError__notMatchingPasswords: "Les mots de passe ne correspondent pas.",
  formFieldError__verificationLinkExpired: "Le lien de v\xE9rification a expir\xE9. Merci de demander un nouveau lien.",
  formFieldHintText__optional: "Optionnel",
  formFieldHintText__slug: "Un slug est un identifiant lisible qui doit \xEAtre unique. Il est souvent utilis\xE9 dans les URL.",
  formFieldInputPlaceholder__apiKeyDescription: "Expliquez pourquoi vous g\xE9n\xE9rez cette cl\xE9",
  formFieldInputPlaceholder__apiKeyExpirationDate: "S\xE9lectionner une date",
  formFieldInputPlaceholder__apiKeyName: "Entrez le nom de votre cl\xE9 secr\xE8te",
  formFieldInputPlaceholder__backupCode: "Code de sauvegarde",
  formFieldInputPlaceholder__confirmDeletionUserAccount: "Supprimer le compte",
  formFieldInputPlaceholder__emailAddress: "Adresse e-mail",
  formFieldInputPlaceholder__emailAddress_username: "Nom d'utilisateur ou adresse e-mail",
  formFieldInputPlaceholder__emailAddresses: "exemple@email.com, exemple2@email.com",
  formFieldInputPlaceholder__firstName: "Pr\xE9nom",
  formFieldInputPlaceholder__lastName: "Nom de famille",
  formFieldInputPlaceholder__organizationDomain: "Domaine de l'organisation",
  formFieldInputPlaceholder__organizationDomainEmailAddress: "Adresse e-mail de l'organisation",
  formFieldInputPlaceholder__organizationName: "Nom de l'organisation",
  formFieldInputPlaceholder__organizationSlug: "Identifiant de l'organisation",
  formFieldInputPlaceholder__password: "Mot de passe",
  formFieldInputPlaceholder__phoneNumber: "Num\xE9ro de t\xE9l\xE9phone",
  formFieldInputPlaceholder__username: "Nom d'utilisateur",
  formFieldInput__emailAddress_format: void 0,
  formFieldLabel__apiKeyDescription: "Description",
  formFieldLabel__apiKeyExpiration: "Expiration",
  formFieldLabel__apiKeyName: "Nom de la cl\xE9 secr\xE8te",
  formFieldLabel__automaticInvitations: "Autoriser les invitations automatiques pour ce domaine",
  formFieldLabel__backupCode: "Code de r\xE9cup\xE9ration",
  formFieldLabel__confirmDeletion: "Confirmation",
  formFieldLabel__confirmPassword: "Confirmer le mot de passe",
  formFieldLabel__currentPassword: "Mot de passe actuel",
  formFieldLabel__emailAddress: "Adresse e-mail",
  formFieldLabel__emailAddress_username: "Adresse e-mail ou nom d'utilisateur",
  formFieldLabel__emailAddresses: "Adresses e-mail",
  formFieldLabel__firstName: "Pr\xE9nom",
  formFieldLabel__lastName: "Nom de famille",
  formFieldLabel__newPassword: "Nouveau mot de passe",
  formFieldLabel__organizationDomain: "Domaine",
  formFieldLabel__organizationDomainDeletePending: "Supprimer les invitations et suggestions en attente",
  formFieldLabel__organizationDomainEmailAddress: "E-mail de v\xE9rification",
  formFieldLabel__organizationDomainEmailAddressDescription: "Entrer une adresse e-mail appartenant \xE0 ce domaine pour recevoir un code et v\xE9rifier ce domaine.",
  formFieldLabel__organizationName: "Nom de l'organisation",
  formFieldLabel__organizationSlug: "Slug URL",
  formFieldLabel__passkeyName: "Nom de la cl\xE9 de s\xE9curit\xE9",
  formFieldLabel__password: "Mot de passe",
  formFieldLabel__phoneNumber: "Num\xE9ro de t\xE9l\xE9phone",
  formFieldLabel__role: "R\xF4le",
  formFieldLabel__signOutOfOtherSessions: "Se d\xE9connecter de tous les autres appareils",
  formFieldLabel__username: "Nom d'utilisateur",
  impersonationFab: {
    action__signOut: "D\xE9connexion",
    title: "Connect\xE9 en tant que {{identifier}}"
  },
  lastAuthenticationStrategy: "Derni\xE8re utilisation",
  maintenanceMode: "Nous effectuons des travaux de maintenance, mais ne vous en inqui\xE9tez pas, cela ne devrait pas prendre plus de quelques minutes.",
  membershipRole__admin: "Administrateur",
  membershipRole__basicMember: "Membre",
  membershipRole__guestMember: "Invit\xE9",
  organizationList: {
    action__createOrganization: "Cr\xE9er une organisation",
    action__invitationAccept: "Rejoindre",
    action__suggestionsAccept: "Demande d'adh\xE9sion",
    createOrganization: "Cr\xE9er une Organisation",
    invitationAcceptedLabel: "Accept\xE9e",
    subtitle: "pour continuer vers {{applicationName}}",
    suggestionsAcceptedLabel: "En attente d\u2019approbation",
    title: "Choisissez un compte",
    titleWithoutPersonal: "Choisissez une organisation"
  },
  organizationProfile: {
    apiKeysPage: {
      title: "Cl\xE9s API"
    },
    badge__automaticInvitation: "Invitations automatiques",
    badge__automaticSuggestion: "Suggestions automatiques",
    badge__manualInvitation: "Pas d'inscription automatique",
    badge__unverified: "Non v\xE9rifi\xE9",
    billingPage: {
      paymentHistorySection: {
        empty: "Aucun historique de paiement",
        notFound: "Tentative de paiement introuvable",
        tableHeader__amount: "Montant",
        tableHeader__date: "Date",
        tableHeader__status: "Statut"
      },
      paymentMethodsSection: {
        actionLabel__default: "Rendre par d\xE9faut",
        actionLabel__remove: "Supprimer",
        add: "Ajouter une nouvelle m\xE9thode de paiement",
        addSubtitle: "Ajoutez une nouvelle m\xE9thode de paiement \xE0 votre compte.",
        cancelButton: "Annuler",
        formButtonPrimary__add: "Ajouter une m\xE9thode de paiement",
        formButtonPrimary__pay: "Payer {{amount}}",
        payWithTestCardButton: "Payer avec une carte de test",
        removeMethod: {
          messageLine1: "{{identifier}} sera supprim\xE9 de ce compte.",
          messageLine2: "Vous ne pourrez plus utiliser cette m\xE9thode de paiement et tous les abonnements r\xE9currents qui en d\xE9pendent ne fonctionneront plus.",
          successMessage: "{{paymentMethod}} a \xE9t\xE9 supprim\xE9 de votre compte.",
          title: "Supprimer une m\xE9thode de paiement"
        },
        title: "M\xE9thodes de paiement"
      },
      start: {
        headerTitle__payments: "Paiements",
        headerTitle__plans: "Plans",
        headerTitle__statements: "Relev\xE9s",
        headerTitle__subscriptions: "Abonnements"
      },
      statementsSection: {
        empty: "Aucun relev\xE9 \xE0 afficher",
        itemCaption__paidForPlan: "Pay\xE9 pour le plan {{plan}} {{period}}",
        itemCaption__proratedCredit: "Cr\xE9dit au prorata pour l\u2019utilisation partielle de l\u2019abonnement pr\xE9c\xE9dent",
        itemCaption__subscribedAndPaidForPlan: "Souscrit et pay\xE9 pour le plan {{plan}} {{period}}",
        notFound: "Relev\xE9 introuvable",
        tableHeader__amount: "Montant",
        tableHeader__date: "Date",
        title: "Relev\xE9s",
        totalPaid: "Total pay\xE9"
      },
      subscriptionsListSection: {
        actionLabel__manageSubscription: "G\xE9rer",
        actionLabel__newSubscription: "S'abonner \xE0 un plan",
        actionLabel__switchPlan: "Changer de plan",
        tableHeader__edit: "Modifier",
        tableHeader__plan: "Plan",
        tableHeader__startDate: "Date de d\xE9but",
        title: "Abonnement"
      },
      subscriptionsSection: {
        actionLabel__default: "G\xE9rer"
      },
      switchPlansSection: {
        title: "Changer de plan"
      },
      title: "Abonnements"
    },
    createDomainPage: {
      subtitle: "Ajoutez le domaine pour le v\xE9rifier. Les utilisateurs poss\xE9dant une adresses e-mail sur ce domaine peuvent rejoindre l'organisation automatiquement ou faire une demande pour y adh\xE9rer.",
      title: "Ajouter un domaine"
    },
    invitePage: {
      detailsTitle__inviteFailed: "Les invitations suivantes n\u2019ont pas pu \xEAtre envoy\xE9es. Veuillez r\xE9gler ce probl\xE8me et r\xE9essayer:",
      formButtonPrimary__continue: "Envoyer des invitations",
      selectDropdown__role: "S\xE9lectionner un r\xF4le",
      subtitle: "Inviter des membres \xE0 rejoindre l\u2019organisation",
      successMessage: "Les invitations ont \xE9t\xE9 envoy\xE9es.",
      title: "Inviter des membres"
    },
    membersPage: {
      action__invite: "Inviter",
      action__search: "Rechercher",
      activeMembersTab: {
        menuAction__remove: "Supprimer",
        tableHeader__actions: "Actions",
        tableHeader__joined: "Rejoint",
        tableHeader__role: "R\xF4le",
        tableHeader__user: "Utilisateur"
      },
      detailsTitle__emptyRow: "Aucun membre",
      invitationsTab: {
        autoInvitations: {
          headerSubtitle: "Invitez des utilisateurs en connectant un domaine de messagerie \xE0 votre organisation. Toute personne s'inscrivant avec une adresses e-mail sur ce domaine pourra rejoindre l'organisation.",
          headerTitle: "Invitations automatiques",
          primaryButton: "G\xE9rer les domaines valid\xE9s"
        },
        table__emptyRow: "Pas d'invitations \xE0 afficher"
      },
      invitedMembersTab: {
        menuAction__revoke: "R\xE9voquer l'invitation",
        tableHeader__invited: "Invit\xE9"
      },
      requestsTab: {
        autoSuggestions: {
          headerSubtitle: "Les utilisateurs qui s'inscrivent avec un domaine de messagerie identique verront une suggestion pour demander \xE0 rejoindre votre organisation.",
          headerTitle: "Suggestions automatiques",
          primaryButton: "G\xE9rer les domaines valid\xE9s"
        },
        menuAction__approve: "Approuver",
        menuAction__reject: "Rejeter",
        tableHeader__requested: "Acc\xE8s demand\xE9",
        table__emptyRow: "Pas de demandes \xE0 afficher"
      },
      start: {
        headerTitle__invitations: "Invitations",
        headerTitle__members: "Membres",
        headerTitle__requests: "Demandes"
      }
    },
    navbar: {
      apiKeys: "Cl\xE9s API",
      billing: "Facturation",
      description: "G\xE9rer votre organisation.",
      general: "G\xE9n\xE9ral",
      members: "Membres",
      title: "Organisation"
    },
    plansPage: {
      alerts: {
        noPermissionsToManageBilling: "Vous n'avez pas les permissions pour g\xE9rer la facturation de cette organisation."
      },
      title: "Plans"
    },
    profilePage: {
      dangerSection: {
        deleteOrganization: {
          actionDescription: "Saisissez {{organizationName}} ci-dessous pour continuer.",
          messageLine1: "\xCAtes-vous s\xFBr(e) de vouloir supprimer cette organisation ?",
          messageLine2: "Cette action est d\xE9finitive et irr\xE9versible.",
          successMessage: "Vous avez supprim\xE9 l'organisation.",
          title: "Supprimer l'organisation"
        },
        leaveOrganization: {
          actionDescription: "Saisissez {{organizationName}} ci-dessous pour continuer.",
          messageLine1: "\xCAtes-vous s\xFBr de vouloir quitter cette organisation ? Vous perdrez l'acc\xE8s \xE0 cette organisation et \xE0 ses applications.",
          messageLine2: "Cette action est permanente et irr\xE9versible.",
          successMessage: "Vous avez quitt\xE9 l'organisation.",
          title: "Quitter l'organisation"
        },
        title: "Danger"
      },
      domainSection: {
        menuAction__manage: "G\xE9rer",
        menuAction__remove: "Supprimer",
        menuAction__verify: "Valider",
        primaryButton: "Ajouter un domaine",
        subtitle: "Permettre aux utilisateurs de rejoindre l'organisation automatiquement ou de faire une demande d'adh\xE9sion si leur domaine de messagerie est v\xE9rifi\xE9.",
        title: "Domaines v\xE9rifi\xE9s"
      },
      successMessage: "L'organisation a \xE9t\xE9 mise \xE0 jour.",
      title: "Profil de l\u2019organisation"
    },
    removeDomainPage: {
      messageLine1: "Le domaine de messagerie {{domain}} sera supprim\xE9.",
      messageLine2: "Les utilisateurs ne pourront plus rejoindre l'organisation automatiquement apr\xE8s cela.",
      successMessage: "{{domain}} a \xE9t\xE9 supprim\xE9.",
      title: "Supprimer un domaine"
    },
    start: {
      headerTitle__general: "G\xE9n\xE9ral",
      headerTitle__members: "Membres",
      profileSection: {
        primaryButton: "Mettre \xE0 jour le profil",
        title: "Profil de l'organisation",
        uploadAction__title: "Logo"
      }
    },
    verifiedDomainPage: {
      dangerTab: {
        calloutInfoLabel: "Supprimer ce domaine affectera les utilisateurs invit\xE9s.",
        removeDomainActionLabel__remove: "Supprimer ce domaine",
        removeDomainSubtitle: "Supprimer ce domaine de vos domaines v\xE9rifi\xE9s",
        removeDomainTitle: "Supprimer un domaine"
      },
      enrollmentTab: {
        automaticInvitationOption__description: "Les utilisateurs sont automatiquement invit\xE9s \xE0 rejoindre l'organisation lors de leur inscription et peuvent la rejoindre \xE0 tout moment.",
        automaticInvitationOption__label: "Invitations automatiques",
        automaticSuggestionOption__description: "Les utilisateurs re\xE7oivent une suggestion d'adh\xE9sion \xE0 l'organisation, mais doivent \xEAtre approuv\xE9s par un administrateur avant de pouvoir y adh\xE9rer.",
        automaticSuggestionOption__label: "Suggestions automatiques",
        calloutInfoLabel: "Changer le mode d'inscription n'affectera que les nouveaux utilisateurs.",
        calloutInvitationCountLabel: "Invitations en attente envoy\xE9es aux utilisateurs : {{count}}",
        calloutSuggestionCountLabel: "Suggestions en attente envoy\xE9es aux utilisateurs : {{count}}",
        manualInvitationOption__description: "Les utilisateurs peuvent uniquement \xEAtre invit\xE9s \xE0 l'organisation manuellement.",
        manualInvitationOption__label: "Pas d'inscription automatique",
        subtitle: "Choisissez comment les utilisateurs de ce domaine peuvent rejoindre l'organisation."
      },
      start: {
        headerTitle__danger: "Danger",
        headerTitle__enrollment: "Option d'inscription"
      },
      subtitle: "Le domaine {{domain}} est maintenant v\xE9rifi\xE9. Poursuivez en s\xE9lectionnant le mode d'inscription.",
      title: "Mettre \xE0 jour {{domain}}"
    },
    verifyDomainPage: {
      formSubtitle: "Saisissez le code de v\xE9rification envoy\xE9 \xE0 votre adresse e-mail",
      formTitle: "Code de v\xE9rification",
      resendButton: "Vous n'avez pas re\xE7u de code ? Renvoyer",
      subtitle: "Le domaine {{domainName}} doit \xEAtre v\xE9rifi\xE9 par e-mail.",
      subtitleVerificationCodeScreen: "Un code de v\xE9rification a \xE9t\xE9 envoy\xE9 \xE0 {{emailAddress}}. Saisissez le code pour continuer.",
      title: "V\xE9rifier un domaine"
    }
  },
  organizationSwitcher: {
    action__closeOrganizationSwitcher: void 0,
    action__createOrganization: "Cr\xE9er une organisation",
    action__invitationAccept: "Rejoindre",
    action__manageOrganization: "G\xE9rer l'organisation",
    action__openOrganizationSwitcher: void 0,
    action__suggestionsAccept: "Demander \xE0 rejoindre",
    notSelected: "Aucune organisation s\xE9lectionn\xE9e",
    personalWorkspace: "Espace de travail personnel",
    suggestionsAcceptedLabel: "En attente d'acceptation"
  },
  paginationButton__next: "Prochain",
  paginationButton__previous: "Pr\xE9c\xE9dent",
  paginationRowText__displaying: "Affichage",
  paginationRowText__of: "de",
  reverification: {
    alternativeMethods: {
      actionLink: "Utiliser une autre m\xE9thode",
      actionText: "Vous ne pouvez pas acc\xE9der \xE0 votre compte ?",
      blockButton__backupCode: "Utiliser un code de r\xE9cup\xE9ration",
      blockButton__emailCode: "Recevoir un code par e-mail",
      blockButton__passkey: "Utiliser une cl\xE9 de s\xE9curit\xE9",
      blockButton__password: "Utiliser le mot de passe",
      blockButton__phoneCode: "Recevoir un code par t\xE9l\xE9phone",
      blockButton__totp: "Utiliser un code d\u2019application d\u2019authentification",
      getHelp: {
        blockButton__emailSupport: "Contacter le support par e-mail",
        content: "Si vous ne pouvez pas acc\xE9der \xE0 votre compte, contactez notre \xE9quipe de support pour obtenir de l'aide.",
        title: "Obtenir de l'aide"
      },
      subtitle: "Choisissez une m\xE9thode alternative pour v\xE9rifier votre identit\xE9.",
      title: "V\xE9rification alternative"
    },
    backupCodeMfa: {
      subtitle: "Entrez l'un de vos codes de r\xE9cup\xE9ration pour v\xE9rifier votre compte.",
      title: "V\xE9rification par code de r\xE9cup\xE9ration"
    },
    emailCode: {
      formTitle: "Entrez le code de v\xE9rification",
      resendButton: "Renvoyer le code",
      subtitle: "Un code a \xE9t\xE9 envoy\xE9 \xE0 votre adresse e-mail.",
      title: "V\xE9rification par e-mail"
    },
    noAvailableMethods: {
      message: "Aucune m\xE9thode de v\xE9rification n'est disponible.",
      subtitle: "Impossible de proc\xE9der \xE0 la v\xE9rification.",
      title: "Aucune m\xE9thode disponible"
    },
    passkey: {
      blockButton__passkey: "Utiliser votre cl\xE9 de s\xE9curit\xE9",
      subtitle: "L'utilisation de votre cl\xE9 de s\xE9curit\xE9 confirme votre identit\xE9. Votre appareil peut vous demander votre empreinte digitale, votre visage ou votre code de s\xE9curit\xE9.",
      title: "Utiliser votre cl\xE9 de s\xE9curit\xE9"
    },
    password: {
      actionLink: "R\xE9initialiser le mot de passe",
      subtitle: "Entrez votre mot de passe pour continuer.",
      title: "V\xE9rification par mot de passe"
    },
    phoneCode: {
      formTitle: "Entrez le code de v\xE9rification",
      resendButton: "Renvoyer le code",
      subtitle: "Un code a \xE9t\xE9 envoy\xE9 \xE0 votre t\xE9l\xE9phone.",
      title: "V\xE9rification par t\xE9l\xE9phone"
    },
    phoneCodeMfa: {
      formTitle: "Entrez le code de v\xE9rification",
      resendButton: "Renvoyer le code",
      subtitle: "Un code a \xE9t\xE9 envoy\xE9 \xE0 votre t\xE9l\xE9phone pour v\xE9rification.",
      title: "V\xE9rification par t\xE9l\xE9phone"
    },
    totpMfa: {
      formTitle: "Entrez le code de v\xE9rification",
      subtitle: "Entrez le code g\xE9n\xE9r\xE9 par votre application d'authentification.",
      title: "V\xE9rification par application d\u2019authentification"
    }
  },
  signIn: {
    accountSwitcher: {
      action__addAccount: "Ajouter un compte",
      action__signOutAll: "Se d\xE9connecter de tous les comptes",
      subtitle: "S\xE9lectionnez le compte avec lequel vous souhaitez continuer.",
      title: "Choisissez un compte"
    },
    alternativeMethods: {
      actionLink: "Obtenir de l'aide",
      actionText: "Aucune de ces m\xE9thode d'authentification ?",
      blockButton__backupCode: "Utiliser un code de r\xE9cup\xE9ration",
      blockButton__emailCode: "Envoyer le code \xE0 {{identifier}}",
      blockButton__emailLink: "Envoyer le lien \xE0 {{identifier}}",
      blockButton__passkey: "Utiliser une cl\xE9 de s\xE9curit\xE9",
      blockButton__password: "Connectez-vous avec votre mot de passe",
      blockButton__phoneCode: "Envoyer le code \xE0 {{identifier}}",
      blockButton__totp: "Utilisez votre application d'authentification",
      getHelp: {
        blockButton__emailSupport: "Assistance par e-mail",
        content: "Si vous rencontrez des difficult\xE9s pour vous connecter \xE0 votre compte, envoyez-nous un e-mail et nous travaillerons avec vous pour r\xE9tablir l'acc\xE8s d\xE8s que possible.",
        title: "Obtenir de l'aide"
      },
      subtitle: "Vous rencontrez des probl\xE8mes ? Vous pouvez utiliser l'une de ces m\xE9thodes pour vous connecter.",
      title: "Utiliser une autre m\xE9thode"
    },
    alternativePhoneCodeProvider: {
      formTitle: "Code de v\xE9rification",
      resendButton: "Vous n\u2019avez pas re\xE7u de code ? Renvoyer",
      subtitle: "pour continuer vers {{applicationName}}",
      title: "V\xE9rifiez votre {{provider}}"
    },
    backupCodeMfa: {
      subtitle: "pour continuer vers {{applicationName}}",
      title: "Entrez un code de r\xE9cup\xE9ration"
    },
    emailCode: {
      formTitle: "Le code de v\xE9rification",
      resendButton: "Renvoyer le code",
      subtitle: "pour continuer vers {{applicationName}}",
      title: "V\xE9rifiez votre messagerie"
    },
    emailLink: {
      clientMismatch: {
        subtitle: "Ce lien ne correspond pas \xE0 la demande en cours.",
        title: "Erreur de correspondance du client"
      },
      expired: {
        subtitle: "Retournez \xE0 l'onglet d'origine pour continuer",
        title: "Ce lien de v\xE9rification a expir\xE9"
      },
      failed: {
        subtitle: "Retourner \xE0 l'onglet original pour continuer",
        title: "Ce lien de v\xE9rification n'est pas valide"
      },
      formSubtitle: "Utilisez le lien de v\xE9rification envoy\xE9 par e-mail",
      formTitle: "lien de v\xE9rification",
      loading: {
        subtitle: "Vous allez bient\xF4t \xEAtre redirig\xE9",
        title: "Signing in..."
      },
      resendButton: "Renvoyer le lien",
      subtitle: "pour continuer vers {{applicationName}}",
      title: "V\xE9rifiez votre messagerie",
      unusedTab: {
        title: "Vous pouvez fermer cet onglet"
      },
      verified: {
        subtitle: "Vous serez bient\xF4t redirig\xE9",
        title: "Connexion r\xE9ussie"
      },
      verifiedSwitchTab: {
        subtitle: "Revenir \xE0 l'onglet d'origine pour continuer",
        subtitleNewTab: "Revenez \xE0 l'onglet nouvellement ouvert pour continuer",
        titleNewTab: "Connect\xE9 sur un autre onglet"
      }
    },
    enterpriseConnections: {
      subtitle: void 0,
      title: void 0
    },
    forgotPassword: {
      formTitle: "Code de r\xE9initialisation du mot de passe",
      resendButton: "Vous n'avez pas re\xE7u de code ? Renvoyer",
      subtitle: "pour r\xE9initialiser votre mot de passe",
      subtitle_email: "Tout d'abord, saisissez le code envoy\xE9 \xE0 votre adresse e-mail.",
      subtitle_phone: "Tout d'abord, saisissez le code envoy\xE9 \xE0 votre t\xE9l\xE9phone.",
      title: "R\xE9initialiser le mot de passe"
    },
    forgotPasswordAlternativeMethods: {
      blockButton__resetPassword: "R\xE9initialiser votre mot de passe",
      label__alternativeMethods: "Ou connectez-vous avec une autre m\xE9thode.",
      title: "Mot de passe oubli\xE9 ?"
    },
    noAvailableMethods: {
      message: "Impossible de poursuivre la connexion. Aucun facteur d'authentification n'est disponible.",
      subtitle: "Une erreur s'est produite",
      title: "Impossible de se connecter"
    },
    passkey: {
      subtitle: "Utilisez une cl\xE9 de s\xE9curit\xE9 pour continuer.",
      title: "Cl\xE9 de s\xE9curit\xE9"
    },
    password: {
      actionLink: "Utiliser une autre m\xE9thode",
      subtitle: "pour continuer vers {{applicationName}}",
      title: "Tapez votre mot de passe"
    },
    passwordPwned: {
      title: "Mot de passe compromis"
    },
    phoneCode: {
      formTitle: "Code de v\xE9rification",
      resendButton: "Vous n'avez pas re\xE7u de code ? Renvoyer",
      subtitle: "pour continuer vers {{applicationName}}",
      title: "V\xE9rifiez votre t\xE9l\xE9phone"
    },
    phoneCodeMfa: {
      formTitle: "Code de v\xE9rification",
      resendButton: "Vous n'avez pas re\xE7u de code ? Renvoyer",
      subtitle: "Entrez le code envoy\xE9 \xE0 votre t\xE9l\xE9phone pour continuer.",
      title: "V\xE9rifiez votre t\xE9l\xE9phone"
    },
    resetPassword: {
      formButtonPrimary: "R\xE9initialiser",
      requiredMessage: "Pour des raisons de s\xE9curit\xE9, il est n\xE9cessaire de r\xE9initialiser votre mot de passe.",
      successMessage: "Votre mot de passe a \xE9t\xE9 modifi\xE9 avec succ\xE8s. Nous vous reconnectons, veuillez patienter un instant.",
      title: "R\xE9initialiser le mot de passe"
    },
    resetPasswordMfa: {
      detailsLabel: "Nous devons v\xE9rifier votre identit\xE9 avant de r\xE9initialiser votre mot de passe."
    },
    start: {
      actionLink: "S'inscrire",
      actionLink__join_waitlist: "Rejoindre la liste d'attente",
      actionLink__use_email: "Utiliser e-mail",
      actionLink__use_email_username: "Utiliser l'e-mail ou le nom d'utilisateur",
      actionLink__use_passkey: "Utiliser une cl\xE9 de s\xE9curit\xE9",
      actionLink__use_phone: "Utiliser t\xE9l\xE9phone",
      actionLink__use_username: "Utiliser le nom d'utilisateur",
      actionText: "Vous n'avez pas encore de compte ?",
      actionText__join_waitlist: "Inscrivez-vous sur la liste d'attente",
      alternativePhoneCodeProvider: {
        actionLink: "Utiliser une autre m\xE9thode",
        label: "Num\xE9ro de t\xE9l\xE9phone {{provider}}",
        subtitle: "Entrez votre num\xE9ro de t\xE9l\xE9phone pour recevoir un code de v\xE9rification sur {{provider}}.",
        title: "Se connecter \xE0 {{applicationName}} avec {{provider}}"
      },
      subtitle: "pour continuer vers {{applicationName}}",
      subtitleCombined: void 0,
      title: "S'identifier",
      titleCombined: "Continuer vers {{applicationName}}"
    },
    totpMfa: {
      formTitle: "Le code de v\xE9rification",
      subtitle: "Entrez le code de l'application d'authentification.",
      title: "V\xE9rification en deux \xE9tapes"
    }
  },
  signInEnterPasswordTitle: "Tapez votre mot de passe",
  signUp: {
    alternativePhoneCodeProvider: {
      resendButton: "Vous n\u2019avez pas re\xE7u de code ? Renvoyer",
      subtitle: "Entrez le code de v\xE9rification envoy\xE9 \xE0 votre {{provider}}",
      title: "V\xE9rifiez votre {{provider}}"
    },
    continue: {
      actionLink: "S'identifier",
      actionText: "Vous avez d\xE9j\xE0 un compte ?",
      subtitle: "pour continuer vers {{applicationName}}",
      title: "Remplir les champs manquants"
    },
    emailCode: {
      formSubtitle: "Entrez le code de v\xE9rification envoy\xE9 \xE0 votre adresse e-mail",
      formTitle: "Le code de v\xE9rification",
      resendButton: "Renvoyer le code",
      subtitle: "pour continuer vers {{applicationName}}",
      title: "V\xE9rifiez votre e-mail"
    },
    emailLink: {
      clientMismatch: {
        subtitle: "Pour continuer, ouvrir le lien de v\xE9rification sur l'appareil et le navigateur depuis lequel vous avez initi\xE9 l'inscription",
        title: "Erreur de correspondance du client"
      },
      formSubtitle: "Utilisez le lien de v\xE9rification envoy\xE9 \xE0 votre adresse e-mail",
      formTitle: "lien de v\xE9rification",
      loading: {
        title: "Cr\xE9ation de votre compte..."
      },
      resendButton: "Renvoyer le lien",
      subtitle: "pour continuer vers {{applicationName}}",
      title: "V\xE9rifiez votre e-mail",
      verified: {
        title: "Compte cr\xE9\xE9"
      },
      verifiedSwitchTab: {
        subtitle: "Revenez \xE0 l'onglet nouvellement ouvert pour continuer",
        subtitleNewTab: "Revenir \xE0 l'onglet pr\xE9c\xE9dent pour continuer",
        title: "Courriel v\xE9rifi\xE9 avec succ\xE8s"
      }
    },
    enterpriseConnections: {
      subtitle: void 0,
      title: void 0
    },
    legalConsent: {
      checkbox: {
        label__onlyPrivacyPolicy: "J'accepte la Politique de confidentialit\xE9.",
        label__onlyTermsOfService: "J'accepte les Conditions d'utilisation.",
        label__termsOfServiceAndPrivacyPolicy: `J'accepte les {{ termsOfServiceLink || link("Conditions d'utilisation") }} et la {{ privacyPolicyLink || link("Politique de confidentialit\xE9") }}.`
      },
      continue: {
        subtitle: "Lisez et acceptez les conditions pour continuer.",
        title: "Consentement l\xE9gal"
      }
    },
    phoneCode: {
      formSubtitle: "Entrez le code de v\xE9rification envoy\xE9 \xE0 votre num\xE9ro de t\xE9l\xE9phone",
      formTitle: "Le code de v\xE9rification",
      resendButton: "Renvoyer le code",
      subtitle: "pour continuer vers {{applicationName}}",
      title: "V\xE9rifiez votre t\xE9l\xE9phone"
    },
    restrictedAccess: {
      actionLink: "Contacter le support",
      actionText: "Acc\xE8s restreint.",
      blockButton__emailSupport: "Contacter le support par e-mail",
      blockButton__joinWaitlist: "Rejoindre la liste d'attente",
      subtitle: "Vous n'avez pas la permission d'acc\xE9der \xE0 cette page.",
      subtitleWaitlist: "Inscrivez-vous pour demander l'acc\xE8s.",
      title: "Acc\xE8s restreint"
    },
    start: {
      actionLink: "S'identifier",
      actionLink__use_email: "Utiliser votre adresse e-mail",
      actionLink__use_phone: "Utiliser votre t\xE9l\xE9phone",
      actionText: "Vous avez d\xE9j\xE0 un compte ?",
      alternativePhoneCodeProvider: {
        actionLink: "Utiliser une autre m\xE9thode",
        label: "Num\xE9ro de t\xE9l\xE9phone {{provider}}",
        subtitle: "Entrez votre num\xE9ro de t\xE9l\xE9phone pour recevoir un code de v\xE9rification sur {{provider}}.",
        title: "S\u2019inscrire \xE0 {{applicationName}} avec {{provider}}"
      },
      subtitle: "pour continuer vers {{applicationName}}",
      subtitleCombined: "pour continuer vers {{applicationName}}",
      title: "Cr\xE9ez votre compte",
      titleCombined: "Cr\xE9ez votre compte"
    }
  },
  socialButtonsBlockButton: "Continuer avec {{provider|titleize}}",
  socialButtonsBlockButtonManyInView: "{{provider|titleize}}",
  taskChooseOrganization: {
    chooseOrganization: {
      action__createOrganization: "Cr\xE9er une nouvelle organisation",
      action__invitationAccept: "Rejoindre",
      action__suggestionsAccept: "Demander \xE0 rejoindre",
      subtitle: "Rejoignez une organisation existante ou cr\xE9ez-en une nouvelle",
      suggestionsAcceptedLabel: "En attente d\u2019approbation",
      title: "Choisir une organisation"
    },
    createOrganization: {
      formButtonReset: "Annuler",
      formButtonSubmit: "Continuer",
      formFieldInputPlaceholder__name: "Mon organisation",
      formFieldInputPlaceholder__slug: "mon-organisation",
      formFieldLabel__name: "Nom",
      formFieldLabel__slug: "Identifiant (slug)",
      subtitle: "Entrez les d\xE9tails de votre organisation pour continuer",
      title: "Configurer votre organisation"
    },
    signOut: {
      actionLink: "Se d\xE9connecter",
      actionText: "Connect\xE9 en tant que {{identifier}}"
    }
  },
  unstable__errors: {
    already_a_member_in_organization: "Vous \xEAtes d\xE9j\xE0 membre de cette organisation.",
    captcha_invalid: "Inscription \xE9chou\xE9e en raison de validations de s\xE9curit\xE9 incorrectes. Veuillez rafra\xEEchir la page pour r\xE9essayer ou contacter le support pour obtenir de l'aide.",
    captcha_unavailable: "Inscription \xE9chou\xE9e en raison d'une validation de captcha non r\xE9ussie. Veuillez actualiser la page pour r\xE9essayer ou contacter le support pour obtenir de l'aide.",
    form_code_incorrect: "Code incorrect",
    form_identifier_exists__email_address: "Cette adresse e-mail existe d\xE9j\xE0.",
    form_identifier_exists__phone_number: "Ce num\xE9ro de t\xE9l\xE9phone existe d\xE9j\xE0.",
    form_identifier_exists__username: "Ce nom d'utilisateur existe d\xE9j\xE0.",
    form_identifier_not_found: "Nous n'avons pas trouv\xE9 de compte avec ces d\xE9tails.",
    form_param_format_invalid: "Le format est invalide",
    form_param_format_invalid__email_address: "L'adresse e-mail doit \xEAtre une adresse e-mail valide.",
    form_param_format_invalid__phone_number: "Le num\xE9ro de t\xE9l\xE9phone doit \xEAtre au format international.",
    form_param_max_length_exceeded__first_name: "Le pr\xE9nom ne doit pas d\xE9passer 256 caract\xE8res.",
    form_param_max_length_exceeded__last_name: "Le nom ne doit pas d\xE9passer 256 caract\xE8res.",
    form_param_max_length_exceeded__name: "Le nom ne doit pas d\xE9passer 256 caract\xE8res.",
    form_param_nil: "Ce champ est requis.",
    form_param_type_invalid: void 0,
    form_param_type_invalid__email_address: void 0,
    form_param_type_invalid__phone_number: void 0,
    form_param_value_invalid: "La valeur fournie est invalide.",
    form_password_incorrect: "Mot de passe incorrect",
    form_password_length_too_short: "Votre mot de passe est trop court.",
    form_password_not_strong_enough: "Votre mot de passe n'est pas assez fort.",
    form_password_pwned: "Ce mot de passe a \xE9t\xE9 compromis et ne peut pas \xEAtre utilis\xE9. Veuillez essayer un autre mot de passe \xE0 la place.",
    form_password_pwned__sign_in: "Mot de passe compromis. Veuillez le r\xE9initialiser.",
    form_password_size_in_bytes_exceeded: "Votre mot de passe a d\xE9pass\xE9 le nombre maximum d'octets autoris\xE9s. Veuillez le raccourcir ou supprimer certains caract\xE8res sp\xE9ciaux.",
    form_password_validation_failed: "Mot de passe incorrect",
    form_username_invalid_character: "L'identifiant contient des caract\xE8res invalides.",
    form_username_invalid_length: "Le nombre de caract\xE8res de l'identifiant est invalide.",
    identification_deletion_failed: "Vous ne pouvez pas supprimer votre derni\xE8re identification.",
    not_allowed_access: "L'adresse e-mail ou le num\xE9ro de t\xE9l\xE9phone n'est pas autoris\xE9e \xE0 s'inscrire. Cela peut \xEAtre d\xFB \xE0 l'utilisation de '+', '=', '#' ou '.' dans votre adresse e-mail, l'utilisation d'un domaine connect\xE9 \xE0 un service de messagerie temporaire ou l'exclusion explicite. Si vous pensez que c'est une erreur, veuillez contacter le support.",
    organization_domain_blocked: "Ce domaine d'organisation est bloqu\xE9.",
    organization_domain_common: "Ce domaine est trop courant pour une organisation.",
    organization_domain_exists_for_enterprise_connection: void 0,
    organization_membership_quota_exceeded: "Le quota de membres de l'organisation a \xE9t\xE9 d\xE9pass\xE9.",
    organization_minimum_permissions_needed: "Permissions minimales n\xE9cessaires pour acc\xE9der \xE0 cette organisation.",
    organization_not_found_or_unauthorized: "Vous n\u2019\xEAtes plus membre de cette organisation. Veuillez en choisir ou en cr\xE9er une autre.",
    organization_not_found_or_unauthorized_with_create_organization_disabled: "Vous n\u2019\xEAtes plus membre de cette organisation. Veuillez en choisir une autre.",
    passkey_already_exists: "Cette cl\xE9 de s\xE9curit\xE9 existe d\xE9j\xE0.",
    passkey_not_supported: "Les cl\xE9s de s\xE9curit\xE9 ne sont pas prises en charge sur cet appareil.",
    passkey_pa_not_supported: "Les cl\xE9s de s\xE9curit\xE9 ne sont pas prises en charge dans cet environnement.",
    passkey_registration_cancelled: "Enregistrement de la cl\xE9 de s\xE9curit\xE9 annul\xE9.",
    passkey_retrieval_cancelled: "R\xE9cup\xE9ration de la cl\xE9 de s\xE9curit\xE9 annul\xE9e.",
    passwordComplexity: {
      maximumLength: "moins de {{length}} caract\xE8res",
      minimumLength: "{{length}} caract\xE8res ou plus",
      requireLowercase: "une lettre minuscule",
      requireNumbers: "un chiffre",
      requireSpecialCharacter: "un caract\xE8re sp\xE9cial",
      requireUppercase: "une lettre majuscule",
      sentencePrefix: "Votre mot de passe doit contenir"
    },
    phone_number_exists: "Ce num\xE9ro de t\xE9l\xE9phone est d\xE9j\xE0 utilis\xE9. Veuillez essayer un autre.",
    session_exists: "Vous \xEAtes d\xE9j\xE0 connect\xE9.",
    web3_missing_identifier: "Aucune extension de portefeuille Web3 trouv\xE9e. Veuillez en installer une pour continuer.",
    zxcvbn: {
      couldBeStronger: "Votre mot de passe fonctionne mais pourrait \xEAtre plus s\xFBr. Essayez d'ajouter des caract\xE8res.",
      goodPassword: "Bien jou\xE9. C'est un excellent mot de passe.",
      notEnough: "Votre mot de passe n'est pas assez fort.",
      suggestions: {
        allUppercase: "Mettez quelques lettres en majuscules, mais pas toutes.",
        anotherWord: "Ajoutez des mots moins courants.",
        associatedYears: "\xC9vitez les ann\xE9es qui vous sont associ\xE9es. (ex: date de naissance)",
        capitalization: "Capitalisez mais pas seulement la premi\xE8re lettre.",
        dates: "\xC9vitez les dates et les ann\xE9es qui vous sont associ\xE9es. (ex: date ou ann\xE9e de naissance)",
        l33t: "\xC9vitez les substitutions de lettres pr\xE9visibles comme '@' pour 'a'.",
        longerKeyboardPattern: "Utilisez des motifs de clavier plus longs et changez de sens de frappe plusieurs fois.",
        noNeed: "Vous pouvez cr\xE9er des mots de passe forts sans utiliser de symboles, de chiffres ou de lettres majuscules.",
        pwned: "Si vous utilisez ce mot de passe ailleurs, vous devriez le modifier.",
        recentYears: "\xC9vitez les derni\xE8res ann\xE9es.",
        repeated: "\xC9vitez les mots et les caract\xE8res r\xE9p\xE9t\xE9s.",
        reverseWords: "\xC9vitez les orthographes invers\xE9es des mots courants",
        sequences: "\xC9vitez les s\xE9quences de caract\xE8res courantes.",
        useWords: "Utilisez plusieurs mots, mais \xE9vitez les phrases courantes."
      },
      warnings: {
        common: "Ce mot de passe est couramment utilis\xE9.",
        commonNames: "Les noms communs et les noms de famille sont faciles \xE0 deviner.",
        dates: "Les dates sont faciles \xE0 deviner.",
        extendedRepeat: "Les caract\xE8res r\xE9p\xE9t\xE9s comme 'abcabcabc' sont faciles \xE0 deviner.",
        keyPattern: "Des motifs de clavier courts sont faciles \xE0 deviner.",
        namesByThemselves: "Des noms ou des pr\xE9noms simples sont faciles \xE0 deviner.",
        pwned: "Votre mot de passe a \xE9t\xE9 divulgu\xE9 suite \xE0 une violation de donn\xE9es sur Internet.",
        recentYears: "Les ann\xE9es r\xE9centes sont faciles \xE0 deviner.",
        sequences: "Des s\xE9quences de caract\xE8res communs comme 'abc' sont faciles \xE0 deviner.",
        similarToCommon: "Ce mot de passe est similaire \xE0 un mot de passe couramment utilis\xE9.",
        simpleRepeat: "Les caract\xE8res r\xE9p\xE9t\xE9s comme 'aaa' sont faciles \xE0 deviner.",
        straightRow: "Les lignes droites de touches de votre clavier sont faciles \xE0 deviner.",
        topHundred: "Ce mot de passe est fr\xE9quemment utilis\xE9.",
        topTen: "Ce mot de passe est tr\xE8s utilis\xE9.",
        userInputs: "Le mot de passe ne doit pas comporter de donn\xE9es personnelles ou li\xE9es au site.",
        wordByItself: "Les mots simples sont faciles \xE0 deviner."
      }
    }
  },
  userButton: {
    action__addAccount: "Ajouter un compte",
    action__closeUserMenu: "Fermer le menu utilisateur",
    action__manageAccount: "G\xE9rer son compte",
    action__openUserMenu: "Ouvrir le menu utilisateur",
    action__signOut: "D\xE9connexion",
    action__signOutAll: "Se d\xE9connecter de tous les comptes"
  },
  userProfile: {
    apiKeysPage: {
      title: "Cl\xE9s API"
    },
    backupCodePage: {
      actionLabel__copied: "Copi\xE9 !",
      actionLabel__copy: "Copier tous les codes",
      actionLabel__download: "T\xE9l\xE9charger en .txt",
      actionLabel__print: "Imprimer",
      infoText1: "Les codes de r\xE9cup\xE9ration seront activ\xE9s pour ce compte.",
      infoText2: "Gardez les codes de r\xE9cup\xE9ration secrets et stockez-les en toute s\xE9curit\xE9. Vous pouvez r\xE9g\xE9n\xE9rer les codes de r\xE9cup\xE9ration si vous pensez qu'ils ont \xE9t\xE9 compromis.",
      subtitle__codelist: "Conservez-les en toute s\xE9curit\xE9 et gardez-les secrets.",
      successMessage: "Les codes de r\xE9cup\xE9ration sont maintenant activ\xE9s. Vous pouvez utiliser l'un d'entre eux pour vous connecter \xE0 votre compte, si vous perdez l'acc\xE8s \xE0 votre dispositif d'authentification. Chaque code ne peut \xEAtre utilis\xE9 qu'une seule fois.",
      successSubtitle: "Vous pouvez utiliser l'un d'entre eux pour vous connecter \xE0 votre compte, si vous perdez l'acc\xE8s \xE0 votre dispositif d'authentification.",
      title: "Ajouter la v\xE9rification du code de r\xE9cup\xE9ration",
      title__codelist: "Codes de r\xE9cup\xE9ration"
    },
    billingPage: {
      paymentHistorySection: {
        empty: "Aucun historique de paiement",
        notFound: "Tentative de paiement introuvable",
        tableHeader__amount: "Montant",
        tableHeader__date: "Date",
        tableHeader__status: "Statut"
      },
      paymentMethodsSection: {
        actionLabel__default: "Rendre par d\xE9faut",
        actionLabel__remove: "Supprimer",
        add: "Ajouter une nouvelle m\xE9thode de paiement",
        addSubtitle: "Ajoutez une nouvelle m\xE9thode de paiement \xE0 votre compte.",
        cancelButton: "Annuler",
        formButtonPrimary__add: "Ajouter une m\xE9thode de paiement",
        formButtonPrimary__pay: "Payer {{amount}}",
        payWithTestCardButton: "Payer avec une carte de test",
        removeMethod: {
          messageLine1: "{{identifier}} sera supprim\xE9 de ce compte.",
          messageLine2: "Vous ne pourrez plus utiliser cette m\xE9thode de paiement et tous les abonnements r\xE9currents qui en d\xE9pendent ne fonctionneront plus.",
          successMessage: "{{paymentMethod}} a \xE9t\xE9 supprim\xE9 de votre compte.",
          title: "Supprimer une m\xE9thode de paiement"
        },
        title: "M\xE9thodes de paiement"
      },
      start: {
        headerTitle__payments: "Paiements",
        headerTitle__plans: "Plans",
        headerTitle__statements: "Relev\xE9s",
        headerTitle__subscriptions: "Abonnement"
      },
      statementsSection: {
        empty: "Aucun relev\xE9 \xE0 afficher",
        itemCaption__paidForPlan: "Pay\xE9 pour le plan {{plan}} {{period}}",
        itemCaption__proratedCredit: "Cr\xE9dit au prorata pour l\u2019utilisation partielle de l\u2019abonnement pr\xE9c\xE9dent",
        itemCaption__subscribedAndPaidForPlan: "Souscrit et pay\xE9 pour le plan {{plan}} {{period}}",
        notFound: "Relev\xE9 introuvable",
        tableHeader__amount: "Montant",
        tableHeader__date: "Date",
        title: "Relev\xE9s",
        totalPaid: "Total pay\xE9"
      },
      subscriptionsListSection: {
        actionLabel__manageSubscription: "G\xE9rer",
        actionLabel__newSubscription: "S'abonner \xE0 un plan",
        actionLabel__switchPlan: "Changer de plan",
        tableHeader__edit: "Modifier",
        tableHeader__plan: "Plan",
        tableHeader__startDate: "Date de d\xE9but",
        title: "Abonnement"
      },
      subscriptionsSection: {
        actionLabel__default: "G\xE9rer"
      },
      switchPlansSection: {
        title: "Changer de plan"
      },
      title: "Facturation"
    },
    connectedAccountPage: {
      formHint: "S\xE9lectionnez un fournisseur pour connecter votre compte.",
      formHint__noAccounts: "Aucun fournisseur de compte externe n'est disponible.",
      removeResource: {
        messageLine1: "{{identifier}} sera supprim\xE9 de ce compte.",
        messageLine2: "Vous ne pourrez plus utiliser ce compte connect\xE9 et toutes les fonctionnalit\xE9s d\xE9pendantes ne fonctionneront plus.",
        successMessage: "{{connectedAccount}} a \xE9t\xE9 supprim\xE9 de votre compte.",
        title: "Supprimer le compte connect\xE9"
      },
      socialButtonsBlockButton: "Connecter {{provider|titleize}} compte",
      successMessage: "Le fournisseur a \xE9t\xE9 ajout\xE9 \xE0 votre compte",
      title: "Ajouter un compte connect\xE9"
    },
    deletePage: {
      actionDescription: 'Saisissez "Supprimer le compte" ci-dessous pour continuer.',
      confirm: "Supprimer le compte",
      messageLine1: "\xCAtes-vous s\xFBr de vouloir supprimer votre compte ? Certaines donn\xE9es associ\xE9es peuvent \xEAtre conserv\xE9es. Pour demander la suppression compl\xE8te des donn\xE9es, veuillez contacter l'assistance.",
      messageLine2: "Cette action est d\xE9finitive et irr\xE9versible.",
      title: "Supprimer le compte"
    },
    emailAddressPage: {
      emailCode: {
        formHint: "Un e-mail contenant un code de v\xE9rification sera envoy\xE9 \xE0 cette adresse e-mail.",
        formSubtitle: "Saisissez le code de v\xE9rification envoy\xE9 \xE0 {{identifier}}",
        formTitle: "Le code de v\xE9rification",
        resendButton: "Renvoyer le lien",
        successMessage: "L'e-mail {{identifier}} a \xE9t\xE9 v\xE9rifi\xE9 et ajout\xE9 \xE0 votre compte."
      },
      emailLink: {
        formHint: "Un e-mail contenant un lien de v\xE9rification sera envoy\xE9 \xE0 cette adresse e-mail.",
        formSubtitle: "Cliquez sur le lien de v\xE9rification dans l'e-mail envoy\xE9 \xE0 {{identifier}}",
        formTitle: "lien de v\xE9rification",
        resendButton: "Renvoyer le lien",
        successMessage: "L'e-mail {{identifier}} a \xE9t\xE9 v\xE9rifi\xE9 et ajout\xE9 \xE0 votre compte."
      },
      enterpriseSSOLink: {
        formButton: "Cliquez pour vous connecter",
        formSubtitle: "Terminez la connexion avec {{identifier}}"
      },
      formHint: "Vous devrez v\xE9rifier cette adresse e-mail avant qu\u2019elle puisse \xEAtre ajout\xE9e \xE0 votre compte.",
      removeResource: {
        messageLine1: "{{identifier}} sera supprim\xE9 de ce compte.",
        messageLine2: "Vous ne pourrez plus vous connecter avec cette adresse e-mail.",
        successMessage: "{{emailAddress}} a \xE9t\xE9 supprim\xE9 de votre compte.",
        title: "Supprimer l'adresse e-mail"
      },
      title: "Ajouter une adresse e-mail",
      verifyTitle: "Verifier un e-mail"
    },
    formButtonPrimary__add: "Ajouter",
    formButtonPrimary__continue: "Continuer",
    formButtonPrimary__finish: "Retour",
    formButtonPrimary__remove: "Supprimer",
    formButtonPrimary__save: "Sauvegarder",
    formButtonReset: "Annuler",
    mfaPage: {
      formHint: "S\xE9lectionnez une m\xE9thode \xE0 ajouter.",
      title: "Ajouter la v\xE9rification en deux \xE9tapes"
    },
    mfaPhoneCodePage: {
      backButton: "Utiliser un num\xE9ro existant",
      primaryButton__addPhoneNumber: "Ajouter un num\xE9ro de t\xE9l\xE9phone",
      removeResource: {
        messageLine1: "{{identifier}} ne recevra plus de codes de validation lors de la connexion.",
        messageLine2: "Votre compte sera moins s\xE9curis\xE9. Souhaitez-vous continuer ?",
        successMessage: "La v\xE9rification en deux \xE9tapes du code SMS a \xE9t\xE9 supprim\xE9e pour {{mfaPhoneCode}}",
        title: "Supprimer la v\xE9rification en deux \xE9tapes"
      },
      subtitle__availablePhoneNumbers: "S\xE9lectionnez un num\xE9ro de t\xE9l\xE9phone pour vous inscrire \xE0 la v\xE9rification en deux \xE9tapes du code SMS.",
      subtitle__unavailablePhoneNumbers: "Il n'y a pas de num\xE9ros de t\xE9l\xE9phone disponibles pour s'inscrire \xE0 la v\xE9rification en deux \xE9tapes du code SMS.",
      successMessage1: "Lors de la connexion, vous devrez entrer un code de v\xE9rification envoy\xE9 \xE0 ce num\xE9ro de t\xE9l\xE9phone comme \xE9tape suppl\xE9mentaire.",
      successMessage2: "Enregistrez ces codes de r\xE9cup\xE9rations et conservez-les en lieu s\xFBr. Si vous perdez l'acc\xE8s \xE0 votre appareil d'authentification, vous pourrez utiliser les codes de r\xE9cup\xE9rations pour vous connecter.",
      successTitle: "V\xE9rification par SMS activ\xE9e",
      title: "Ajouter la v\xE9rification du code SMS"
    },
    mfaTOTPPage: {
      authenticatorApp: {
        buttonAbleToScan__nonPrimary: "Scannez le QR code \xE0 la place",
        buttonUnableToScan__nonPrimary: "Vous ne pouvez pas scanner le QR code ?",
        infoText__ableToScan: "Configurez une nouvelle m\xE9thode de connexion dans votre application d'authentification et scannez le QR code suivant pour le lier \xE0 votre compte.",
        infoText__unableToScan: "Configurez une nouvelle m\xE9thode de connexion dans votre authentificateur et entrez la cl\xE9 fournie ci-dessous.",
        inputLabel__unableToScan1: "Assurez-vous que les mots de passe bas\xE9s sur le temps ou \xE0 usage unique sont activ\xE9s, puis terminez la liaison de votre compte.",
        inputLabel__unableToScan2: "Alternativement, si votre authentificateur prend en charge les URI TOTP, vous pouvez \xE9galement copier l'URI complet."
      },
      removeResource: {
        messageLine1: "Les codes de v\xE9rification de cet authentificateur ne seront plus requis lors de la connexion.",
        messageLine2: "Votre compte sera moins s\xE9curis\xE9. Souhaitez-vous continuer ?",
        successMessage: "La v\xE9rification en deux \xE9tapes via l'application d'authentification a \xE9t\xE9 supprim\xE9e.",
        title: "Supprimer la v\xE9rification en deux \xE9tapes"
      },
      successMessage: "La v\xE9rification en deux \xE9tapes est maintenant activ\xE9e. Lors de la connexion, vous devrez saisir un code de v\xE9rification de cet authentificateur comme \xE9tape suppl\xE9mentaire.",
      title: "Ajouter une application d'authentification",
      verifySubtitle: "Entrez le code de v\xE9rification g\xE9n\xE9r\xE9 par votre application d'authentification",
      verifyTitle: "Le code de v\xE9rification"
    },
    mobileButton__menu: "Menu",
    navbar: {
      account: "Compte",
      apiKeys: "Cl\xE9s API",
      billing: "Facturation",
      description: "G\xE9rer votre compte.",
      security: "S\xE9curit\xE9",
      title: "Profil"
    },
    passkeyScreen: {
      removeResource: {
        messageLine1: "\xCAtes-vous s\xFBr de vouloir supprimer cette cl\xE9 de s\xE9curit\xE9 ?",
        title: "Supprimer la cl\xE9 de s\xE9curit\xE9"
      },
      subtitle__rename: "Renommez votre cl\xE9 de s\xE9curit\xE9 pour une identification facile.",
      title__rename: "Renommer la cl\xE9 de s\xE9curit\xE9"
    },
    passwordPage: {
      checkboxInfoText__signOutOfOtherSessions: "Il est recommand\xE9 de se d\xE9connecter de tous les autres appareils qui pourraient avoir utilis\xE9 votre ancien mot de passe.",
      readonly: "Votre mot de passe ne peut pas \xEAtre modifi\xE9 pour l'instant car vous ne pouvez vous connecter qu'\xE0 l'aide de la connexion entreprise.",
      successMessage__set: "Votre mot de passe a \xE9t\xE9 mis \xE0 jour.",
      successMessage__signOutOfOtherSessions: "Tous les autres appareils ont \xE9t\xE9 d\xE9connect\xE9s.",
      successMessage__update: "Votre mot de passe a \xE9t\xE9 mis \xE0 jour.",
      title__set: "Mettre \xE0 jour le mot de passe",
      title__update: "Changer le mot de passe"
    },
    phoneNumberPage: {
      infoText: "Un SMS contenant un lien de v\xE9rification sera envoy\xE9 \xE0 ce num\xE9ro de t\xE9l\xE9phone.",
      removeResource: {
        messageLine1: "{{identifier}} sera supprim\xE9 de ce compte.",
        messageLine2: "Vous ne pourrez plus vous connecter avec ce num\xE9ro de t\xE9l\xE9phone.",
        successMessage: "{{phoneNumber}} a \xE9t\xE9 supprim\xE9 de votre compte.",
        title: "Supprimer le num\xE9ro de t\xE9l\xE9phone"
      },
      successMessage: "{{identifier}} a \xE9t\xE9 v\xE9rifi\xE9 et ajout\xE9 \xE0 votre compte.",
      title: "Ajouter un num\xE9ro de t\xE9l\xE9phone",
      verifySubtitle: "Saisissez le code de v\xE9rification envoy\xE9 \xE0 {{identifier}}",
      verifyTitle: "V\xE9rification du num\xE9ro de t\xE9l\xE9phone"
    },
    plansPage: {
      title: "Plans"
    },
    profilePage: {
      fileDropAreaHint: "T\xE9l\xE9chargez une image JPG, PNG, GIF ou WEBP inf\xE9rieure \xE0 10 Mo",
      imageFormDestructiveActionSubtitle: "Supprimer l'image",
      imageFormSubtitle: "T\xE9l\xE9charger une image",
      imageFormTitle: "Photo de profil",
      readonly: "Les informations de votre profil ont \xE9t\xE9 fournies par la connexion d'entreprise et ne peuvent pas \xEAtre modifi\xE9es.",
      successMessage: "Votre profil a \xE9t\xE9 mis a jour.",
      title: "Mettre \xE0 jour le profil"
    },
    start: {
      activeDevicesSection: {
        destructiveAction: "Se d\xE9connecter de l'appareil",
        title: "Appareils actifs"
      },
      connectedAccountsSection: {
        actionLabel__connectionFailed: "R\xE9essayer",
        actionLabel__reauthorize: "Autoriser maintenant",
        destructiveActionTitle: "Retirer",
        primaryButton: "Connecter le compte",
        subtitle__disconnected: "Compte d\xE9connect\xE9. Connectez-vous \xE0 nouveau pour acc\xE9der aux fonctionnalit\xE9s.",
        subtitle__reauthorize: "Les autorisations requises ont \xE9t\xE9 mises \xE0 jour, ce qui peut entra\xEEner des fonctionnalit\xE9s limit\xE9es. Veuillez r\xE9-autoriser cette application pour \xE9viter tout probl\xE8me.",
        title: "Comptes connect\xE9s"
      },
      dangerSection: {
        deleteAccountButton: "Supprimer le compte",
        title: "Danger"
      },
      emailAddressesSection: {
        destructiveAction: "Supprimer l'adresse e-mail",
        detailsAction__nonPrimary: "D\xE9finir comme principale",
        detailsAction__primary: "Compl\xE9ter la v\xE9rification",
        detailsAction__unverified: "Compl\xE9ter la v\xE9rification",
        primaryButton: "Ajouter une adresse e-mail",
        title: "Adresses e-mail"
      },
      enterpriseAccountsSection: {
        title: "Comptes entreprises"
      },
      headerTitle__account: "Compte",
      headerTitle__security: "S\xE9curit\xE9",
      mfaSection: {
        backupCodes: {
          actionLabel__regenerate: "R\xE9g\xE9n\xE9rer les codes",
          headerTitle: "Codes de r\xE9cup\xE9ration",
          subtitle__regenerate: "Obtenez de nouveaux codes de r\xE9cup\xE9ration s\xE9curis\xE9s. Les codes de r\xE9cup\xE9ration ant\xE9rieurs seront supprim\xE9s et ne pourront pas \xEAtre utilis\xE9s.",
          title__regenerate: "R\xE9g\xE9n\xE9rer les codes de r\xE9cup\xE9ration"
        },
        phoneCode: {
          actionLabel__setDefault: "D\xE9finir par d\xE9faut",
          destructiveActionLabel: "Supprimer le num\xE9ro de t\xE9l\xE9phone"
        },
        primaryButton: "Ajouter la v\xE9rification en deux \xE9tapes",
        title: "V\xE9rification en deux \xE9tapes",
        totp: {
          destructiveActionTitle: "D\xE9sactiver",
          headerTitle: "Application d'authentification"
        }
      },
      passkeysSection: {
        menuAction__destructive: "Supprimer",
        menuAction__rename: "Renommer",
        primaryButton: "Ajouter une cl\xE9 de s\xE9curit\xE9",
        title: "Cl\xE9s de s\xE9curit\xE9"
      },
      passwordSection: {
        primaryButton__setPassword: "D\xE9finir le mot de passe",
        primaryButton__updatePassword: "Changer le mot de passe",
        title: "Mot de passe"
      },
      phoneNumbersSection: {
        destructiveAction: "Supprimer le num\xE9ro de t\xE9l\xE9phone",
        detailsAction__nonPrimary: "D\xE9finir comme principale",
        detailsAction__primary: "Compl\xE9ter la v\xE9rification",
        detailsAction__unverified: "Compl\xE9ter la v\xE9rification",
        primaryButton: "Ajouter un num\xE9ro de t\xE9l\xE9phone",
        title: "Num\xE9ros de t\xE9l\xE9phone"
      },
      profileSection: {
        primaryButton: "Mettre \xE0 jour le profil",
        title: "Profil"
      },
      usernameSection: {
        primaryButton__setUsername: "D\xE9finir le nom d'utilisateur",
        primaryButton__updateUsername: "Changer le nom d'utilisateur",
        title: "Nom d'utilisateur"
      },
      web3WalletsSection: {
        destructiveAction: "Supprimer le portefeuille",
        detailsAction__nonPrimary: "D\xE9finir comme principal",
        primaryButton: "Portefeuilles Web3",
        title: "Portefeuilles Web3"
      }
    },
    usernamePage: {
      successMessage: "Votre nom d'utilisateur a \xE9t\xE9 mis \xE0 jour.",
      title__set: "Mettre \xE0 jour le nom d'utilisateur",
      title__update: "Mettre \xE0 jour le nom d'utilisateur"
    },
    web3WalletPage: {
      removeResource: {
        messageLine1: "{{identifier}} sera supprim\xE9 de ce compte.",
        messageLine2: "Vous ne pourrez plus vous connecter avec ce portefeuille web3.",
        successMessage: "{{web3Wallet}} a \xE9t\xE9 supprim\xE9 de votre compte.",
        title: "Supprimer le portefeuille Web3"
      },
      subtitle__availableWallets: "S\xE9lectionnez un portefeuille Web3 pour vous connecter \xE0 votre compte.",
      subtitle__unavailableWallets: "Il n'y a pas de portefeuilles Web3 disponibles.",
      successMessage: "Le portefeuille a \xE9t\xE9 ajout\xE9 \xE0 votre compte.",
      title: "Ajouter un portefeuille Web3",
      web3WalletButtonsBlockButton: "Utiliser un portefeuille Web3"
    }
  },
  waitlist: {
    start: {
      actionLink: "Rejoindre la liste d'attente",
      actionText: "Int\xE9ress\xE9 ? Rejoignez-nous !",
      formButton: "S'inscrire",
      subtitle: "Soyez parmi les premiers \xE0 d\xE9couvrir notre nouveau service.",
      title: "Rejoindre la liste d'attente"
    },
    success: {
      message: "Merci ! Vous avez \xE9t\xE9 ajout\xE9 \xE0 la liste d'attente.",
      subtitle: "Nous vous contacterons bient\xF4t avec plus de d\xE9tails.",
      title: "Inscription r\xE9ussie"
    }
  }
};
// Annotate the CommonJS export names for ESM import in node:
0 && (module.exports = {
  frFR
});
