// @clerk/localizations/dist/he-IL.mjs
// src/he-IL.ts
var heIL = {
  locale: "he-IL",
  apiKeys: {
    action__add: void 0,
    action__search: void 0,
    createdAndExpirationStatus__expiresOn: void 0,
    createdAndExpirationStatus__never: void 0,
    detailsTitle__emptyRow: void 0,
    formButtonPrimary__add: void 0,
    formFieldCaption__expiration__expiresOn: void 0,
    formFieldCaption__expiration__never: void 0,
    formFieldOption__expiration__180d: void 0,
    formFieldOption__expiration__1d: void 0,
    formFieldOption__expiration__1y: void 0,
    formFieldOption__expiration__30d: void 0,
    formFieldOption__expiration__60d: void 0,
    formFieldOption__expiration__7d: void 0,
    formFieldOption__expiration__90d: void 0,
    formFieldOption__expiration__never: void 0,
    formHint: void 0,
    formTitle: void 0,
    lastUsed__days: void 0,
    lastUsed__hours: void 0,
    lastUsed__minutes: void 0,
    lastUsed__months: void 0,
    lastUsed__seconds: void 0,
    lastUsed__years: void 0,
    menuAction__revoke: void 0,
    revokeConfirmation: {
      confirmationText: void 0,
      formButtonPrimary__revoke: void 0,
      formHint: void 0,
      formTitle: void 0
    }
  },
  backButton: "\u05D7\u05D6\u05D5\u05E8",
  badge__activePlan: void 0,
  badge__canceledEndsAt: void 0,
  badge__currentPlan: void 0,
  badge__default: "\u05D1\u05E8\u05D9\u05E8\u05EA \u05DE\u05D7\u05D3\u05DC",
  badge__endsAt: void 0,
  badge__expired: void 0,
  badge__freeTrial: void 0,
  badge__otherImpersonatorDevice: "\u05DE\u05DB\u05E9\u05D9\u05E8 \u05DE\u05D7\u05E7\u05D4 \u05D0\u05D7\u05E8",
  badge__pastDueAt: void 0,
  badge__pastDuePlan: void 0,
  badge__primary: "\u05E8\u05D0\u05E9\u05D9",
  badge__renewsAt: void 0,
  badge__requiresAction: "\u05D3\u05D5\u05E8\u05E9 \u05E4\u05E2\u05D5\u05DC\u05D4",
  badge__startsAt: void 0,
  badge__thisDevice: "\u05DE\u05DB\u05E9\u05D9\u05E8 \u05D6\u05D4",
  badge__trialEndsAt: void 0,
  badge__unverified: "\u05DC\u05D0 \u05DE\u05D0\u05D5\u05DE\u05EA",
  badge__upcomingPlan: void 0,
  badge__userDevice: "\u05DE\u05DB\u05E9\u05D9\u05E8 \u05DE\u05E9\u05EA\u05DE\u05E9",
  badge__you: "\u05D0\u05EA\u05D4",
  billing: {
    addPaymentMethod__label: void 0,
    alwaysFree: void 0,
    annually: void 0,
    availableFeatures: void 0,
    billedAnnually: void 0,
    billedMonthlyOnly: void 0,
    cancelFreeTrial: void 0,
    cancelFreeTrialAccessUntil: void 0,
    cancelFreeTrialTitle: void 0,
    cancelSubscription: void 0,
    cancelSubscriptionAccessUntil: void 0,
    cancelSubscriptionNoCharge: void 0,
    cancelSubscriptionPastDue: void 0,
    cancelSubscriptionTitle: void 0,
    cannotSubscribeMonthly: void 0,
    cannotSubscribeUnrecoverable: void 0,
    checkout: {
      description__paymentSuccessful: void 0,
      description__subscriptionSuccessful: void 0,
      downgradeNotice: void 0,
      emailForm: {
        subtitle: void 0,
        title: void 0
      },
      lineItems: {
        title__freeTrialEndsAt: void 0,
        title__paymentMethod: void 0,
        title__statementId: void 0,
        title__subscriptionBegins: void 0,
        title__totalPaid: void 0
      },
      pastDueNotice: void 0,
      perMonth: void 0,
      title: void 0,
      title__paymentSuccessful: void 0,
      title__subscriptionSuccessful: void 0,
      title__trialSuccess: void 0,
      totalDueAfterTrial: void 0
    },
    credit: void 0,
    creditRemainder: void 0,
    defaultFreePlanActive: void 0,
    free: void 0,
    getStarted: void 0,
    keepFreeTrial: void 0,
    keepSubscription: void 0,
    manage: void 0,
    manageSubscription: void 0,
    month: void 0,
    monthly: void 0,
    pastDue: void 0,
    pay: void 0,
    paymentMethod: {
      applePayDescription: {
        annual: void 0,
        monthly: void 0
      },
      dev: {
        anyNumbers: void 0,
        cardNumber: void 0,
        cvcZip: void 0,
        developmentMode: void 0,
        expirationDate: void 0,
        testCardInfo: void 0
      }
    },
    paymentMethods__label: void 0,
    popular: void 0,
    pricingTable: {
      billingCycle: void 0,
      included: void 0
    },
    reSubscribe: void 0,
    seeAllFeatures: void 0,
    startFreeTrial: void 0,
    startFreeTrial__days: void 0,
    subscribe: void 0,
    subscriptionDetails: {
      beginsOn: void 0,
      currentBillingCycle: void 0,
      endsOn: void 0,
      firstPaymentAmount: void 0,
      firstPaymentOn: void 0,
      nextPaymentAmount: void 0,
      nextPaymentOn: void 0,
      pastDueAt: void 0,
      renewsAt: void 0,
      subscribedOn: void 0,
      title: void 0,
      trialEndsOn: void 0,
      trialStartedOn: void 0
    },
    subtotal: void 0,
    switchPlan: void 0,
    switchToAnnual: void 0,
    switchToAnnualWithAnnualPrice: void 0,
    switchToMonthly: void 0,
    switchToMonthlyWithPrice: void 0,
    totalDue: void 0,
    totalDueToday: void 0,
    viewFeatures: void 0,
    viewPayment: void 0,
    year: void 0
  },
  createOrganization: {
    formButtonSubmit: "\u05E6\u05D5\u05E8 \u05D0\u05E8\u05D2\u05D5\u05DF",
    invitePage: {
      formButtonReset: "\u05D3\u05DC\u05D2"
    },
    title: "\u05E6\u05D5\u05E8 \u05D0\u05E8\u05D2\u05D5\u05DF"
  },
  dates: {
    lastDay: "\u05D0\u05EA\u05DE\u05D5\u05DC \u05D1-{{ date | timeString('iw') }}",
    next6Days: "{{ date | weekday('iw','long') }} \u05D1-{{ date | timeString('iw') }}",
    nextDay: "\u05DE\u05D7\u05E8 \u05D1-{{ date | timeString('iw') }}",
    numeric: "{{ date | numeric('iw') }}",
    previous6Days: "{{ date | weekday('iw','long') }} \u05D4\u05D0\u05D7\u05E8\u05D5\u05DF \u05D1-{{ date | timeString('iw') }}",
    sameDay: "\u05D4\u05D9\u05D5\u05DD \u05D1-{{ date | timeString('iw') }}"
  },
  dividerText: "\u05D0\u05D5",
  footerActionLink__alternativePhoneCodeProvider: void 0,
  footerActionLink__useAnotherMethod: "\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05E9\u05D9\u05D8\u05D4 \u05D0\u05D7\u05E8\u05EA",
  footerPageLink__help: "\u05E2\u05D6\u05E8\u05D4",
  footerPageLink__privacy: "\u05E4\u05E8\u05D8\u05D9\u05D5\u05EA",
  footerPageLink__terms: "\u05EA\u05E0\u05D0\u05D9\u05DD",
  formButtonPrimary: "\u05D4\u05DE\u05E9\u05DA",
  formButtonPrimary__verify: "Verify",
  formFieldAction__forgotPassword: "\u05E9\u05DB\u05D7\u05EA \u05E1\u05D9\u05E1\u05DE\u05D4?",
  formFieldError__matchingPasswords: "\u05D4\u05E1\u05D9\u05E1\u05DE\u05D0\u05D5\u05EA \u05EA\u05D5\u05D0\u05DE\u05D5\u05EA.",
  formFieldError__notMatchingPasswords: "\u05D4\u05E1\u05D9\u05E1\u05DE\u05D0\u05D5\u05EA \u05D0\u05D9\u05E0\u05DF \u05EA\u05D5\u05D0\u05DE\u05D5\u05EA.",
  formFieldError__verificationLinkExpired: "\u05D4\u05E7\u05D9\u05E9\u05D5\u05E8 \u05DC\u05D0\u05D9\u05DE\u05D5\u05EA \u05E4\u05D2 \u05EA\u05D5\u05E7\u05E3. \u05D0\u05E0\u05D0 \u05D1\u05E7\u05E9/\u05D9 \u05E7\u05D9\u05E9\u05D5\u05E8 \u05D7\u05D3\u05E9.",
  formFieldHintText__optional: "\u05D0\u05D5\u05E4\u05E6\u05D9\u05D5\u05E0\u05DC\u05D9",
  formFieldHintText__slug: "(slug) \u05D4\u05D5\u05D0 \u05DE\u05D6\u05D4\u05D4 \u05E7\u05E8\u05D9\u05D0 \u05E9\u05DE\u05D9\u05D5\u05E2\u05D3 \u05DC\u05D4\u05D9\u05D5\u05EA \u05D9\u05D9\u05D7\u05D5\u05D3\u05D9. \u05D4\u05D5\u05D0 \u05DE\u05E9\u05DE\u05E9 \u05DC\u05E2\u05D9\u05EA\u05D9\u05DD \u05E7\u05E8\u05D5\u05D1\u05D5\u05EA \u05D1\u05DB\u05EA\u05D5\u05D1\u05D5\u05EA URL.",
  formFieldInputPlaceholder__apiKeyDescription: void 0,
  formFieldInputPlaceholder__apiKeyExpirationDate: void 0,
  formFieldInputPlaceholder__apiKeyName: void 0,
  formFieldInputPlaceholder__backupCode: void 0,
  formFieldInputPlaceholder__confirmDeletionUserAccount: "\u05DE\u05D7\u05D9\u05E7\u05EA \u05D7\u05E9\u05D1\u05D5\u05DF",
  formFieldInputPlaceholder__emailAddress: void 0,
  formFieldInputPlaceholder__emailAddress_username: void 0,
  formFieldInputPlaceholder__emailAddresses: "example@email.com, example2@email.com",
  formFieldInputPlaceholder__firstName: void 0,
  formFieldInputPlaceholder__lastName: void 0,
  formFieldInputPlaceholder__organizationDomain: void 0,
  formFieldInputPlaceholder__organizationDomainEmailAddress: void 0,
  formFieldInputPlaceholder__organizationName: void 0,
  formFieldInputPlaceholder__organizationSlug: "\u05D4\u05D0\u05E8\u05D2\u05D5\u05DF-\u05E9\u05DC\u05D9",
  formFieldInputPlaceholder__password: void 0,
  formFieldInputPlaceholder__phoneNumber: void 0,
  formFieldInputPlaceholder__username: void 0,
  formFieldInput__emailAddress_format: void 0,
  formFieldLabel__apiKeyDescription: void 0,
  formFieldLabel__apiKeyExpiration: void 0,
  formFieldLabel__apiKeyName: void 0,
  formFieldLabel__automaticInvitations: "\u05D4\u05E4\u05E2\u05DC \u05D4\u05D6\u05DE\u05E0\u05D5\u05EA \u05D0\u05D5\u05D8\u05D5\u05DE\u05D8\u05D9\u05D5\u05EA \u05DC\u05D3\u05D5\u05DE\u05D9\u05D9\u05DF \u05D4\u05D6\u05D4",
  formFieldLabel__backupCode: "\u05E7\u05D5\u05D3 \u05D2\u05D9\u05D1\u05D5\u05D9",
  formFieldLabel__confirmDeletion: "\u05D0\u05D9\u05E9\u05D5\u05E8",
  formFieldLabel__confirmPassword: "\u05D0\u05DE\u05EA \u05E1\u05D9\u05E1\u05DE\u05D4",
  formFieldLabel__currentPassword: "\u05E1\u05D9\u05E1\u05DE\u05D4 \u05E0\u05D5\u05DB\u05D7\u05D9\u05EA",
  formFieldLabel__emailAddress: '\u05DB\u05EA\u05D5\u05D1\u05EA \u05D3\u05D5\u05D0"\u05DC',
  formFieldLabel__emailAddress_username: '\u05DB\u05EA\u05D5\u05D1\u05EA \u05D3\u05D5\u05D0"\u05DC \u05D0\u05D5 \u05E9\u05DD \u05DE\u05E9\u05EA\u05DE\u05E9',
  formFieldLabel__emailAddresses: '\u05DB\u05EA\u05D5\u05D1\u05D5\u05EA \u05D3\u05D5\u05D0"\u05DC',
  formFieldLabel__firstName: "\u05E9\u05DD \u05E4\u05E8\u05D8\u05D9",
  formFieldLabel__lastName: "\u05E9\u05DD \u05DE\u05E9\u05E4\u05D7\u05D4",
  formFieldLabel__newPassword: "\u05E1\u05D9\u05E1\u05DE\u05D4 \u05D7\u05D3\u05E9\u05D4",
  formFieldLabel__organizationDomain: "\u05D3\u05D5\u05DE\u05D9\u05D9\u05DF",
  formFieldLabel__organizationDomainDeletePending: "\u05DE\u05D7\u05E7 \u05D4\u05D6\u05DE\u05E0\u05D5\u05EA \u05D5\u05D4\u05E6\u05E2\u05D5\u05EA \u05DE\u05DE\u05EA\u05D9\u05E0\u05D5\u05EA",
  formFieldLabel__organizationDomainEmailAddress: "\u05DB\u05EA\u05D5\u05D1\u05EA \u05D0\u05D9\u05DE\u05D9\u05D9\u05DC \u05DC\u05D0\u05D9\u05DE\u05D5\u05EA",
  formFieldLabel__organizationDomainEmailAddressDescription: "\u05D4\u05D6\u05DF \u05DB\u05EA\u05D5\u05D1\u05EA \u05D0\u05D9\u05DE\u05D9\u05D9\u05DC \u05EA\u05D7\u05EA \u05D3\u05D5\u05DE\u05D9\u05D9\u05DF \u05D6\u05D4 \u05DB\u05D3\u05D9 \u05DC\u05E7\u05D1\u05DC \u05E7\u05D5\u05D3 \u05D5\u05DC\u05D0\u05DE\u05EA \u05D0\u05EA \u05D4\u05D3\u05D5\u05DE\u05D9\u05D9\u05DF.",
  formFieldLabel__organizationName: "\u05E9\u05DD \u05D4\u05D0\u05E8\u05D2\u05D5\u05DF",
  formFieldLabel__organizationSlug: "\u05DB\u05EA\u05D5\u05D1\u05EA URL \u05E9\u05DC \u05D4\u05D0\u05E8\u05D2\u05D5\u05DF",
  formFieldLabel__passkeyName: "\u05E9\u05DD \u05D0\u05D5 \u05DE\u05E4\u05EA\u05D7 \u05E1\u05D9\u05E1\u05DE\u05D4",
  formFieldLabel__password: "\u05E1\u05D9\u05E1\u05DE\u05D4",
  formFieldLabel__phoneNumber: "\u05DE\u05E1\u05E4\u05E8 \u05D8\u05DC\u05E4\u05D5\u05DF",
  formFieldLabel__role: "\u05EA\u05E4\u05E7\u05D9\u05D3",
  formFieldLabel__signOutOfOtherSessions: "\u05D4\u05EA\u05E0\u05EA\u05E7 \u05DE\u05DB\u05DC \u05D4\u05DE\u05DB\u05E9\u05D9\u05E8\u05D9\u05DD \u05D4\u05D0\u05D7\u05E8\u05D9\u05DD",
  formFieldLabel__username: "\u05E9\u05DD \u05DE\u05E9\u05EA\u05DE\u05E9",
  impersonationFab: {
    action__signOut: "\u05D4\u05EA\u05E0\u05EA\u05E7",
    title: "\u05DE\u05D7\u05D5\u05D1\u05E8 \u05DB{{identifier}}"
  },
  lastAuthenticationStrategy: "\u05D1\u05E9\u05D9\u05DE\u05D5\u05E9 \u05D0\u05D7\u05E8\u05D5\u05DF",
  maintenanceMode: void 0,
  membershipRole__admin: "\u05DE\u05E0\u05D4\u05DC",
  membershipRole__basicMember: "\u05D7\u05D1\u05E8",
  membershipRole__guestMember: "\u05D0\u05D5\u05E8\u05D7",
  organizationList: {
    action__createOrganization: "\u05E6\u05D5\u05E8 \u05D0\u05E8\u05D2\u05D5\u05DF",
    action__invitationAccept: "\u05D4\u05E6\u05D8\u05E8\u05E3",
    action__suggestionsAccept: "\u05D1\u05E7\u05E9\u05D4 \u05DC\u05D4\u05E6\u05D8\u05E8\u05E4\u05D5\u05EA",
    createOrganization: "\u05E6\u05D5\u05E8 \u05D0\u05E8\u05D2\u05D5\u05DF",
    invitationAcceptedLabel: "\u05D4\u05E6\u05D8\u05E8\u05E3",
    subtitle: "\u05DB\u05D3\u05D9 \u05DC\u05D4\u05DE\u05E9\u05D9\u05DA \u05DC{{applicationName}}",
    suggestionsAcceptedLabel: "\u05DE\u05DE\u05EA\u05D9\u05DF \u05DC\u05D0\u05D9\u05E9\u05D5\u05E8",
    title: "\u05D1\u05D7\u05E8 \u05D7\u05E9\u05D1\u05D5\u05DF",
    titleWithoutPersonal: "\u05D1\u05D7\u05E8 \u05D0\u05E8\u05D2\u05D5\u05DF"
  },
  organizationProfile: {
    apiKeysPage: {
      title: void 0
    },
    badge__automaticInvitation: "\u05D4\u05D6\u05DE\u05E0\u05D5\u05EA \u05D0\u05D5\u05D8\u05D5\u05DE\u05D8\u05D9\u05D5\u05EA",
    badge__automaticSuggestion: "\u05D4\u05E6\u05E2\u05D5\u05EA \u05D0\u05D5\u05D8\u05D5\u05DE\u05D8\u05D9\u05D5\u05EA",
    badge__manualInvitation: "\u05DC\u05DC\u05D0 \u05D4\u05E8\u05E9\u05DE\u05D4 \u05D0\u05D5\u05D8\u05D5\u05DE\u05D8\u05D9\u05EA",
    badge__unverified: "\u05DC\u05D0 \u05DE\u05D0\u05D5\u05DE\u05EA",
    billingPage: {
      paymentHistorySection: {
        empty: void 0,
        notFound: void 0,
        tableHeader__amount: void 0,
        tableHeader__date: void 0,
        tableHeader__status: void 0
      },
      paymentMethodsSection: {
        actionLabel__default: void 0,
        actionLabel__remove: void 0,
        add: void 0,
        addSubtitle: void 0,
        cancelButton: void 0,
        formButtonPrimary__add: void 0,
        formButtonPrimary__pay: void 0,
        payWithTestCardButton: void 0,
        removeMethod: {
          messageLine1: void 0,
          messageLine2: void 0,
          successMessage: void 0,
          title: void 0
        },
        title: void 0
      },
      start: {
        headerTitle__payments: void 0,
        headerTitle__plans: void 0,
        headerTitle__statements: void 0,
        headerTitle__subscriptions: void 0
      },
      statementsSection: {
        empty: void 0,
        itemCaption__paidForPlan: void 0,
        itemCaption__proratedCredit: void 0,
        itemCaption__subscribedAndPaidForPlan: void 0,
        notFound: void 0,
        tableHeader__amount: void 0,
        tableHeader__date: void 0,
        title: void 0,
        totalPaid: void 0
      },
      subscriptionsListSection: {
        actionLabel__manageSubscription: void 0,
        actionLabel__newSubscription: void 0,
        actionLabel__switchPlan: void 0,
        tableHeader__edit: void 0,
        tableHeader__plan: void 0,
        tableHeader__startDate: void 0,
        title: void 0
      },
      subscriptionsSection: {
        actionLabel__default: void 0
      },
      switchPlansSection: {
        title: void 0
      },
      title: void 0
    },
    createDomainPage: {
      subtitle: "\u05D4\u05D5\u05E1\u05E3 \u05D0\u05EA \u05D4\u05D3\u05D5\u05DE\u05D9\u05D9\u05DF \u05DC\u05D0\u05D9\u05DE\u05D5\u05EA. \u05DE\u05E9\u05EA\u05DE\u05E9\u05D9\u05DD \u05E2\u05DD \u05DB\u05EA\u05D5\u05D1\u05D5\u05EA \u05D0\u05D9\u05DE\u05D9\u05D9\u05DC \u05EA\u05D7\u05EA \u05D3\u05D5\u05DE\u05D9\u05D9\u05DF \u05D6\u05D4 \u05D9\u05DB\u05D5\u05DC\u05D9\u05DD \u05DC\u05D4\u05E6\u05D8\u05E8\u05E3 \u05DC\u05D0\u05E8\u05D2\u05D5\u05DF \u05D1\u05D0\u05D5\u05E4\u05DF \u05D0\u05D5\u05D8\u05D5\u05DE\u05D8\u05D9 \u05D0\u05D5 \u05DC\u05D1\u05E7\u05E9 \u05DC\u05D4\u05E6\u05D8\u05E8\u05E3.",
      title: "\u05D4\u05D5\u05E1\u05E3 \u05D3\u05D5\u05DE\u05D9\u05D9\u05DF"
    },
    invitePage: {
      detailsTitle__inviteFailed: "\u05DC\u05D0 \u05D4\u05D9\u05D4 \u05E0\u05D9\u05EA\u05DF \u05DC\u05E9\u05DC\u05D5\u05D7 \u05D0\u05EA \u05D4\u05D4\u05D6\u05DE\u05E0\u05D5\u05EA. \u05EA\u05E7\u05DF \u05D0\u05EA \u05D4\u05D1\u05E2\u05D9\u05D5\u05EA \u05D4\u05D1\u05D0\u05D5\u05EA \u05D5\u05E0\u05E1\u05D4 \u05E9\u05D5\u05D1:",
      formButtonPrimary__continue: "\u05E9\u05DC\u05D7 \u05D4\u05D6\u05DE\u05E0\u05D5\u05EA",
      selectDropdown__role: "\u05D1\u05D7\u05E8 \u05EA\u05E4\u05E7\u05D9\u05D3",
      subtitle: "\u05D4\u05D6\u05DE\u05DF \u05D7\u05D1\u05E8\u05D9\u05DD \u05D7\u05D3\u05E9\u05D9\u05DD \u05DC\u05D0\u05E8\u05D2\u05D5\u05DF \u05D6\u05D4",
      successMessage: "\u05D4\u05D4\u05D6\u05DE\u05E0\u05D5\u05EA \u05E0\u05E9\u05DC\u05D7\u05D5 \u05D1\u05D4\u05E6\u05DC\u05D7\u05D4",
      title: "\u05D4\u05D6\u05DE\u05DF \u05D7\u05D1\u05E8\u05D9\u05DD"
    },
    membersPage: {
      action__invite: "\u05D4\u05D6\u05DE\u05DF",
      action__search: void 0,
      activeMembersTab: {
        menuAction__remove: "\u05D4\u05E1\u05E8 \u05D7\u05D1\u05E8",
        tableHeader__actions: void 0,
        tableHeader__joined: "\u05D4\u05E6\u05D8\u05E8\u05E3",
        tableHeader__role: "\u05EA\u05E4\u05E7\u05D9\u05D3",
        tableHeader__user: "\u05DE\u05E9\u05EA\u05DE\u05E9"
      },
      detailsTitle__emptyRow: "\u05D0\u05D9\u05DF \u05D7\u05D1\u05E8\u05D9\u05DD \u05DC\u05D4\u05E6\u05D2\u05D4",
      invitationsTab: {
        autoInvitations: {
          headerSubtitle: "\u05D4\u05D6\u05DE\u05DF \u05DE\u05E9\u05EA\u05DE\u05E9\u05D9\u05DD \u05E2\u05DC \u05D9\u05D3\u05D9 \u05D7\u05D9\u05D1\u05D5\u05E8 \u05D3\u05D5\u05DE\u05D9\u05D9\u05DF \u05D0\u05D9\u05DE\u05D9\u05D9\u05DC \u05DC\u05D0\u05E8\u05D2\u05D5\u05DF \u05E9\u05DC\u05DA. \u05DB\u05DC \u05DE\u05D9 \u05E9\u05E0\u05E8\u05E9\u05DD \u05E2\u05DD \u05D3\u05D5\u05DE\u05D9\u05D9\u05DF \u05D0\u05D9\u05DE\u05D9\u05D9\u05DC \u05EA\u05D5\u05D0\u05DD \u05D9\u05D5\u05DB\u05DC \u05DC\u05D4\u05E6\u05D8\u05E8\u05E3 \u05DC\u05D0\u05E8\u05D2\u05D5\u05DF \u05D1\u05DB\u05DC \u05E2\u05EA.",
          headerTitle: "\u05D4\u05D6\u05DE\u05E0\u05D5\u05EA \u05D0\u05D5\u05D8\u05D5\u05DE\u05D8\u05D9\u05D5\u05EA",
          primaryButton: "\u05E0\u05D4\u05DC \u05D3\u05D5\u05DE\u05D9\u05D9\u05E0\u05D9\u05DD \u05DE\u05D0\u05D5\u05DE\u05EA\u05D9\u05DD"
        },
        table__emptyRow: "\u05D0\u05D9\u05DF \u05D4\u05D6\u05DE\u05E0\u05D5\u05EA \u05DC\u05D4\u05E6\u05D2\u05D4"
      },
      invitedMembersTab: {
        menuAction__revoke: "\u05D1\u05D8\u05DC \u05D4\u05D6\u05DE\u05E0\u05D4",
        tableHeader__invited: "\u05D4\u05D5\u05D6\u05DE\u05DF"
      },
      requestsTab: {
        autoSuggestions: {
          headerSubtitle: "\u05DE\u05E9\u05EA\u05DE\u05E9\u05D9\u05DD \u05E9\u05E0\u05E8\u05E9\u05DE\u05D9\u05DD \u05E2\u05DD \u05D3\u05D5\u05DE\u05D9\u05D9\u05DF \u05D0\u05D9\u05DE\u05D9\u05D9\u05DC \u05EA\u05D5\u05D0\u05DD \u05D9\u05D5\u05DB\u05DC\u05D5 \u05DC\u05E8\u05D0\u05D5\u05EA \u05D4\u05E6\u05E2\u05D4 \u05DC\u05D1\u05E7\u05E9 \u05DC\u05D4\u05E6\u05D8\u05E8\u05E3 \u05DC\u05D0\u05E8\u05D2\u05D5\u05DF \u05E9\u05DC\u05DA.",
          headerTitle: "\u05D4\u05E6\u05E2\u05D5\u05EA \u05D0\u05D5\u05D8\u05D5\u05DE\u05D8\u05D9\u05D5\u05EA",
          primaryButton: "\u05E0\u05D4\u05DC \u05D3\u05D5\u05DE\u05D9\u05D9\u05E0\u05D9\u05DD \u05DE\u05D0\u05D5\u05DE\u05EA\u05D9\u05DD"
        },
        menuAction__approve: "\u05D0\u05E9\u05E8",
        menuAction__reject: "\u05D3\u05D7\u05D4",
        tableHeader__requested: "\u05D1\u05E7\u05E9\u05EA \u05D2\u05D9\u05E9\u05D4",
        table__emptyRow: "\u05D0\u05D9\u05DF \u05D1\u05E7\u05E9\u05D5\u05EA \u05DC\u05D4\u05E6\u05D2\u05D4"
      },
      start: {
        headerTitle__invitations: "\u05D4\u05D6\u05DE\u05E0\u05D5\u05EA",
        headerTitle__members: "\u05D7\u05D1\u05E8\u05D9\u05DD",
        headerTitle__requests: "\u05D1\u05E7\u05E9\u05D5\u05EA"
      }
    },
    navbar: {
      apiKeys: void 0,
      billing: void 0,
      description: "\u05E0\u05D4\u05DC \u05D0\u05EA \u05D4\u05D0\u05E8\u05D2\u05D5\u05DF \u05E9\u05DC\u05DA.",
      general: "\u05DB\u05DC\u05DC\u05D9",
      members: "\u05D7\u05D1\u05E8\u05D9\u05DD",
      title: "\u05D0\u05E8\u05D2\u05D5\u05DF"
    },
    plansPage: {
      alerts: {
        noPermissionsToManageBilling: void 0
      },
      title: void 0
    },
    profilePage: {
      dangerSection: {
        deleteOrganization: {
          actionDescription: '\u05D4\u05E7\u05DC\u05D3 "{{organizationName}}" \u05DC\u05DE\u05D8\u05D4 \u05DB\u05D3\u05D9 \u05DC\u05D4\u05DE\u05E9\u05D9\u05DA.',
          messageLine1: "\u05D4\u05D0\u05DD \u05D0\u05EA\u05D4 \u05D1\u05D8\u05D5\u05D7 \u05E9\u05D0\u05EA\u05D4 \u05E8\u05D5\u05E6\u05D4 \u05DC\u05DE\u05D7\u05D5\u05E7 \u05D0\u05EA \u05D4\u05D0\u05E8\u05D2\u05D5\u05DF \u05D4\u05D6\u05D4?",
          messageLine2: "\u05E4\u05E2\u05D5\u05DC\u05D4 \u05D6\u05D5 \u05D4\u05D9\u05D0 \u05E1\u05D5\u05E4\u05D9\u05EA \u05D5\u05D1\u05DC\u05EA\u05D9 \u05D4\u05E4\u05D9\u05DB\u05D4.",
          successMessage: "\u05DE\u05D7\u05E7\u05EA \u05D0\u05EA \u05D4\u05D0\u05E8\u05D2\u05D5\u05DF.",
          title: "\u05DE\u05D7\u05E7 \u05D0\u05E8\u05D2\u05D5\u05DF"
        },
        leaveOrganization: {
          actionDescription: '\u05D4\u05E7\u05DC\u05D3 "{{organizationName}}" \u05DC\u05DE\u05D8\u05D4 \u05DB\u05D3\u05D9 \u05DC\u05D4\u05DE\u05E9\u05D9\u05DA.',
          messageLine1: "\u05D4\u05D0\u05DD \u05D0\u05EA\u05D4 \u05D1\u05D8\u05D5\u05D7 \u05E9\u05D0\u05EA\u05D4 \u05E8\u05D5\u05E6\u05D4 \u05DC\u05E2\u05D6\u05D5\u05D1 \u05D0\u05EA \u05D4\u05D0\u05E8\u05D2\u05D5\u05DF \u05D4\u05D6\u05D4? \u05EA\u05D0\u05D1\u05D3 \u05D2\u05D9\u05E9\u05D4 \u05DC\u05D0\u05E8\u05D2\u05D5\u05DF \u05D6\u05D4 \u05D5\u05DC\u05D9\u05D9\u05E9\u05D5\u05DE\u05D9\u05DD \u05E9\u05DC\u05D5.",
          messageLine2: "\u05E4\u05E2\u05D5\u05DC\u05D4 \u05D6\u05D5 \u05D4\u05D9\u05D0 \u05E1\u05D5\u05E4\u05D9\u05EA \u05D5\u05D1\u05DC\u05EA\u05D9 \u05D4\u05E4\u05D9\u05DB\u05D4.",
          successMessage: "\u05E2\u05D6\u05D1\u05EA \u05D0\u05EA \u05D4\u05D0\u05E8\u05D2\u05D5\u05DF.",
          title: "\u05E2\u05D6\u05D5\u05D1 \u05D0\u05EA \u05D4\u05D0\u05E8\u05D2\u05D5\u05DF"
        },
        title: "\u05E1\u05D9\u05DB\u05D5\u05DF"
      },
      domainSection: {
        menuAction__manage: "\u05E0\u05D4\u05DC",
        menuAction__remove: "\u05DE\u05D7\u05E7",
        menuAction__verify: "\u05D0\u05DE\u05EA",
        primaryButton: "\u05D4\u05D5\u05E1\u05E3 \u05D3\u05D5\u05DE\u05D9\u05D9\u05DF",
        subtitle: "\u05D0\u05E4\u05E9\u05E8 \u05DC\u05DE\u05E9\u05EA\u05DE\u05E9\u05D9\u05DD \u05DC\u05D4\u05E6\u05D8\u05E8\u05E3 \u05DC\u05D0\u05E8\u05D2\u05D5\u05DF \u05D1\u05D0\u05D5\u05E4\u05DF \u05D0\u05D5\u05D8\u05D5\u05DE\u05D8\u05D9 \u05D0\u05D5 \u05DC\u05D1\u05E7\u05E9 \u05DC\u05D4\u05E6\u05D8\u05E8\u05E3 \u05E2\u05DC \u05D1\u05E1\u05D9\u05E1 \u05D3\u05D5\u05DE\u05D9\u05D9\u05DF \u05D0\u05D9\u05DE\u05D9\u05D9\u05DC \u05DE\u05D0\u05D5\u05DE\u05EA.",
        title: "\u05D3\u05D5\u05DE\u05D9\u05D9\u05E0\u05D9\u05DD \u05DE\u05D0\u05D5\u05DE\u05EA\u05D9\u05DD"
      },
      successMessage: "\u05D4\u05D0\u05E8\u05D2\u05D5\u05DF \u05E2\u05D5\u05D3\u05DB\u05DF.",
      title: "\u05E4\u05E8\u05D5\u05E4\u05D9\u05DC \u05D0\u05E8\u05D2\u05D5\u05DF"
    },
    removeDomainPage: {
      messageLine1: "\u05D3\u05D5\u05DE\u05D9\u05D9\u05DF \u05D4\u05D0\u05D9\u05DE\u05D9\u05D9\u05DC {{domain}} \u05D9\u05D5\u05E1\u05E8.",
      messageLine2: "\u05DE\u05E9\u05EA\u05DE\u05E9\u05D9\u05DD \u05DC\u05D0 \u05D9\u05D5\u05DB\u05DC\u05D5 \u05DC\u05D4\u05E6\u05D8\u05E8\u05E3 \u05DC\u05D0\u05E8\u05D2\u05D5\u05DF \u05D1\u05D0\u05D5\u05E4\u05DF \u05D0\u05D5\u05D8\u05D5\u05DE\u05D8\u05D9 \u05DC\u05D0\u05D7\u05E8 \u05E4\u05E2\u05D5\u05DC\u05D4 \u05D6\u05D5.",
      successMessage: "{{domain}} \u05D4\u05D5\u05E1\u05E8.",
      title: "\u05D4\u05E1\u05E8 \u05D3\u05D5\u05DE\u05D9\u05D9\u05DF"
    },
    start: {
      headerTitle__general: "\u05DB\u05DC\u05DC\u05D9",
      headerTitle__members: "\u05D7\u05D1\u05E8\u05D9\u05DD",
      profileSection: {
        primaryButton: "\u05E2\u05D3\u05DB\u05DF \u05E4\u05E8\u05D5\u05E4\u05D9\u05DC",
        title: "\u05E4\u05E8\u05D5\u05E4\u05D9\u05DC \u05D0\u05E8\u05D2\u05D5\u05DF",
        uploadAction__title: "\u05DC\u05D5\u05D2\u05D5"
      }
    },
    verifiedDomainPage: {
      dangerTab: {
        calloutInfoLabel: "\u05D4\u05E1\u05E8\u05EA \u05D3\u05D5\u05DE\u05D9\u05D9\u05DF \u05D6\u05D4 \u05EA\u05E9\u05E4\u05D9\u05E2 \u05E2\u05DC \u05D4\u05DE\u05E9\u05EA\u05DE\u05E9\u05D9\u05DD \u05D4\u05DE\u05D5\u05D6\u05DE\u05E0\u05D9\u05DD.",
        removeDomainActionLabel__remove: "\u05D4\u05E1\u05E8 \u05D3\u05D5\u05DE\u05D9\u05D9\u05DF",
        removeDomainSubtitle: "\u05D4\u05E1\u05E8 \u05D3\u05D5\u05DE\u05D9\u05D9\u05DF \u05D6\u05D4 \u05DE\u05E8\u05E9\u05D9\u05DE\u05EA \u05D4\u05D3\u05D5\u05DE\u05D9\u05D9\u05E0\u05D9\u05DD \u05D4\u05DE\u05D0\u05D5\u05DE\u05EA\u05D9\u05DD \u05E9\u05DC\u05DA",
        removeDomainTitle: "\u05D4\u05E1\u05E8 \u05D3\u05D5\u05DE\u05D9\u05D9\u05DF"
      },
      enrollmentTab: {
        automaticInvitationOption__description: "\u05DE\u05E9\u05EA\u05DE\u05E9\u05D9\u05DD \u05DE\u05D5\u05D6\u05DE\u05E0\u05D9\u05DD \u05D0\u05D5\u05D8\u05D5\u05DE\u05D8\u05D9\u05EA \u05DC\u05D4\u05E6\u05D8\u05E8\u05E3 \u05DC\u05D0\u05E8\u05D2\u05D5\u05DF \u05DB\u05E9\u05D4\u05DD \u05E0\u05E8\u05E9\u05DE\u05D9\u05DD \u05D5\u05D9\u05DB\u05D5\u05DC\u05D9\u05DD \u05DC\u05D4\u05E6\u05D8\u05E8\u05E3 \u05D1\u05DB\u05DC \u05E2\u05EA.",
        automaticInvitationOption__label: "\u05D4\u05D6\u05DE\u05E0\u05D5\u05EA \u05D0\u05D5\u05D8\u05D5\u05DE\u05D8\u05D9\u05D5\u05EA",
        automaticSuggestionOption__description: "\u05DE\u05E9\u05EA\u05DE\u05E9\u05D9\u05DD \u05DE\u05E7\u05D1\u05DC\u05D9\u05DD \u05D4\u05E6\u05E2\u05D4 \u05DC\u05D1\u05E7\u05E9 \u05DC\u05D4\u05E6\u05D8\u05E8\u05E3, \u05D0\u05DA \u05D7\u05D9\u05D9\u05D1\u05D9\u05DD \u05DC\u05D4\u05D9\u05D5\u05EA \u05DE\u05D0\u05D5\u05E9\u05E8\u05D9\u05DD \u05E2\u05DC \u05D9\u05D3\u05D9 \u05DE\u05E0\u05D4\u05DC \u05DC\u05E4\u05E0\u05D9 \u05E9\u05D9\u05D5\u05DB\u05DC\u05D5 \u05DC\u05D4\u05E6\u05D8\u05E8\u05E3 \u05DC\u05D0\u05E8\u05D2\u05D5\u05DF.",
        automaticSuggestionOption__label: "\u05D4\u05E6\u05E2\u05D5\u05EA \u05D0\u05D5\u05D8\u05D5\u05DE\u05D8\u05D9\u05D5\u05EA",
        calloutInfoLabel: "\u05E9\u05D9\u05E0\u05D5\u05D9 \u05DE\u05E6\u05D1 \u05D4\u05D4\u05E8\u05E9\u05DE\u05D4 \u05D9\u05E9\u05E4\u05D9\u05E2 \u05E8\u05E7 \u05E2\u05DC \u05DE\u05E9\u05EA\u05DE\u05E9\u05D9\u05DD \u05D7\u05D3\u05E9\u05D9\u05DD.",
        calloutInvitationCountLabel: "\u05D4\u05D6\u05DE\u05E0\u05D5\u05EA \u05DE\u05DE\u05EA\u05D9\u05E0\u05D5\u05EA \u05E9\u05E0\u05E9\u05DC\u05D7\u05D5 \u05DC\u05DE\u05E9\u05EA\u05DE\u05E9\u05D9\u05DD: {{count}}",
        calloutSuggestionCountLabel: "\u05D4\u05E6\u05E2\u05D5\u05EA \u05DE\u05DE\u05EA\u05D9\u05E0\u05D5\u05EA \u05E9\u05E0\u05E9\u05DC\u05D7\u05D5 \u05DC\u05DE\u05E9\u05EA\u05DE\u05E9\u05D9\u05DD: {{count}}",
        manualInvitationOption__description: "\u05DE\u05E9\u05EA\u05DE\u05E9\u05D9\u05DD \u05D9\u05DB\u05D5\u05DC\u05D9\u05DD \u05DC\u05D4\u05E6\u05D8\u05E8\u05E3 \u05DC\u05D0\u05E8\u05D2\u05D5\u05DF \u05E8\u05E7 \u05D1\u05D0\u05DE\u05E6\u05E2\u05D5\u05EA \u05D4\u05D6\u05DE\u05E0\u05D4 \u05D9\u05D3\u05E0\u05D9\u05EA.",
        manualInvitationOption__label: "\u05DC\u05DC\u05D0 \u05D4\u05E8\u05E9\u05DE\u05D4 \u05D0\u05D5\u05D8\u05D5\u05DE\u05D8\u05D9\u05EA",
        subtitle: "\u05D1\u05D7\u05E8 \u05DB\u05D9\u05E6\u05D3 \u05DE\u05E9\u05EA\u05DE\u05E9\u05D9\u05DD \u05DE\u05D3\u05D5\u05DE\u05D9\u05D9\u05DF \u05D6\u05D4 \u05D9\u05DB\u05D5\u05DC\u05D9\u05DD \u05DC\u05D4\u05E6\u05D8\u05E8\u05E3 \u05DC\u05D0\u05E8\u05D2\u05D5\u05DF."
      },
      start: {
        headerTitle__danger: "\u05E1\u05D9\u05DB\u05D5\u05DF",
        headerTitle__enrollment: "\u05D0\u05E4\u05E9\u05E8\u05D5\u05D9\u05D5\u05EA \u05D4\u05E8\u05E9\u05DE\u05D4"
      },
      subtitle: "\u05D4\u05D3\u05D5\u05DE\u05D9\u05D9\u05DF {{domain}} \u05D0\u05D5\u05DE\u05EA \u05D1\u05D4\u05E6\u05DC\u05D7\u05D4. \u05D4\u05DE\u05E9\u05DA \u05D1\u05D1\u05D7\u05D9\u05E8\u05EA \u05DE\u05E6\u05D1 \u05D4\u05E8\u05E9\u05DE\u05D4.",
      title: "\u05E2\u05D3\u05DB\u05DF {{domain}}"
    },
    verifyDomainPage: {
      formSubtitle: "\u05D4\u05D6\u05DF \u05D0\u05EA \u05E7\u05D5\u05D3 \u05D4\u05D0\u05D9\u05DE\u05D5\u05EA \u05E9\u05E0\u05E9\u05DC\u05D7 \u05DC\u05DB\u05EA\u05D5\u05D1\u05EA \u05D4\u05D0\u05D9\u05DE\u05D9\u05D9\u05DC \u05E9\u05DC\u05DA",
      formTitle: "\u05E7\u05D5\u05D3 \u05D0\u05D9\u05DE\u05D5\u05EA",
      resendButton: "\u05DC\u05D0 \u05E7\u05D9\u05D1\u05DC\u05EA \u05E7\u05D5\u05D3? \u05E9\u05DC\u05D7 \u05E9\u05D5\u05D1",
      subtitle: "\u05D4\u05D3\u05D5\u05DE\u05D9\u05D9\u05DF {{domainName}} \u05E6\u05E8\u05D9\u05DA \u05DC\u05D4\u05D9\u05D5\u05EA \u05DE\u05D0\u05D5\u05DE\u05EA \u05D1\u05D0\u05DE\u05E6\u05E2\u05D5\u05EA \u05D0\u05D9\u05DE\u05D9\u05D9\u05DC.",
      subtitleVerificationCodeScreen: "\u05E7\u05D5\u05D3 \u05D0\u05D9\u05DE\u05D5\u05EA \u05E0\u05E9\u05DC\u05D7 \u05DC{{emailAddress}}. \u05D4\u05D6\u05DF \u05D0\u05EA \u05D4\u05E7\u05D5\u05D3 \u05DB\u05D3\u05D9 \u05DC\u05D4\u05DE\u05E9\u05D9\u05DA.",
      title: "\u05D0\u05DE\u05EA \u05D3\u05D5\u05DE\u05D9\u05D9\u05DF"
    }
  },
  organizationSwitcher: {
    action__closeOrganizationSwitcher: void 0,
    action__createOrganization: "\u05E6\u05D5\u05E8 \u05D0\u05E8\u05D2\u05D5\u05DF",
    action__invitationAccept: "\u05D4\u05E6\u05D8\u05E8\u05E3",
    action__manageOrganization: "\u05E0\u05D4\u05DC \u05D0\u05E8\u05D2\u05D5\u05DF",
    action__openOrganizationSwitcher: void 0,
    action__suggestionsAccept: "\u05D1\u05E7\u05E9\u05D4 \u05DC\u05D4\u05E6\u05D8\u05E8\u05E4\u05D5\u05EA",
    notSelected: "\u05DC\u05D0 \u05E0\u05D1\u05D7\u05E8 \u05D0\u05E8\u05D2\u05D5\u05DF",
    personalWorkspace: "\u05D0\u05D6\u05D5\u05E8 \u05D0\u05D9\u05E9\u05D9",
    suggestionsAcceptedLabel: "\u05DE\u05DE\u05EA\u05D9\u05DF \u05DC\u05D0\u05D9\u05E9\u05D5\u05E8"
  },
  paginationButton__next: "\u05D4\u05D1\u05D0",
  paginationButton__previous: "\u05D4\u05E7\u05D5\u05D3\u05DD",
  paginationRowText__displaying: "\u05DE\u05E6\u05D9\u05D2",
  paginationRowText__of: "\u05DE\u05EA\u05D5\u05DA",
  reverification: {
    alternativeMethods: {
      actionLink: "\u05D4\u05E9\u05D2 \u05E2\u05D6\u05E8\u05D4",
      actionText: "\u05D0\u05D9\u05DF \u05DC\u05DA \u05D0\u05E3 \u05D0\u05D7\u05D3 \u05DE\u05D0\u05DC\u05D4?",
      blockButton__backupCode: "\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05E7\u05D5\u05D3 \u05D2\u05D9\u05D1\u05D5\u05D9",
      blockButton__emailCode: "\u05E7\u05D5\u05D3 \u05D0\u05D9\u05DE\u05D9\u05D9\u05DC \u05DC {{identifier}}",
      blockButton__passkey: void 0,
      blockButton__password: "\u05D4\u05DE\u05E9\u05DA \u05E2\u05DD \u05D4\u05E1\u05D9\u05E1\u05DE\u05D4 \u05E9\u05DC\u05DA",
      blockButton__phoneCode: "\u05E9\u05DC\u05D7 \u05E7\u05D5\u05D3 SMS \u05DC {{identifier}}",
      blockButton__totp: "\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05D0\u05E4\u05DC\u05D9\u05E7\u05E6\u05D9\u05D9\u05EA \u05D4\u05D0\u05D9\u05DE\u05D5\u05EA \u05E9\u05DC\u05DA",
      getHelp: {
        blockButton__emailSupport: "\u05EA\u05DE\u05D9\u05DB\u05D4 \u05D1\u05D0\u05D9\u05DE\u05D9\u05D9\u05DC",
        content: "\u05D0\u05DD \u05D9\u05E9 \u05DC\u05DA \u05D1\u05E2\u05D9\u05D4 \u05D1\u05D0\u05D9\u05DE\u05D5\u05EA \u05D7\u05E9\u05D1\u05D5\u05E0\u05DA, \u05E9\u05DC\u05D7 \u05DC\u05E0\u05D5 \u05D0\u05D9\u05DE\u05D9\u05D9\u05DC \u05D5\u05D0\u05E0\u05D7\u05E0\u05D5 \u05E0\u05D9\u05E6\u05D5\u05E8 \u05D0\u05D9\u05EA\u05DA \u05E7\u05E9\u05E8 \u05E2\u05DC \u05DE\u05E0\u05EA \u05DC\u05E9\u05D7\u05D6\u05E8 \u05D0\u05EA \u05D4\u05D2\u05D9\u05E9\u05D4 \u05D1\u05D4\u05E7\u05D3\u05DD \u05D4\u05D0\u05E4\u05E9\u05E8\u05D9",
        title: "\u05D4\u05E9\u05D2 \u05E2\u05D6\u05E8\u05D4"
      },
      subtitle: "\u05E0\u05EA\u05E7\u05DC\u05EA \u05D1\u05D1\u05E2\u05D9\u05D5\u05EA? \u05D0\u05EA\u05D4 \u05D9\u05DB\u05D5\u05DC \u05DC\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05DB\u05DC \u05D0\u05D7\u05EA \u05DE\u05D4\u05E9\u05D9\u05D8\u05D5\u05EA \u05D4\u05DC\u05DC\u05D5 \u05E2\u05D1\u05D5\u05E8 \u05D0\u05D9\u05DE\u05D5\u05EA",
      title: "\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05E9\u05D9\u05D8\u05D4 \u05D0\u05D7\u05E8\u05EA"
    },
    backupCodeMfa: {
      subtitle: "\u05E7\u05D5\u05D3 \u05D4\u05D2\u05D9\u05D1\u05D5\u05D9 \u05E9\u05DC\u05DA \u05D4\u05D5\u05D0 \u05D4\u05E7\u05D5\u05D3 \u05E9\u05E7\u05D9\u05D1\u05DC\u05EA \u05DB\u05D0\u05E9\u05E8 \u05D4\u05D2\u05D3\u05E8\u05EA \u05D0\u05EA \u05D4\u05D0\u05D9\u05DE\u05D5\u05EA \u05D4\u05D3\u05D5-\u05E9\u05DC\u05D1\u05D9",
      title: "\u05D4\u05DB\u05E0\u05E1 \u05D0\u05EA \u05E7\u05D5\u05D3 \u05D4\u05D2\u05D9\u05D1\u05D5\u05D9"
    },
    emailCode: {
      formTitle: "\u05E7\u05D5\u05D3 \u05D0\u05D9\u05DE\u05D5\u05EA",
      resendButton: "\u05DC\u05D0 \u05E7\u05D9\u05D1\u05DC\u05EA \u05E7\u05D5\u05D3? \u05E9\u05DC\u05D7 \u05E9\u05D5\u05D1",
      subtitle: "\u05D4\u05DE\u05E9\u05DA \u05DC {{applicationName}}",
      title: "\u05D1\u05D3\u05D5\u05E7 \u05D0\u05EA \u05D4\u05D0\u05D9\u05DE\u05D9\u05D9\u05DC \u05E9\u05DC\u05DA"
    },
    noAvailableMethods: {
      message: "\u05DC\u05D0 \u05E0\u05D9\u05EA\u05DF \u05DC\u05D4\u05DE\u05E9\u05D9\u05DA \u05E2\u05DD \u05D4\u05D0\u05D9\u05DE\u05D5\u05EA. \u05D0\u05D9\u05DF \u05D2\u05D5\u05E8\u05DD \u05D0\u05D9\u05DE\u05D5\u05EA \u05D6\u05DE\u05D9\u05DF",
      subtitle: "\u05E7\u05E8\u05EA\u05D4 \u05EA\u05E7\u05DC\u05D4",
      title: "\u05DC\u05D0 \u05E0\u05D9\u05EA\u05DF \u05DC\u05D0\u05DE\u05EA \u05D0\u05EA \u05D7\u05E9\u05D1\u05D5\u05E0\u05DA"
    },
    passkey: {
      blockButton__passkey: void 0,
      subtitle: void 0,
      title: void 0
    },
    password: {
      actionLink: "\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05E9\u05D9\u05D8\u05D4 \u05D0\u05D7\u05E8\u05EA",
      subtitle: "\u05D4\u05DB\u05E0\u05E1 \u05D0\u05EA \u05D4\u05E1\u05D9\u05E1\u05DE\u05D4 \u05D4\u05DE\u05E7\u05D5\u05E9\u05E8\u05EA \u05E2\u05DD \u05D7\u05E9\u05D1\u05D5\u05E0\u05DA",
      title: "\u05D4\u05DB\u05E0\u05E1 \u05E1\u05D9\u05E1\u05DE\u05D4"
    },
    phoneCode: {
      formTitle: "\u05E7\u05D5\u05D3 \u05D0\u05D9\u05DE\u05D5\u05EA",
      resendButton: "\u05DC\u05D0 \u05E7\u05D9\u05D1\u05DC\u05EA \u05E7\u05D5\u05D3? \u05E9\u05DC\u05D7 \u05E9\u05D5\u05D1",
      subtitle: "\u05DC\u05D4\u05DE\u05E9\u05DA \u05DC {{applicationName}}",
      title: "\u05D1\u05D3\u05D5\u05E7 \u05D0\u05EA \u05D4\u05D8\u05DC\u05E4\u05D5\u05DF \u05E9\u05DC\u05DA"
    },
    phoneCodeMfa: {
      formTitle: "\u05E7\u05D5\u05D3 \u05D0\u05D9\u05DE\u05D5\u05EA",
      resendButton: "\u05DC\u05D0 \u05E7\u05D9\u05D1\u05DC\u05EA \u05E7\u05D5\u05D3? \u05E9\u05DC\u05D7 \u05E9\u05D5\u05D1",
      subtitle: "\u05DC\u05D4\u05DE\u05E9\u05DA, \u05D0\u05E0\u05D0 \u05D4\u05DB\u05E0\u05E1 \u05D0\u05EA \u05E7\u05D5\u05D3 \u05D4\u05D0\u05D9\u05DE\u05D5\u05EA \u05E9\u05E0\u05E9\u05DC\u05D7 \u05DC\u05D8\u05DC\u05E4\u05D5\u05DF \u05E9\u05DC\u05DA",
      title: "\u05D1\u05D3\u05D5\u05E7 \u05D0\u05EA \u05D4\u05D8\u05DC\u05E4\u05D5\u05DF \u05E9\u05DC\u05DA"
    },
    totpMfa: {
      formTitle: "\u05E7\u05D5\u05D3 \u05D0\u05D9\u05DE\u05D5\u05EA",
      subtitle: "\u05DC\u05D4\u05DE\u05E9\u05DA, \u05D0\u05E0\u05D0 \u05D4\u05DB\u05E0\u05E1 \u05D0\u05EA \u05E7\u05D5\u05D3 \u05D4\u05D0\u05D9\u05DE\u05D5\u05EA \u05E9\u05E0\u05D5\u05E6\u05E8 \u05E2\u05DC \u05D9\u05D3\u05D9 \u05D0\u05E4\u05DC\u05D9\u05E7\u05E6\u05D9\u05D9\u05EA \u05D4\u05D0\u05D9\u05DE\u05D5\u05EA \u05E9\u05DC\u05DA",
      title: "\u05D0\u05D9\u05DE\u05D5\u05EA \u05D3\u05D5-\u05E9\u05DC\u05D1\u05D9"
    }
  },
  signIn: {
    accountSwitcher: {
      action__addAccount: "\u05D4\u05D5\u05E1\u05E3 \u05D7\u05E9\u05D1\u05D5\u05DF",
      action__signOutAll: "\u05D4\u05EA\u05E0\u05EA\u05E7 \u05DE\u05DB\u05DC \u05D4\u05D7\u05E9\u05D1\u05D5\u05E0\u05D5\u05EA",
      subtitle: "\u05D1\u05D7\u05E8 \u05D0\u05EA \u05D4\u05D7\u05E9\u05D1\u05D5\u05DF \u05D0\u05D9\u05EA\u05D5 \u05EA\u05E8\u05E6\u05D4 \u05DC\u05D4\u05DE\u05E9\u05D9\u05DA.",
      title: "\u05D1\u05D7\u05E8 \u05D7\u05E9\u05D1\u05D5\u05DF"
    },
    alternativeMethods: {
      actionLink: "\u05E7\u05D1\u05DC \u05E2\u05D6\u05E8\u05D4",
      actionText: "\u05D0\u05D9\u05DF \u05DC\u05DA \u05D0\u05E3 \u05D0\u05D7\u05EA \u05DE\u05D4\u05E9\u05D9\u05D8\u05D5\u05EA \u05D4\u05D0\u05DC\u05D4?",
      blockButton__backupCode: "\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05E7\u05D5\u05D3 \u05D2\u05D9\u05D1\u05D5\u05D9",
      blockButton__emailCode: "\u05E9\u05DC\u05D7 \u05E7\u05D5\u05D3 \u05D1\u05D0\u05D9\u05DE\u05D9\u05D9\u05DC \u05DC-{{identifier}}",
      blockButton__emailLink: "\u05E9\u05DC\u05D7 \u05E7\u05D9\u05E9\u05D5\u05E8 \u05D1\u05D0\u05D9\u05DE\u05D9\u05D9\u05DC \u05DC-{{identifier}}",
      blockButton__passkey: "\u05D4\u05D9\u05DB\u05E0\u05E1 \u05E2\u05DD \u05D4\u05DE\u05E4\u05EA\u05D7 \u05E9\u05DC\u05DA",
      blockButton__password: "\u05D4\u05EA\u05D7\u05D1\u05E8 \u05E2\u05DD \u05D4\u05E1\u05D9\u05E1\u05DE\u05D4 \u05E9\u05DC\u05DA",
      blockButton__phoneCode: "\u05E9\u05DC\u05D7 \u05E7\u05D5\u05D3 SMS \u05DC-{{identifier}}",
      blockButton__totp: "\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05D0\u05E4\u05DC\u05D9\u05E7\u05E6\u05D9\u05D9\u05EA \u05D4\u05D0\u05D9\u05DE\u05D5\u05EA \u05E9\u05DC\u05DA",
      getHelp: {
        blockButton__emailSupport: "\u05DE\u05D9\u05D9\u05DC \u05DC\u05EA\u05DE\u05D9\u05DB\u05D4",
        content: "\u05D0\u05DD \u05D0\u05EA\u05D4 \u05E0\u05EA\u05E7\u05DC \u05D1\u05E7\u05E9\u05D9\u05D9\u05DD \u05D1\u05D4\u05EA\u05D7\u05D1\u05E8\u05D5\u05EA \u05DC\u05D7\u05E9\u05D1\u05D5\u05E0\u05DA, \u05E9\u05DC\u05D7 \u05DC\u05E0\u05D5 \u05DE\u05D9\u05D9\u05DC \u05D5\u05E0\u05E2\u05D1\u05D5\u05D3 \u05D0\u05D9\u05EA\u05DA \u05DB\u05D3\u05D9 \u05DC\u05E9\u05D7\u05D6\u05E8 \u05D0\u05EA \u05D4\u05D2\u05D9\u05E9\u05D4 \u05D1\u05D4\u05E7\u05D3\u05DD \u05D4\u05D0\u05E4\u05E9\u05E8\u05D9.",
        title: "\u05E7\u05D1\u05DC \u05E2\u05D6\u05E8\u05D4"
      },
      subtitle: "\u05E0\u05EA\u05E7\u05DC\u05EA \u05D1\u05D1\u05E2\u05D9\u05D4? \u05EA\u05D5\u05DB\u05DC \u05DC\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05D0\u05D7\u05EA \u05DE\u05D4\u05E9\u05D9\u05D8\u05D5\u05EA \u05D4\u05D0\u05DC\u05D4 \u05DB\u05D3\u05D9 \u05DC\u05D4\u05EA\u05D7\u05D1\u05E8.",
      title: "\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05E9\u05D9\u05D8\u05D4 \u05D0\u05D7\u05E8\u05EA"
    },
    alternativePhoneCodeProvider: {
      formTitle: void 0,
      resendButton: void 0,
      subtitle: void 0,
      title: void 0
    },
    backupCodeMfa: {
      subtitle: "\u05DC\u05D4\u05DE\u05E9\u05D9\u05DA \u05D0\u05DC {{applicationName}}",
      title: "\u05D4\u05DB\u05E0\u05E1 \u05E7\u05D5\u05D3 \u05D2\u05D9\u05D1\u05D5\u05D9"
    },
    emailCode: {
      formTitle: "\u05E7\u05D5\u05D3 \u05D0\u05D9\u05DE\u05D5\u05EA",
      resendButton: "\u05E9\u05DC\u05D7 \u05E7\u05D5\u05D3 \u05E9\u05D5\u05D1",
      subtitle: "\u05DC\u05D4\u05DE\u05E9\u05D9\u05DA \u05D0\u05DC {{applicationName}}",
      title: '\u05D1\u05D3\u05D5\u05E7 \u05D0\u05EA \u05D4\u05D3\u05D5\u05D0"\u05DC \u05E9\u05DC\u05DA'
    },
    emailLink: {
      clientMismatch: {
        subtitle: "\u05DC\u05D4\u05DE\u05E9\u05DA, \u05E4\u05EA\u05D7 \u05D0\u05EA \u05E7\u05D5\u05D3 \u05D4\u05D0\u05D9\u05DE\u05D5\u05EA \u05DE\u05D4\u05DE\u05DB\u05E9\u05D9\u05E8 \u05D5\u05D4\u05D3\u05E4\u05D3\u05E4\u05DF \u05DE\u05DE\u05E0\u05D5 \u05D0\u05EA\u05D4 \u05DE\u05EA\u05DB\u05D5\u05D5\u05DF \u05DC\u05D1\u05E6\u05E2 \u05DB\u05E0\u05D9\u05E1\u05D4",
        title: "\u05E7\u05D9\u05E9\u05D5\u05E8 \u05D4\u05D0\u05D9\u05DE\u05D5\u05EA \u05D4\u05D6\u05D4 \u05DC\u05D0 \u05D7\u05D5\u05E7\u05D9 \u05E2\u05D1\u05D5\u05E8 \u05DE\u05DB\u05E9\u05D9\u05E8 \u05D6\u05D4"
      },
      expired: {
        subtitle: "\u05D7\u05D6\u05D5\u05E8 \u05DC\u05DB\u05E8\u05D8\u05D9\u05E1\u05D9\u05D9\u05D4 \u05D4\u05DE\u05E7\u05D5\u05E8\u05D9\u05EA \u05DC\u05D4\u05DE\u05E9\u05DA.",
        title: "\u05E7\u05D9\u05E9\u05D5\u05E8 \u05D4\u05D0\u05D9\u05DE\u05D5\u05EA \u05D4\u05D6\u05D4 \u05E4\u05D2 \u05EA\u05D5\u05E7\u05E3"
      },
      failed: {
        subtitle: "\u05D7\u05D6\u05D5\u05E8 \u05DC\u05DB\u05E8\u05D8\u05D9\u05E1\u05D9\u05D9\u05D4 \u05D4\u05DE\u05E7\u05D5\u05E8\u05D9\u05EA \u05DC\u05D4\u05DE\u05E9\u05DA.",
        title: "\u05E7\u05D9\u05E9\u05D5\u05E8 \u05D4\u05D0\u05D9\u05DE\u05D5\u05EA \u05D4\u05D6\u05D4 \u05DC\u05D0 \u05D7\u05D5\u05E7\u05D9"
      },
      formSubtitle: '\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05E7\u05D9\u05E9\u05D5\u05E8 \u05D4\u05D0\u05D9\u05DE\u05D5\u05EA \u05E9\u05E0\u05E9\u05DC\u05D7 \u05DC\u05D3\u05D5\u05D0"\u05DC \u05E9\u05DC\u05DA',
      formTitle: "\u05E7\u05D9\u05E9\u05D5\u05E8 \u05D0\u05D9\u05DE\u05D5\u05EA",
      loading: {
        subtitle: "\u05EA\u05D5\u05E2\u05D1\u05E8 \u05D1\u05E7\u05E8\u05D5\u05D1",
        title: "\u05DE\u05EA\u05D7\u05D1\u05E8..."
      },
      resendButton: "\u05E9\u05DC\u05D7 \u05E7\u05D9\u05E9\u05D5\u05E8 \u05E9\u05D5\u05D1",
      subtitle: "\u05DC\u05D4\u05DE\u05E9\u05D9\u05DA \u05D0\u05DC {{applicationName}}",
      title: '\u05D1\u05D3\u05D5\u05E7 \u05D0\u05EA \u05D4\u05D3\u05D5\u05D0"\u05DC \u05E9\u05DC\u05DA',
      unusedTab: {
        title: "\u05D0\u05EA\u05D4 \u05D9\u05DB\u05D5\u05DC \u05DC\u05E1\u05D2\u05D5\u05E8 \u05D0\u05EA \u05D4\u05DB\u05E8\u05D8\u05D9\u05E1\u05D9\u05D9\u05D4 \u05D4\u05D6\u05D5"
      },
      verified: {
        subtitle: "\u05EA\u05D5\u05E2\u05D1\u05E8 \u05D1\u05E7\u05E8\u05D5\u05D1",
        title: "\u05E0\u05DB\u05E0\u05E1\u05EA \u05D1\u05D4\u05E6\u05DC\u05D7\u05D4"
      },
      verifiedSwitchTab: {
        subtitle: "\u05D7\u05D6\u05D5\u05E8 \u05DC\u05DB\u05E8\u05D8\u05D9\u05E1\u05D9\u05D9\u05D4 \u05D4\u05DE\u05E7\u05D5\u05E8\u05D9\u05EA \u05DC\u05D4\u05DE\u05E9\u05DA",
        subtitleNewTab: "\u05D7\u05D6\u05D5\u05E8 \u05DC\u05DB\u05E8\u05D8\u05D9\u05E1\u05D9\u05D9\u05D4 \u05E9\u05E0\u05E4\u05EA\u05D7\u05D4 \u05D7\u05D3\u05E9\u05D4 \u05DC\u05D4\u05DE\u05E9\u05DA",
        titleNewTab: "\u05E0\u05DB\u05E0\u05E1 \u05D1\u05DB\u05E8\u05D8\u05D9\u05E1\u05D9\u05D9\u05D4 \u05D0\u05D7\u05E8\u05EA"
      }
    },
    enterpriseConnections: {
      subtitle: void 0,
      title: void 0
    },
    forgotPassword: {
      formTitle: "\u05D0\u05E4\u05E1 \u05E7\u05D5\u05D3 \u05D4\u05E1\u05D9\u05E1\u05DE\u05D4",
      resendButton: "\u05E9\u05DC\u05D7 \u05E7\u05D5\u05D3 \u05E9\u05D5\u05D1",
      subtitle: "\u05DB\u05D3\u05D9 \u05DC\u05D0\u05E4\u05E1 \u05D0\u05EA \u05D4\u05E1\u05D9\u05E1\u05DE\u05D4 \u05E9\u05DC\u05DA",
      subtitle_email: "\u05E8\u05D0\u05E9\u05D9\u05EA, \u05D4\u05DB\u05E0\u05E1 \u05D0\u05EA \u05D4\u05E7\u05D5\u05D3 \u05E9\u05E0\u05E9\u05DC\u05D7 \u05DC\u05D0\u05D9\u05DE\u05D9\u05D9\u05DC \u05E9\u05DC\u05DA",
      subtitle_phone: "\u05E8\u05D0\u05E9\u05D9\u05EA, \u05D4\u05DB\u05E0\u05E1 \u05D0\u05EA \u05D4\u05E7\u05D5\u05D3 \u05E9\u05E0\u05E9\u05DC\u05D7 \u05DC\u05D8\u05DC\u05E4\u05D5\u05DF \u05E9\u05DC\u05DA",
      title: "\u05D0\u05D9\u05E4\u05D5\u05E1 \u05E1\u05D9\u05E1\u05DE\u05D4"
    },
    forgotPasswordAlternativeMethods: {
      blockButton__resetPassword: "\u05D0\u05E4\u05E1 \u05D0\u05EA \u05D4\u05E1\u05D9\u05E1\u05DE\u05D4 \u05E9\u05DC\u05DA",
      label__alternativeMethods: "\u05D0\u05D5, \u05D4\u05EA\u05D7\u05D1\u05E8 \u05D1\u05D0\u05DE\u05E6\u05E2\u05D5\u05EA \u05E9\u05D9\u05D8\u05D4 \u05D0\u05D7\u05E8\u05EA.",
      title: "\u05E9\u05DB\u05D7\u05EA \u05E1\u05D9\u05E1\u05DE\u05D4?"
    },
    noAvailableMethods: {
      message: "\u05DC\u05D0 \u05E0\u05D9\u05EA\u05DF \u05DC\u05D4\u05DE\u05E9\u05D9\u05DA \u05D1\u05D4\u05EA\u05D7\u05D1\u05E8\u05D5\u05EA. \u05D0\u05D9\u05DF \u05D2\u05D5\u05E8\u05DD \u05D0\u05D9\u05DE\u05D5\u05EA \u05D6\u05DE\u05D9\u05DF.",
      subtitle: "\u05D0\u05D9\u05E8\u05E2\u05D4 \u05E9\u05D2\u05D9\u05D0\u05D4",
      title: "\u05DC\u05D0 \u05E0\u05D9\u05EA\u05DF \u05DC\u05D4\u05EA\u05D7\u05D1\u05E8"
    },
    passkey: {
      subtitle: void 0,
      title: void 0
    },
    password: {
      actionLink: "\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05E9\u05D9\u05D8\u05D4 \u05D0\u05D7\u05E8\u05EA",
      subtitle: "\u05DC\u05D4\u05DE\u05E9\u05D9\u05DA \u05D0\u05DC {{applicationName}}",
      title: "\u05D4\u05DB\u05E0\u05E1 \u05D0\u05EA \u05E1\u05D9\u05E1\u05DE\u05EA\u05DA"
    },
    passwordPwned: {
      title: void 0
    },
    phoneCode: {
      formTitle: "\u05E7\u05D5\u05D3 \u05D0\u05D9\u05DE\u05D5\u05EA",
      resendButton: "\u05E9\u05DC\u05D7 \u05D0\u05EA \u05D4\u05E7\u05D5\u05D3 \u05E9\u05D5\u05D1",
      subtitle: "\u05DC\u05D4\u05DE\u05E9\u05D9\u05DA \u05D0\u05DC {{applicationName}}",
      title: "\u05D1\u05D3\u05D5\u05E7 \u05D0\u05EA \u05D4\u05D8\u05DC\u05E4\u05D5\u05DF \u05E9\u05DC\u05DA"
    },
    phoneCodeMfa: {
      formTitle: "\u05E7\u05D5\u05D3 \u05D0\u05D9\u05DE\u05D5\u05EA",
      resendButton: "\u05E9\u05DC\u05D7 \u05D0\u05EA \u05D4\u05E7\u05D5\u05D3 \u05E9\u05D5\u05D1",
      subtitle: void 0,
      title: "\u05D1\u05D3\u05D5\u05E7 \u05D0\u05EA \u05D4\u05D8\u05DC\u05E4\u05D5\u05DF \u05E9\u05DC\u05DA"
    },
    resetPassword: {
      formButtonPrimary: "\u05D0\u05E4\u05E1 \u05E1\u05D9\u05E1\u05DE\u05D4",
      requiredMessage: "\u05DE\u05D8\u05E2\u05DE\u05D9 \u05D0\u05D1\u05D8\u05D7\u05D4, \u05D9\u05E9 \u05DC\u05D0\u05E4\u05E1 \u05D0\u05EA \u05D4\u05E1\u05D9\u05E1\u05DE\u05D4 \u05E9\u05DC\u05DA.",
      successMessage: "\u05D4\u05E1\u05D9\u05E1\u05DE\u05D4 \u05E9\u05DC\u05DA \u05E9\u05D5\u05E0\u05EA\u05D4 \u05D1\u05D4\u05E6\u05DC\u05D7\u05D4. \u05DE\u05D7\u05D1\u05E8 \u05D0\u05D5\u05EA\u05DA, \u05D0\u05E0\u05D0 \u05D4\u05DE\u05EA\u05DF \u05E8\u05D2\u05E2.",
      title: "\u05D0\u05E4\u05E1 \u05E1\u05D9\u05E1\u05DE\u05D4"
    },
    resetPasswordMfa: {
      detailsLabel: "\u05D0\u05E0\u05D7\u05E0\u05D5 \u05E6\u05E8\u05D9\u05DB\u05D9\u05DD \u05DC\u05D0\u05DE\u05EA \u05D0\u05EA \u05D6\u05D4\u05D5\u05EA\u05DA \u05DC\u05E4\u05E0\u05D9 \u05E9\u05E0\u05D0\u05E4\u05E1 \u05D0\u05EA \u05D4\u05E1\u05D9\u05E1\u05DE\u05D4 \u05E9\u05DC\u05DA."
    },
    start: {
      actionLink: "\u05D4\u05E8\u05E9\u05DE\u05D4",
      actionLink__join_waitlist: void 0,
      actionLink__use_email: '\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05D3\u05D5\u05D0"\u05DC',
      actionLink__use_email_username: '\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05D3\u05D5\u05D0"\u05DC \u05D0\u05D5 \u05E9\u05DD \u05DE\u05E9\u05EA\u05DE\u05E9',
      actionLink__use_passkey: "\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05DE\u05E4\u05EA\u05D7 \u05E1\u05D9\u05E1\u05DE\u05D4 \u05D1\u05DE\u05E7\u05D5\u05DD",
      actionLink__use_phone: "\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05D8\u05DC\u05E4\u05D5\u05DF",
      actionLink__use_username: "\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05E9\u05DD \u05DE\u05E9\u05EA\u05DE\u05E9",
      actionText: "\u05D0\u05D9\u05DF \u05DC\u05DA \u05D7\u05E9\u05D1\u05D5\u05DF?",
      actionText__join_waitlist: void 0,
      alternativePhoneCodeProvider: {
        actionLink: void 0,
        label: void 0,
        subtitle: void 0,
        title: void 0
      },
      subtitle: "\u05DC\u05D4\u05DE\u05E9\u05D9\u05DA \u05D0\u05DC {{applicationName}}",
      subtitleCombined: void 0,
      title: "\u05D4\u05EA\u05D7\u05D1\u05E8",
      titleCombined: void 0
    },
    totpMfa: {
      formTitle: "\u05E7\u05D5\u05D3 \u05D0\u05D9\u05DE\u05D5\u05EA",
      subtitle: "\u05DC\u05D4\u05DE\u05E9\u05DA, \u05D0\u05E0\u05D0 \u05D4\u05DB\u05E0\u05E1 \u05D0\u05EA \u05E7\u05D5\u05D3 \u05D4\u05D0\u05D9\u05DE\u05D5\u05EA \u05E9\u05E0\u05D5\u05E6\u05E8 \u05E2\u05DC \u05D9\u05D3\u05D9 \u05D0\u05E4\u05DC\u05D9\u05E7\u05E6\u05D9\u05D9\u05EA \u05D4\u05D0\u05D9\u05DE\u05D5\u05EA \u05E9\u05DC\u05DA",
      title: "\u05D0\u05D9\u05DE\u05D5\u05EA \u05E9\u05E0\u05D9 \u05E9\u05DC\u05D1\u05D9\u05DD"
    }
  },
  signInEnterPasswordTitle: "\u05D4\u05D6\u05DF \u05D0\u05EA \u05D4\u05E1\u05D9\u05E1\u05DE\u05D4 \u05E9\u05DC\u05DA",
  signUp: {
    alternativePhoneCodeProvider: {
      resendButton: void 0,
      subtitle: void 0,
      title: void 0
    },
    continue: {
      actionLink: "\u05D4\u05EA\u05D7\u05D1\u05E8",
      actionText: "\u05D9\u05E9 \u05DC\u05DA \u05D7\u05E9\u05D1\u05D5\u05DF?",
      subtitle: "\u05DC\u05D4\u05DE\u05E9\u05D9\u05DA \u05D0\u05DC {{applicationName}}",
      title: "\u05DE\u05DC\u05D0 \u05E9\u05D3\u05D5\u05EA \u05D7\u05E1\u05E8\u05D9\u05DD"
    },
    emailCode: {
      formSubtitle: '\u05D4\u05DB\u05E0\u05E1 \u05D0\u05EA \u05E7\u05D5\u05D3 \u05D4\u05D0\u05D9\u05DE\u05D5\u05EA \u05E9\u05E0\u05E9\u05DC\u05D7 \u05DC\u05DB\u05EA\u05D5\u05D1\u05EA \u05D4\u05D3\u05D5\u05D0"\u05DC \u05E9\u05DC\u05DA',
      formTitle: "\u05E7\u05D5\u05D3 \u05D0\u05D9\u05DE\u05D5\u05EA",
      resendButton: "\u05E9\u05DC\u05D7 \u05E7\u05D5\u05D3 \u05E9\u05D5\u05D1",
      subtitle: "\u05DC\u05D4\u05DE\u05E9\u05D9\u05DA \u05D0\u05DC {{applicationName}}",
      title: '\u05D0\u05DE\u05EA \u05D0\u05EA \u05DB\u05EA\u05D5\u05D1\u05EA \u05D4\u05D3\u05D5\u05D0"\u05DC \u05E9\u05DC\u05DA'
    },
    emailLink: {
      clientMismatch: {
        subtitle: "\u05DC\u05D4\u05DE\u05E9\u05DA, \u05E4\u05EA\u05D7 \u05D0\u05EA \u05E7\u05D5\u05D3 \u05D4\u05D0\u05D9\u05DE\u05D5\u05EA \u05DE\u05D4\u05DE\u05DB\u05E9\u05D9\u05E8 \u05D5\u05D4\u05D3\u05E4\u05D3\u05E4\u05DF \u05DE\u05DE\u05E0\u05D5 \u05D0\u05EA\u05D4 \u05DE\u05EA\u05DB\u05D5\u05D5\u05DF \u05DC\u05D1\u05E6\u05E2 \u05D0\u05EA \u05D4\u05E8\u05D9\u05E9\u05D5\u05DD",
        title: "\u05E7\u05D9\u05E9\u05D5\u05E8 \u05D4\u05D0\u05D9\u05DE\u05D5\u05EA \u05D0\u05D9\u05E0\u05D5 \u05D7\u05D5\u05E7\u05D9 \u05E2\u05D1\u05D5\u05E8 \u05DE\u05DB\u05E9\u05D9\u05E8 \u05D6\u05D4"
      },
      formSubtitle: '\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05E7\u05D9\u05E9\u05D5\u05E8 \u05D4\u05D0\u05D9\u05DE\u05D5\u05EA \u05E9\u05E0\u05E9\u05DC\u05D7 \u05DC\u05DB\u05EA\u05D5\u05D1\u05EA \u05D4\u05D3\u05D5\u05D0"\u05DC \u05E9\u05DC\u05DA',
      formTitle: "\u05E7\u05D9\u05E9\u05D5\u05E8 \u05DC\u05D0\u05D9\u05DE\u05D5\u05EA",
      loading: {
        title: "\u05DE\u05EA\u05D7\u05D9\u05DC \u05DC\u05D4\u05D9\u05E8\u05E9\u05DD..."
      },
      resendButton: "\u05E9\u05DC\u05D7 \u05E7\u05D9\u05E9\u05D5\u05E8 \u05E9\u05D5\u05D1",
      subtitle: "\u05DC\u05D4\u05DE\u05E9\u05D9\u05DA \u05D0\u05DC {{applicationName}}",
      title: '\u05D0\u05DE\u05EA \u05D0\u05EA \u05DB\u05EA\u05D5\u05D1\u05EA \u05D4\u05D3\u05D5\u05D0"\u05DC \u05E9\u05DC\u05DA',
      verified: {
        title: "\u05E0\u05E8\u05E9\u05DE\u05EA \u05D1\u05D4\u05E6\u05DC\u05D7\u05D4"
      },
      verifiedSwitchTab: {
        subtitle: "\u05D7\u05D6\u05D5\u05E8 \u05DC\u05DB\u05E8\u05D8\u05D9\u05E1\u05D9\u05D9\u05D4 \u05E9\u05E0\u05E4\u05EA\u05D7\u05D4 \u05DC\u05D0\u05D7\u05E8\u05D5\u05E0\u05D4 \u05DC\u05D4\u05DE\u05E9\u05D9\u05DA",
        subtitleNewTab: "\u05D7\u05D6\u05D5\u05E8 \u05DC\u05DB\u05E8\u05D8\u05D9\u05E1\u05D9\u05D9\u05D4 \u05D4\u05E7\u05D5\u05D3\u05DE\u05EA \u05DC\u05D4\u05DE\u05E9\u05D9\u05DA",
        title: '\u05D0\u05D9\u05DE\u05D5\u05EA \u05D3\u05D5\u05D0"\u05DC \u05D4\u05E6\u05DC\u05D9\u05D7'
      }
    },
    enterpriseConnections: {
      subtitle: void 0,
      title: void 0
    },
    legalConsent: {
      checkbox: {
        label__onlyPrivacyPolicy: void 0,
        label__onlyTermsOfService: void 0,
        label__termsOfServiceAndPrivacyPolicy: void 0
      },
      continue: {
        subtitle: void 0,
        title: void 0
      }
    },
    phoneCode: {
      formSubtitle: "\u05D4\u05DB\u05E0\u05E1 \u05D0\u05EA \u05E7\u05D5\u05D3 \u05D4\u05D0\u05D9\u05DE\u05D5\u05EA \u05E9\u05E0\u05E9\u05DC\u05D7 \u05DC\u05DE\u05E1\u05E4\u05E8 \u05D4\u05D8\u05DC\u05E4\u05D5\u05DF \u05E9\u05DC\u05DA",
      formTitle: "\u05E7\u05D5\u05D3 \u05D0\u05D9\u05DE\u05D5\u05EA",
      resendButton: "\u05E9\u05DC\u05D7 \u05E7\u05D5\u05D3 \u05E9\u05D5\u05D1",
      subtitle: "\u05DC\u05D4\u05DE\u05E9\u05D9\u05DA \u05D0\u05DC {{applicationName}}",
      title: "\u05D0\u05DE\u05EA \u05D0\u05EA \u05DE\u05E1\u05E4\u05E8 \u05D4\u05D8\u05DC\u05E4\u05D5\u05DF \u05E9\u05DC\u05DA"
    },
    restrictedAccess: {
      actionLink: void 0,
      actionText: void 0,
      blockButton__emailSupport: void 0,
      blockButton__joinWaitlist: void 0,
      subtitle: void 0,
      subtitleWaitlist: void 0,
      title: void 0
    },
    start: {
      actionLink: "\u05D4\u05EA\u05D7\u05D1\u05E8",
      actionLink__use_email: void 0,
      actionLink__use_phone: void 0,
      actionText: "\u05D9\u05E9 \u05DC\u05DA \u05D7\u05E9\u05D1\u05D5\u05DF?",
      alternativePhoneCodeProvider: {
        actionLink: void 0,
        label: void 0,
        subtitle: void 0,
        title: void 0
      },
      subtitle: "\u05DC\u05D4\u05DE\u05E9\u05D9\u05DA \u05D0\u05DC {{applicationName}}",
      subtitleCombined: "\u05DC\u05D4\u05DE\u05E9\u05D9\u05DA \u05D0\u05DC {{applicationName}}",
      title: "\u05E6\u05D5\u05E8 \u05D0\u05EA \u05D4\u05D7\u05E9\u05D1\u05D5\u05DF \u05E9\u05DC\u05DA",
      titleCombined: "\u05E6\u05D5\u05E8 \u05D0\u05EA \u05D4\u05D7\u05E9\u05D1\u05D5\u05DF \u05E9\u05DC\u05DA"
    }
  },
  socialButtonsBlockButton: "\u05D4\u05DE\u05E9\u05DA \u05E2\u05DD {{provider|titleize}}",
  socialButtonsBlockButtonManyInView: "{{provider|titleize}}",
  taskChooseOrganization: {
    chooseOrganization: {
      action__createOrganization: void 0,
      action__invitationAccept: void 0,
      action__suggestionsAccept: void 0,
      subtitle: void 0,
      suggestionsAcceptedLabel: void 0,
      title: void 0
    },
    createOrganization: {
      formButtonReset: void 0,
      formButtonSubmit: void 0,
      formFieldInputPlaceholder__name: void 0,
      formFieldInputPlaceholder__slug: void 0,
      formFieldLabel__name: void 0,
      formFieldLabel__slug: void 0,
      subtitle: void 0,
      title: void 0
    },
    signOut: {
      actionLink: void 0,
      actionText: void 0
    }
  },
  unstable__errors: {
    already_a_member_in_organization: "{{email}} \u05DB\u05D1\u05E8 \u05D7\u05D1\u05E8 \u05D1\u05D0\u05E8\u05D2\u05D5\u05DF",
    captcha_invalid: "\u05D4\u05D4\u05E8\u05E9\u05DE\u05D4 \u05E0\u05DB\u05E9\u05DC\u05D4 \u05E2\u05E7\u05D1 \u05DB\u05E9\u05DC \u05D1\u05D0\u05D9\u05DE\u05D5\u05EA \u05D4\u05D0\u05D1\u05D8\u05D7\u05D4. \u05D0\u05E0\u05D0 \u05E8\u05E2\u05E0\u05DF \u05D0\u05EA \u05D4\u05D3\u05E3 \u05D5\u05E0\u05E1\u05D4 \u05E9\u05D5\u05D1, \u05D0\u05D5 \u05E4\u05E0\u05D4 \u05DC\u05EA\u05DE\u05D9\u05DB\u05D4 \u05DC\u05E2\u05D6\u05E8\u05D4 \u05E0\u05D5\u05E1\u05E4\u05EA.",
    captcha_unavailable: "\u05D4\u05D4\u05E8\u05E9\u05DE\u05D4 \u05E0\u05DB\u05E9\u05DC\u05D4 \u05E2\u05E7\u05D1 \u05DB\u05E9\u05DC \u05D1\u05D0\u05D9\u05DE\u05D5\u05EA \u05E0\u05D2\u05D3 \u05D1\u05D5\u05D8\u05D9\u05DD. \u05D0\u05E0\u05D0 \u05E8\u05E2\u05E0\u05DF \u05D0\u05EA \u05D4\u05D3\u05E3 \u05D5\u05E0\u05E1\u05D4 \u05E9\u05D5\u05D1, \u05D0\u05D5 \u05E4\u05E0\u05D4 \u05DC\u05EA\u05DE\u05D9\u05DB\u05D4 \u05DC\u05E2\u05D6\u05E8\u05D4 \u05E0\u05D5\u05E1\u05E4\u05EA.",
    form_code_incorrect: void 0,
    form_identifier_exists__email_address: "\u05DB\u05EA\u05D5\u05D1\u05EA \u05D4\u05DE\u05D9\u05D9\u05DC \u05D4\u05D6\u05D0\u05EA \u05DB\u05D1\u05E8 \u05EA\u05E4\u05D5\u05E1\u05D4. \u05D0\u05E0\u05D0 \u05E0\u05E1\u05D4 \u05D0\u05D7\u05E8\u05EA.",
    form_identifier_exists__phone_number: "\u05DE\u05E1\u05E4\u05E8 \u05D4\u05D8\u05DC\u05E4\u05D5\u05DF \u05D4\u05D6\u05D4 \u05DB\u05D1\u05E8 \u05EA\u05E4\u05D5\u05E1. \u05D0\u05E0\u05D0 \u05E0\u05E1\u05D4 \u05DE\u05E1\u05E4\u05E8 \u05D0\u05D7\u05E8.",
    form_identifier_exists__username: "\u05E9\u05DD \u05D4\u05DE\u05E9\u05EA\u05DE\u05E9 \u05D4\u05D6\u05D4 \u05DB\u05D1\u05E8 \u05EA\u05E4\u05D5\u05E1. \u05D0\u05E0\u05D0 \u05E0\u05E1\u05D4 \u05E9\u05DD \u05DE\u05E9\u05EA\u05DE\u05E9 \u05D0\u05D7\u05E8",
    form_identifier_not_found: "\u05DC\u05D0 \u05E0\u05D9\u05EA\u05DF \u05DC\u05DE\u05E6\u05D5\u05D0 \u05D7\u05E9\u05D1\u05D5\u05DF \u05E2\u05DD \u05D0\u05DC\u05D5 \u05D4\u05E4\u05E8\u05D8\u05D9\u05DD.",
    form_param_format_invalid: void 0,
    form_param_format_invalid__email_address: "\u05DB\u05EA\u05D5\u05D1\u05EA \u05D4\u05D0\u05D9\u05DE\u05D9\u05D9\u05DC \u05D7\u05D9\u05D9\u05D1\u05EA \u05DC\u05D4\u05D9\u05D5\u05EA \u05DB\u05EA\u05D5\u05D1\u05EA \u05D0\u05D9\u05DE\u05D9\u05D9\u05DC \u05EA\u05E7\u05D9\u05E0\u05D4.",
    form_param_format_invalid__phone_number: "\u05DE\u05E1\u05E4\u05E8 \u05D4\u05D8\u05DC\u05E4\u05D5\u05DF \u05D7\u05D9\u05D9\u05D1 \u05DC\u05D4\u05D9\u05D5\u05EA \u05D1\u05E4\u05D5\u05E8\u05DE\u05D8 \u05D1\u05D9\u05E0\u05DC\u05D0\u05D5\u05DE\u05D9 \u05EA\u05E7\u05D9\u05DF.",
    form_param_max_length_exceeded__first_name: "\u05E9\u05DD \u05E4\u05E8\u05D8\u05D9 \u05DC\u05D0 \u05E6\u05E8\u05D9\u05DA \u05DC\u05E2\u05DC\u05D5\u05EA \u05E2\u05DC 256 \u05EA\u05D5\u05D5\u05D9\u05DD.",
    form_param_max_length_exceeded__last_name: "\u05E9\u05DD \u05DE\u05E9\u05E4\u05D7\u05D4 \u05DC\u05D0 \u05E6\u05E8\u05D9\u05DA \u05DC\u05E2\u05DC\u05D5\u05EA \u05E2\u05DC 256 \u05EA\u05D5\u05D5\u05D9\u05DD.",
    form_param_max_length_exceeded__name: "\u05E9\u05DD \u05DC\u05D0 \u05E6\u05E8\u05D9\u05DA \u05DC\u05E2\u05DC\u05D5\u05EA \u05E2\u05DC 256 \u05EA\u05D5\u05D5\u05D9\u05DD.",
    form_param_nil: void 0,
    form_param_type_invalid: void 0,
    form_param_type_invalid__email_address: void 0,
    form_param_type_invalid__phone_number: void 0,
    form_param_value_invalid: void 0,
    form_password_incorrect: void 0,
    form_password_length_too_short: void 0,
    form_password_not_strong_enough: "\u05D4\u05E1\u05D9\u05E1\u05DE\u05D4 \u05E9\u05DC\u05DA \u05D0\u05D9\u05E0\u05D4 \u05DE\u05E1\u05E4\u05D9\u05E7 \u05D7\u05D6\u05E7\u05D4.",
    form_password_pwned: "\u05D4\u05E1\u05D9\u05E1\u05DE\u05D4 \u05D4\u05D6\u05D5 \u05E0\u05DE\u05E6\u05D0\u05D4 \u05DB\u05D7\u05DC\u05E7 \u05DE\u05D4\u05E4\u05E8\u05D8\u05D9\u05DD \u05E9\u05E0\u05D7\u05E9\u05E4\u05D5 \u05D1\u05D4\u05E4\u05E8\u05EA \u05E0\u05EA\u05D5\u05E0\u05D9\u05DD \u05D5\u05DC\u05D0 \u05E0\u05D9\u05EA\u05DF \u05DC\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05D4, \u05E0\u05E1\u05D4 \u05E1\u05D9\u05E1\u05DE\u05D4 \u05D0\u05D7\u05E8\u05EA \u05D1\u05DE\u05E7\u05D5\u05DD.",
    form_password_pwned__sign_in: "\u05D4\u05E1\u05D9\u05E1\u05DE\u05D4 \u05D4\u05D6\u05D5 \u05E0\u05DE\u05E6\u05D0\u05D4 \u05DB\u05D7\u05DC\u05E7 \u05DE\u05D4\u05E4\u05E8\u05D8\u05D9\u05DD \u05E9\u05E0\u05D7\u05E9\u05E4\u05D5 \u05D1\u05D4\u05E4\u05E8\u05EA \u05E0\u05EA\u05D5\u05E0\u05D9\u05DD \u05D5\u05DC\u05D0 \u05E0\u05D9\u05EA\u05DF \u05DC\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05D4, \u05D0\u05E0\u05D0 \u05D1\u05E6\u05E2 \u05D0\u05D9\u05EA\u05D7\u05D5\u05DC \u05DC\u05E1\u05D9\u05E1\u05DE\u05D4 \u05E9\u05DC\u05DA.",
    form_password_size_in_bytes_exceeded: "\u05D4\u05E1\u05D9\u05E1\u05DE\u05D4 \u05E9\u05DC\u05DA \u05D7\u05D5\u05E8\u05D2\u05EA \u05DE\u05DE\u05E1\u05E4\u05E8 \u05D4\u05D1\u05D9\u05D9\u05D8\u05D9\u05DD \u05D4\u05DE\u05E8\u05D1\u05D9 \u05D4\u05DE\u05D5\u05EA\u05E8, \u05E0\u05E1\u05D4 \u05DC\u05E7\u05E6\u05E8 \u05D0\u05D5\u05EA\u05D4 \u05D0\u05D5 \u05DC\u05D4\u05E1\u05D9\u05E8 \u05DB\u05DE\u05D4 \u05EA\u05D5\u05D5\u05D9\u05DD \u05DE\u05D9\u05D5\u05D7\u05D3\u05D9\u05DD.",
    form_password_validation_failed: "\u05E1\u05D9\u05E1\u05DE\u05D4 \u05E9\u05D2\u05D5\u05D9\u05D4",
    form_username_invalid_character: void 0,
    form_username_invalid_length: void 0,
    identification_deletion_failed: "\u05DC\u05D0 \u05E0\u05D9\u05EA\u05DF \u05DC\u05DE\u05D7\u05D5\u05E7 \u05D0\u05EA \u05D4\u05D6\u05D9\u05D4\u05D5\u05D9 \u05D4\u05D0\u05D7\u05E8\u05D5\u05DF \u05E9\u05DC\u05DA.",
    not_allowed_access: "\u05D4\u05D0\u05D9\u05DE\u05D9\u05D9\u05DC \u05D0\u05D5 \u05DE\u05E1\u05E4\u05E8 \u05D4\u05D8\u05DC\u05E4\u05D5\u05DF \u05D0\u05D9\u05E0\u05D5 \u05DE\u05D5\u05EA\u05E8 \u05DC\u05D4\u05E8\u05E9\u05DE\u05D4. \u05D6\u05D4 \u05E2\u05E9\u05D5\u05D9 \u05DC\u05D4\u05D9\u05D5\u05EA \u05D1\u05D2\u05DC\u05DC \u05D4\u05E9\u05D9\u05DE\u05D5\u05E9 \u05D1-'+', '=', '#' \u05D0\u05D5 '.' \u05D1\u05DB\u05EA\u05D5\u05D1\u05EA \u05D4\u05D0\u05D9\u05DE\u05D9\u05D9\u05DC \u05E9\u05DC\u05DA, \u05D4\u05E9\u05D9\u05DE\u05D5\u05E9 \u05D1\u05EA\u05D7\u05D5\u05DD \u05D4\u05DE\u05D7\u05D5\u05D1\u05E8 \u05DC\u05E9\u05D9\u05E8\u05D5\u05EA \u05D0\u05D9\u05DE\u05D9\u05D9\u05DC \u05D6\u05DE\u05E0\u05D9 \u05D0\u05D5 \u05D4\u05E4\u05E8\u05E1\u05D8 \u05D1\u05DB\u05EA\u05D5\u05D1\u05EA \u05D4\u05D0\u05D9\u05DE\u05D9\u05D9\u05DC \u05E9\u05DC\u05DA. \u05D0\u05DD \u05DE\u05D7\u05D6\u05E8\u05D9\u05DD \u05E9\u05D2\u05D9\u05D0\u05D4, \u05E0\u05D0 \u05DC\u05D9\u05E6\u05D5\u05E8 \u05E7\u05E9\u05E8 \u05E2\u05DD \u05EA\u05DE\u05D9\u05DB\u05D4.",
    organization_domain_blocked: "\u05D4\u05D3\u05D5\u05DE\u05D9\u05D9\u05DF \u05E9\u05DC \u05E1\u05E4\u05E7 \u05D4\u05D0\u05D9\u05DE\u05D9\u05D9\u05DC \u05D4\u05D6\u05D4 \u05D7\u05E1\u05D5\u05DD. \u05D0\u05E0\u05D0 \u05E0\u05E1\u05D4 \u05D0\u05D7\u05D3 \u05E9\u05D5\u05E0\u05D4.",
    organization_domain_common: "\u05D4\u05D3\u05D5\u05DE\u05D9\u05D9\u05DF \u05E9\u05DC \u05E1\u05E4\u05E7 \u05D4\u05D0\u05D9\u05DE\u05D9\u05D9\u05DC \u05D4\u05D6\u05D4 \u05E0\u05E4\u05D5\u05E5. \u05D0\u05E0\u05D0 \u05E0\u05E1\u05D4 \u05D0\u05D7\u05D3 \u05E9\u05D5\u05E0\u05D4.",
    organization_domain_exists_for_enterprise_connection: void 0,
    organization_membership_quota_exceeded: "\u05D4\u05D2\u05E2\u05EA \u05DC\u05DE\u05D2\u05D1\u05DC\u05EA \u05D4\u05D7\u05D1\u05E8\u05D5\u05EA \u05D1\u05D0\u05E8\u05D2\u05D5\u05DF, \u05DB\u05D5\u05DC\u05DC \u05D4\u05D6\u05DE\u05E0\u05D5\u05EA \u05D9\u05D5\u05E6\u05D0\u05D5\u05EA \u05D3\u05D5\u05E4\u05DF.",
    organization_minimum_permissions_needed: "\u05D7\u05D9\u05D9\u05D1 \u05DC\u05D4\u05D9\u05D5\u05EA \u05D7\u05D1\u05E8 \u05D0\u05E8\u05D2\u05D5\u05DF \u05D0\u05D7\u05D3 \u05DC\u05E4\u05D7\u05D5\u05EA \u05E2\u05DD \u05D4\u05D4\u05E8\u05E9\u05D0\u05D5\u05EA \u05D4\u05DE\u05D9\u05E0\u05D9\u05DE\u05DC\u05D9\u05D5\u05EA \u05D4\u05E0\u05D3\u05E8\u05E9\u05D5\u05EA.",
    organization_not_found_or_unauthorized: void 0,
    organization_not_found_or_unauthorized_with_create_organization_disabled: void 0,
    passkey_already_exists: "\u05DE\u05E4\u05EA\u05D7 \u05D4\u05E1\u05D9\u05E1\u05DE\u05D4 \u05DB\u05D1\u05E8 \u05E8\u05E9\u05D5\u05DD \u05D1\u05DE\u05DB\u05E9\u05D9\u05E8 \u05D6\u05D4.",
    passkey_not_supported: "\u05DE\u05E4\u05EA\u05D7\u05D5\u05EA \u05E1\u05D9\u05E1\u05DE\u05D4 \u05D0\u05D9\u05E0\u05DD \u05E0\u05EA\u05DE\u05DB\u05D9\u05DD \u05D1\u05DE\u05DB\u05E9\u05D9\u05E8 \u05D6\u05D4.",
    passkey_pa_not_supported: "\u05D4\u05D4\u05E8\u05E9\u05DE\u05D4 \u05D3\u05D5\u05E8\u05E9\u05EA \u05DE\u05D0\u05DE\u05EA \u05E4\u05DC\u05D8\u05E4\u05D5\u05E8\u05DE\u05D4 \u05D0\u05DA \u05D4\u05DE\u05DB\u05E9\u05D9\u05E8 \u05D0\u05D9\u05E0\u05D5 \u05EA\u05D5\u05DE\u05DA \u05D1\u05DB\u05DA.",
    passkey_registration_cancelled: "\u05E8\u05D9\u05E9\u05D5\u05DD \u05DE\u05E4\u05EA\u05D7 \u05D4\u05E1\u05D9\u05E1\u05DE\u05D4 \u05D1\u05D5\u05D8\u05DC \u05D0\u05D5 \u05E4\u05D2 \u05D4\u05D6\u05DE\u05DF \u05D4\u05E7\u05E6\u05D5\u05D1.",
    passkey_retrieval_cancelled: "\u05D0\u05D9\u05DE\u05D5\u05EA \u05DE\u05E4\u05EA\u05D7 \u05D4\u05E1\u05D9\u05E1\u05DE\u05D4 \u05D1\u05D5\u05D8\u05DC \u05D0\u05D5 \u05E4\u05D2 \u05D4\u05D6\u05DE\u05DF \u05D4\u05E7\u05E6\u05D5\u05D1.",
    passwordComplexity: {
      maximumLength: "\u05E4\u05D7\u05D5\u05EA \u05DE-{{length}} \u05EA\u05D5\u05D5\u05D9\u05DD",
      minimumLength: "{{length}} \u05EA\u05D5\u05D5\u05D9\u05DD \u05D0\u05D5 \u05D9\u05D5\u05EA\u05E8",
      requireLowercase: "\u05D0\u05D5\u05EA \u05E7\u05D8\u05E0\u05D4",
      requireNumbers: "\u05DE\u05E1\u05E4\u05E8",
      requireSpecialCharacter: "\u05EA\u05D5 \u05DE\u05D9\u05D5\u05D7\u05D3",
      requireUppercase: "\u05D0\u05D5\u05EA \u05D2\u05D3\u05D5\u05DC\u05D4",
      sentencePrefix: "\u05D4\u05E1\u05D9\u05E1\u05DE\u05D4 \u05E9\u05DC\u05DA \u05D7\u05D9\u05D9\u05D1\u05EA \u05DC\u05D4\u05DB\u05D9\u05DC"
    },
    phone_number_exists: "\u05DE\u05E1\u05E4\u05E8 \u05D4\u05D8\u05DC\u05E4\u05D5\u05DF \u05D4\u05D6\u05D4 \u05DB\u05D1\u05E8 \u05D1\u05E9\u05D9\u05DE\u05D5\u05E9. \u05D0\u05E0\u05D0 \u05E0\u05E1\u05D4 \u05DE\u05E1\u05E4\u05E8 \u05D0\u05D7\u05E8.",
    session_exists: "\u05D0\u05EA\u05D4 \u05DB\u05D1\u05E8 \u05DE\u05D7\u05D5\u05D1\u05E8 \u05DC\u05D7\u05E9\u05D1\u05D5\u05DF.",
    web3_missing_identifier: void 0,
    zxcvbn: {
      couldBeStronger: "\u05D4\u05E1\u05D9\u05E1\u05DE\u05D4 \u05E9\u05DC\u05DA \u05EA\u05E7\u05E4\u05D4, \u05D0\u05DA \u05D9\u05DB\u05D5\u05DC\u05D4 \u05DC\u05D4\u05D9\u05D5\u05EA \u05D7\u05D6\u05E7\u05D4 \u05D9\u05D5\u05EA\u05E8. \u05E0\u05E1\u05D4 \u05DC\u05D4\u05D5\u05E1\u05D9\u05E3 \u05D9\u05D5\u05EA\u05E8 \u05EA\u05D5\u05D5\u05D9\u05DD.",
      goodPassword: "\u05E2\u05D1\u05D5\u05D3\u05D4 \u05D8\u05D5\u05D1\u05D4. \u05D6\u05D5 \u05E1\u05D9\u05E1\u05DE\u05D4 \u05DE\u05E6\u05D5\u05D9\u05E0\u05EA.",
      notEnough: "\u05D4\u05E1\u05D9\u05E1\u05DE\u05D4 \u05E9\u05DC\u05DA \u05D0\u05D9\u05E0\u05D4 \u05DE\u05E1\u05E4\u05D9\u05E7 \u05D7\u05D6\u05E7\u05D4.",
      suggestions: {
        allUppercase: "\u05D4\u05D2\u05D3\u05DC \u05DB\u05DE\u05D4, \u05D0\u05DA \u05DC\u05D0 \u05D0\u05EA \u05DB\u05DC \u05D4\u05D0\u05D5\u05EA\u05D9\u05D5\u05EA.",
        anotherWord: "\u05D4\u05D5\u05E1\u05E3 \u05E2\u05D5\u05D3 \u05DE\u05D9\u05DC\u05D9\u05DD \u05E9\u05D4\u05DF \u05E4\u05D7\u05D5\u05EA \u05E0\u05E4\u05D5\u05E6\u05D5\u05EA.",
        associatedYears: "\u05D4\u05D9\u05DE\u05E0\u05E2 \u05DE\u05E9\u05E0\u05D9\u05DD \u05E9\u05E7\u05E9\u05D5\u05E8\u05D5\u05EA \u05D0\u05DC\u05D9\u05DA.",
        capitalization: "\u05D4\u05D2\u05D3\u05DC \u05D9\u05D5\u05EA\u05E8 \u05DE\u05D0\u05D5\u05EA \u05D0\u05D7\u05EA.",
        dates: "\u05D4\u05DE\u05E0\u05E2 \u05DE\u05EA\u05D0\u05E8\u05D9\u05DB\u05D9\u05DD \u05D5\u05E9\u05E0\u05D9\u05DD \u05E9\u05E7\u05E9\u05D5\u05E8\u05D9\u05DD \u05D0\u05DC\u05D9\u05DA.",
        l33t: "\u05D4\u05DE\u05E0\u05E2 \u05DE\u05D4\u05D7\u05DC\u05E4\u05D5\u05EA \u05EA\u05D5\u05D5\u05D9\u05DD \u05E6\u05E4\u05D5\u05D9\u05D5\u05EA \u05DB\u05DE\u05D5 '@' \u05D1\u05DE\u05E7\u05D5\u05DD 'a'.",
        longerKeyboardPattern: "\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05D3\u05E4\u05D5\u05E1\u05D9 \u05DE\u05E7\u05DC\u05D3\u05EA \u05D0\u05E8\u05D5\u05DB\u05D9\u05DD \u05D9\u05D5\u05EA\u05E8 \u05D5\u05E9\u05E0\u05D4 \u05D0\u05EA \u05DB\u05D9\u05D5\u05D5\u05DF \u05D4\u05D4\u05E7\u05DC\u05D3\u05D4 \u05DE\u05E1\u05E4\u05E8 \u05E4\u05E2\u05DE\u05D9\u05DD.",
        noNeed: "\u05D0\u05EA\u05D4 \u05D9\u05DB\u05D5\u05DC \u05DC\u05D9\u05E6\u05D5\u05E8 \u05E1\u05D9\u05E1\u05DE\u05D0\u05D5\u05EA \u05D7\u05D6\u05E7\u05D5\u05EA \u05DC\u05DC\u05D0 \u05E9\u05D9\u05DE\u05D5\u05E9 \u05D1\u05E1\u05DE\u05DC\u05D9\u05DD, \u05DE\u05E1\u05E4\u05E8\u05D9\u05DD, \u05D0\u05D5 \u05D0\u05D5\u05EA\u05D9\u05D5\u05EA \u05D2\u05D3\u05D5\u05DC\u05D5\u05EA.",
        pwned: "\u05D0\u05DD \u05D0\u05EA\u05D4 \u05DE\u05E9\u05EA\u05DE\u05E9 \u05D1\u05E1\u05D9\u05E1\u05DE\u05D4 \u05D6\u05D5 \u05D1\u05DE\u05E7\u05D5\u05DE\u05D5\u05EA \u05D0\u05D7\u05E8\u05D9\u05DD, \u05E2\u05DC\u05D9\u05DA \u05DC\u05E9\u05E0\u05D5\u05EA\u05D4.",
        recentYears: "\u05D4\u05D9\u05DE\u05E0\u05E2 \u05DE\u05E9\u05E0\u05D9\u05DD \u05D0\u05D7\u05E8\u05D5\u05E0\u05D5\u05EA.",
        repeated: "\u05D4\u05D9\u05DE\u05E0\u05E2 \u05DE\u05DE\u05D9\u05DC\u05D9\u05DD \u05D5\u05EA\u05D5\u05D5\u05D9\u05DD \u05DE\u05D5\u05D7\u05D6\u05E8\u05D9\u05DD.",
        reverseWords: "\u05D4\u05D9\u05DE\u05E0\u05E2 \u05DE\u05DB\u05EA\u05D9\u05D1\u05D4 \u05D4\u05E4\u05D5\u05DB\u05D4 \u05E9\u05DC \u05DE\u05D9\u05DC\u05D9\u05DD \u05E0\u05E4\u05D5\u05E6\u05D5\u05EA.",
        sequences: "\u05D4\u05DE\u05E0\u05E2 \u05DE\u05E8\u05E6\u05E4\u05D5\u05EA \u05EA\u05D5\u05D5\u05D9\u05DD \u05E0\u05E4\u05D5\u05E6\u05D5\u05EA.",
        useWords: "\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05DE\u05D9\u05DC\u05D9\u05DD \u05DE\u05E8\u05D5\u05D1\u05D5\u05EA, \u05D0\u05DA \u05D4\u05D9\u05DE\u05E0\u05E2 \u05DE\u05D1\u05D9\u05D8\u05D5\u05D9\u05D9\u05DD \u05E0\u05E4\u05D5\u05E6\u05D9\u05DD."
      },
      warnings: {
        common: "\u05D6\u05D5 \u05E1\u05D9\u05E1\u05DE\u05D4 \u05E9\u05E0\u05DE\u05E6\u05D0\u05EA \u05D1\u05E9\u05D9\u05DE\u05D5\u05E9 \u05E0\u05E4\u05D5\u05E5.",
        commonNames: "\u05E9\u05DE\u05D5\u05EA \u05E0\u05E4\u05D5\u05E6\u05D9\u05DD \u05D5\u05E9\u05DE\u05D5\u05EA \u05DE\u05E9\u05E4\u05D7\u05D4 \u05E7\u05DC\u05D9\u05DD \u05DC\u05E0\u05D9\u05D7\u05D5\u05E9.",
        dates: "\u05EA\u05D0\u05E8\u05D9\u05DB\u05D9\u05DD \u05E7\u05DC\u05D9\u05DD \u05DC\u05E0\u05D9\u05D7\u05D5\u05E9.",
        extendedRepeat: '\u05D3\u05E4\u05D5\u05E1\u05D9\u05DD \u05DE\u05D5\u05D7\u05D6\u05E8\u05D9\u05DD \u05E9\u05DC \u05EA\u05D5\u05D5\u05D9\u05DD \u05DB\u05DE\u05D5 "abcabcabc" \u05E7\u05DC\u05D9\u05DD \u05DC\u05E0\u05D9\u05D7\u05D5\u05E9.',
        keyPattern: "\u05D3\u05E4\u05D5\u05E1\u05D9\u05DD \u05E7\u05E6\u05E8\u05D9\u05DD \u05E9\u05DC \u05DE\u05E7\u05DC\u05D3\u05EA \u05E7\u05DC\u05D9\u05DD \u05DC\u05E0\u05D9\u05D7\u05D5\u05E9.",
        namesByThemselves: "\u05E9\u05DE\u05D5\u05EA \u05D1\u05D5\u05D3\u05D3\u05D9\u05DD \u05D0\u05D5 \u05E9\u05DE\u05D5\u05EA \u05DE\u05E9\u05E4\u05D7\u05D4 \u05E7\u05DC\u05D9\u05DD \u05DC\u05E0\u05D9\u05D7\u05D5\u05E9.",
        pwned: "\u05D4\u05E1\u05D9\u05E1\u05DE\u05D4 \u05E9\u05DC\u05DA \u05D4\u05D5\u05D7\u05E9\u05E4\u05D4 \u05D1\u05DE\u05D4\u05DC\u05DA \u05D4\u05E4\u05E8\u05EA \u05E0\u05EA\u05D5\u05E0\u05D9\u05DD \u05D1\u05D0\u05D9\u05E0\u05D8\u05E8\u05E0\u05D8.",
        recentYears: "\u05E9\u05E0\u05D9\u05DD \u05D0\u05D7\u05E8\u05D5\u05E0\u05D5\u05EA \u05E7\u05DC\u05D5\u05EA \u05DC\u05E0\u05D9\u05D7\u05D5\u05E9.",
        sequences: '\u05E8\u05E6\u05E4\u05D5\u05EA \u05EA\u05D5\u05D5\u05D9\u05DD \u05E0\u05E4\u05D5\u05E6\u05D5\u05EA \u05DB\u05DE\u05D5 "abc" \u05E7\u05DC\u05D5\u05EA \u05DC\u05E0\u05D9\u05D7\u05D5\u05E9.',
        similarToCommon: "\u05D6\u05D5 \u05E1\u05D9\u05E1\u05DE\u05D4 \u05D3\u05D5\u05DE\u05D4 \u05DC\u05E1\u05D9\u05E1\u05DE\u05D4 \u05E9\u05E0\u05DE\u05E6\u05D0\u05EA \u05D1\u05E9\u05D9\u05DE\u05D5\u05E9 \u05E0\u05E4\u05D5\u05E5.",
        simpleRepeat: '\u05EA\u05D5\u05D5\u05D9\u05DD \u05DE\u05D5\u05D7\u05D6\u05E8\u05D9\u05DD \u05DB\u05DE\u05D5 "aaa" \u05E7\u05DC\u05D9\u05DD \u05DC\u05E0\u05D9\u05D7\u05D5\u05E9.',
        straightRow: "\u05E9\u05D5\u05E8\u05D5\u05EA \u05D9\u05E9\u05E8\u05D5\u05EA \u05E9\u05DC \u05DE\u05E7\u05E9\u05D9\u05DD \u05D1\u05DE\u05E7\u05DC\u05D3\u05EA \u05E9\u05DC\u05DA \u05E7\u05DC\u05D5\u05EA \u05DC\u05E0\u05D9\u05D7\u05D5\u05E9.",
        topHundred: "\u05D6\u05D5 \u05E1\u05D9\u05E1\u05DE\u05D4 \u05E9\u05E0\u05DE\u05E6\u05D0\u05EA \u05D1\u05E9\u05D9\u05DE\u05D5\u05E9 \u05EA\u05DB\u05D5\u05E3.",
        topTen: "\u05D6\u05D5 \u05E1\u05D9\u05E1\u05DE\u05D4 \u05E9\u05E0\u05DE\u05E6\u05D0\u05EA \u05D1\u05E9\u05D9\u05DE\u05D5\u05E9 \u05DB\u05D1\u05D3.",
        userInputs: "\u05D0\u05D9\u05DF \u05DC\u05D4\u05D6\u05D9\u05DF \u05E0\u05EA\u05D5\u05E0\u05D9\u05DD \u05D0\u05D9\u05E9\u05D9\u05D9\u05DD \u05D0\u05D5 \u05E7\u05E9\u05D5\u05E8\u05D9\u05DD \u05DC\u05D3\u05E3.",
        wordByItself: "\u05DE\u05D9\u05DC\u05D9\u05DD \u05D1\u05D5\u05D3\u05D3\u05D5\u05EA \u05E7\u05DC\u05D5\u05EA \u05DC\u05E0\u05D9\u05D7\u05D5\u05E9."
      }
    }
  },
  userButton: {
    action__addAccount: "\u05D4\u05D5\u05E1\u05E3 \u05D7\u05E9\u05D1\u05D5\u05DF",
    action__closeUserMenu: void 0,
    action__manageAccount: "\u05E0\u05D4\u05DC \u05D7\u05E9\u05D1\u05D5\u05DF",
    action__openUserMenu: void 0,
    action__signOut: "\u05D4\u05EA\u05E0\u05EA\u05E7",
    action__signOutAll: "\u05D4\u05EA\u05E0\u05EA\u05E7 \u05DE\u05DB\u05DC \u05D4\u05D7\u05E9\u05D1\u05D5\u05E0\u05D5\u05EA"
  },
  userProfile: {
    apiKeysPage: {
      title: void 0
    },
    backupCodePage: {
      actionLabel__copied: "\u05D4\u05D5\u05E2\u05EA\u05E7!",
      actionLabel__copy: "\u05D4\u05E2\u05EA\u05E7 \u05D4\u05DB\u05DC",
      actionLabel__download: "\u05D4\u05D5\u05E8\u05D3 .txt",
      actionLabel__print: "\u05D4\u05D3\u05E4\u05E1",
      infoText1: "\u05E7\u05D5\u05D3\u05D9 \u05D2\u05D9\u05D1\u05D5\u05D9 \u05D9\u05D4\u05D9\u05D5 \u05DE\u05D5\u05E4\u05E2\u05DC\u05D9\u05DD \u05DC\u05D7\u05E9\u05D1\u05D5\u05DF \u05D6\u05D4.",
      infoText2: "\u05E9\u05DE\u05D5\u05E8 \u05D0\u05EA \u05E7\u05D5\u05D3\u05D9 \u05D4\u05D2\u05D9\u05D1\u05D5\u05D9 \u05D1\u05E1\u05D5\u05D3 \u05D5\u05D0\u05D7\u05E1\u05DF \u05D0\u05D5\u05EA\u05DD \u05D1\u05D1\u05D8\u05D7\u05D4. \u05EA\u05D5\u05DB\u05DC \u05DC\u05D7\u05D5\u05DC\u05DC \u05DE\u05D7\u05D3\u05E9 \u05E7\u05D5\u05D3\u05D9 \u05D2\u05D9\u05D1\u05D5\u05D9 \u05D0\u05DD \u05D0\u05EA\u05D4 \u05D7\u05D5\u05E9\u05D3 \u05E9\u05D4\u05DD \u05E0\u05E4\u05D2\u05E2\u05D5.",
      subtitle__codelist: "\u05D0\u05D7\u05E1\u05DF \u05D0\u05D5\u05EA\u05DD \u05D1\u05D1\u05D8\u05D7\u05D4 \u05D5\u05E9\u05DE\u05D5\u05E8 \u05E2\u05DC\u05D9\u05D4\u05DD \u05D1\u05E1\u05D5\u05D3.",
      successMessage: "\u05E7\u05D5\u05D3\u05D9 \u05D4\u05D2\u05D9\u05D1\u05D5\u05D9 \u05DE\u05D5\u05E4\u05E2\u05DC\u05D9\u05DD \u05DB\u05E2\u05EA. \u05EA\u05D5\u05DB\u05DC \u05DC\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05D0\u05D7\u05D3 \u05DE\u05D0\u05DC\u05D4 \u05DB\u05D3\u05D9 \u05DC\u05D4\u05EA\u05D7\u05D1\u05E8 \u05DC\u05D7\u05E9\u05D1\u05D5\u05DF \u05E9\u05DC\u05DA, \u05D0\u05DD \u05D0\u05EA\u05D4 \u05DE\u05D0\u05D1\u05D3 \u05D2\u05D9\u05E9\u05D4 \u05DC\u05DE\u05DB\u05E9\u05D9\u05E8 \u05D4\u05D0\u05D9\u05DE\u05D5\u05EA \u05E9\u05DC\u05DA. \u05DB\u05DC \u05E7\u05D5\u05D3 \u05D9\u05DB\u05D5\u05DC \u05DC\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05D5 \u05E8\u05E7 \u05E4\u05E2\u05DD \u05D0\u05D7\u05EA.",
      successSubtitle: "\u05EA\u05D5\u05DB\u05DC \u05DC\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05D0\u05D7\u05D3 \u05DE\u05D0\u05DC\u05D4 \u05DB\u05D3\u05D9 \u05DC\u05D4\u05EA\u05D7\u05D1\u05E8 \u05DC\u05D7\u05E9\u05D1\u05D5\u05DF \u05E9\u05DC\u05DA, \u05D0\u05DD \u05D0\u05EA\u05D4 \u05DE\u05D0\u05D1\u05D3 \u05D2\u05D9\u05E9\u05D4 \u05DC\u05DE\u05DB\u05E9\u05D9\u05E8 \u05D4\u05D0\u05D9\u05DE\u05D5\u05EA \u05E9\u05DC\u05DA.",
      title: "\u05D4\u05D5\u05E1\u05E3 \u05D0\u05D9\u05DE\u05D5\u05EA \u05E7\u05D5\u05D3 \u05D2\u05D9\u05D1\u05D5\u05D9",
      title__codelist: "\u05E7\u05D5\u05D3\u05D9 \u05D2\u05D9\u05D1\u05D5\u05D9"
    },
    billingPage: {
      paymentHistorySection: {
        empty: void 0,
        notFound: void 0,
        tableHeader__amount: void 0,
        tableHeader__date: void 0,
        tableHeader__status: void 0
      },
      paymentMethodsSection: {
        actionLabel__default: void 0,
        actionLabel__remove: void 0,
        add: void 0,
        addSubtitle: void 0,
        cancelButton: void 0,
        formButtonPrimary__add: void 0,
        formButtonPrimary__pay: void 0,
        payWithTestCardButton: void 0,
        removeMethod: {
          messageLine1: void 0,
          messageLine2: void 0,
          successMessage: void 0,
          title: void 0
        },
        title: void 0
      },
      start: {
        headerTitle__payments: void 0,
        headerTitle__plans: void 0,
        headerTitle__statements: void 0,
        headerTitle__subscriptions: void 0
      },
      statementsSection: {
        empty: void 0,
        itemCaption__paidForPlan: void 0,
        itemCaption__proratedCredit: void 0,
        itemCaption__subscribedAndPaidForPlan: void 0,
        notFound: void 0,
        tableHeader__amount: void 0,
        tableHeader__date: void 0,
        title: void 0,
        totalPaid: void 0
      },
      subscriptionsListSection: {
        actionLabel__manageSubscription: void 0,
        actionLabel__newSubscription: void 0,
        actionLabel__switchPlan: void 0,
        tableHeader__edit: void 0,
        tableHeader__plan: void 0,
        tableHeader__startDate: void 0,
        title: void 0
      },
      subscriptionsSection: {
        actionLabel__default: void 0
      },
      switchPlansSection: {
        title: void 0
      },
      title: void 0
    },
    connectedAccountPage: {
      formHint: "\u05D1\u05D7\u05E8 \u05E1\u05E4\u05E7 \u05DB\u05D3\u05D9 \u05DC\u05D7\u05D1\u05E8 \u05D0\u05EA \u05D4\u05D7\u05E9\u05D1\u05D5\u05DF \u05E9\u05DC\u05DA.",
      formHint__noAccounts: "\u05D0\u05D9\u05DF \u05E1\u05E4\u05E7\u05D9\u05DD \u05D6\u05DE\u05D9\u05E0\u05D9\u05DD \u05DC\u05D7\u05E9\u05D1\u05D5\u05E0\u05D5\u05EA \u05D7\u05D9\u05E6\u05D5\u05E0\u05D9\u05D9\u05DD.",
      removeResource: {
        messageLine1: "{{identifier}} \u05D9\u05D5\u05E1\u05E8 \u05DE\u05D4\u05D7\u05E9\u05D1\u05D5\u05DF \u05D4\u05D6\u05D4.",
        messageLine2: "\u05DC\u05D0 \u05EA\u05D5\u05DB\u05DC \u05D9\u05D5\u05EA\u05E8 \u05DC\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05D7\u05E9\u05D1\u05D5\u05DF \u05DE\u05D7\u05D5\u05D1\u05E8 \u05D6\u05D4 \u05D5\u05DB\u05DC \u05D4\u05EA\u05DB\u05D5\u05E0\u05D5\u05EA \u05D4\u05EA\u05DC\u05D5\u05D9\u05D5\u05EA \u05DC\u05D0 \u05D9\u05E2\u05D1\u05D3\u05D5.",
        successMessage: "{{connectedAccount}} \u05D4\u05D5\u05E1\u05E8 \u05DE\u05D4\u05D7\u05E9\u05D1\u05D5\u05DF \u05E9\u05DC\u05DA.",
        title: "\u05D4\u05E1\u05E8 \u05D7\u05E9\u05D1\u05D5\u05DF \u05DE\u05D7\u05D5\u05D1\u05E8"
      },
      socialButtonsBlockButton: "\u05D7\u05D1\u05E8 \u05D7\u05E9\u05D1\u05D5\u05DF {{provider|titleize}}",
      successMessage: "\u05D4\u05E1\u05E4\u05E7 \u05D4\u05EA\u05D5\u05D5\u05E1\u05E3 \u05DC\u05D7\u05E9\u05D1\u05D5\u05DF \u05E9\u05DC\u05DA",
      title: "\u05D4\u05D5\u05E1\u05E3 \u05D7\u05E9\u05D1\u05D5\u05DF \u05DE\u05D7\u05D5\u05D1\u05E8"
    },
    deletePage: {
      actionDescription: '\u05D4\u05E7\u05DC\u05D3 "\u05DE\u05D7\u05E7 \u05D7\u05E9\u05D1\u05D5\u05DF" \u05DC\u05DE\u05D8\u05D4 \u05DB\u05D3\u05D9 \u05DC\u05D4\u05DE\u05E9\u05D9\u05DA.',
      confirm: "\u05DE\u05D7\u05E7 \u05D7\u05E9\u05D1\u05D5\u05DF",
      messageLine1: "\u05D4\u05D0\u05DD \u05D0\u05EA\u05D4 \u05D1\u05D8\u05D5\u05D7 \u05E9\u05D1\u05E8\u05E6\u05D5\u05E0\u05DA \u05DC\u05DE\u05D7\u05D5\u05E7 \u05D0\u05EA \u05D4\u05D7\u05E9\u05D1\u05D5\u05DF \u05E9\u05DC\u05DA? \u05D7\u05DC\u05E7 \u05DE\u05D4\u05E0\u05EA\u05D5\u05E0\u05D9\u05DD \u05D4\u05DE\u05E9\u05D5\u05D9\u05DB\u05D9\u05DD \u05E2\u05E9\u05D5\u05D9\u05D9\u05DD \u05DC\u05D4\u05D9\u05E9\u05DE\u05E8. \u05DB\u05D3\u05D9 \u05DC\u05D1\u05E7\u05E9 \u05DE\u05D7\u05D9\u05E7\u05EA \u05E0\u05EA\u05D5\u05E0\u05D9\u05DD \u05DE\u05DC\u05D0\u05D4, \u05D0\u05E0\u05D0 \u05E6\u05D5\u05E8 \u05E7\u05E9\u05E8 \u05E2\u05DD \u05D4\u05EA\u05DE\u05D9\u05DB\u05D4.",
      messageLine2: "\u05E4\u05E2\u05D5\u05DC\u05D4 \u05D6\u05D5 \u05D4\u05D9\u05D0 \u05E1\u05D5\u05E4\u05D9\u05EA \u05D5\u05D1\u05DC\u05EA\u05D9 \u05D4\u05E4\u05D9\u05DB\u05D4.",
      title: "\u05DE\u05D7\u05E7 \u05D7\u05E9\u05D1\u05D5\u05DF"
    },
    emailAddressPage: {
      emailCode: {
        formHint: "\u05D0\u05D9\u05DE\u05D9\u05D9\u05DC \u05E9\u05DE\u05DB\u05D9\u05DC \u05E7\u05D5\u05D3 \u05D0\u05D9\u05DE\u05D5\u05EA \u05D9\u05E9\u05DC\u05D7 \u05DC\u05DB\u05EA\u05D5\u05D1\u05EA \u05D6\u05D5.",
        formSubtitle: "\u05D4\u05DB\u05E0\u05E1 \u05D0\u05EA \u05E7\u05D5\u05D3 \u05D4\u05D0\u05D9\u05DE\u05D5\u05EA \u05E9\u05E0\u05E9\u05DC\u05D7 \u05DC-{{identifier}}",
        formTitle: "\u05E7\u05D5\u05D3 \u05D0\u05D9\u05DE\u05D5\u05EA",
        resendButton: "\u05E9\u05DC\u05D7 \u05E7\u05D5\u05D3 \u05DE\u05D7\u05D3\u05E9",
        successMessage: "\u05D4\u05D0\u05D9\u05DE\u05D9\u05D9\u05DC {{identifier}} \u05D4\u05EA\u05D5\u05D5\u05E1\u05E3 \u05DC\u05D7\u05E9\u05D1\u05D5\u05DF \u05E9\u05DC\u05DA."
      },
      emailLink: {
        formHint: "\u05D0\u05D9\u05DE\u05D9\u05D9\u05DC \u05E9\u05DE\u05DB\u05D9\u05DC \u05E7\u05D9\u05E9\u05D5\u05E8 \u05DC\u05D0\u05D9\u05DE\u05D5\u05EA \u05D9\u05E9\u05DC\u05D7 \u05DC\u05DB\u05EA\u05D5\u05D1\u05EA \u05D6\u05D5.",
        formSubtitle: "\u05DC\u05D7\u05E5 \u05E2\u05DC \u05E7\u05D9\u05E9\u05D5\u05E8 \u05D4\u05D0\u05D9\u05DE\u05D5\u05EA \u05D1\u05D0\u05D9\u05DE\u05D9\u05D9\u05DC \u05E9\u05E0\u05E9\u05DC\u05D7 \u05DC-{{identifier}}",
        formTitle: "\u05E7\u05D9\u05E9\u05D5\u05E8 \u05DC\u05D0\u05D9\u05DE\u05D5\u05EA",
        resendButton: "\u05E9\u05DC\u05D7 \u05E7\u05D9\u05E9\u05D5\u05E8 \u05DE\u05D7\u05D3\u05E9",
        successMessage: "\u05D4\u05D0\u05D9\u05DE\u05D9\u05D9\u05DC {{identifier}} \u05D4\u05EA\u05D5\u05D5\u05E1\u05E3 \u05DC\u05D7\u05E9\u05D1\u05D5\u05DF \u05E9\u05DC\u05DA."
      },
      enterpriseSSOLink: {
        formButton: void 0,
        formSubtitle: void 0
      },
      formHint: void 0,
      removeResource: {
        messageLine1: "{{identifier}} \u05D9\u05D5\u05E1\u05E8 \u05DE\u05D4\u05D7\u05E9\u05D1\u05D5\u05DF \u05D4\u05D6\u05D4.",
        messageLine2: "\u05DC\u05D0 \u05EA\u05D5\u05DB\u05DC \u05D9\u05D5\u05EA\u05E8 \u05DC\u05D4\u05EA\u05D7\u05D1\u05E8 \u05D1\u05D0\u05DE\u05E6\u05E2\u05D5\u05EA \u05DB\u05EA\u05D5\u05D1\u05EA \u05D0\u05D9\u05DE\u05D9\u05D9\u05DC \u05D6\u05D5.",
        successMessage: "\u05D4\u05D0\u05D9\u05DE\u05D9\u05D9\u05DC {{emailAddress}} \u05D4\u05D5\u05E1\u05E8 \u05DE\u05D4\u05D7\u05E9\u05D1\u05D5\u05DF \u05E9\u05DC\u05DA.",
        title: "\u05D4\u05E1\u05E8 \u05DB\u05EA\u05D5\u05D1\u05EA \u05D0\u05D9\u05DE\u05D9\u05D9\u05DC"
      },
      title: "\u05D4\u05D5\u05E1\u05E3 \u05DB\u05EA\u05D5\u05D1\u05EA \u05D0\u05D9\u05DE\u05D9\u05D9\u05DC",
      verifyTitle: "\u05D0\u05DE\u05EA \u05DB\u05EA\u05D5\u05D1\u05EA \u05D0\u05D9\u05DE\u05D9\u05D9\u05DC"
    },
    formButtonPrimary__add: "\u05D4\u05D5\u05E1\u05E3",
    formButtonPrimary__continue: "\u05D4\u05DE\u05E9\u05DA",
    formButtonPrimary__finish: "\u05E1\u05D9\u05D9\u05DD",
    formButtonPrimary__remove: "\u05DE\u05D7\u05E7",
    formButtonPrimary__save: "\u05E9\u05DE\u05D5\u05E8",
    formButtonReset: "\u05D1\u05D8\u05DC",
    mfaPage: {
      formHint: "\u05D1\u05D7\u05E8 \u05E9\u05D9\u05D8\u05D4 \u05DC\u05D4\u05D5\u05E1\u05E4\u05D4.",
      title: "\u05D4\u05D5\u05E1\u05E3 \u05D0\u05D9\u05DE\u05D5\u05EA \u05D3\u05D5-\u05E9\u05DC\u05D1\u05D9"
    },
    mfaPhoneCodePage: {
      backButton: "\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05DE\u05E1\u05E4\u05E8 \u05E7\u05D9\u05D9\u05DD",
      primaryButton__addPhoneNumber: "\u05D4\u05D5\u05E1\u05E3 \u05DE\u05E1\u05E4\u05E8 \u05D8\u05DC\u05E4\u05D5\u05DF",
      removeResource: {
        messageLine1: "{{identifier}} \u05DC\u05D0 \u05D9\u05E7\u05D1\u05DC \u05D9\u05D5\u05EA\u05E8 \u05E7\u05D5\u05D3\u05D9\u05DD \u05DC\u05D0\u05D9\u05DE\u05D5\u05EA \u05D1\u05E2\u05EA \u05D4\u05D4\u05EA\u05D7\u05D1\u05E8\u05D5\u05EA.",
        messageLine2: "\u05D4\u05D7\u05E9\u05D1\u05D5\u05DF \u05E9\u05DC\u05DA \u05E2\u05E9\u05D5\u05D9 \u05DC\u05D0 \u05DC\u05D4\u05D9\u05D5\u05EA \u05D1\u05D8\u05D5\u05D7 \u05DB\u05DE\u05D5 \u05E9\u05D4\u05D5\u05D0. \u05D4\u05D0\u05DD \u05D0\u05EA\u05D4 \u05D1\u05D8\u05D5\u05D7 \u05E9\u05D0\u05EA\u05D4 \u05E8\u05D5\u05E6\u05D4 \u05DC\u05D4\u05DE\u05E9\u05D9\u05DA?",
        successMessage: "\u05D0\u05D9\u05DE\u05D5\u05EA \u05E7\u05D5\u05D3 SMS \u05D3\u05D5-\u05E9\u05DC\u05D1\u05D9 \u05D4\u05D5\u05E1\u05E8 \u05DC{{mfaPhoneCode}}",
        title: "\u05D4\u05E1\u05E8 \u05D0\u05D9\u05DE\u05D5\u05EA \u05D3\u05D5-\u05E9\u05DC\u05D1\u05D9"
      },
      subtitle__availablePhoneNumbers: "\u05D1\u05D7\u05E8 \u05DE\u05E1\u05E4\u05E8 \u05D8\u05DC\u05E4\u05D5\u05DF \u05DC\u05D4\u05E8\u05E9\u05DE\u05D4 \u05DC\u05D0\u05D9\u05DE\u05D5\u05EA \u05E7\u05D5\u05D3 SMS \u05D3\u05D5-\u05E9\u05DC\u05D1\u05D9.",
      subtitle__unavailablePhoneNumbers: "\u05D0\u05D9\u05DF \u05DE\u05E1\u05E4\u05E8\u05D9 \u05D8\u05DC\u05E4\u05D5\u05DF \u05D6\u05DE\u05D9\u05E0\u05D9\u05DD \u05DC\u05D4\u05E8\u05E9\u05DE\u05D4 \u05DC\u05D0\u05D9\u05DE\u05D5\u05EA \u05E7\u05D5\u05D3 SMS \u05D3\u05D5-\u05E9\u05DC\u05D1\u05D9.",
      successMessage1: "\u05D1\u05E2\u05EA \u05D4\u05D4\u05EA\u05D7\u05D1\u05E8\u05D5\u05EA, \u05EA\u05E6\u05D8\u05E8\u05DA \u05DC\u05D4\u05D6\u05D9\u05DF \u05E7\u05D5\u05D3 \u05D0\u05D9\u05DE\u05D5\u05EA \u05E9\u05E0\u05E9\u05DC\u05D7 \u05DC\u05DE\u05E1\u05E4\u05E8 \u05D8\u05DC\u05E4\u05D5\u05DF \u05D6\u05D4 \u05DB\u05E9\u05DC\u05D1 \u05E0\u05D5\u05E1\u05E3.",
      successMessage2: "\u05E9\u05DE\u05D5\u05E8 \u05E7\u05D5\u05D3\u05D9 \u05D2\u05D9\u05D1\u05D5\u05D9 \u05D0\u05DC\u05D5 \u05D1\u05DE\u05E7\u05D5\u05DD \u05D1\u05D8\u05D5\u05D7. \u05D0\u05DD \u05EA\u05D0\u05D1\u05D3 \u05D2\u05D9\u05E9\u05D4 \u05DC\u05DE\u05DB\u05E9\u05D9\u05E8 \u05D4\u05D0\u05D9\u05DE\u05D5\u05EA \u05E9\u05DC\u05DA, \u05EA\u05D5\u05DB\u05DC \u05DC\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05E7\u05D5\u05D3\u05D9 \u05D2\u05D9\u05D1\u05D5\u05D9 \u05DB\u05D3\u05D9 \u05DC\u05D4\u05EA\u05D7\u05D1\u05E8.",
      successTitle: "\u05D0\u05D9\u05DE\u05D5\u05EA \u05E7\u05D5\u05D3 SMS \u05D4\u05D5\u05E4\u05E2\u05DC",
      title: "\u05D4\u05D5\u05E1\u05E3 \u05D0\u05D9\u05DE\u05D5\u05EA \u05E7\u05D5\u05D3 SMS"
    },
    mfaTOTPPage: {
      authenticatorApp: {
        buttonAbleToScan__nonPrimary: "\u05E1\u05E8\u05D5\u05E7 \u05E7\u05D5\u05D3 QR \u05D1\u05DE\u05E7\u05D5\u05DD",
        buttonUnableToScan__nonPrimary: "\u05DC\u05D0 \u05D9\u05DB\u05D5\u05DC \u05DC\u05E1\u05E8\u05D5\u05E7 \u05E7\u05D5\u05D3 QR?",
        infoText__ableToScan: "\u05D4\u05D2\u05D3\u05E8 \u05E9\u05D9\u05D8\u05EA \u05D4\u05EA\u05D7\u05D1\u05E8\u05D5\u05EA \u05D7\u05D3\u05E9\u05D4 \u05D1\u05D0\u05E4\u05DC\u05D9\u05E7\u05E6\u05D9\u05D9\u05EA \u05D4\u05D0\u05D5\u05EA\u05E0\u05D8\u05D9\u05E7\u05D8\u05D5\u05E8 \u05E9\u05DC\u05DA \u05D5\u05E1\u05E8\u05D5\u05E7 \u05D0\u05EA \u05E7\u05D5\u05D3 \u05D4-QR \u05D4\u05D1\u05D0 \u05DB\u05D3\u05D9 \u05DC\u05D7\u05D1\u05E8\u05D5 \u05DC\u05D7\u05E9\u05D1\u05D5\u05DF \u05E9\u05DC\u05DA.",
        infoText__unableToScan: "\u05D4\u05D2\u05D3\u05E8 \u05E9\u05D9\u05D8\u05EA \u05D4\u05EA\u05D7\u05D1\u05E8\u05D5\u05EA \u05D7\u05D3\u05E9\u05D4 \u05D1\u05D0\u05D5\u05EA\u05E0\u05D8\u05D9\u05E7\u05D8\u05D5\u05E8 \u05E9\u05DC\u05DA \u05D5\u05D4\u05DB\u05E0\u05E1 \u05D0\u05EA \u05D4\u05DE\u05E4\u05EA\u05D7 \u05E9\u05E1\u05D5\u05E4\u05E7 \u05DC\u05DE\u05D8\u05D4.",
        inputLabel__unableToScan1: "\u05D5\u05D5\u05D3\u05D0 \u05E9\u05E1\u05D9\u05E1\u05DE\u05D0\u05D5\u05EA \u05DE\u05D1\u05D5\u05E1\u05E1\u05D5\u05EA-\u05D6\u05DE\u05DF \u05D0\u05D5 \u05D7\u05D3-\u05E4\u05E2\u05DE\u05D9\u05D5\u05EA \u05DE\u05D5\u05E4\u05E2\u05DC\u05D5\u05EA, \u05D5\u05D0\u05D6 \u05E1\u05D9\u05D9\u05DD \u05DC\u05D7\u05D1\u05E8 \u05D0\u05EA \u05D4\u05D7\u05E9\u05D1\u05D5\u05DF \u05E9\u05DC\u05DA.",
        inputLabel__unableToScan2: "\u05DC\u05D7\u05DC\u05D5\u05E4\u05D9\u05DF, \u05D0\u05DD \u05D4\u05D0\u05D5\u05EA\u05E0\u05D8\u05D9\u05E7\u05D8\u05D5\u05E8 \u05E9\u05DC\u05DA \u05EA\u05D5\u05DE\u05DA \u05D1\u05DB\u05EA\u05D5\u05D1\u05EA URI \u05E9\u05DC TOTP, \u05EA\u05D5\u05DB\u05DC \u05D2\u05DD \u05DC\u05D4\u05E2\u05EA\u05D9\u05E7 \u05D0\u05EA \u05D4\u05DB\u05EA\u05D5\u05D1\u05EA \u05D4\u05DE\u05DC\u05D0\u05D4."
      },
      removeResource: {
        messageLine1: "\u05E7\u05D5\u05D3\u05D9 \u05D4\u05D0\u05D9\u05DE\u05D5\u05EA \u05DE\u05D4\u05D0\u05D5\u05EA\u05E0\u05D8\u05D9\u05E7\u05D8\u05D5\u05E8 \u05D4\u05D6\u05D4 \u05DC\u05D0 \u05D9\u05D4\u05D9\u05D5 \u05E0\u05D3\u05E8\u05E9\u05D9\u05DD \u05D9\u05D5\u05EA\u05E8 \u05D1\u05E2\u05EA \u05D4\u05D4\u05EA\u05D7\u05D1\u05E8\u05D5\u05EA.",
        messageLine2: "\u05D4\u05D7\u05E9\u05D1\u05D5\u05DF \u05E9\u05DC\u05DA \u05E2\u05E9\u05D5\u05D9 \u05DC\u05D0 \u05DC\u05D4\u05D9\u05D5\u05EA \u05D1\u05D8\u05D5\u05D7 \u05DB\u05DE\u05D5 \u05E9\u05D4\u05D5\u05D0. \u05D4\u05D0\u05DD \u05D0\u05EA\u05D4 \u05D1\u05D8\u05D5\u05D7 \u05E9\u05D0\u05EA\u05D4 \u05E8\u05D5\u05E6\u05D4 \u05DC\u05D4\u05DE\u05E9\u05D9\u05DA?",
        successMessage: "\u05D0\u05D9\u05DE\u05D5\u05EA \u05D3\u05D5-\u05E9\u05DC\u05D1\u05D9 \u05D1\u05D0\u05DE\u05E6\u05E2\u05D5\u05EA \u05D0\u05E4\u05DC\u05D9\u05E7\u05E6\u05D9\u05EA \u05D4\u05D0\u05D5\u05EA\u05E0\u05D8\u05D9\u05E7\u05D8\u05D5\u05E8 \u05D4\u05D5\u05E1\u05E8.",
        title: "\u05D4\u05E1\u05E8 \u05D0\u05D9\u05DE\u05D5\u05EA \u05D3\u05D5-\u05E9\u05DC\u05D1\u05D9"
      },
      successMessage: "\u05D4\u05D0\u05D9\u05DE\u05D5\u05EA \u05D4\u05D3\u05D5-\u05E9\u05DC\u05D1\u05D9 \u05DE\u05D5\u05E4\u05E2\u05DC \u05DB\u05E2\u05EA. \u05D1\u05E2\u05EA \u05D4\u05D4\u05EA\u05D7\u05D1\u05E8\u05D5\u05EA, \u05EA\u05D9\u05D3\u05E8\u05E9 \u05DC\u05D4\u05DB\u05E0\u05D9\u05E1 \u05E7\u05D5\u05D3 \u05D0\u05D9\u05DE\u05D5\u05EA \u05DE\u05D4\u05D0\u05D5\u05EA\u05E0\u05D8\u05D9\u05E7\u05D8\u05D5\u05E8 \u05D6\u05D4 \u05DB\u05E9\u05DC\u05D1 \u05E0\u05D5\u05E1\u05E3.",
      title: "\u05D4\u05D5\u05E1\u05E3 \u05D0\u05E4\u05DC\u05D9\u05E7\u05E6\u05D9\u05D9\u05EA \u05D0\u05D9\u05DE\u05D5\u05EA",
      verifySubtitle: "\u05D4\u05DB\u05E0\u05E1 \u05D0\u05EA \u05E7\u05D5\u05D3 \u05D4\u05D0\u05D9\u05DE\u05D5\u05EA \u05E9\u05E0\u05D5\u05E6\u05E8 \u05E2\u05DC \u05D9\u05D3\u05D9 \u05D4\u05D0\u05D5\u05EA\u05E0\u05D8\u05D9\u05E7\u05D8\u05D5\u05E8 \u05E9\u05DC\u05DA",
      verifyTitle: "\u05E7\u05D5\u05D3 \u05D0\u05D9\u05DE\u05D5\u05EA"
    },
    mobileButton__menu: "\u05EA\u05E4\u05E8\u05D9\u05D8",
    navbar: {
      account: "\u05E4\u05E8\u05D5\u05E4\u05D9\u05DC",
      apiKeys: void 0,
      billing: void 0,
      description: "\u05E0\u05D4\u05DC \u05D0\u05EA \u05E4\u05E8\u05D8\u05D9 \u05D4\u05D7\u05E9\u05D1\u05D5\u05DF \u05E9\u05DC\u05DA.",
      security: "\u05D0\u05D1\u05D8\u05D7\u05D4",
      title: "\u05D7\u05E9\u05D1\u05D5\u05DF"
    },
    passkeyScreen: {
      removeResource: {
        messageLine1: "{{name}} \u05D9\u05D5\u05E1\u05E8 \u05DE\u05D7\u05E9\u05D1\u05D5\u05DF \u05D6\u05D4",
        title: "\u05D4\u05E1\u05E8 \u05DE\u05E4\u05EA\u05D7 \u05E1\u05D9\u05E1\u05DE\u05D4"
      },
      subtitle__rename: "\u05D0\u05EA\u05D4 \u05D9\u05DB\u05D5\u05DC \u05DC\u05E9\u05E0\u05D5\u05EA \u05D0\u05EA \u05E9\u05DD \u05DE\u05E4\u05EA\u05D7 \u05D4\u05E1\u05D9\u05E1\u05DE\u05D4 \u05E2\u05DC \u05DE\u05E0\u05EA \u05DC\u05DE\u05E6\u05D5\u05D0 \u05D0\u05D5\u05EA\u05D5 \u05D1\u05E7\u05DC\u05D5\u05EA \u05D9\u05D5\u05EA\u05E8",
      title__rename: "\u05E9\u05E0\u05D4 \u05E9\u05DD \u05DC\u05DE\u05E4\u05EA\u05D7 \u05E1\u05D9\u05E1\u05DE\u05D4"
    },
    passwordPage: {
      checkboxInfoText__signOutOfOtherSessions: "\u05DE\u05D5\u05DE\u05DC\u05E5 \u05DC\u05D4\u05EA\u05E0\u05EA\u05E7 \u05DE\u05DB\u05DC \u05D4\u05DE\u05DB\u05E9\u05D9\u05E8\u05D9\u05DD \u05D4\u05D0\u05D7\u05E8\u05D9\u05DD \u05E9\u05E2\u05DC\u05D5\u05DC\u05D9\u05DD \u05D4\u05D9\u05D5 \u05DC\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05E1\u05D9\u05E1\u05DE\u05D4 \u05D4\u05D9\u05E9\u05E0\u05D4 \u05E9\u05DC\u05DA.",
      readonly: "\u05DB\u05E8\u05D2\u05E2 \u05DC\u05D0 \u05E0\u05D9\u05EA\u05DF \u05DC\u05E2\u05E8\u05D5\u05DA \u05D0\u05EA \u05D4\u05E1\u05D9\u05E1\u05DE\u05D4 \u05E9\u05DC\u05DA \u05DE\u05DB\u05D9\u05D5\u05D5\u05DF \u05E9\u05E0\u05D9\u05EA\u05DF \u05DC\u05D4\u05EA\u05D7\u05D1\u05E8 \u05E8\u05E7 \u05D3\u05E8\u05DA \u05D7\u05D9\u05D1\u05D5\u05E8 \u05D0\u05E8\u05D2\u05D5\u05E0\u05D9.",
      successMessage__set: "\u05D4\u05E1\u05D9\u05E1\u05DE\u05D4 \u05E9\u05DC\u05DA \u05D4\u05D5\u05D2\u05D3\u05E8\u05D4.",
      successMessage__signOutOfOtherSessions: "\u05DB\u05DC \u05D4\u05DE\u05DB\u05E9\u05D9\u05E8\u05D9\u05DD \u05D4\u05D0\u05D7\u05E8\u05D9\u05DD \u05D4\u05EA\u05E0\u05EA\u05E7\u05D5.",
      successMessage__update: "\u05D4\u05E1\u05D9\u05E1\u05DE\u05D4 \u05E9\u05DC\u05DA \u05E2\u05D5\u05D3\u05DB\u05E0\u05D4.",
      title__set: "\u05D4\u05D2\u05D3\u05E8 \u05E1\u05D9\u05E1\u05DE\u05D4",
      title__update: "\u05E9\u05E0\u05D4 \u05E1\u05D9\u05E1\u05DE\u05D4"
    },
    phoneNumberPage: {
      infoText: "\u05D4\u05D5\u05D3\u05E2\u05EA \u05D8\u05E7\u05E1\u05D8 \u05E9\u05DE\u05DB\u05D9\u05DC\u05D4 \u05E7\u05D9\u05E9\u05D5\u05E8 \u05DC\u05D0\u05D9\u05DE\u05D5\u05EA \u05EA\u05D9\u05E9\u05DC\u05D7 \u05DC\u05DE\u05E1\u05E4\u05E8 \u05D8\u05DC\u05E4\u05D5\u05DF \u05D6\u05D4.",
      removeResource: {
        messageLine1: "{{identifier}} \u05D9\u05D5\u05E1\u05E8 \u05DE\u05D4\u05D7\u05E9\u05D1\u05D5\u05DF \u05D4\u05D6\u05D4.",
        messageLine2: "\u05DC\u05D0 \u05EA\u05D5\u05DB\u05DC \u05D9\u05D5\u05EA\u05E8 \u05DC\u05D4\u05EA\u05D7\u05D1\u05E8 \u05D1\u05D0\u05DE\u05E6\u05E2\u05D5\u05EA \u05DE\u05E1\u05E4\u05E8 \u05D8\u05DC\u05E4\u05D5\u05DF \u05D6\u05D4.",
        successMessage: "{{phoneNumber}} \u05D4\u05D5\u05E1\u05E8 \u05DE\u05D4\u05D7\u05E9\u05D1\u05D5\u05DF \u05E9\u05DC\u05DA.",
        title: "\u05D4\u05E1\u05E8 \u05DE\u05E1\u05E4\u05E8 \u05D8\u05DC\u05E4\u05D5\u05DF"
      },
      successMessage: "{{identifier}} \u05D4\u05EA\u05D5\u05D5\u05E1\u05E3 \u05DC\u05D7\u05E9\u05D1\u05D5\u05DF \u05E9\u05DC\u05DA.",
      title: "\u05D4\u05D5\u05E1\u05E3 \u05DE\u05E1\u05E4\u05E8 \u05D8\u05DC\u05E4\u05D5\u05DF",
      verifySubtitle: "\u05D4\u05D6\u05DF \u05D0\u05EA \u05E7\u05D5\u05D3 \u05D4\u05D0\u05D9\u05DE\u05D5\u05EA \u05E9\u05E0\u05E9\u05DC\u05D7 \u05DC{{identifier}}",
      verifyTitle: "\u05D0\u05DE\u05EA \u05DE\u05E1\u05E4\u05E8 \u05D8\u05DC\u05E4\u05D5\u05DF"
    },
    plansPage: {
      title: void 0
    },
    profilePage: {
      fileDropAreaHint: '\u05D4\u05E2\u05DC\u05D4 \u05EA\u05DE\u05D5\u05E0\u05D4 \u05D1\u05E4\u05D5\u05E8\u05DE\u05D8 JPG, PNG, GIF, \u05D0\u05D5 WEBP \u05D4\u05E7\u05D8\u05E0\u05D4 \u05DE-10 \u05DE"\u05D1',
      imageFormDestructiveActionSubtitle: "\u05D4\u05E1\u05E8 \u05EA\u05DE\u05D5\u05E0\u05D4",
      imageFormSubtitle: "\u05D4\u05E2\u05DC\u05D4 \u05EA\u05DE\u05D5\u05E0\u05D4",
      imageFormTitle: "\u05EA\u05DE\u05D5\u05E0\u05EA \u05E4\u05E8\u05D5\u05E4\u05D9\u05DC",
      readonly: "\u05E4\u05E8\u05D8\u05D9 \u05D4\u05E4\u05E8\u05D5\u05E4\u05D9\u05DC \u05E9\u05DC\u05DA \u05E0\u05D9\u05EA\u05E0\u05D9\u05DD \u05E2\u05DC \u05D9\u05D3\u05D9 \u05D4\u05D7\u05D9\u05D1\u05D5\u05E8 \u05D4\u05D0\u05E8\u05D2\u05D5\u05E0\u05D9 \u05D5\u05D0\u05D9\u05E0\u05DD \u05E0\u05D9\u05EA\u05E0\u05D9\u05DD \u05DC\u05E2\u05E8\u05D9\u05DB\u05D4.",
      successMessage: "\u05D4\u05E4\u05E8\u05D5\u05E4\u05D9\u05DC \u05E9\u05DC\u05DA \u05E2\u05D5\u05D3\u05DB\u05DF.",
      title: "\u05E2\u05D3\u05DB\u05DF \u05E4\u05E8\u05D5\u05E4\u05D9\u05DC"
    },
    start: {
      activeDevicesSection: {
        destructiveAction: "\u05D4\u05EA\u05E0\u05EA\u05E7 \u05DE\u05D4\u05DE\u05DB\u05E9\u05D9\u05E8",
        title: "\u05DE\u05DB\u05E9\u05D9\u05E8\u05D9\u05DD \u05E4\u05E2\u05D9\u05DC\u05D9\u05DD"
      },
      connectedAccountsSection: {
        actionLabel__connectionFailed: "\u05E0\u05E1\u05D4 \u05E9\u05D5\u05D1",
        actionLabel__reauthorize: "\u05D0\u05E9\u05E8 \u05E2\u05DB\u05E9\u05D9\u05D5",
        destructiveActionTitle: "\u05D4\u05E1\u05E8",
        primaryButton: "\u05D7\u05D1\u05E8 \u05D7\u05E9\u05D1\u05D5\u05DF",
        subtitle__disconnected: "\u05D4\u05D7\u05E9\u05D1\u05D5\u05DF \u05D4\u05D6\u05D4 \u05E0\u05D5\u05EA\u05E7",
        subtitle__reauthorize: "\u05D4\u05D4\u05E8\u05E9\u05D0\u05D5\u05EA \u05D4\u05E0\u05D3\u05E8\u05E9\u05D5\u05EA \u05E2\u05D5\u05D3\u05DB\u05E0\u05D5, \u05D9\u05D9\u05EA\u05DB\u05DF \u05E9\u05D0\u05EA\u05D4 \u05D7\u05D5\u05D5\u05D4 \u05E4\u05D5\u05E0\u05E7\u05E6\u05D9\u05D5\u05E0\u05DC\u05D9\u05D5\u05EA \u05DE\u05D5\u05D2\u05D1\u05DC\u05EA. \u05D0\u05E0\u05D0 \u05D0\u05E9\u05E8 \u05DE\u05D7\u05D3\u05E9 \u05D0\u05EA \u05D4\u05D0\u05E4\u05DC\u05D9\u05E7\u05E6\u05D9\u05D4 \u05DB\u05D3\u05D9 \u05DC\u05D4\u05D9\u05DE\u05E0\u05E2 \u05DE\u05D1\u05E2\u05D9\u05D5\u05EA.",
        title: "\u05D7\u05E9\u05D1\u05D5\u05E0\u05D5\u05EA \u05DE\u05D7\u05D5\u05D1\u05E8\u05D9\u05DD"
      },
      dangerSection: {
        deleteAccountButton: "\u05DE\u05D7\u05E7 \u05D7\u05E9\u05D1\u05D5\u05DF",
        title: "\u05DE\u05E1\u05D5\u05DB\u05DF"
      },
      emailAddressesSection: {
        destructiveAction: "\u05D4\u05E1\u05E8 \u05DB\u05EA\u05D5\u05D1\u05EA \u05D0\u05D9\u05DE\u05D9\u05D9\u05DC",
        detailsAction__nonPrimary: "\u05D4\u05D2\u05D3\u05E8 \u05DB\u05E2\u05D9\u05E7\u05E8\u05D9\u05EA",
        detailsAction__primary: "\u05D4\u05E9\u05DC\u05DD \u05D0\u05D9\u05DE\u05D5\u05EA",
        detailsAction__unverified: "\u05D4\u05E9\u05DC\u05DD \u05D0\u05D9\u05DE\u05D5\u05EA",
        primaryButton: "\u05D4\u05D5\u05E1\u05E3 \u05DB\u05EA\u05D5\u05D1\u05EA \u05D0\u05D9\u05DE\u05D9\u05D9\u05DC",
        title: "\u05DB\u05EA\u05D5\u05D1\u05EA \u05D0\u05D9\u05DE\u05D9\u05D9\u05DC"
      },
      enterpriseAccountsSection: {
        title: "\u05D7\u05E9\u05D1\u05D5\u05E0\u05D5\u05EA \u05D0\u05E8\u05D2\u05D5\u05E0\u05D9\u05D9\u05DD"
      },
      headerTitle__account: "\u05D7\u05E9\u05D1\u05D5\u05DF",
      headerTitle__security: "\u05D0\u05D1\u05D8\u05D7\u05D4",
      mfaSection: {
        backupCodes: {
          actionLabel__regenerate: "\u05E6\u05D5\u05E8 \u05E7\u05D5\u05D3\u05D9\u05DD \u05DE\u05D7\u05D3\u05E9",
          headerTitle: "\u05E7\u05D5\u05D3 \u05D2\u05D9\u05D1\u05D5\u05D9",
          subtitle__regenerate: "\u05E7\u05D1\u05DC \u05E7\u05D1\u05D5\u05E6\u05D4 \u05D7\u05D3\u05E9\u05D4 \u05E9\u05DC \u05E7\u05D5\u05D3\u05D9 \u05D2\u05D9\u05D1\u05D5\u05D9 \u05DE\u05D0\u05D5\u05D1\u05D8\u05D7\u05D9\u05DD. \u05E7\u05D5\u05D3\u05D9 \u05D2\u05D9\u05D1\u05D5\u05D9 \u05E7\u05D5\u05D3\u05DE\u05D9\u05DD \u05D9\u05D9\u05DE\u05D7\u05E7\u05D5 \u05D5\u05DC\u05D0 \u05E0\u05D9\u05EA\u05DF \u05DC\u05D4\u05E9\u05EA\u05DE\u05E9 \u05D1\u05D4\u05DD.",
          title__regenerate: "\u05E6\u05D5\u05E8 \u05DE\u05D7\u05D3\u05E9 \u05E7\u05D5\u05D3\u05D9 \u05D2\u05D9\u05D1\u05D5\u05D9"
        },
        phoneCode: {
          actionLabel__setDefault: "\u05D4\u05D2\u05D3\u05E8 \u05DB\u05D1\u05E8\u05D9\u05E8\u05EA \u05DE\u05D7\u05D3\u05DC",
          destructiveActionLabel: "\u05D4\u05E1\u05E8 \u05DE\u05E1\u05E4\u05E8 \u05D8\u05DC\u05E4\u05D5\u05DF"
        },
        primaryButton: "\u05D4\u05D5\u05E1\u05E3 \u05D0\u05D9\u05DE\u05D5\u05EA \u05D3\u05D5-\u05E9\u05DC\u05D1\u05D9",
        title: "\u05D0\u05D9\u05DE\u05D5\u05EA \u05D3\u05D5-\u05E9\u05DC\u05D1\u05D9",
        totp: {
          destructiveActionTitle: "\u05D4\u05E1\u05E8",
          headerTitle: "\u05D0\u05E4\u05DC\u05D9\u05E7\u05E6\u05D9\u05D9\u05EA \u05DE\u05D0\u05DE\u05EA"
        }
      },
      passkeysSection: {
        menuAction__destructive: "\u05D4\u05D2\u05D3\u05E8 \u05E1\u05D9\u05E1\u05DE\u05D4",
        menuAction__rename: "\u05E2\u05D3\u05DB\u05DF \u05E1\u05D9\u05E1\u05DE\u05D4",
        primaryButton: void 0,
        title: "\u05E1\u05D9\u05E1\u05DE\u05D4"
      },
      passwordSection: {
        primaryButton__setPassword: "\u05D4\u05D2\u05D3\u05E8 \u05E1\u05D9\u05E1\u05DE\u05D4",
        primaryButton__updatePassword: "\u05E9\u05E0\u05D4 \u05E1\u05D9\u05E1\u05DE\u05D4",
        title: "\u05E1\u05D9\u05E1\u05DE\u05D4"
      },
      phoneNumbersSection: {
        destructiveAction: "\u05D4\u05E1\u05E8 \u05DE\u05E1\u05E4\u05E8 \u05D8\u05DC\u05E4\u05D5\u05DF",
        detailsAction__nonPrimary: "\u05D4\u05D2\u05D3\u05E8 \u05DB\u05E8\u05D0\u05E9\u05D9",
        detailsAction__primary: "\u05D4\u05E9\u05DC\u05DD \u05D0\u05D9\u05DE\u05D5\u05EA",
        detailsAction__unverified: "\u05D4\u05E9\u05DC\u05DD \u05D0\u05D9\u05DE\u05D5\u05EA",
        primaryButton: "\u05D4\u05D5\u05E1\u05E3 \u05DE\u05E1\u05E4\u05E8 \u05D8\u05DC\u05E4\u05D5\u05DF",
        title: "\u05DE\u05E1\u05E4\u05E8\u05D9 \u05D8\u05DC\u05E4\u05D5\u05DF"
      },
      profileSection: {
        primaryButton: "\u05E2\u05D3\u05DB\u05DF \u05E4\u05E8\u05D5\u05E4\u05D9\u05DC",
        title: "\u05E4\u05E8\u05D5\u05E4\u05D9\u05DC"
      },
      usernameSection: {
        primaryButton__setUsername: "\u05D4\u05D2\u05D3\u05E8 \u05E9\u05DD \u05DE\u05E9\u05EA\u05DE\u05E9",
        primaryButton__updateUsername: "\u05E9\u05E0\u05D4 \u05E9\u05DD \u05DE\u05E9\u05EA\u05DE\u05E9",
        title: "\u05E9\u05DD \u05DE\u05E9\u05EA\u05DE\u05E9"
      },
      web3WalletsSection: {
        destructiveAction: "\u05D4\u05E1\u05E8 \u05D0\u05E8\u05E0\u05E7",
        detailsAction__nonPrimary: void 0,
        primaryButton: "\u05D0\u05E8\u05E0\u05E7\u05D9 Web3",
        title: "\u05D0\u05E8\u05E0\u05E7\u05D9 Web3"
      }
    },
    usernamePage: {
      successMessage: "\u05E9\u05DD \u05D4\u05DE\u05E9\u05EA\u05DE\u05E9 \u05E9\u05DC\u05DA \u05E2\u05D5\u05D3\u05DB\u05DF.",
      title__set: "\u05E2\u05D3\u05DB\u05DF \u05E9\u05DD \u05DE\u05E9\u05EA\u05DE\u05E9",
      title__update: "\u05E2\u05D3\u05DB\u05DF \u05E9\u05DD \u05DE\u05E9\u05EA\u05DE\u05E9"
    },
    web3WalletPage: {
      removeResource: {
        messageLine1: "{{identifier}} \u05D9\u05D5\u05E1\u05E8 \u05DE\u05D4\u05D7\u05E9\u05D1\u05D5\u05DF \u05D4\u05D6\u05D4.",
        messageLine2: "\u05DC\u05D0 \u05EA\u05D5\u05DB\u05DC \u05D9\u05D5\u05EA\u05E8 \u05DC\u05D4\u05EA\u05D7\u05D1\u05E8 \u05D1\u05D0\u05DE\u05E6\u05E2\u05D5\u05EA \u05D0\u05E8\u05E0\u05E7 web3 \u05D6\u05D4.",
        successMessage: "{{web3Wallet}} \u05D4\u05D5\u05E1\u05E8 \u05DE\u05D4\u05D7\u05E9\u05D1\u05D5\u05DF \u05E9\u05DC\u05DA.",
        title: "\u05D4\u05E1\u05E8 \u05D0\u05E8\u05E0\u05E7 web3"
      },
      subtitle__availableWallets: "\u05D1\u05D7\u05E8 \u05D0\u05E8\u05E0\u05E7 web3 \u05DC\u05D7\u05D9\u05D1\u05D5\u05E8 \u05DC\u05D7\u05E9\u05D1\u05D5\u05DF \u05E9\u05DC\u05DA.",
      subtitle__unavailableWallets: "\u05D0\u05D9\u05DF \u05D0\u05E8\u05E0\u05E7\u05D9 web3 \u05D6\u05DE\u05D9\u05E0\u05D9\u05DD.",
      successMessage: "\u05D4\u05D0\u05E8\u05E0\u05E7 \u05D4\u05EA\u05D5\u05D5\u05E1\u05E3 \u05DC\u05D7\u05E9\u05D1\u05D5\u05DF \u05E9\u05DC\u05DA.",
      title: "\u05D4\u05D5\u05E1\u05E3 \u05D0\u05E8\u05E0\u05E7 web3",
      web3WalletButtonsBlockButton: "{{provider|titleize}}"
    }
  },
  waitlist: {
    start: {
      actionLink: void 0,
      actionText: void 0,
      formButton: void 0,
      subtitle: void 0,
      title: void 0
    },
    success: {
      message: void 0,
      subtitle: void 0,
      title: void 0
    }
  }
};
export {
  heIL
};
