// @clerk/localizations/dist/ro-RO.js
"use strict";
var __defProp = Object.defineProperty;
var __getOwnPropDesc = Object.getOwnPropertyDescriptor;
var __getOwnPropNames = Object.getOwnPropertyNames;
var __hasOwnProp = Object.prototype.hasOwnProperty;
var __export = (target, all) => {
  for (var name in all)
    __defProp(target, name, { get: all[name], enumerable: true });
};
var __copyProps = (to, from, except, desc) => {
  if (from && typeof from === "object" || typeof from === "function") {
    for (let key of __getOwnPropNames(from))
      if (!__hasOwnProp.call(to, key) && key !== except)
        __defProp(to, key, { get: () => from[key], enumerable: !(desc = __getOwnPropDesc(from, key)) || desc.enumerable });
  }
  return to;
};
var __toCommonJS = (mod) => __copyProps(__defProp({}, "__esModule", { value: true }), mod);

// src/ro-RO.ts
var ro_RO_exports = {};
__export(ro_RO_exports, {
  roRO: () => roRO
});
module.exports = __toCommonJS(ro_RO_exports);
var roRO = {
  locale: "ro-RO",
  apiKeys: {
    action__add: "Adaug\u0103 cheie nou\u0103",
    action__search: "Caut\u0103 chei",
    createdAndExpirationStatus__expiresOn: "Creat\u0103 {{ createdDate | shortDate('ro-RO') }} \u2022 Expir\u0103 {{ expiresDate | longDate('ro-RO') }}",
    createdAndExpirationStatus__never: "Creat\u0103 {{ createdDate | shortDate('ro-RO') }} \u2022 Nu expir\u0103 niciodat\u0103",
    detailsTitle__emptyRow: "Nu au fost g\u0103site chei API",
    formButtonPrimary__add: "Creeaz\u0103 cheie",
    formFieldCaption__expiration__expiresOn: "Expir\u0103 la {{ date }}",
    formFieldCaption__expiration__never: "Aceast\u0103 cheie nu va expira",
    formFieldOption__expiration__180d: "180 de zile",
    formFieldOption__expiration__1d: "1 zi",
    formFieldOption__expiration__1y: "1 an",
    formFieldOption__expiration__30d: "30 de zile",
    formFieldOption__expiration__60d: "60 de zile",
    formFieldOption__expiration__7d: "7 zile",
    formFieldOption__expiration__90d: "90 de zile",
    formFieldOption__expiration__never: "Niciodat\u0103",
    formHint: "Furnizeaz\u0103 un nume pentru a genera o cheie nou\u0103. O vei putea revoca oric\xE2nd.",
    formTitle: "Adaug\u0103 cheie API nou\u0103",
    lastUsed__days: "acum {{days}}z",
    lastUsed__hours: "acum {{hours}}h",
    lastUsed__minutes: "acum {{minutes}}m",
    lastUsed__months: "acum {{months}}l",
    lastUsed__seconds: "acum {{seconds}}s",
    lastUsed__years: "acum {{years}}a",
    menuAction__revoke: "Revoc\u0103 cheia",
    revokeConfirmation: {
      confirmationText: "Revoc\u0103",
      formButtonPrimary__revoke: "Revoc\u0103 cheia",
      formHint: "Sigur vrei s\u0103 \u0219tergi aceast\u0103 cheie Secret?",
      formTitle: "Revoci cheia secret\u0103 \u201E{{apiKeyName}}\u201D?"
    }
  },
  backButton: "\xCEnapoi",
  badge__activePlan: "Activ",
  badge__canceledEndsAt: "Anulat \u2022 Se \xEEncheie {{ date | shortDate('ro-RO') }}",
  badge__currentPlan: "Plan curent",
  badge__default: "Implicit",
  badge__endsAt: "Se \xEEncheie {{ date | shortDate('ro-RO') }}",
  badge__expired: "Expirat",
  badge__freeTrial: "Prob\u0103 gratuit\u0103",
  badge__otherImpersonatorDevice: "Alt dispozitiv de impersonare",
  badge__pastDueAt: "Restan\u021B\u0103 din {{ date | shortDate('ro-RO') }}",
  badge__pastDuePlan: "Restan\u021B\u0103",
  badge__primary: "Principal",
  badge__renewsAt: "Re\xEEnnoire {{ date | shortDate('ro-RO') }}",
  badge__requiresAction: "Necesit\u0103 ac\u021Biune",
  badge__startsAt: "\xCEncepe {{ date | shortDate('ro-RO') }}",
  badge__thisDevice: "Acest dispozitiv",
  badge__trialEndsAt: "Perioada de prob\u0103 se \xEEncheie {{ date | shortDate('ro-RO') }}",
  badge__unverified: "Neverificat",
  badge__upcomingPlan: "Urmeaz\u0103",
  badge__userDevice: "Dispozitiv utilizator",
  badge__you: "Tu",
  billing: {
    addPaymentMethod__label: "Adaug\u0103 metod\u0103 de plat\u0103",
    alwaysFree: "\xCEntotdeauna gratuit",
    annually: "Anual",
    availableFeatures: "Func\u021Bii disponibile",
    billedAnnually: "Facturat anual",
    billedMonthlyOnly: "Doar facturare lunar\u0103",
    cancelFreeTrial: "Anuleaz\u0103 perioada de prob\u0103",
    cancelFreeTrialAccessUntil: "Perioada de prob\u0103 r\u0103m\xE2ne activ\u0103 p\xE2n\u0103 la {{ date | longDate('ro-RO') }}. Dup\u0103 aceea, vei pierde accesul la func\u021Biile de prob\u0103. Nu vei fi taxat.",
    cancelFreeTrialTitle: "Anulezi perioada de prob\u0103 pentru planul {{plan}}?",
    cancelSubscription: "Anuleaz\u0103 abonamentul",
    cancelSubscriptionAccessUntil: "Po\u021Bi folosi \xEEn continuare func\u021Biile \u201E{{plan}}\u201D p\xE2n\u0103 la {{ date | longDate('ro-RO') }}, dup\u0103 care nu vei mai avea acces.",
    cancelSubscriptionNoCharge: "Nu vei fi taxat pentru acest abonament.",
    cancelSubscriptionPastDue: void 0,
    cancelSubscriptionTitle: "Anulezi abonamentul {{plan}}?",
    cannotSubscribeMonthly: "Nu te po\u021Bi abona la acest plan cu plat\u0103 lunar\u0103. Pentru acest plan este necesar\u0103 plata anual\u0103.",
    cannotSubscribeUnrecoverable: "Nu te po\u021Bi abona la acest plan. Abonamentul t\u0103u actual este mai scump dec\xE2t acest plan.",
    checkout: {
      description__paymentSuccessful: "Plata ta a fost efectuat\u0103 cu succes.",
      description__subscriptionSuccessful: "Noul t\u0103u abonament este configurat.",
      downgradeNotice: "Vei p\u0103stra abonamentul curent \u0219i func\u021Biile sale p\xE2n\u0103 la finalul ciclului de facturare, apoi vei fi schimbat la acest abonament.",
      emailForm: {
        subtitle: "\xCEnainte de a finaliza achizi\u021Bia, adaug\u0103 o adres\u0103 de email unde vor fi trimise chitan\u021Bele.",
        title: "Adaug\u0103 o adres\u0103 de email"
      },
      lineItems: {
        title__freeTrialEndsAt: "Perioada de prob\u0103 se \xEEncheie pe",
        title__paymentMethod: "Metod\u0103 de plat\u0103",
        title__statementId: "ID extras",
        title__subscriptionBegins: "Abonamentul \xEEncepe",
        title__totalPaid: "Total pl\u0103tit"
      },
      pastDueNotice: "Abonamentul anterior era restant, f\u0103r\u0103 plat\u0103.",
      perMonth: "pe lun\u0103",
      title: "Plat\u0103",
      title__paymentSuccessful: "Plata a reu\u0219it!",
      title__subscriptionSuccessful: "Succes!",
      title__trialSuccess: "Perioada de prob\u0103 a \xEEnceput!",
      totalDueAfterTrial: "Total de plat\u0103 dup\u0103 ce perioada de prob\u0103 se \xEEncheie \xEEn {{days}} zile"
    },
    credit: "Credit",
    creditRemainder: "Credit pentru restul abonamentului curent.",
    defaultFreePlanActive: "\xCEn prezent e\u0219ti pe planul Gratuit",
    free: "Gratuit",
    getStarted: "\xCEncepe",
    keepFreeTrial: "P\u0103streaz\u0103 perioada de prob\u0103",
    keepSubscription: "P\u0103streaz\u0103 abonamentul",
    manage: "Gestioneaz\u0103",
    manageSubscription: "Gestioneaz\u0103 abonamentul",
    month: "Lun\u0103",
    monthly: "Lunar",
    pastDue: "Restan\u021B\u0103",
    pay: "Pl\u0103te\u0219te {{amount}}",
    paymentMethod: {
      applePayDescription: {
        annual: "Plat\u0103 anual\u0103",
        monthly: "Plat\u0103 lunar\u0103"
      },
      dev: {
        anyNumbers: "Orice numere",
        cardNumber: "Num\u0103r card",
        cvcZip: "CVC, cod po\u0219tal",
        developmentMode: "Mod de dezvoltare",
        expirationDate: "Data expir\u0103rii",
        testCardInfo: "Informa\u021Bii card de test"
      }
    },
    paymentMethods__label: "Metode de plat\u0103",
    popular: "Popular",
    pricingTable: {
      billingCycle: "Ciclu de facturare",
      included: "Inclus"
    },
    reSubscribe: "Re-aboneaz\u0103-te",
    seeAllFeatures: "Vezi toate func\u021Biile",
    startFreeTrial: "\xCEncepe perioada de prob\u0103",
    startFreeTrial__days: "\xCEncepe perioada de prob\u0103 de {{days}} zile",
    subscribe: "Aboneaz\u0103-te",
    subscriptionDetails: {
      beginsOn: "\xCEncepe pe",
      currentBillingCycle: "Ciclu de facturare curent",
      endsOn: "Se \xEEncheie pe",
      firstPaymentAmount: "Prima plat\u0103",
      firstPaymentOn: "Prima plat\u0103 pe",
      nextPaymentAmount: "Urm\u0103toarea plat\u0103",
      nextPaymentOn: "Urm\u0103toarea plat\u0103 pe",
      pastDueAt: "Restan\u021B\u0103 pe",
      renewsAt: "Se re\xEEnnoie\u0219te la",
      subscribedOn: "Abonat pe",
      title: "Abonament",
      trialEndsOn: "Perioada de prob\u0103 se \xEEncheie pe",
      trialStartedOn: "Perioada de prob\u0103 a \xEEnceput pe"
    },
    subtotal: "Subtotal",
    switchPlan: "Schimb\u0103 pe acest plan",
    switchToAnnual: "Treci la anual",
    switchToAnnualWithAnnualPrice: "Treci la anual {{currency}}{{price}} / an",
    switchToMonthly: "Treci la lunar",
    switchToMonthlyWithPrice: "Treci la lunar {{currency}}{{price}} / lun\u0103",
    totalDue: "Total de plat\u0103",
    totalDueToday: "Total de plat\u0103 ast\u0103zi",
    viewFeatures: "Vezi func\u021Biile",
    viewPayment: "Vezi plata",
    year: "An"
  },
  createOrganization: {
    formButtonSubmit: "Creeaz\u0103 organiza\u021Bie",
    invitePage: {
      formButtonReset: "Omite"
    },
    title: "Creeaz\u0103 organiza\u021Bie"
  },
  dates: {
    lastDay: "Ieri la {{ date | timeString('ro-RO') }}",
    next6Days: "{{ date | weekday('ro-RO','long') }} la {{ date | timeString('ro-RO') }}",
    nextDay: "M\xE2ine la {{ date | timeString('ro-RO') }}",
    numeric: "{{ date | numeric('ro-RO') }}",
    previous6Days: "{{ date | weekday('ro-RO','long') }} trecut la {{ date | timeString('ro-RO') }}",
    sameDay: "Ast\u0103zi la {{ date | timeString('ro-RO') }}"
  },
  dividerText: "sau",
  footerActionLink__alternativePhoneCodeProvider: "Trimite codul prin SMS \xEEn schimb",
  footerActionLink__useAnotherMethod: "Folose\u0219te alt\u0103 metod\u0103",
  footerPageLink__help: "Ajutor",
  footerPageLink__privacy: "Confiden\u021Bialitate",
  footerPageLink__terms: "Termeni",
  formButtonPrimary: "Continu\u0103",
  formButtonPrimary__verify: "Verific\u0103",
  formFieldAction__forgotPassword: "Ai uitat parola?",
  formFieldError__matchingPasswords: "Parolele coincid.",
  formFieldError__notMatchingPasswords: "Parolele nu coincid.",
  formFieldError__verificationLinkExpired: "Linkul de verificare a expirat. Te rug\u0103m s\u0103 solici\u021Bi unul nou.",
  formFieldHintText__optional: "Op\u021Bional",
  formFieldHintText__slug: "Un \u201Eslug\u201D este un ID u\u0219or de citit care trebuie s\u0103 fie unic. Este folosit adesea \xEEn URL-uri.",
  formFieldInputPlaceholder__apiKeyDescription: "Explic\u0103 de ce generezi aceast\u0103 cheie",
  formFieldInputPlaceholder__apiKeyExpirationDate: "Selecteaz\u0103 data",
  formFieldInputPlaceholder__apiKeyName: "Introdu numele cheii secrete",
  formFieldInputPlaceholder__backupCode: "Introdu codul de rezerv\u0103",
  formFieldInputPlaceholder__confirmDeletionUserAccount: "\u0218terge contul",
  formFieldInputPlaceholder__emailAddress: "Introdu adresa de email",
  formFieldInputPlaceholder__emailAddress_username: "Introdu email sau nume de utilizator",
  formFieldInputPlaceholder__emailAddresses: "exemplu@email.com, exemplu2@email.com",
  formFieldInputPlaceholder__firstName: "Prenume",
  formFieldInputPlaceholder__lastName: "Nume",
  formFieldInputPlaceholder__organizationDomain: "exemplu.com",
  formFieldInputPlaceholder__organizationDomainEmailAddress: "tu@exemplu.com",
  formFieldInputPlaceholder__organizationName: "Numele organiza\u021Biei",
  formFieldInputPlaceholder__organizationSlug: "org-mea",
  formFieldInputPlaceholder__password: "Introdu parola",
  formFieldInputPlaceholder__phoneNumber: "Introdu num\u0103rul de telefon",
  formFieldInputPlaceholder__username: void 0,
  formFieldInput__emailAddress_format: void 0,
  formFieldLabel__apiKeyDescription: "Descriere",
  formFieldLabel__apiKeyExpiration: "Expirare",
  formFieldLabel__apiKeyName: "Numele cheii secrete",
  formFieldLabel__automaticInvitations: "Activeaz\u0103 invita\u021Bii automate pentru acest domeniu",
  formFieldLabel__backupCode: "Cod de rezerv\u0103",
  formFieldLabel__confirmDeletion: "Confirmare",
  formFieldLabel__confirmPassword: "Confirm\u0103 parola",
  formFieldLabel__currentPassword: "Parola actual\u0103",
  formFieldLabel__emailAddress: "Adres\u0103 de email",
  formFieldLabel__emailAddress_username: "Adres\u0103 de email sau nume de utilizator",
  formFieldLabel__emailAddresses: "Adrese de email",
  formFieldLabel__firstName: "Prenume",
  formFieldLabel__lastName: "Nume",
  formFieldLabel__newPassword: "Parol\u0103 nou\u0103",
  formFieldLabel__organizationDomain: "Domeniu",
  formFieldLabel__organizationDomainDeletePending: "\u0218terge invita\u021Biile \u0219i sugestiile \xEEn a\u0219teptare",
  formFieldLabel__organizationDomainEmailAddress: "Adres\u0103 email de verificare",
  formFieldLabel__organizationDomainEmailAddressDescription: "Introdu o adres\u0103 de email sub acest domeniu pentru a primi un cod \u0219i a verifica domeniul.",
  formFieldLabel__organizationName: "Nume",
  formFieldLabel__organizationSlug: "Slug",
  formFieldLabel__passkeyName: "Numele cheii de acces",
  formFieldLabel__password: "Parol\u0103",
  formFieldLabel__phoneNumber: "Num\u0103r de telefon",
  formFieldLabel__role: "Rol",
  formFieldLabel__signOutOfOtherSessions: "Deconecteaz\u0103 celelalte dispozitive",
  formFieldLabel__username: "Nume de utilizator",
  impersonationFab: {
    action__signOut: "Deconecteaz\u0103-te",
    title: "Autentificat ca {{identifier}}"
  },
  lastAuthenticationStrategy: "Ultima utilizare",
  maintenanceMode: "Efectu\u0103m lucr\u0103ri de mentenan\u021B\u0103, dar nu va dura mai mult de c\xE2teva minute.",
  membershipRole__admin: "Administrator",
  membershipRole__basicMember: "Membru",
  membershipRole__guestMember: "Invitat",
  organizationList: {
    action__createOrganization: "Creeaz\u0103 organiza\u021Bie",
    action__invitationAccept: "Al\u0103tur\u0103-te",
    action__suggestionsAccept: "Solicit\u0103 al\u0103turarea",
    createOrganization: "Creeaz\u0103 organiza\u021Bie",
    invitationAcceptedLabel: "Te-ai al\u0103turat",
    subtitle: "pentru a continua la {{applicationName}}",
    suggestionsAcceptedLabel: "\xCEn a\u0219teptare",
    title: "Alege un cont",
    titleWithoutPersonal: "Alege o organiza\u021Bie"
  },
  organizationProfile: {
    apiKeysPage: {
      title: "Chei API"
    },
    badge__automaticInvitation: "Invita\u021Bii automate",
    badge__automaticSuggestion: "Sugestii automate",
    badge__manualInvitation: "F\u0103r\u0103 \xEEnscriere automat\u0103",
    badge__unverified: "Neverificat",
    billingPage: {
      paymentHistorySection: {
        empty: "Nu exist\u0103 istoric de pl\u0103\u021Bi",
        notFound: "Plata nu a fost g\u0103sit\u0103",
        tableHeader__amount: "Sum\u0103",
        tableHeader__date: "Dat\u0103",
        tableHeader__status: "Stare"
      },
      paymentMethodsSection: {
        actionLabel__default: "Seteaz\u0103 ca implicit\u0103",
        actionLabel__remove: "Elimin\u0103",
        add: "Adaug\u0103 metod\u0103 de plat\u0103 nou\u0103",
        addSubtitle: "Adaug\u0103 o nou\u0103 metod\u0103 de plat\u0103 la cont.",
        cancelButton: "Anuleaz\u0103",
        formButtonPrimary__add: "Adaug\u0103 metoda de plat\u0103",
        formButtonPrimary__pay: "Pl\u0103te\u0219te {{amount}}",
        payWithTestCardButton: "Pl\u0103te\u0219te cu card de test",
        removeMethod: {
          messageLine1: "{{identifier}} va fi eliminat(\u0103) din acest cont.",
          messageLine2: "Nu vei mai putea folosi aceast\u0103 metod\u0103 de plat\u0103, iar abonamentele recurente dependente de ea nu vor mai func\u021Biona.",
          successMessage: "{{paymentMethod}} a fost eliminat\u0103 din contul t\u0103u.",
          title: "Elimin\u0103 metoda de plat\u0103"
        },
        title: "Metode de plat\u0103"
      },
      start: {
        headerTitle__payments: "Pl\u0103\u021Bi",
        headerTitle__plans: "Planuri",
        headerTitle__statements: "Extrase",
        headerTitle__subscriptions: "Abonament"
      },
      statementsSection: {
        empty: "Nu exist\u0103 extrase de afi\u0219at",
        itemCaption__paidForPlan: "Pl\u0103tit pentru planul {{plan}} ({{period}})",
        itemCaption__proratedCredit: "Credit propor\u021Bional pentru utilizarea par\u021Bial\u0103 a abonamentului anterior",
        itemCaption__subscribedAndPaidForPlan: "Abonat \u0219i pl\u0103tit pentru planul {{plan}} ({{period}})",
        notFound: "Extrasul nu a fost g\u0103sit",
        tableHeader__amount: "Sum\u0103",
        tableHeader__date: "Dat\u0103",
        title: "Extrase",
        totalPaid: "Total pl\u0103tit"
      },
      subscriptionsListSection: {
        actionLabel__manageSubscription: void 0,
        actionLabel__newSubscription: "Aboneaz\u0103-te la un plan",
        actionLabel__switchPlan: "Schimb\u0103 planurile",
        tableHeader__edit: "Editeaz\u0103",
        tableHeader__plan: "Plan",
        tableHeader__startDate: "Data \xEEnceperii",
        title: "Abonament"
      },
      subscriptionsSection: {
        actionLabel__default: "Gestioneaz\u0103"
      },
      switchPlansSection: {
        title: "Schimb\u0103 planurile"
      },
      title: "Facturare"
    },
    createDomainPage: {
      subtitle: "Adaug\u0103 domeniul pentru verificare. Utilizatorii cu adrese de email pe acest domeniu se pot al\u0103tura automat sau pot solicita al\u0103turarea.",
      title: "Adaug\u0103 domeniu"
    },
    invitePage: {
      detailsTitle__inviteFailed: "Invita\u021Biile nu au putut fi trimise. Exist\u0103 deja invita\u021Bii \xEEn a\u0219teptare pentru urm\u0103toarele adrese: {{email_addresses}}.",
      formButtonPrimary__continue: "Trimite invita\u021Biile",
      selectDropdown__role: "Selecteaz\u0103 rolul",
      subtitle: "Introdu sau lipe\u0219te una sau mai multe adrese de email, separate prin spa\u021Bii sau virgule.",
      successMessage: "Invita\u021Biile au fost trimise cu succes",
      title: "Invit\u0103 membri noi"
    },
    membersPage: {
      action__invite: "Invit\u0103",
      action__search: "Caut\u0103",
      activeMembersTab: {
        menuAction__remove: "Elimin\u0103 membrul",
        tableHeader__actions: "Ac\u021Biuni",
        tableHeader__joined: "Al\u0103turat",
        tableHeader__role: "Rol",
        tableHeader__user: "Utilizator"
      },
      detailsTitle__emptyRow: "Nu sunt membri de afi\u0219at",
      invitationsTab: {
        autoInvitations: {
          headerSubtitle: "Invit\u0103 utilizatori conect\xE2nd un domeniu de email la organiza\u021Bie. Oricine se \xEEnscrie cu un email ce se potrive\u0219te va putea intra oric\xE2nd.",
          headerTitle: "Invita\u021Bii automate",
          primaryButton: "Gestioneaz\u0103 domeniile verificate"
        },
        table__emptyRow: "Nu sunt invita\u021Bii de afi\u0219at"
      },
      invitedMembersTab: {
        menuAction__revoke: "Revoc\u0103 invita\u021Bia",
        tableHeader__invited: "Invitat"
      },
      requestsTab: {
        autoSuggestions: {
          headerSubtitle: "Utilizatorii care se \xEEnscriu cu un email ce se potrive\u0219te pot primi sugestia de a cere acces.",
          headerTitle: "Sugestii automate",
          primaryButton: "Gestioneaz\u0103 domeniile verificate"
        },
        menuAction__approve: "Aprob\u0103",
        menuAction__reject: "Respinge",
        tableHeader__requested: "Acces solicitat",
        table__emptyRow: "Nu sunt cereri de afi\u0219at"
      },
      start: {
        headerTitle__invitations: "Invita\u021Bii",
        headerTitle__members: "Membri",
        headerTitle__requests: "Ceri de acces"
      }
    },
    navbar: {
      apiKeys: "Chei API",
      billing: "Facturare",
      description: "Gestioneaz\u0103 organiza\u021Bia.",
      general: "General",
      members: "Membri",
      title: "Organiza\u021Bie"
    },
    plansPage: {
      alerts: {
        noPermissionsToManageBilling: "Nu ai permisiuni pentru a gestiona facturarea acestei organiza\u021Bii."
      },
      title: "Planuri"
    },
    profilePage: {
      dangerSection: {
        deleteOrganization: {
          actionDescription: "Tasteaz\u0103 \u201E{{organizationName}}\u201D mai jos pentru a continua.",
          messageLine1: "Sigur vrei s\u0103 \u0219tergi aceast\u0103 organiza\u021Bie?",
          messageLine2: "Aceast\u0103 ac\u021Biune este permanent\u0103 \u0219i ireversibil\u0103.",
          successMessage: "Ai \u0219ters organiza\u021Bia.",
          title: "\u0218terge organiza\u021Bia"
        },
        leaveOrganization: {
          actionDescription: "Tasteaz\u0103 \u201E{{organizationName}}\u201D mai jos pentru a continua.",
          messageLine1: "Sigur vrei s\u0103 p\u0103r\u0103se\u0219ti aceast\u0103 organiza\u021Bie? Vei pierde accesul la organiza\u021Bie \u0219i aplica\u021Biile ei.",
          messageLine2: "Aceast\u0103 ac\u021Biune este permanent\u0103 \u0219i ireversibil\u0103.",
          successMessage: "Ai p\u0103r\u0103sit organiza\u021Bia.",
          title: "P\u0103r\u0103se\u0219te organiza\u021Bia"
        },
        title: "Pericol"
      },
      domainSection: {
        menuAction__manage: "Gestioneaz\u0103",
        menuAction__remove: "\u0218terge",
        menuAction__verify: "Verific\u0103",
        primaryButton: "Adaug\u0103 domeniu",
        subtitle: "Permite utilizatorilor s\u0103 se al\u0103ture automat sau s\u0103 solicite al\u0103turarea pe baza unui domeniu de email verificat.",
        title: "Domenii verificate"
      },
      successMessage: "Organiza\u021Bia a fost actualizat\u0103.",
      title: "Actualizeaz\u0103 profilul"
    },
    removeDomainPage: {
      messageLine1: "Domeniul de email {{domain}} va fi eliminat.",
      messageLine2: "Utilizatorii nu se vor mai putea al\u0103tura automat organiza\u021Biei dup\u0103 aceasta.",
      successMessage: "{{domain}} a fost eliminat.",
      title: "Elimin\u0103 domeniul"
    },
    start: {
      headerTitle__general: "General",
      headerTitle__members: "Membri",
      profileSection: {
        primaryButton: "Actualizeaz\u0103 profilul",
        title: "Profilul organiza\u021Biei",
        uploadAction__title: "Logo"
      }
    },
    verifiedDomainPage: {
      dangerTab: {
        calloutInfoLabel: "Eliminarea acestui domeniu va afecta utilizatorii invita\u021Bi.",
        removeDomainActionLabel__remove: "Elimin\u0103 domeniul",
        removeDomainSubtitle: "Elimin\u0103 acest domeniu din domeniile verificate",
        removeDomainTitle: "Elimin\u0103 domeniul"
      },
      enrollmentTab: {
        automaticInvitationOption__description: "Utilizatorii sunt invita\u021Bi automat s\u0103 se al\u0103ture \u0219i pot intra oric\xE2nd.",
        automaticInvitationOption__label: "Invita\u021Bii automate",
        automaticSuggestionOption__description: "Utilizatorii primesc o sugestie de a solicita al\u0103turarea, dar trebuie aproba\u021Bi de un administrator.",
        automaticSuggestionOption__label: "Sugestii automate",
        calloutInfoLabel: "Schimbarea modului de \xEEnscriere va afecta doar utilizatorii noi.",
        calloutInvitationCountLabel: "Invita\u021Bii \xEEn a\u0219teptare trimise utilizatorilor: {{count}}",
        calloutSuggestionCountLabel: "Sugestii \xEEn a\u0219teptare trimise utilizatorilor: {{count}}",
        manualInvitationOption__description: "Utilizatorii pot fi invita\u021Bi doar manual \xEEn organiza\u021Bie.",
        manualInvitationOption__label: "F\u0103r\u0103 \xEEnscriere automat\u0103",
        subtitle: "Alege cum pot utilizatorii din acest domeniu s\u0103 se al\u0103ture organiza\u021Biei."
      },
      start: {
        headerTitle__danger: "Pericol",
        headerTitle__enrollment: "Op\u021Biuni de \xEEnscriere"
      },
      subtitle: "Domeniul {{domain}} a fost verificat. Continu\u0103 select\xE2nd modul de \xEEnscriere.",
      title: "Actualizeaz\u0103 {{domain}}"
    },
    verifyDomainPage: {
      formSubtitle: "Introdu codul de verificare trimis la adresa ta de email",
      formTitle: "Cod de verificare",
      resendButton: "Nu ai primit un cod? Retrimite",
      subtitle: "Domeniul {{domainName}} trebuie verificat prin email.",
      subtitleVerificationCodeScreen: "Un cod de verificare a fost trimis la {{emailAddress}}. Introdu codul pentru a continua.",
      title: "Verific\u0103 domeniul"
    }
  },
  organizationSwitcher: {
    action__closeOrganizationSwitcher: void 0,
    action__createOrganization: "Creeaz\u0103 organiza\u021Bie",
    action__invitationAccept: "Al\u0103tur\u0103-te",
    action__manageOrganization: "Gestioneaz\u0103",
    action__openOrganizationSwitcher: void 0,
    action__suggestionsAccept: "Solicit\u0103 al\u0103turarea",
    notSelected: "Nicio organiza\u021Bie selectat\u0103",
    personalWorkspace: "Cont personal",
    suggestionsAcceptedLabel: "\xCEn a\u0219teptare"
  },
  paginationButton__next: "\xCEnainte",
  paginationButton__previous: "\xCEnapoi",
  paginationRowText__displaying: "Afi\u0219are",
  paginationRowText__of: "din",
  reverification: {
    alternativeMethods: {
      actionLink: "Ob\u021Bine ajutor",
      actionText: "Nu ai niciuna dintre acestea?",
      blockButton__backupCode: "Folose\u0219te un cod de rezerv\u0103",
      blockButton__emailCode: "Trimite cod pe email c\u0103tre {{identifier}}",
      blockButton__passkey: "Folose\u0219te cheia de acces",
      blockButton__password: "Continu\u0103 cu parola",
      blockButton__phoneCode: "Trimite cod prin SMS la {{identifier}}",
      blockButton__totp: "Folose\u0219te aplica\u021Bia de autentificare",
      getHelp: {
        blockButton__emailSupport: "Trimite email la suport",
        content: "Dac\u0103 ai probleme la verificare, scrie-ne pe email \u0219i te ajut\u0103m s\u0103-\u021Bi recape\u021Bi accesul c\xE2t mai repede.",
        title: "Ajutor"
      },
      subtitle: "Ai probleme? Po\u021Bi folosi oricare dintre aceste metode pentru verificare.",
      title: "Folose\u0219te alt\u0103 metod\u0103"
    },
    backupCodeMfa: {
      subtitle: "Introdu codul de rezerv\u0103 primit c\xE2nd ai configurat autentificarea \xEEn doi pa\u0219i",
      title: "Introdu un cod de rezerv\u0103"
    },
    emailCode: {
      formTitle: "Cod de verificare",
      resendButton: "Nu ai primit un cod? Retrimite",
      subtitle: "Introdu codul trimis pe email pentru a continua",
      title: "Verificare necesar\u0103"
    },
    noAvailableMethods: {
      message: "Nu se poate continua. Nu exist\u0103 un factor de autentificare configurat.",
      subtitle: "A ap\u0103rut o eroare",
      title: "Nu \xEE\u021Bi putem verifica contul"
    },
    passkey: {
      blockButton__passkey: "Folose\u0219te cheia de acces",
      subtitle: "Folosirea cheii de acces confirm\u0103 identitatea ta. Dispozitivul \xEE\u021Bi poate cere amprenta, fa\u021Ba sau codul de ecran.",
      title: "Folose\u0219te cheia de acces"
    },
    password: {
      actionLink: "Folose\u0219te alt\u0103 metod\u0103",
      subtitle: "Introdu parola curent\u0103 pentru a continua",
      title: "Verificare necesar\u0103"
    },
    phoneCode: {
      formTitle: "Cod de verificare",
      resendButton: "Nu ai primit un cod? Retrimite",
      subtitle: "Introdu codul trimis pe telefon pentru a continua",
      title: "Verificare necesar\u0103"
    },
    phoneCodeMfa: {
      formTitle: "Cod de verificare",
      resendButton: "Nu ai primit un cod? Retrimite",
      subtitle: "Introdu codul trimis pe telefon pentru a continua",
      title: "Verificare necesar\u0103"
    },
    totpMfa: {
      formTitle: "Cod de verificare",
      subtitle: "Introdu codul generat de aplica\u021Bia ta de autentificare pentru a continua",
      title: "Verificare \xEEn doi pa\u0219i"
    }
  },
  signIn: {
    accountSwitcher: {
      action__addAccount: "Adaug\u0103 cont",
      action__signOutAll: "Deconecteaz\u0103 toate conturile",
      subtitle: "Selecteaz\u0103 contul cu care vrei s\u0103 continui.",
      title: "Alege un cont"
    },
    alternativeMethods: {
      actionLink: "Ajutor",
      actionText: "Nu ai niciuna dintre acestea?",
      blockButton__backupCode: "Folose\u0219te un cod de rezerv\u0103",
      blockButton__emailCode: "Trimite cod pe email c\u0103tre {{identifier}}",
      blockButton__emailLink: "Trimite link pe email c\u0103tre {{identifier}}",
      blockButton__passkey: "Autentificare cu cheia de acces",
      blockButton__password: "Autentificare cu parola",
      blockButton__phoneCode: "Trimite cod prin SMS la {{identifier}}",
      blockButton__totp: "Folose\u0219te aplica\u021Bia de autentificare",
      getHelp: {
        blockButton__emailSupport: "Trimite email la suport",
        content: "Dac\u0103 ai probleme la autentificare, scrie-ne pe email \u0219i te ajut\u0103m s\u0103-\u021Bi recape\u021Bi accesul c\xE2t mai repede.",
        title: "Ajutor"
      },
      subtitle: "Ai probleme? Po\u021Bi folosi oricare dintre aceste metode pentru autentificare.",
      title: "Folose\u0219te alt\u0103 metod\u0103"
    },
    alternativePhoneCodeProvider: {
      formTitle: "Cod de verificare",
      resendButton: "Nu ai primit un cod? Retrimite",
      subtitle: "pentru a continua la {{applicationName}}",
      title: "Verific\u0103-\u021Bi {{provider}}"
    },
    backupCodeMfa: {
      subtitle: "Codul de rezerv\u0103 este cel pe care l-ai primit la configurarea autentific\u0103rii \xEEn doi pa\u0219i.",
      title: "Introdu un cod de rezerv\u0103"
    },
    emailCode: {
      formTitle: "Cod de verificare",
      resendButton: "Nu ai primit un cod? Retrimite",
      subtitle: "pentru a continua la {{applicationName}}",
      title: "Verific\u0103-\u021Bi emailul"
    },
    emailLink: {
      clientMismatch: {
        subtitle: "Pentru a continua, deschide linkul de verificare pe dispozitivul \u0219i \xEEn browserul de pe care ai \xEEnceput autentificarea",
        title: "Linkul de verificare nu este valid pe acest dispozitiv"
      },
      expired: {
        subtitle: "Revino \xEEn fila ini\u021Bial\u0103 pentru a continua.",
        title: "Linkul de verificare a expirat"
      },
      failed: {
        subtitle: "Revino \xEEn fila ini\u021Bial\u0103 pentru a continua.",
        title: "Linkul de verificare este invalid"
      },
      formSubtitle: "Folose\u0219te linkul de verificare trimis pe email",
      formTitle: "Link de verificare",
      loading: {
        subtitle: "Vei fi redirec\u021Bionat \xEEn cur\xE2nd",
        title: "Autentificare..."
      },
      resendButton: "Nu ai primit un link? Retrimite",
      subtitle: "pentru a continua la {{applicationName}}",
      title: "Verific\u0103-\u021Bi emailul",
      unusedTab: {
        title: "Po\u021Bi \xEEnchide aceast\u0103 fil\u0103"
      },
      verified: {
        subtitle: "Vei fi redirec\u021Bionat \xEEn cur\xE2nd",
        title: "Autentificat cu succes"
      },
      verifiedSwitchTab: {
        subtitle: "Revino \xEEn fila original\u0103 pentru a continua",
        subtitleNewTab: "Revino \xEEn noua fil\u0103 deschis\u0103 pentru a continua",
        titleNewTab: "Autentificat \xEEn alt\u0103 fil\u0103"
      }
    },
    enterpriseConnections: {
      subtitle: void 0,
      title: void 0
    },
    forgotPassword: {
      formTitle: "Cod pentru resetarea parolei",
      resendButton: "Nu ai primit un cod? Retrimite",
      subtitle: "pentru a-\u021Bi reseta parola",
      subtitle_email: "Mai \xEEnt\xE2i introdu codul trimis la adresa de email",
      subtitle_phone: "Mai \xEEnt\xE2i introdu codul trimis pe telefon",
      title: "Reseteaz\u0103 parola"
    },
    forgotPasswordAlternativeMethods: {
      blockButton__resetPassword: "Reseteaz\u0103 parola",
      label__alternativeMethods: "Sau autentific\u0103-te cu alt\u0103 metod\u0103",
      title: "Ai uitat parola?"
    },
    noAvailableMethods: {
      message: "Nu se poate continua autentificarea. Nu exist\u0103 niciun factor de autentificare disponibil.",
      subtitle: "A ap\u0103rut o eroare",
      title: "Nu te po\u021Bi autentifica"
    },
    passkey: {
      subtitle: "Folosirea cheii de acces confirm\u0103 c\u0103 e\u0219ti tu. Dispozitivul \xEE\u021Bi poate cere amprenta, fa\u021Ba sau blocarea ecranului.",
      title: "Folose\u0219te cheia de acces"
    },
    password: {
      actionLink: "Folose\u0219te alt\u0103 metod\u0103",
      subtitle: "Introdu parola asociat\u0103 contului t\u0103u",
      title: "Introdu parola"
    },
    passwordPwned: {
      title: "Parola este compromis\u0103"
    },
    phoneCode: {
      formTitle: "Cod de verificare",
      resendButton: "Nu ai primit un cod? Retrimite",
      subtitle: "pentru a continua la {{applicationName}}",
      title: "Verific\u0103-\u021Bi telefonul"
    },
    phoneCodeMfa: {
      formTitle: "Cod de verificare",
      resendButton: "Nu ai primit un cod? Retrimite",
      subtitle: "Pentru a continua, introdu codul de verificare trimis pe telefon",
      title: "Verific\u0103-\u021Bi telefonul"
    },
    resetPassword: {
      formButtonPrimary: "Reseteaz\u0103 parola",
      requiredMessage: "Din motive de securitate, este necesar s\u0103 \xEE\u021Bi resetezi parola.",
      successMessage: "Parola a fost schimbat\u0103 cu succes. Te autentific\u0103m, te rug\u0103m a\u0219teapt\u0103 un moment.",
      title: "Seteaz\u0103 parol\u0103 nou\u0103"
    },
    resetPasswordMfa: {
      detailsLabel: "Trebuie s\u0103 \xEE\u021Bi verific\u0103m identitatea \xEEnainte de a reseta parola."
    },
    start: {
      actionLink: "\xCEnregistreaz\u0103-te",
      actionLink__join_waitlist: "\xCEnscrie-te pe lista de a\u0219teptare",
      actionLink__use_email: "Folose\u0219te emailul",
      actionLink__use_email_username: "Folose\u0219te email sau nume de utilizator",
      actionLink__use_passkey: "Folose\u0219te cheia de acces",
      actionLink__use_phone: "Folose\u0219te telefonul",
      actionLink__use_username: "Folose\u0219te numele de utilizator",
      actionText: "Nu ai cont?",
      actionText__join_waitlist: "Vrei acces timpuriu?",
      alternativePhoneCodeProvider: {
        actionLink: "Folose\u0219te alt\u0103 metod\u0103",
        label: "Num\u0103r de telefon {{provider}}",
        subtitle: "Introdu num\u0103rul t\u0103u de telefon pentru a primi un cod de verificare pe {{provider}}.",
        title: "Autentific\u0103-te \xEEn {{applicationName}} cu {{provider}}"
      },
      subtitle: "Bine ai revenit! Te rug\u0103m s\u0103 te autentifici pentru a continua",
      subtitleCombined: void 0,
      title: "Autentific\u0103-te \xEEn {{applicationName}}",
      titleCombined: "Continu\u0103 c\u0103tre {{applicationName}}"
    },
    totpMfa: {
      formTitle: "Cod de verificare",
      subtitle: "Pentru a continua, introdu codul generat de aplica\u021Bia ta de autentificare",
      title: "Verificare \xEEn doi pa\u0219i"
    }
  },
  signInEnterPasswordTitle: "Introdu parola",
  signUp: {
    alternativePhoneCodeProvider: {
      resendButton: "Nu ai primit un cod? Retrimite",
      subtitle: "Introdu codul de verificare trimis pe {{provider}}",
      title: "Verific\u0103-\u021Bi {{provider}}"
    },
    continue: {
      actionLink: "Autentific\u0103-te",
      actionText: "Ai deja cont?",
      subtitle: "Completeaz\u0103 c\xE2mpurile r\u0103mase pentru a continua.",
      title: "Completeaz\u0103 c\xE2mpurile lips\u0103"
    },
    emailCode: {
      formSubtitle: "Introdu codul de verificare trimis la adresa ta de email",
      formTitle: "Cod de verificare",
      resendButton: "Nu ai primit un cod? Retrimite",
      subtitle: "Introdu codul de verificare trimis pe email",
      title: "Verific\u0103-\u021Bi emailul"
    },
    emailLink: {
      clientMismatch: {
        subtitle: "Pentru a continua, deschide linkul de verificare pe dispozitivul \u0219i \xEEn browserul de pe care ai \xEEnceput \xEEnregistrarea",
        title: "Linkul de verificare nu este valid pe acest dispozitiv"
      },
      formSubtitle: "Folose\u0219te linkul de verificare trimis la adresa ta de email",
      formTitle: "Link de verificare",
      loading: {
        title: "\xCEnregistrare..."
      },
      resendButton: "Nu ai primit un link? Retrimite",
      subtitle: "pentru a continua la {{applicationName}}",
      title: "Verific\u0103-\u021Bi emailul",
      verified: {
        title: "\xCEnregistrare reu\u0219it\u0103"
      },
      verifiedSwitchTab: {
        subtitle: "Revino \xEEn noua fil\u0103 deschis\u0103 pentru a continua",
        subtitleNewTab: "Revino \xEEn fila precedent\u0103 pentru a continua",
        title: "Email verificat cu succes"
      }
    },
    enterpriseConnections: {
      subtitle: void 0,
      title: void 0
    },
    legalConsent: {
      checkbox: {
        label__onlyPrivacyPolicy: 'Sunt de acord cu {{ privacyPolicyLink || link("Politica de confiden\u021Bialitate") }}',
        label__onlyTermsOfService: 'Sunt de acord cu {{ termsOfServiceLink || link("Termenii \u0219i condi\u021Biile") }}',
        label__termsOfServiceAndPrivacyPolicy: 'Sunt de acord cu {{ termsOfServiceLink || link("Termenii \u0219i condi\u021Biile") }} \u0219i {{ privacyPolicyLink || link("Politica de confiden\u021Bialitate") }}'
      },
      continue: {
        subtitle: "Cite\u0219te \u0219i accept\u0103 termenii pentru a continua",
        title: "Consim\u021B\u0103m\xE2nt legal"
      }
    },
    phoneCode: {
      formSubtitle: "Introdu codul de verificare trimis pe telefon",
      formTitle: "Cod de verificare",
      resendButton: "Nu ai primit un cod? Retrimite",
      subtitle: "Introdu codul de verificare trimis pe telefon",
      title: "Verific\u0103-\u021Bi telefonul"
    },
    restrictedAccess: {
      actionLink: "Autentific\u0103-te",
      actionText: "Ai deja cont?",
      blockButton__emailSupport: "Trimite email la suport",
      blockButton__joinWaitlist: "\xCEnscrie-te pe lista de a\u0219teptare",
      subtitle: "\xCEnregistr\u0103rile sunt momentan dezactivate. Dac\u0103 crezi c\u0103 ar trebui s\u0103 ai acces, contacteaz\u0103 suportul.",
      subtitleWaitlist: "\xCEnregistr\u0103rile sunt momentan dezactivate. Pentru a fi primul care afl\u0103 c\xE2nd lans\u0103m, \xEEnscrie-te pe lista de a\u0219teptare.",
      title: "Acces restric\u021Bionat"
    },
    start: {
      actionLink: "Autentific\u0103-te",
      actionLink__use_email: "Folose\u0219te emailul",
      actionLink__use_phone: "Folose\u0219te telefonul",
      actionText: "Ai deja cont?",
      alternativePhoneCodeProvider: {
        actionLink: "Folose\u0219te alt\u0103 metod\u0103",
        label: "Num\u0103r de telefon {{provider}}",
        subtitle: "Introdu num\u0103rul t\u0103u de telefon pentru a primi un cod de verificare pe {{provider}}.",
        title: "\xCEnregistreaz\u0103-te \xEEn {{applicationName}} cu {{provider}}"
      },
      subtitle: "Bun venit! Completeaz\u0103 detaliile pentru a \xEEncepe.",
      subtitleCombined: "Bun venit! Completeaz\u0103 detaliile pentru a \xEEncepe.",
      title: "Creeaz\u0103-\u021Bi contul",
      titleCombined: "Creeaz\u0103-\u021Bi contul"
    }
  },
  socialButtonsBlockButton: "Continu\u0103 cu {{provider|titleize}}",
  socialButtonsBlockButtonManyInView: "{{provider|titleize}}",
  taskChooseOrganization: {
    chooseOrganization: {
      action__createOrganization: "Creeaz\u0103 organiza\u021Bie nou\u0103",
      action__invitationAccept: "Al\u0103tur\u0103-te",
      action__suggestionsAccept: "Solicit\u0103 al\u0103turarea",
      subtitle: "Al\u0103tur\u0103-te unei organiza\u021Bii existente sau creeaz\u0103 una nou\u0103",
      suggestionsAcceptedLabel: "\xCEn a\u0219teptare",
      title: "Alege o organiza\u021Bie"
    },
    createOrganization: {
      formButtonReset: "Anuleaz\u0103",
      formButtonSubmit: "Continu\u0103",
      formFieldInputPlaceholder__name: "Organiza\u021Bia mea",
      formFieldInputPlaceholder__slug: "organiza\u021Bia-mea",
      formFieldLabel__name: "Nume",
      formFieldLabel__slug: "Slug",
      subtitle: "Introdu detaliile organiza\u021Biei pentru a continua",
      title: "Configureaz\u0103-\u021Bi organiza\u021Bia"
    },
    signOut: {
      actionLink: "Deconecteaz\u0103-te",
      actionText: "Autentificat ca {{identifier}}"
    }
  },
  unstable__errors: {
    already_a_member_in_organization: "{{email}} este deja membru al organiza\u021Biei.",
    captcha_invalid: void 0,
    captcha_unavailable: "\xCEnregistrarea a e\u0219uat din cauza valid\u0103rii anti-bot. Re\xEEmprosp\u0103teaz\u0103 pagina sau contacteaz\u0103 suportul.",
    form_code_incorrect: void 0,
    form_identifier_exists__email_address: void 0,
    form_identifier_exists__phone_number: void 0,
    form_identifier_exists__username: void 0,
    form_identifier_not_found: void 0,
    form_param_format_invalid: void 0,
    form_param_format_invalid__email_address: void 0,
    form_param_format_invalid__phone_number: void 0,
    form_param_max_length_exceeded__first_name: void 0,
    form_param_max_length_exceeded__last_name: void 0,
    form_param_max_length_exceeded__name: void 0,
    form_param_nil: void 0,
    form_param_type_invalid: void 0,
    form_param_type_invalid__email_address: void 0,
    form_param_type_invalid__phone_number: void 0,
    form_param_value_invalid: void 0,
    form_password_incorrect: void 0,
    form_password_length_too_short: "Parola este prea scurt\u0103. Trebuie s\u0103 aib\u0103 cel pu\u021Bin 8 caractere.",
    form_password_not_strong_enough: "Parola ta nu este suficient de puternic\u0103.",
    form_password_pwned: "Aceast\u0103 parol\u0103 a fost g\u0103sit\u0103 \xEEntr-o bre\u0219\u0103 de securitate \u0219i nu poate fi folosit\u0103. Te rug\u0103m alege alta.",
    form_password_pwned__sign_in: "Aceast\u0103 parol\u0103 a fost g\u0103sit\u0103 \xEEntr-o bre\u0219\u0103 de securitate \u0219i nu poate fi folosit\u0103. Te rug\u0103m reseteaz\u0103 parola.",
    form_password_size_in_bytes_exceeded: void 0,
    form_password_validation_failed: void 0,
    form_username_invalid_character: void 0,
    form_username_invalid_length: "Numele de utilizator trebuie s\u0103 aib\u0103 \xEEntre {{min_length}} \u0219i {{max_length}} caractere.",
    identification_deletion_failed: void 0,
    not_allowed_access: void 0,
    organization_domain_blocked: void 0,
    organization_domain_common: void 0,
    organization_domain_exists_for_enterprise_connection: void 0,
    organization_membership_quota_exceeded: void 0,
    organization_minimum_permissions_needed: void 0,
    organization_not_found_or_unauthorized: "Nu mai e\u0219ti membru al acestei organiza\u021Bii. Te rug\u0103m alege sau creeaz\u0103 alta.",
    organization_not_found_or_unauthorized_with_create_organization_disabled: "Nu mai e\u0219ti membru al acestei organiza\u021Bii. Te rug\u0103m alege alta.",
    passkey_already_exists: "Exist\u0103 deja o cheie de acces \xEEnregistrat\u0103 pe acest dispozitiv.",
    passkey_not_supported: "Cheile de acces nu sunt acceptate pe acest dispozitiv.",
    passkey_pa_not_supported: "\xCEnregistrarea necesit\u0103 un autentificator de platform\u0103, dar dispozitivul nu \xEEl suport\u0103.",
    passkey_registration_cancelled: "\xCEnregistrarea cheii de acces a fost anulat\u0103 sau a expirat.",
    passkey_retrieval_cancelled: "Verificarea cu cheia de acces a fost anulat\u0103 sau a expirat.",
    passwordComplexity: {
      maximumLength: "mai pu\u021Bin de {{length}} caractere",
      minimumLength: "{{length}} sau mai multe caractere",
      requireLowercase: "o liter\u0103 mic\u0103",
      requireNumbers: "o cifr\u0103",
      requireSpecialCharacter: "un caracter special",
      requireUppercase: "o liter\u0103 mare",
      sentencePrefix: "Parola ta trebuie s\u0103 con\u021Bin\u0103"
    },
    phone_number_exists: void 0,
    session_exists: void 0,
    web3_missing_identifier: "Nu am g\u0103sit o extensie pentru portofel Web3. Te rug\u0103m instaleaz\u0103 una pentru a continua.",
    zxcvbn: {
      couldBeStronger: "Parola ta func\u021Bioneaz\u0103, dar ar putea fi mai puternic\u0103. \xCEncearc\u0103 s\u0103 adaugi mai multe caractere.",
      goodPassword: "Parola ta \xEEndepline\u0219te toate cerin\u021Bele necesare.",
      notEnough: "Parola ta nu este suficient de puternic\u0103.",
      suggestions: {
        allUppercase: "Scrie cu majuscule unele, dar nu toate literele.",
        anotherWord: "Adaug\u0103 mai multe cuvinte mai pu\u021Bin comune.",
        associatedYears: "Evit\u0103 anii asocia\u021Bi cu tine.",
        capitalization: "Folose\u0219te majuscule \u0219i \xEEn alte pozi\u021Bii dec\xE2t prima liter\u0103.",
        dates: "Evit\u0103 datele \u0219i anii asocia\u021Bi cu tine.",
        l33t: "Evit\u0103 substitu\u021Biile previzibile, precum '@' pentru 'a'.",
        longerKeyboardPattern: "Folose\u0219te modele de tast\u0103 mai lungi \u0219i schimb\u0103 direc\u021Bia.",
        noNeed: "Po\u021Bi crea parole puternice f\u0103r\u0103 simboluri, cifre sau litere mari.",
        pwned: "Dac\u0103 folose\u0219ti aceast\u0103 parol\u0103 \xEEn alt\u0103 parte, ar trebui s\u0103 o schimbi.",
        recentYears: "Evit\u0103 anii recen\u021Bi.",
        repeated: "Evit\u0103 cuvintele \u0219i caracterele repetate.",
        reverseWords: "Evit\u0103 inversarea ortografiei cuvintelor comune.",
        sequences: "Evit\u0103 secven\u021Bele comune de caractere.",
        useWords: "Folose\u0219te mai multe cuvinte, dar evit\u0103 expresiile comune."
      },
      warnings: {
        common: "Aceasta este o parol\u0103 des folosit\u0103.",
        commonNames: "Numele \u0219i prenumele comune sunt u\u0219or de ghicit.",
        dates: "Datele sunt u\u0219or de ghicit.",
        extendedRepeat: "Modelele repetate de caractere precum \u201Eabcabcabc\u201D sunt u\u0219or de ghicit.",
        keyPattern: "Modelele scurte pe tastatur\u0103 sunt u\u0219or de ghicit.",
        namesByThemselves: "Numele sau prenumele singure sunt u\u0219or de ghicit.",
        pwned: "Parola ta a fost expus\u0103 \xEEntr-o bre\u0219\u0103 de date pe Internet.",
        recentYears: "Anul recent este u\u0219or de ghicit.",
        sequences: "Secven\u021Bele comune (de ex. \u201Eabc\u201D) sunt u\u0219or de ghicit.",
        similarToCommon: "Este similar\u0103 cu o parol\u0103 des folosit\u0103.",
        simpleRepeat: "Caracterele repetate (de ex. \u201Eaaa\u201D) sunt u\u0219or de ghicit.",
        straightRow: "\u0218irurile drepte de taste sunt u\u0219or de ghicit.",
        topHundred: "Aceasta este o parol\u0103 frecvent folosit\u0103.",
        topTen: "Aceasta este o parol\u0103 foarte des folosit\u0103.",
        userInputs: "Nu trebuie s\u0103 con\u021Bin\u0103 date personale sau ale paginii.",
        wordByItself: "Cuvintele singulare sunt u\u0219or de ghicit."
      }
    }
  },
  userButton: {
    action__addAccount: "Adaug\u0103 cont",
    action__closeUserMenu: void 0,
    action__manageAccount: "Gestioneaz\u0103 contul",
    action__openUserMenu: void 0,
    action__signOut: "Deconecteaz\u0103-te",
    action__signOutAll: "Deconecteaz\u0103 toate conturile"
  },
  userProfile: {
    apiKeysPage: {
      title: "Chei API"
    },
    backupCodePage: {
      actionLabel__copied: "Copiat!",
      actionLabel__copy: "Copiaz\u0103 toate",
      actionLabel__download: "Descarc\u0103 .txt",
      actionLabel__print: "Printeaz\u0103",
      infoText1: "Codurile de rezerv\u0103 vor fi activate pentru acest cont.",
      infoText2: "P\u0103streaz\u0103 codurile de rezerv\u0103 secrete \u0219i stocheaz\u0103-le \xEEn siguran\u021B\u0103. Le po\u021Bi regenera dac\u0103 suspectezi c\u0103 au fost compromise.",
      subtitle__codelist: "Stocheaz\u0103-le \xEEn siguran\u021B\u0103 \u0219i p\u0103streaz\u0103-le secrete.",
      successMessage: "Codurile de rezerv\u0103 sunt acum activate. Po\u021Bi folosi unul dintre ele pentru a te autentifica, dac\u0103 pierzi accesul la dispozitivul t\u0103u de autentificare. Fiecare cod poate fi folosit o singur\u0103 dat\u0103.",
      successSubtitle: "Po\u021Bi folosi unul dintre acestea pentru a te autentifica \xEEn cont dac\u0103 pierzi accesul la dispozitivul de autentificare.",
      title: "Adaug\u0103 verificare cu cod de rezerv\u0103",
      title__codelist: "Coduri de rezerv\u0103"
    },
    billingPage: {
      paymentHistorySection: {
        empty: "Nu exist\u0103 istoric de pl\u0103\u021Bi",
        notFound: "\xCEncercarea de plat\u0103 nu a fost g\u0103sit\u0103",
        tableHeader__amount: "Sum\u0103",
        tableHeader__date: "Dat\u0103",
        tableHeader__status: "Stare"
      },
      paymentMethodsSection: {
        actionLabel__default: "Seteaz\u0103 ca implicit\u0103",
        actionLabel__remove: "Elimin\u0103",
        add: "Adaug\u0103 metod\u0103 de plat\u0103 nou\u0103",
        addSubtitle: "Adaug\u0103 o nou\u0103 metod\u0103 de plat\u0103 \xEEn contul t\u0103u.",
        cancelButton: "Anuleaz\u0103",
        formButtonPrimary__add: "Adaug\u0103 metoda de plat\u0103",
        formButtonPrimary__pay: "Pl\u0103te\u0219te {{amount}}",
        payWithTestCardButton: "Pl\u0103te\u0219te cu card de test",
        removeMethod: {
          messageLine1: "{{identifier}} va fi eliminat(\u0103) din acest cont.",
          messageLine2: "Nu vei mai putea utiliza aceast\u0103 surs\u0103 de plat\u0103, iar orice abonamente recurente care depind de ea nu vor mai func\u021Biona.",
          successMessage: "{{paymentMethod}} a fost eliminat\u0103 din contul t\u0103u.",
          title: "Elimin\u0103 metoda de plat\u0103"
        },
        title: "Metode de plat\u0103"
      },
      start: {
        headerTitle__payments: "Pl\u0103\u021Bi",
        headerTitle__plans: "Planuri",
        headerTitle__statements: "Extrase",
        headerTitle__subscriptions: "Abonament"
      },
      statementsSection: {
        empty: "Nu exist\u0103 extrase de afi\u0219at",
        itemCaption__paidForPlan: "Pl\u0103tit pentru planul {{plan}} {{period}}",
        itemCaption__proratedCredit: "Credit propor\u021Bional pentru utilizarea par\u021Bial\u0103 a abonamentului anterior",
        itemCaption__subscribedAndPaidForPlan: "Abonat \u0219i pl\u0103tit pentru planul {{plan}} {{period}}",
        notFound: "Extrasul nu a fost g\u0103sit",
        tableHeader__amount: "Sum\u0103",
        tableHeader__date: "Dat\u0103",
        title: "Extrase",
        totalPaid: "Total pl\u0103tit"
      },
      subscriptionsListSection: {
        actionLabel__manageSubscription: void 0,
        actionLabel__newSubscription: "Aboneaz\u0103-te la un plan",
        actionLabel__switchPlan: "Schimb\u0103 planurile",
        tableHeader__edit: "Editeaz\u0103",
        tableHeader__plan: "Plan",
        tableHeader__startDate: "Data \xEEnceperii",
        title: "Abonament"
      },
      subscriptionsSection: {
        actionLabel__default: "Gestioneaz\u0103"
      },
      switchPlansSection: {
        title: "Schimb\u0103 planurile"
      },
      title: "Facturare"
    },
    connectedAccountPage: {
      formHint: "Selecteaz\u0103 un furnizor pentru a-\u021Bi conecta contul.",
      formHint__noAccounts: "Nu exist\u0103 furnizori externi disponibili.",
      removeResource: {
        messageLine1: "{{identifier}} va fi eliminat din acest cont.",
        messageLine2: "Nu vei mai putea folosi acest cont conectat, iar func\u021Biile dependente nu vor mai func\u021Biona.",
        successMessage: "{{connectedAccount}} a fost eliminat din contul t\u0103u.",
        title: "Elimin\u0103 contul conectat"
      },
      socialButtonsBlockButton: "{{provider|titleize}}",
      successMessage: "Furnizorul a fost ad\u0103ugat la contul t\u0103u",
      title: "Adaug\u0103 cont conectat"
    },
    deletePage: {
      actionDescription: "Tasteaz\u0103 \u201E\u0218terge contul\u201D mai jos pentru a continua.",
      confirm: "\u0218terge contul",
      messageLine1: "Sigur vrei s\u0103-\u021Bi \u0219tergi contul? Este posibil ca unele date asociate s\u0103 fie p\u0103strate. Pentru a solicita \u0219tergerea complet\u0103 a datelor, te rug\u0103m s\u0103 contactezi suportul.",
      messageLine2: "Aceast\u0103 ac\u021Biune este permanent\u0103 \u0219i ireversibil\u0103.",
      title: "\u0218terge contul"
    },
    emailAddressPage: {
      emailCode: {
        formHint: "Un email cu un cod de verificare va fi trimis la aceast\u0103 adres\u0103.",
        formSubtitle: "Introdu codul de verificare trimis la {{identifier}}",
        formTitle: "Cod de verificare",
        resendButton: "Nu ai primit un cod? Retrimite",
        successMessage: "Adresa {{identifier}} a fost ad\u0103ugat\u0103 la contul t\u0103u."
      },
      emailLink: {
        formHint: "Un email cu un link de verificare va fi trimis la aceast\u0103 adres\u0103.",
        formSubtitle: "Apas\u0103 pe linkul de verificare din emailul trimis la {{identifier}}",
        formTitle: "Link de verificare",
        resendButton: "Nu ai primit un link? Retrimite",
        successMessage: "Adresa {{identifier}} a fost ad\u0103ugat\u0103 la contul t\u0103u."
      },
      enterpriseSSOLink: {
        formButton: "Apas\u0103 pentru autentificare",
        formSubtitle: "Finalizeaz\u0103 autentificarea cu {{identifier}}"
      },
      formHint: "Va trebui s\u0103 verifici aceast\u0103 adres\u0103 de email \xEEnainte de a o ad\u0103uga la cont.",
      removeResource: {
        messageLine1: "{{identifier}} va fi eliminat\u0103 din acest cont.",
        messageLine2: "Nu te vei mai putea autentifica folosind aceast\u0103 adres\u0103 de email.",
        successMessage: "{{emailAddress}} a fost eliminat\u0103 din contul t\u0103u.",
        title: "Elimin\u0103 adresa de email"
      },
      title: "Adaug\u0103 adres\u0103 de email",
      verifyTitle: "Verific\u0103 adresa de email"
    },
    formButtonPrimary__add: "Adaug\u0103",
    formButtonPrimary__continue: "Continu\u0103",
    formButtonPrimary__finish: "Finalizeaz\u0103",
    formButtonPrimary__remove: "Elimin\u0103",
    formButtonPrimary__save: "Salveaz\u0103",
    formButtonReset: "Anuleaz\u0103",
    mfaPage: {
      formHint: "Selecteaz\u0103 o metod\u0103 de ad\u0103ugat.",
      title: "Adaug\u0103 verificare \xEEn doi pa\u0219i"
    },
    mfaPhoneCodePage: {
      backButton: "Folose\u0219te num\u0103rul existent",
      primaryButton__addPhoneNumber: "Adaug\u0103 num\u0103r de telefon",
      removeResource: {
        messageLine1: "{{identifier}} nu va mai primi coduri de verificare la autentificare.",
        messageLine2: "Contul t\u0103u ar putea fi mai pu\u021Bin sigur. Sigur vrei s\u0103 continui?",
        successMessage: "Verificarea \xEEn doi pa\u0219i prin cod SMS a fost eliminat\u0103 pentru {{mfaPhoneCode}}",
        title: "Elimin\u0103 verificarea \xEEn doi pa\u0219i"
      },
      subtitle__availablePhoneNumbers: "Selecteaz\u0103 un num\u0103r existent pentru verificare prin SMS sau adaug\u0103 unul nou.",
      subtitle__unavailablePhoneNumbers: "Nu exist\u0103 numere disponibile pentru verificare prin SMS; te rug\u0103m adaug\u0103 unul nou.",
      successMessage1: "La autentificare, va trebui s\u0103 introduci un cod trimis la acest num\u0103r de telefon ca pas suplimentar.",
      successMessage2: "Salveaz\u0103 aceste coduri de rezerv\u0103 \xEEntr-un loc sigur. Dac\u0103 pierzi accesul la dispozitivul de autentificare, le po\u021Bi folosi pentru a te autentifica.",
      successTitle: "Verificare prin SMS activat\u0103",
      title: "Adaug\u0103 verificare prin SMS"
    },
    mfaTOTPPage: {
      authenticatorApp: {
        buttonAbleToScan__nonPrimary: "Scaneaz\u0103 codul QR \xEEn schimb",
        buttonUnableToScan__nonPrimary: "Nu po\u021Bi scana codul QR?",
        infoText__ableToScan: "Adaug\u0103 o metod\u0103 nou\u0103 de autentificare \xEEn aplica\u021Bia ta \u0219i scaneaz\u0103 codul QR de mai jos pentru a o lega de cont.",
        infoText__unableToScan: "Adaug\u0103 o metod\u0103 nou\u0103 \xEEn aplica\u021Bia ta de autentificare \u0219i introdu Cheia furnizat\u0103 mai jos.",
        inputLabel__unableToScan1: "Asigur\u0103-te c\u0103 parolele \u201ETime-based\u201D sau \u201EOne-time\u201D sunt activate, apoi finalizeaz\u0103 conectarea contului.",
        inputLabel__unableToScan2: "Alternativ, dac\u0103 aplica\u021Bia ta suport\u0103 URI TOTP, po\u021Bi copia URI-ul complet."
      },
      removeResource: {
        messageLine1: "Codurile de verificare din aceast\u0103 aplica\u021Bie nu vor mai fi necesare la autentificare.",
        messageLine2: "Contul t\u0103u ar putea fi mai pu\u021Bin sigur. Sigur vrei s\u0103 continui?",
        successMessage: "Verificarea \xEEn doi pa\u0219i prin aplica\u021Bie de autentificare a fost eliminat\u0103.",
        title: "Elimin\u0103 verificarea \xEEn doi pa\u0219i"
      },
      successMessage: "Verificarea \xEEn doi pa\u0219i este acum activat\u0103. La autentificare va trebui s\u0103 introduci un cod din aceast\u0103 aplica\u021Bie.",
      title: "Adaug\u0103 aplica\u021Bie de autentificare",
      verifySubtitle: "Introdu codul generat de aplica\u021Bia ta de autentificare",
      verifyTitle: "Cod de verificare"
    },
    mobileButton__menu: "Meniu",
    navbar: {
      account: "Profil",
      apiKeys: "Chei API",
      billing: "Facturare",
      description: "Gestioneaz\u0103 informa\u021Biile contului.",
      security: "Securitate",
      title: "Cont"
    },
    passkeyScreen: {
      removeResource: {
        messageLine1: "{{name}} va fi eliminat\u0103 din acest cont.",
        title: "Elimin\u0103 cheia de acces"
      },
      subtitle__rename: "Po\u021Bi schimba numele cheii de acces pentru a o g\u0103si mai u\u0219or.",
      title__rename: "Redenume\u0219te cheia de acces"
    },
    passwordPage: {
      checkboxInfoText__signOutOfOtherSessions: "Se recomand\u0103 s\u0103 te deconectezi de pe celelalte dispozitive care ar fi putut folosi parola veche.",
      readonly: "Parola nu poate fi editat\u0103 momentan deoarece te po\u021Bi autentifica doar prin conexiunea enterprise.",
      successMessage__set: "Parola a fost setat\u0103.",
      successMessage__signOutOfOtherSessions: "Toate celelalte dispozitive au fost deconectate.",
      successMessage__update: "Parola a fost actualizat\u0103.",
      title__set: "Seteaz\u0103 parola",
      title__update: "Actualizeaz\u0103 parola"
    },
    phoneNumberPage: {
      infoText: "Un mesaj text cu un cod de verificare va fi trimis la acest num\u0103r. Pot exista costuri pentru mesaje \u0219i date.",
      removeResource: {
        messageLine1: "{{identifier}} va fi eliminat din acest cont.",
        messageLine2: "Nu te vei mai putea autentifica folosind acest num\u0103r de telefon.",
        successMessage: "{{phoneNumber}} a fost eliminat din contul t\u0103u.",
        title: "Elimin\u0103 num\u0103rul de telefon"
      },
      successMessage: "{{identifier}} a fost ad\u0103ugat la contul t\u0103u.",
      title: "Adaug\u0103 num\u0103r de telefon",
      verifySubtitle: "Introdu codul trimis la {{identifier}}",
      verifyTitle: "Verific\u0103 num\u0103rul de telefon"
    },
    plansPage: {
      title: "Planuri"
    },
    profilePage: {
      fileDropAreaHint: "Dimensiune recomandat\u0103 1:1, p\xE2n\u0103 la 10MB.",
      imageFormDestructiveActionSubtitle: "Elimin\u0103",
      imageFormSubtitle: "\xCEncarc\u0103",
      imageFormTitle: "Imagine de profil",
      readonly: "Informa\u021Biile profilului sunt furnizate de conexiunea enterprise \u0219i nu pot fi editate.",
      successMessage: "Profilul t\u0103u a fost actualizat.",
      title: "Actualizeaz\u0103 profilul"
    },
    start: {
      activeDevicesSection: {
        destructiveAction: "Deconecteaz\u0103 dispozitivul",
        title: "Dispozitive active"
      },
      connectedAccountsSection: {
        actionLabel__connectionFailed: "Reconecteaz\u0103",
        actionLabel__reauthorize: "Autorizeaz\u0103 acum",
        destructiveActionTitle: "Elimin\u0103",
        primaryButton: "Conecteaz\u0103 cont",
        subtitle__disconnected: "Acest cont a fost deconectat.",
        subtitle__reauthorize: "Permisiunile necesare au fost actualizate \u0219i po\u021Bi \xEEnt\xE2mpina func\u021Bionalitate limitat\u0103. Te rug\u0103m reautorizeaz\u0103 aplica\u021Bia pentru a evita problemele.",
        title: "Conturi conectate"
      },
      dangerSection: {
        deleteAccountButton: "\u0218terge contul",
        title: "\u0218terge contul"
      },
      emailAddressesSection: {
        destructiveAction: "Elimin\u0103 emailul",
        detailsAction__nonPrimary: "Seteaz\u0103 ca principal",
        detailsAction__primary: "Finalizeaz\u0103 verificarea",
        detailsAction__unverified: "Verific\u0103",
        primaryButton: "Adaug\u0103 adres\u0103 de email",
        title: "Adrese de email"
      },
      enterpriseAccountsSection: {
        title: "Conturi enterprise"
      },
      headerTitle__account: "Detalii profil",
      headerTitle__security: "Securitate",
      mfaSection: {
        backupCodes: {
          actionLabel__regenerate: "Genereaz\u0103 din nou",
          headerTitle: "Coduri de rezerv\u0103",
          subtitle__regenerate: "Ob\u021Bine un set nou de coduri de rezerv\u0103. Codurile anterioare vor fi \u0219terse \u0219i nu vor mai putea fi folosite.",
          title__regenerate: "Genereaz\u0103 din nou codurile de rezerv\u0103"
        },
        phoneCode: {
          actionLabel__setDefault: "Seteaz\u0103 ca implicit",
          destructiveActionLabel: "Elimin\u0103"
        },
        primaryButton: "Adaug\u0103 verificare \xEEn doi pa\u0219i",
        title: "Verificare \xEEn doi pa\u0219i",
        totp: {
          destructiveActionTitle: "Elimin\u0103",
          headerTitle: "Aplica\u021Bie de autentificare"
        }
      },
      passkeysSection: {
        menuAction__destructive: "Elimin\u0103",
        menuAction__rename: "Redenume\u0219te",
        primaryButton: "Adaug\u0103 o cheie de acces",
        title: "Chei de acces"
      },
      passwordSection: {
        primaryButton__setPassword: "Seteaz\u0103 parola",
        primaryButton__updatePassword: "Actualizeaz\u0103 parola",
        title: "Parol\u0103"
      },
      phoneNumbersSection: {
        destructiveAction: "Elimin\u0103 num\u0103rul de telefon",
        detailsAction__nonPrimary: "Seteaz\u0103 ca principal",
        detailsAction__primary: "Finalizeaz\u0103 verificarea",
        detailsAction__unverified: "Verific\u0103 num\u0103rul de telefon",
        primaryButton: "Adaug\u0103 num\u0103r de telefon",
        title: "Numere de telefon"
      },
      profileSection: {
        primaryButton: "Actualizeaz\u0103 profilul",
        title: "Profil"
      },
      usernameSection: {
        primaryButton__setUsername: "Seteaz\u0103 numele de utilizator",
        primaryButton__updateUsername: "Actualizeaz\u0103 numele de utilizator",
        title: "Nume de utilizator"
      },
      web3WalletsSection: {
        destructiveAction: "Elimin\u0103 portofelul",
        detailsAction__nonPrimary: "Seteaz\u0103 ca principal",
        primaryButton: "Conecteaz\u0103 portofel",
        title: "Portofele Web3"
      }
    },
    usernamePage: {
      successMessage: "Numele de utilizator a fost actualizat.",
      title__set: "Seteaz\u0103 numele de utilizator",
      title__update: "Actualizeaz\u0103 numele de utilizator"
    },
    web3WalletPage: {
      removeResource: {
        messageLine1: "{{identifier}} va fi eliminat din acest cont.",
        messageLine2: "Nu te vei mai putea autentifica folosind acest portofel web3.",
        successMessage: "{{web3Wallet}} a fost eliminat din contul t\u0103u.",
        title: "Elimin\u0103 portofelul web3"
      },
      subtitle__availableWallets: "Selecteaz\u0103 un portofel web3 pentru a-l conecta la cont.",
      subtitle__unavailableWallets: "Nu exist\u0103 portofele web3 disponibile.",
      successMessage: "Portofelul a fost ad\u0103ugat la contul t\u0103u.",
      title: "Adaug\u0103 portofel web3",
      web3WalletButtonsBlockButton: "{{provider|titleize}}"
    }
  },
  waitlist: {
    start: {
      actionLink: "Autentific\u0103-te",
      actionText: "Ai deja acces?",
      formButton: "\xCEnscrie-te pe lista de a\u0219teptare",
      subtitle: "Introdu adresa ta de email \u0219i te vom anun\u021Ba c\xE2nd \xEE\u021Bi vine r\xE2ndul",
      title: "\xCEnscrie-te pe lista de a\u0219teptare"
    },
    success: {
      message: "Vei fi redirec\u021Bionat \xEEn cur\xE2nd...",
      subtitle: "Te vom contacta c\xE2nd \xEE\u021Bi vine r\xE2ndul",
      title: "Mul\u021Bumim pentru \xEEnscriere!"
    }
  }
};
// Annotate the CommonJS export names for ESM import in node:
0 && (module.exports = {
  roRO
});
