// @clerk/localizations/dist/sk-SK.js
"use strict";
var __defProp = Object.defineProperty;
var __getOwnPropDesc = Object.getOwnPropertyDescriptor;
var __getOwnPropNames = Object.getOwnPropertyNames;
var __hasOwnProp = Object.prototype.hasOwnProperty;
var __export = (target, all) => {
  for (var name in all)
    __defProp(target, name, { get: all[name], enumerable: true });
};
var __copyProps = (to, from, except, desc) => {
  if (from && typeof from === "object" || typeof from === "function") {
    for (let key of __getOwnPropNames(from))
      if (!__hasOwnProp.call(to, key) && key !== except)
        __defProp(to, key, { get: () => from[key], enumerable: !(desc = __getOwnPropDesc(from, key)) || desc.enumerable });
  }
  return to;
};
var __toCommonJS = (mod) => __copyProps(__defProp({}, "__esModule", { value: true }), mod);

// src/sk-SK.ts
var sk_SK_exports = {};
__export(sk_SK_exports, {
  skSK: () => skSK
});
module.exports = __toCommonJS(sk_SK_exports);
var skSK = {
  locale: "sk-SK",
  apiKeys: {
    action__add: void 0,
    action__search: void 0,
    createdAndExpirationStatus__expiresOn: void 0,
    createdAndExpirationStatus__never: void 0,
    detailsTitle__emptyRow: void 0,
    formButtonPrimary__add: void 0,
    formFieldCaption__expiration__expiresOn: void 0,
    formFieldCaption__expiration__never: void 0,
    formFieldOption__expiration__180d: void 0,
    formFieldOption__expiration__1d: void 0,
    formFieldOption__expiration__1y: void 0,
    formFieldOption__expiration__30d: void 0,
    formFieldOption__expiration__60d: void 0,
    formFieldOption__expiration__7d: void 0,
    formFieldOption__expiration__90d: void 0,
    formFieldOption__expiration__never: void 0,
    formHint: void 0,
    formTitle: void 0,
    lastUsed__days: void 0,
    lastUsed__hours: void 0,
    lastUsed__minutes: void 0,
    lastUsed__months: void 0,
    lastUsed__seconds: void 0,
    lastUsed__years: void 0,
    menuAction__revoke: void 0,
    revokeConfirmation: {
      confirmationText: void 0,
      formButtonPrimary__revoke: void 0,
      formHint: void 0,
      formTitle: void 0
    }
  },
  backButton: "Sp\xE4\u0165",
  badge__activePlan: void 0,
  badge__canceledEndsAt: void 0,
  badge__currentPlan: "Aktu\xE1lny bal\xEDk",
  badge__default: "Predvolen\xE9",
  badge__endsAt: void 0,
  badge__expired: void 0,
  badge__freeTrial: void 0,
  badge__otherImpersonatorDevice: "In\xE9 zariadenie z\xE1stupcu",
  badge__pastDueAt: void 0,
  badge__pastDuePlan: void 0,
  badge__primary: "Hlavn\xFD",
  badge__renewsAt: void 0,
  badge__requiresAction: "Vy\u017Eaduje akciu",
  badge__startsAt: void 0,
  badge__thisDevice: "Toto zariadenie",
  badge__trialEndsAt: void 0,
  badge__unverified: "Nepotvrden\xE9",
  badge__upcomingPlan: void 0,
  badge__userDevice: "Zariadenie pou\u017E\xEDvate\u013Ea",
  badge__you: "Vy",
  billing: {
    addPaymentMethod__label: void 0,
    alwaysFree: void 0,
    annually: void 0,
    availableFeatures: void 0,
    billedAnnually: "Ro\u010Dne",
    billedMonthlyOnly: void 0,
    cancelFreeTrial: void 0,
    cancelFreeTrialAccessUntil: void 0,
    cancelFreeTrialTitle: void 0,
    cancelSubscription: void 0,
    cancelSubscriptionAccessUntil: void 0,
    cancelSubscriptionNoCharge: void 0,
    cancelSubscriptionPastDue: void 0,
    cancelSubscriptionTitle: void 0,
    cannotSubscribeMonthly: void 0,
    cannotSubscribeUnrecoverable: void 0,
    checkout: {
      description__paymentSuccessful: void 0,
      description__subscriptionSuccessful: void 0,
      downgradeNotice: void 0,
      emailForm: {
        subtitle: void 0,
        title: void 0
      },
      lineItems: {
        title__freeTrialEndsAt: void 0,
        title__paymentMethod: void 0,
        title__statementId: void 0,
        title__subscriptionBegins: void 0,
        title__totalPaid: void 0
      },
      pastDueNotice: void 0,
      perMonth: void 0,
      title: void 0,
      title__paymentSuccessful: void 0,
      title__subscriptionSuccessful: void 0,
      title__trialSuccess: void 0,
      totalDueAfterTrial: void 0
    },
    credit: void 0,
    creditRemainder: void 0,
    defaultFreePlanActive: void 0,
    free: "Bezplatne",
    getStarted: "Za\u010Da\u0165",
    keepFreeTrial: void 0,
    keepSubscription: void 0,
    manage: void 0,
    manageSubscription: "Spravova\u0165 \u010Dlenstvo",
    month: "Mesiac",
    monthly: void 0,
    pastDue: void 0,
    pay: void 0,
    paymentMethod: {
      applePayDescription: {
        annual: void 0,
        monthly: void 0
      },
      dev: {
        anyNumbers: void 0,
        cardNumber: void 0,
        cvcZip: void 0,
        developmentMode: void 0,
        expirationDate: void 0,
        testCardInfo: void 0
      }
    },
    paymentMethods__label: void 0,
    popular: void 0,
    pricingTable: {
      billingCycle: void 0,
      included: void 0
    },
    reSubscribe: void 0,
    seeAllFeatures: void 0,
    startFreeTrial: void 0,
    startFreeTrial__days: void 0,
    subscribe: void 0,
    subscriptionDetails: {
      beginsOn: void 0,
      currentBillingCycle: void 0,
      endsOn: void 0,
      firstPaymentAmount: void 0,
      firstPaymentOn: void 0,
      nextPaymentAmount: void 0,
      nextPaymentOn: void 0,
      pastDueAt: void 0,
      renewsAt: void 0,
      subscribedOn: void 0,
      title: void 0,
      trialEndsOn: void 0,
      trialStartedOn: void 0
    },
    subtotal: void 0,
    switchPlan: "Zmeni\u0165 bal\xEDk",
    switchToAnnual: void 0,
    switchToAnnualWithAnnualPrice: void 0,
    switchToMonthly: void 0,
    switchToMonthlyWithPrice: void 0,
    totalDue: void 0,
    totalDueToday: void 0,
    viewFeatures: void 0,
    viewPayment: void 0,
    year: void 0
  },
  createOrganization: {
    formButtonSubmit: "Vytvori\u0165 organiz\xE1ciu",
    invitePage: {
      formButtonReset: "Presko\u010Di\u0165"
    },
    title: "Vytvori\u0165 organiz\xE1ciu"
  },
  dates: {
    lastDay: "V\u010Dera o {{ date | timeString('sk-SK') }}",
    next6Days: "P\u0159\xED\u0161t\xED {{ date | weekday('sk-SK','long') }} o {{ date | timeString('sk-SK') }}",
    nextDay: "Zajtra o {{ date | timeString('sk-SK') }}",
    numeric: "{{ date | numeric('sk-SK') }}",
    previous6Days: "Minul\xFD {{ date | weekday('sk-SK','long') }} o {{ date | timeString('sk-SK') }}",
    sameDay: "Dnes o {{ date | timeString('sk-SK') }}"
  },
  dividerText: "alebo",
  footerActionLink__alternativePhoneCodeProvider: void 0,
  footerActionLink__useAnotherMethod: "Pou\u017Ei\u0165 in\xFA met\xF3du",
  footerPageLink__help: "Pomoc",
  footerPageLink__privacy: "Ochrana s\xFAkromia",
  footerPageLink__terms: "Podmienky",
  formButtonPrimary: "Pokra\u010Dova\u0165",
  formButtonPrimary__verify: "Overi\u0165",
  formFieldAction__forgotPassword: "Zabudli ste heslo?",
  formFieldError__matchingPasswords: "Hesl\xE1 sa musia zhodova\u0165.",
  formFieldError__notMatchingPasswords: "Hesl\xE1 sa nezhoduj\xFA.",
  formFieldError__verificationLinkExpired: "Overovac\xED odkaz vypr\u0161al. Vy\u017Eiadajte si nov\xFD odkaz.",
  formFieldHintText__optional: "Volite\u013En\xE9",
  formFieldHintText__slug: "Slug je \u010Ditate\u013En\xE9 ID, ktor\xE9 mus\xED by\u0165 jedine\u010Dn\xE9. \u010Casto sa pou\u017E\xEDva v URL adres\xE1ch.",
  formFieldInputPlaceholder__apiKeyDescription: void 0,
  formFieldInputPlaceholder__apiKeyExpirationDate: void 0,
  formFieldInputPlaceholder__apiKeyName: void 0,
  formFieldInputPlaceholder__backupCode: "Zadajte z\xE1lo\u017En\xFD k\xF3d",
  formFieldInputPlaceholder__confirmDeletionUserAccount: "Zmaza\u0165 \xFA\u010Det",
  formFieldInputPlaceholder__emailAddress: "Zadajte svoj e-mail",
  formFieldInputPlaceholder__emailAddress_username: "Zadajte e-mail alebo pou\u017E\xEDvate\u013Esk\xE9 meno",
  formFieldInputPlaceholder__emailAddresses: "Zadajte alebo vlo\u017Ete jednu alebo viac emailov\xFDch adries oddelen\xFDch medzerou alebo \u010Diarkou",
  formFieldInputPlaceholder__firstName: "Krstn\xE9 meno",
  formFieldInputPlaceholder__lastName: "Priezvisko",
  formFieldInputPlaceholder__organizationDomain: "priklad.sk",
  formFieldInputPlaceholder__organizationDomainEmailAddress: "vy@priklad.sk",
  formFieldInputPlaceholder__organizationName: "N\xE1zov organiz\xE1cie",
  formFieldInputPlaceholder__organizationSlug: "moja-organizacia",
  formFieldInputPlaceholder__password: "Zadajte heslo",
  formFieldInputPlaceholder__phoneNumber: "Zadajte telef\xF3nne \u010D\xEDslo",
  formFieldInputPlaceholder__username: "Zadajte u\u017E\xEDvate\u013Esk\xE9 meno",
  formFieldInput__emailAddress_format: void 0,
  formFieldLabel__apiKeyDescription: void 0,
  formFieldLabel__apiKeyExpiration: void 0,
  formFieldLabel__apiKeyName: void 0,
  formFieldLabel__automaticInvitations: "Povoli\u0165 automatick\xE9 pozv\xE1nky pre t\xFAto dom\xE9nu",
  formFieldLabel__backupCode: "Z\xE1lo\u017En\xFD k\xF3d",
  formFieldLabel__confirmDeletion: "Potvrdenie",
  formFieldLabel__confirmPassword: "Potvrdi\u0165 heslo",
  formFieldLabel__currentPassword: "S\xFA\u010Dasn\xE9 heslo",
  formFieldLabel__emailAddress: "Emailov\xE1 adresa",
  formFieldLabel__emailAddress_username: "Emailov\xE1 adresa alebo u\u017E\xEDvate\u013Esk\xE9 meno",
  formFieldLabel__emailAddresses: "Emailov\xE9 adresy",
  formFieldLabel__firstName: "Meno",
  formFieldLabel__lastName: "Priezvisko",
  formFieldLabel__newPassword: "Nov\xE9 heslo",
  formFieldLabel__organizationDomain: "Dom\xE9na",
  formFieldLabel__organizationDomainDeletePending: "Zmaza\u0165 \u010Dakaj\xFAce pozv\xE1nky a n\xE1vrhy",
  formFieldLabel__organizationDomainEmailAddress: "Overovacia e-mailov\xE1 adresa",
  formFieldLabel__organizationDomainEmailAddressDescription: "Zadajte e-mailov\xFA adresu pod touto dom\xE9nou na prijatie k\xF3du a overenie tejto dom\xE9ny.",
  formFieldLabel__organizationName: "N\xE1zov organiz\xE1cie",
  formFieldLabel__organizationSlug: "URL adresa",
  formFieldLabel__passkeyName: "N\xE1zov Passkey",
  formFieldLabel__password: "Heslo",
  formFieldLabel__phoneNumber: "Telef\xF3nne \u010D\xEDslo",
  formFieldLabel__role: "Rola",
  formFieldLabel__signOutOfOtherSessions: "Odhl\xE1si\u0165 sa zo v\u0161etk\xFDch ostatn\xFDch zariaden\xED",
  formFieldLabel__username: "U\u017E\xEDvate\u013Esk\xE9 meno",
  impersonationFab: {
    action__signOut: "Odhl\xE1si\u0165 sa",
    title: "Prihl\xE1sen\xFD(\xE1) ako {{identifier}}"
  },
  lastAuthenticationStrategy: "Naposledy pou\u017Eit\xE9",
  maintenanceMode: void 0,
  membershipRole__admin: "Spr\xE1vca",
  membershipRole__basicMember: "\u010Clen",
  membershipRole__guestMember: "Host",
  organizationList: {
    action__createOrganization: "Vytvori\u0165 organiz\xE1ciu",
    action__invitationAccept: "Pripoji\u0165 sa",
    action__suggestionsAccept: "Po\u017Eiada\u0165 o pripojenie",
    createOrganization: "Vytvori\u0165 organiz\xE1ciu",
    invitationAcceptedLabel: "Pripojen\xE9",
    subtitle: "na pokra\u010Dovanie do {{applicationName}}",
    suggestionsAcceptedLabel: "\u010Cak\xE1 na schv\xE1lenie",
    title: "Vyberte \xFA\u010Det",
    titleWithoutPersonal: "Vyberte organiz\xE1ciu"
  },
  organizationProfile: {
    apiKeysPage: {
      title: "K\u013E\xFA\u010De API"
    },
    badge__automaticInvitation: "Automatick\xE9 pozv\xE1nky",
    badge__automaticSuggestion: "Automatick\xE9 n\xE1vrhy",
    badge__manualInvitation: "Bez automatickej registr\xE1cie",
    badge__unverified: "Neoveren\xE9",
    billingPage: {
      paymentHistorySection: {
        empty: void 0,
        notFound: void 0,
        tableHeader__amount: void 0,
        tableHeader__date: void 0,
        tableHeader__status: void 0
      },
      paymentMethodsSection: {
        actionLabel__default: void 0,
        actionLabel__remove: void 0,
        add: void 0,
        addSubtitle: void 0,
        cancelButton: void 0,
        formButtonPrimary__add: void 0,
        formButtonPrimary__pay: void 0,
        payWithTestCardButton: void 0,
        removeMethod: {
          messageLine1: void 0,
          messageLine2: void 0,
          successMessage: void 0,
          title: void 0
        },
        title: void 0
      },
      start: {
        headerTitle__payments: void 0,
        headerTitle__plans: void 0,
        headerTitle__statements: void 0,
        headerTitle__subscriptions: void 0
      },
      statementsSection: {
        empty: void 0,
        itemCaption__paidForPlan: void 0,
        itemCaption__proratedCredit: void 0,
        itemCaption__subscribedAndPaidForPlan: void 0,
        notFound: void 0,
        tableHeader__amount: void 0,
        tableHeader__date: void 0,
        title: void 0,
        totalPaid: void 0
      },
      subscriptionsListSection: {
        actionLabel__manageSubscription: void 0,
        actionLabel__newSubscription: void 0,
        actionLabel__switchPlan: void 0,
        tableHeader__edit: void 0,
        tableHeader__plan: void 0,
        tableHeader__startDate: void 0,
        title: void 0
      },
      subscriptionsSection: {
        actionLabel__default: void 0
      },
      switchPlansSection: {
        title: void 0
      },
      title: void 0
    },
    createDomainPage: {
      subtitle: "Pridajte dom\xE9nu na overenie. Pou\u017E\xEDvatelia s e-mailov\xFDmi adresami v tejto dom\xE9ne sa m\xF4\u017Eu automaticky pripoji\u0165 k organiz\xE1cii alebo po\u017Eiada\u0165 o pripojenie.",
      title: "Prida\u0165 dom\xE9nu"
    },
    invitePage: {
      detailsTitle__inviteFailed: "Pozv\xE1nky sa nepodarilo odosla\u0165. Opravte nasleduj\xFAce a sk\xFAste to znovu:",
      formButtonPrimary__continue: "Odosla\u0165 pozv\xE1nky",
      selectDropdown__role: "Vyberte rolu",
      subtitle: "Pozva\u0165 nov\xFDch \u010Dlenov do tejto organiz\xE1cie",
      successMessage: "Pozv\xE1nky boli \xFAspe\u0161ne odoslan\xE9.",
      title: "Pozva\u0165 \u010Dlenov"
    },
    membersPage: {
      action__invite: "Pozva\u0165",
      action__search: "H\u013Eada\u0165",
      activeMembersTab: {
        menuAction__remove: "Odstr\xE1ni\u0165 \u010Dlena",
        tableHeader__actions: "Akcie",
        tableHeader__joined: "Pripojil sa",
        tableHeader__role: "Rola",
        tableHeader__user: "U\u017E\xEDvate\u013E"
      },
      detailsTitle__emptyRow: "\u017Diadni \u010Dlenovia na zobrazenie",
      invitationsTab: {
        autoInvitations: {
          headerSubtitle: "Pozvite pou\u017E\xEDvate\u013Eov prepojen\xEDm e-mailovej dom\xE9ny s va\u0161ou organiz\xE1ciou. Ka\u017Ed\xFD, kto sa zaregistruje s e-mailovou dom\xE9nou, ktor\xE1 sa zhoduje, sa m\xF4\u017Ee k organiz\xE1cii pripoji\u0165 kedyko\u013Evek.",
          headerTitle: "Automatick\xE9 pozv\xE1nky",
          primaryButton: "Spravova\u0165 overen\xE9 dom\xE9ny"
        },
        table__emptyRow: "\u017Diadne pozv\xE1nky na zobrazenie"
      },
      invitedMembersTab: {
        menuAction__revoke: "Zru\u0161i\u0165 pozvanie",
        tableHeader__invited: "Pozvan\xED"
      },
      requestsTab: {
        autoSuggestions: {
          headerSubtitle: "Pou\u017E\xEDvatelia, ktor\xED sa zaregistruj\xFA s e-mailovou dom\xE9nou, ktor\xE1 sa zhoduje, uvidia n\xE1vrh na po\u017Eiadanie o pripojenie do va\u0161ej organiz\xE1cie.",
          headerTitle: "Automatick\xE9 n\xE1vrhy",
          primaryButton: "Spravova\u0165 overen\xE9 dom\xE9ny"
        },
        menuAction__approve: "Schv\xE1li\u0165",
        menuAction__reject: "Odmietnu\u0165",
        tableHeader__requested: "Po\u017Eiadali o pr\xEDstup",
        table__emptyRow: "\u017Diadne po\u017Eiadavky na zobrazenie"
      },
      start: {
        headerTitle__invitations: "Pozv\xE1nky",
        headerTitle__members: "\u010Clenovia",
        headerTitle__requests: "Po\u017Eiadavky"
      }
    },
    navbar: {
      apiKeys: void 0,
      billing: void 0,
      description: "Spravujte svoju organiz\xE1ciu.",
      general: "V\u0161eobecn\xE9",
      members: "\u010Clenovia",
      title: "Organiz\xE1cia"
    },
    plansPage: {
      alerts: {
        noPermissionsToManageBilling: void 0
      },
      title: void 0
    },
    profilePage: {
      dangerSection: {
        deleteOrganization: {
          actionDescription: 'Pre pokra\u010Dovanie zadajte "{{organizationName}}" ni\u017E\u0161ie.',
          messageLine1: "Ste si ist\xED, \u017Ee chcete odstr\xE1ni\u0165 t\xFAto organiz\xE1ciu?",
          messageLine2: "Tento krok je trval\xFD a nevratn\xFD.",
          successMessage: "Organiz\xE1cia bola odstr\xE1nen\xE1.",
          title: "Odstr\xE1ni\u0165 organiz\xE1ciu"
        },
        leaveOrganization: {
          actionDescription: 'Pre pokra\u010Dovanie zadajte "{{organizationName}}" ni\u017E\u0161ie.',
          messageLine1: "Naozaj chcete opusti\u0165 t\xFAto organiz\xE1ciu? Strat\xEDte pr\xEDstup k tejto organiz\xE1cii a jej aplik\xE1ci\xE1m.",
          messageLine2: "T\xE1to akcia je trval\xE1 a nezvratn\xE1.",
          successMessage: "Opustili ste organiz\xE1ciu.",
          title: "Opusti\u0165 organiz\xE1ciu"
        },
        title: "Upozornenie"
      },
      domainSection: {
        menuAction__manage: "Spravova\u0165",
        menuAction__remove: "Odstr\xE1ni\u0165",
        menuAction__verify: "Overi\u0165",
        primaryButton: "Prida\u0165 dom\xE9nu",
        subtitle: "Umo\u017Enite pou\u017E\xEDvate\u013Eom pripoji\u0165 sa k organiz\xE1cii automaticky alebo po\u017Eiada\u0165 o pripojenie na z\xE1klade overenej e-mailovej dom\xE9ny.",
        title: "Overen\xE9 dom\xE9ny"
      },
      successMessage: "Organiz\xE1cia bola aktualizovan\xE1.",
      title: "Profil organiz\xE1cie"
    },
    removeDomainPage: {
      messageLine1: "E-mailov\xE1 dom\xE9na {{domain}} bude odstr\xE1nen\xE1.",
      messageLine2: "Pou\u017E\xEDvatelia u\u017E nebud\xFA m\xF4c\u0165 automaticky pripoji\u0165 organiz\xE1ciu po tomto kroku.",
      successMessage: "{{domain}} bola odstr\xE1nen\xE1.",
      title: "Odstr\xE1ni\u0165 dom\xE9nu"
    },
    start: {
      headerTitle__general: "V\u0161eobecn\xE9",
      headerTitle__members: "\u010Clenovia",
      profileSection: {
        primaryButton: "Aktualizova\u0165 profil",
        title: "Profil organiz\xE1cie",
        uploadAction__title: "Logo"
      }
    },
    verifiedDomainPage: {
      dangerTab: {
        calloutInfoLabel: "Odstr\xE1nenie tejto dom\xE9ny ovplyvn\xED pozvan\xFDch pou\u017E\xEDvate\u013Eov.",
        removeDomainActionLabel__remove: "Odstr\xE1ni\u0165 dom\xE9nu",
        removeDomainSubtitle: "Odstr\xE1ni\u0165 t\xFAto dom\xE9nu z overen\xFDch dom\xE9n",
        removeDomainTitle: "Odstr\xE1ni\u0165 dom\xE9nu"
      },
      enrollmentTab: {
        automaticInvitationOption__description: "Pou\u017E\xEDvatelia s\xFA automaticky poz\xFDvan\xED na pripojenie sa k organiz\xE1cii, ke\u010F sa zaregistruj\xFA, a m\xF4\u017Eu sa pripoji\u0165 kedyko\u013Evek.",
        automaticInvitationOption__label: "Automatick\xE9 pozv\xE1nky",
        automaticSuggestionOption__description: "Pou\u017E\xEDvatelia dostan\xFA n\xE1vrh na po\u017Eiadanie o pripojenie, ale musia by\u0165 schv\xE1len\xED administr\xE1torom predt\xFDm, ako sa m\xF4\u017Eu pripoji\u0165 k organiz\xE1cii.",
        automaticSuggestionOption__label: "Automatick\xE9 n\xE1vrhy",
        calloutInfoLabel: "Zmena re\u017Eimu z\xE1pisu ovplyvn\xED iba nov\xFDch pou\u017E\xEDvate\u013Eov.",
        calloutInvitationCountLabel: "\u010Cakaj\xFAce pozv\xE1nky zaslan\xE9 pou\u017E\xEDvate\u013Eom: {{count}}",
        calloutSuggestionCountLabel: "\u010Cakaj\xFAce n\xE1vrhy zaslan\xE9 pou\u017E\xEDvate\u013Eom: {{count}}",
        manualInvitationOption__description: "Pou\u017E\xEDvatelia m\xF4\u017Eu by\u0165 pozvan\xED iba manu\xE1lne do organiz\xE1cie.",
        manualInvitationOption__label: "\u017Diadny automatick\xFD z\xE1pis",
        subtitle: "Vyberte, ako m\xF4\u017Eu pou\u017E\xEDvatelia z tejto dom\xE9ny vst\xFApi\u0165 do organiz\xE1cie."
      },
      start: {
        headerTitle__danger: "Nebezpe\u010Die",
        headerTitle__enrollment: "Mo\u017Enosti z\xE1pisu"
      },
      subtitle: "Dom\xE9na {{domain}} bola teraz overen\xE1. Pokra\u010Dujte v\xFDberom re\u017Eimu z\xE1pisu.",
      title: "Aktualizova\u0165 {{domain}}"
    },
    verifyDomainPage: {
      formSubtitle: "Zadajte overovac\xED k\xF3d zaslan\xFD na va\u0161u e-mailov\xFA adresu",
      formTitle: "Overovac\xED k\xF3d",
      resendButton: "Nedostali ste k\xF3d? Znova odosla\u0165",
      subtitle: "Dom\xE9na {{domainName}} mus\xED by\u0165 overen\xE1 cez e-mail.",
      subtitleVerificationCodeScreen: "Overovac\xED k\xF3d bol zaslan\xFD na {{emailAddress}}. Pre pokra\u010Dovanie ho zadajte ni\u017E\u0161ie.",
      title: "Overi\u0165 dom\xE9nu"
    }
  },
  organizationSwitcher: {
    action__closeOrganizationSwitcher: void 0,
    action__createOrganization: "Vytvori\u0165 organiz\xE1ciu",
    action__invitationAccept: "Pripoji\u0165 sa",
    action__manageOrganization: "Spravova\u0165 organiz\xE1ciu",
    action__openOrganizationSwitcher: void 0,
    action__suggestionsAccept: "Po\u017Eiada\u0165 o pripojenie",
    notSelected: "Nie je vybran\xE1 \u017Eiadna organiz\xE1cia",
    personalWorkspace: "Osobn\xFD pracovn\xFD priestor",
    suggestionsAcceptedLabel: "\u010Cak\xE1 na schv\xE1lenie"
  },
  paginationButton__next: "\u010Eal\u0161ie",
  paginationButton__previous: "Predch\xE1dzaj\xFAce",
  paginationRowText__displaying: "Zobrazuje sa",
  paginationRowText__of: "z",
  reverification: {
    alternativeMethods: {
      actionLink: "Z\xEDska\u0165 pomoc",
      actionText: "Nem\xE1te pr\xEDstup k \u017Eiadnej z t\xFDchto met\xF3d?",
      blockButton__backupCode: "Pou\u017Ei\u0165 z\xE1lo\u017En\xFD k\xF3d",
      blockButton__emailCode: "Odosla\u0165 overovac\xED k\xF3d na email {{identifier}}",
      blockButton__passkey: "Pou\u017Ei\u0165 Passkey",
      blockButton__password: "Pokra\u010Dova\u0165 pomocou hesla",
      blockButton__phoneCode: "Posla\u0165 SMS k\xF3d na telef\xF3nne \u010D\xEDslo {{identifier}}",
      blockButton__totp: "Pou\u017Ei\u0165 autentifika\u010Dn\xFA aplik\xE1ciu",
      getHelp: {
        blockButton__emailSupport: "Podpora cez email",
        content: "Ak m\xE1te probl\xE9my s overen\xEDm svojho \xFA\u010Dtu, kontaktujte n\xE1s emailom a pok\xFAsime sa v\xE1m \u010Do najsk\xF4r obnovi\u0165 pr\xEDstup.",
        title: "Z\xEDska\u0165 pomoc"
      },
      subtitle: "M\xE1te probl\xE9my? Na overenie m\xF4\u017Eete pou\u017Ei\u0165 jednu z t\xFDchto met\xF3d.",
      title: "Pou\u017Ei\u0165 in\xFA met\xF3du"
    },
    backupCodeMfa: {
      subtitle: "Zadajte z\xE1lo\u017En\xFD k\xF3d, ktor\xFD ste dostali pri nastavovan\xED dvojfaktorov\xE9ho overenia.",
      title: "Zadajte z\xE1lo\u017En\xFD k\xF3d"
    },
    emailCode: {
      formTitle: "Overovac\xED k\xF3d",
      resendButton: "Nedostali ste k\xF3d? Znova odosla\u0165",
      subtitle: "Pre pokra\u010Dovanie vlo\u017Ete k\xF3d odoslan\xFD na v\xE1\u0161 email",
      title: "Vy\u017Eaduje sa overenie"
    },
    noAvailableMethods: {
      message: "Nemo\u017Eno pokra\u010Dova\u0165 v overen\xED. Nie je nastaven\xE1 \u017Eiadna dostupn\xE1 autentifika\u010Dn\xE1 met\xF3da.",
      subtitle: "Do\u0161lo k chybe",
      title: "Nepodarilo sa overi\u0165 v\xE1\u0161 \xFA\u010Det"
    },
    passkey: {
      blockButton__passkey: "Pou\u017Ei\u0165 Passkey",
      subtitle: "Pou\u017Eitie passkey potvrd\xED va\u0161u identitu. Va\u0161e zariadenie v\xE1s m\xF4\u017Ee vyzva\u0165 na potvrdenie odtla\u010Dkom, tv\xE1rou alebo k\xF3dom.",
      title: "Pou\u017Ei\u0165 Passkey"
    },
    password: {
      actionLink: "Pou\u017Ei\u0165 in\xFA met\xF3du",
      subtitle: "Pre pokra\u010Dovanie vlo\u017Ete svoje aktu\xE1lne heslo",
      title: "Vy\u017Eaduje sa overenie"
    },
    phoneCode: {
      formTitle: "Overovac\xED k\xF3d",
      resendButton: "Nedostali ste k\xF3d? Znova odosla\u0165",
      subtitle: "Pre pokra\u010Dovanie vlo\u017Ete overovac\xED k\xF3d odoslan\xFD na va\u0161e telef\xF3nne \u010D\xEDslo",
      title: "Vy\u017Eaduje sa overenie"
    },
    phoneCodeMfa: {
      formTitle: "Overovac\xED k\xF3d",
      resendButton: "Nedostali ste k\xF3d? Znova odosla\u0165",
      subtitle: "Pre pokra\u010Dovanie vlo\u017Ete overovac\xED k\xF3d odoslan\xFD na va\u0161e telef\xF3nne \u010D\xEDslo",
      title: "Vy\u017Eaduje sa overenie"
    },
    totpMfa: {
      formTitle: "Overovac\xED k\xF3d",
      subtitle: "Vlo\u017Ete overovac\xED k\xF3d z va\u0161ej autentifika\u010Dnej aplik\xE1cie",
      title: "Vy\u017Eaduje sa overenie"
    }
  },
  signIn: {
    accountSwitcher: {
      action__addAccount: "Prida\u0165 \xFA\u010Det",
      action__signOutAll: "Odhl\xE1si\u0165 sa zo v\u0161etk\xFDch \xFA\u010Dtov",
      subtitle: "Vyberte \xFA\u010Det, ktor\xFD chcete pou\u017Ei\u0165",
      title: "Vyberte \xFA\u010Det"
    },
    alternativeMethods: {
      actionLink: "Z\xEDska\u0165 pomoc",
      actionText: "M\xE1te probl\xE9my s prihl\xE1sen\xEDm?",
      blockButton__backupCode: "Pou\u017Ei\u0165 z\xE1lo\u017En\xFD k\xF3d",
      blockButton__emailCode: "Odosla\u0165 overovac\xED k\xF3d na email {{identifier}}",
      blockButton__emailLink: "Odosla\u0165 odkaz na email {{identifier}}",
      blockButton__passkey: "Prihl\xE1si\u0165 sa pomocou Passkey",
      blockButton__password: "Prihl\xE1si\u0165 sa pomocou hesla",
      blockButton__phoneCode: "Posla\u0165 SMS k\xF3d na telef\xF3nne \u010D\xEDslo {{identifier}}",
      blockButton__totp: "Pou\u017Ei\u0165 autentifika\u010Dn\xFA aplik\xE1ciu",
      getHelp: {
        blockButton__emailSupport: "Podpora cez email",
        content: "Ak m\xE1te probl\xE9my s prihl\xE1sen\xEDm do svojho \xFA\u010Dtu, kontaktujte n\xE1s emailom a pok\xFAsime sa v\xE1m \u010Do najsk\xF4r obnovi\u0165 pr\xEDstup.",
        title: "Z\xEDska\u0165 pomoc"
      },
      subtitle: "M\xE1te probl\xE9my? Na prihl\xE1senie m\xF4\u017Eete pou\u017Ei\u0165 jednu z t\xFDchto met\xF3d.",
      title: "Pou\u017Ei\u0165 in\xFA met\xF3du"
    },
    alternativePhoneCodeProvider: {
      formTitle: void 0,
      resendButton: void 0,
      subtitle: void 0,
      title: void 0
    },
    backupCodeMfa: {
      subtitle: "pre pokra\u010Dovanie do {{applicationName}}",
      title: "Zadajte z\xE1lo\u017En\xFD k\xF3d"
    },
    emailCode: {
      formTitle: "Overovac\xED k\xF3d",
      resendButton: "Znovu posla\u0165 k\xF3d",
      subtitle: "pre pokra\u010Dovanie do {{applicationName}}",
      title: "Skontrolujte svoj email"
    },
    emailLink: {
      clientMismatch: {
        subtitle: "Pre pokra\u010Dovanie otvorte overovac\xED odkaz na zariaden\xED a prehliada\u010Di pou\u017Eitom na prihl\xE1senie.",
        title: "Overovac\xED odkaz nie je platn\xFD pre toto zariadenie"
      },
      expired: {
        subtitle: "Vr\xE1\u0165te sa do p\xF4vodn\xE9ho okna pre pokra\u010Dovanie.",
        title: "Tento overovac\xED odkaz vypr\u0161al"
      },
      failed: {
        subtitle: "Vr\xE1\u0165te sa do p\xF4vodn\xE9ho okna pre pokra\u010Dovanie.",
        title: "Tento overovac\xED odkaz je neplatn\xFD"
      },
      formSubtitle: "Pou\u017Eite overovac\xED odkaz zaslan\xFD na v\xE1\u0161 email",
      formTitle: "Overovac\xED odkaz",
      loading: {
        subtitle: "\u010Coskoro budete presmerovan\xED",
        title: "Prihlasujem..."
      },
      resendButton: "Znovu posla\u0165 odkaz",
      subtitle: "pre pokra\u010Dovanie do {{applicationName}}",
      title: "Skontrolujte svoj email",
      unusedTab: {
        title: "M\xF4\u017Eete zatvori\u0165 toto okno"
      },
      verified: {
        subtitle: "\u010Coskoro budete presmerovan\xED",
        title: "\xDAspe\u0161ne prihl\xE1sen\xE9"
      },
      verifiedSwitchTab: {
        subtitle: "Vr\xE1\u0165te sa do p\xF4vodn\xE9ho okna pre pokra\u010Dovanie",
        subtitleNewTab: "Vr\xE1\u0165te sa do novootvoren\xE9ho okna pre pokra\u010Dovanie",
        titleNewTab: "Prihl\xE1sen\xE9 v inom okne"
      }
    },
    enterpriseConnections: {
      subtitle: void 0,
      title: void 0
    },
    forgotPassword: {
      formTitle: "Overovac\xED k\xF3d pre obnovenie hesla",
      resendButton: "Znovu posla\u0165 k\xF3d",
      subtitle: "pre obnovenie hesla",
      subtitle_email: "Najprv zadajte k\xF3d zaslan\xFD na v\xE1\u0161 email",
      subtitle_phone: "Najprv zadajte k\xF3d zaslan\xFD na va\u0161e telef\xF3nne \u010D\xEDslo",
      title: "Obnovi\u0165 heslo"
    },
    forgotPasswordAlternativeMethods: {
      blockButton__resetPassword: "Obnovi\u0165 heslo",
      label__alternativeMethods: "Alebo sa prihl\xE1ste pomocou inej met\xF3dy.",
      title: "Zabudli ste heslo?"
    },
    noAvailableMethods: {
      message: "Nemo\u017Eno pokra\u010Dova\u0165 v prihl\xE1sen\xED. Nie je k dispoz\xEDcii \u017Eiadna dostupn\xE1 autentifika\u010Dn\xE1 met\xF3da.",
      subtitle: "Do\u0161lo k chybe",
      title: "Nie je mo\u017En\xE9 sa prihl\xE1si\u0165"
    },
    passkey: {
      subtitle: "Pou\u017Eitie passkey potvrd\xED va\u0161u identitu. Va\u0161e zariadenie v\xE1s m\xF4\u017Ee vyzva\u0165 na potvrdenie odtla\u010Dkom, tv\xE1rou alebo k\xF3dom.",
      title: "Prihl\xE1si\u0165 sa pomocou Passkey"
    },
    password: {
      actionLink: "Pou\u017Ei\u0165 in\xFA met\xF3du",
      subtitle: "pre pokra\u010Dovanie do {{applicationName}}",
      title: "Zadajte svoje heslo"
    },
    passwordPwned: {
      title: void 0
    },
    phoneCode: {
      formTitle: "Overovac\xED k\xF3d",
      resendButton: "Znova odosla\u0165 k\xF3d",
      subtitle: "pre pokra\u010Dovanie do {{applicationName}}",
      title: "Skontrolujte v\xE1\u0161 telef\xF3n"
    },
    phoneCodeMfa: {
      formTitle: "Overovac\xED k\xF3d",
      resendButton: "Znova odosla\u0165 k\xF3d",
      subtitle: "Pre pokra\u010Dovanie zadajte overovac\xED k\xF3d zaslan\xFD na va\u0161e telef\xF3nne \u010D\xEDslo",
      title: "Skontrolujte v\xE1\u0161 telef\xF3n"
    },
    resetPassword: {
      formButtonPrimary: "Obnovi\u0165 heslo",
      requiredMessage: "Z bezpe\u010Dnostn\xFDch d\xF4vodov je potrebn\xE9 obnovenie v\xE1\u0161ho hesla.",
      successMessage: "Va\u0161e heslo bolo \xFAspe\u0161ne zmenen\xE9. Prihlasujem v\xE1s, pros\xEDm po\u010Dkajte okam\u017Eite.",
      title: "Nastavi\u0165 nov\xE9 heslo"
    },
    resetPasswordMfa: {
      detailsLabel: "Pred obnoven\xEDm hesla je potrebn\xE9 overi\u0165 va\u0161u toto\u017Enos\u0165."
    },
    start: {
      actionLink: "Registrova\u0165 sa",
      actionLink__join_waitlist: "Prida\u0165 sa na waitlist",
      actionLink__use_email: "Pou\u017Ei\u0165 email",
      actionLink__use_email_username: "Pou\u017Ei\u0165 email alebo u\u017E\xEDvate\u013Esk\xE9 meno",
      actionLink__use_passkey: "Pou\u017Ei\u0165 Passkey",
      actionLink__use_phone: "Pou\u017Ei\u0165 telef\xF3n",
      actionLink__use_username: "Pou\u017Ei\u0165 u\u017E\xEDvate\u013Esk\xE9 meno",
      actionText: "Nem\xE1te \xFA\u010Det?",
      actionText__join_waitlist: "Chcete predbe\u017En\xFD pr\xEDstup?",
      alternativePhoneCodeProvider: {
        actionLink: void 0,
        label: void 0,
        subtitle: void 0,
        title: void 0
      },
      subtitle: "pre pokra\u010Dovanie do {{applicationName}}",
      subtitleCombined: void 0,
      title: "Prihl\xE1si\u0165 sa do {{applicationName}}",
      titleCombined: "Pokra\u010Dova\u0165 do {{applicationName}}"
    },
    totpMfa: {
      formTitle: "Overovac\xED k\xF3d",
      subtitle: "Pre pokra\u010Dovanie zadajte overovac\xED k\xF3d z va\u0161ej autentifika\u010Dnej aplik\xE1cie",
      title: "Dvojfaktorov\xE9 overenie"
    }
  },
  signInEnterPasswordTitle: "Zadajte svoje heslo",
  signUp: {
    alternativePhoneCodeProvider: {
      resendButton: void 0,
      subtitle: void 0,
      title: void 0
    },
    continue: {
      actionLink: "Prihl\xE1si\u0165 sa",
      actionText: "M\xE1te \xFA\u010Det?",
      subtitle: "Pre pokra\u010Dovanie pros\xEDm dopl\u0148te zvy\u0161n\xE9 inform\xE1cie.",
      title: "Vypl\u0148te ch\xFDbaj\xFAce polia"
    },
    emailCode: {
      formSubtitle: "Zadajte overovac\xED k\xF3d poslan\xFD na va\u0161u emailov\xFA adresu",
      formTitle: "Overovac\xED k\xF3d",
      resendButton: "Znovu posla\u0165 k\xF3d",
      subtitle: "Vlo\u017Ete overovac\xED k\xF3d zaslan\xFD na va\u0161u emailov\xFA adresu",
      title: "Overte svoj email"
    },
    emailLink: {
      clientMismatch: {
        subtitle: "Pre pokra\u010Dovanie otvorte overovac\xED odkaz na zariaden\xED a prehliada\u010Di pou\u017Eitom na registr\xE1ciu.",
        title: "Overovac\xED odkaz nie je platn\xFD pre toto zariadenie"
      },
      formSubtitle: "Pou\u017Eite overovac\xED odkaz poslan\xFD na va\u0161u emailov\xFA adresu",
      formTitle: "Overovac\xED odkaz",
      loading: {
        title: "Prebieha registr\xE1cia..."
      },
      resendButton: "Znovu posla\u0165 odkaz",
      subtitle: "pre pokra\u010Dovanie do {{applicationName}}",
      title: "Overte svoj email",
      verified: {
        title: "\xDAspe\u0161ne zaregistrovan\xE9"
      },
      verifiedSwitchTab: {
        subtitle: "Vr\xE1\u0165te sa do novootvoren\xE9ho okna pre pokra\u010Dovanie",
        subtitleNewTab: "Vr\xE1\u0165te sa do predch\xE1dzaj\xFAceho okna pre pokra\u010Dovanie",
        title: "Email \xFAspe\u0161ne overen\xFD"
      }
    },
    enterpriseConnections: {
      subtitle: void 0,
      title: void 0
    },
    legalConsent: {
      checkbox: {
        label__onlyPrivacyPolicy: 'S\xFAhlas\xEDm s {{ privacyPolicyLink || link("podmienkami o ochrane osobn\xFDch \xFAdajov") }}',
        label__onlyTermsOfService: 'S\xFAhlas\xEDm s {{ termsOfServiceLink || link("podmienkami pou\u017E\xEDvania") }}',
        label__termsOfServiceAndPrivacyPolicy: 'S\xFAhlas\xEDm s {{ termsOfServiceLink || link("podmienkami pou\u017E\xEDvania") }} a {{ privacyPolicyLink || link("podmienkami o ochrane osobn\xFDch \xFAdajov") }}'
      },
      continue: {
        subtitle: "Pros\xEDm, pre\u010D\xEDtajte si a s\xFAhlaste s na\u0161imi podmienkami pou\u017E\xEDvania a podmienkami o ochrane osobn\xFDch \xFAdajov.",
        title: "S\xFAhlas s podmienkami"
      }
    },
    phoneCode: {
      formSubtitle: "Zadajte overovac\xED k\xF3d poslan\xFD na va\u0161e telef\xF3nne \u010D\xEDslo",
      formTitle: "Overovac\xED k\xF3d",
      resendButton: "Znovu posla\u0165 k\xF3d",
      subtitle: "Pre pokra\u010Dovanie zadajte overovac\xED k\xF3d zaslan\xFD na va\u0161e telef\xF3nne \u010D\xEDslo",
      title: "Overte svoj telef\xF3n"
    },
    restrictedAccess: {
      actionLink: "Prihl\xE1si\u0165 sa",
      actionText: "M\xE1te \xFA\u010Det?",
      blockButton__emailSupport: "Podpora cez email",
      blockButton__joinWaitlist: "Prida\u0165 sa na waitlist",
      subtitle: "Registr\xE1cie aktu\xE1lne nie s\xFA povolen\xE9. Ak si mysl\xEDte, \u017Ee by ste mali ma\u0165 pr\xEDstup, kontaktujte na\u0161u podporu.",
      subtitleWaitlist: "Registr\xE1cie aktu\xE1lne nie s\xFA povolen\xE9. Ak chete by\u0165 informovan\xED o spusten\xED medzi prv\xFDmi, pridajte sa na waitlist.",
      title: "Zablokovan\xFD pr\xEDstup"
    },
    start: {
      actionLink: "Prihl\xE1si\u0165 sa",
      actionLink__use_email: "Pou\u017Ei\u0165 email",
      actionLink__use_phone: "Pou\u017Ei\u0165 telef\xF3n",
      actionText: "M\xE1te \xFA\u010Det?",
      alternativePhoneCodeProvider: {
        actionLink: void 0,
        label: void 0,
        subtitle: void 0,
        title: void 0
      },
      subtitle: "Vitajte! Pre pokra\u010Dovanie pros\xEDm vypl\u0148te inform\xE1cie.",
      subtitleCombined: "Vitajte! Pre pokra\u010Dovanie pros\xEDm vypl\u0148te inform\xE1cie.",
      title: "Vytvorte si \xFA\u010Det",
      titleCombined: "Vytvorte si \xFA\u010Det"
    }
  },
  socialButtonsBlockButton: "Pokra\u010Dova\u0165 s {{provider|titleize}}",
  socialButtonsBlockButtonManyInView: void 0,
  taskChooseOrganization: {
    chooseOrganization: {
      action__createOrganization: void 0,
      action__invitationAccept: void 0,
      action__suggestionsAccept: void 0,
      subtitle: void 0,
      suggestionsAcceptedLabel: void 0,
      title: void 0
    },
    createOrganization: {
      formButtonReset: void 0,
      formButtonSubmit: void 0,
      formFieldInputPlaceholder__name: void 0,
      formFieldInputPlaceholder__slug: void 0,
      formFieldLabel__name: void 0,
      formFieldLabel__slug: void 0,
      subtitle: void 0,
      title: void 0
    },
    signOut: {
      actionLink: void 0,
      actionText: void 0
    }
  },
  unstable__errors: {
    already_a_member_in_organization: void 0,
    captcha_invalid: "Registr\xE1cia zlyhala z d\xF4vodu ne\xFAspe\u0161n\xE9ho overenia zabezpe\u010Denia. Obnovte str\xE1nku a sk\xFAste to znova alebo kontaktujte podporu pre \u010Fal\u0161iu pomoc.",
    captcha_unavailable: "Registr\xE1cia zlyhala z d\xF4vodu ne\xFAspe\u0161n\xE9ho overenia proti botom. Obnovte str\xE1nku a sk\xFAste to znova alebo kontaktujte podporu pre \u010Fal\u0161iu pomoc.",
    form_code_incorrect: void 0,
    form_identifier_exists__email_address: "T\xE1to emailov\xE1 adresa je u\u017E obsaden\xE1. Sk\xFAste pros\xEDm in\xFA.",
    form_identifier_exists__phone_number: "Toto telef\xF3nne \u010D\xEDslo je u\u017E obsaden\xE9. Sk\xFAste pros\xEDm in\xE9.",
    form_identifier_exists__username: "T\xE1to pou\u017Eivate\u013Esk\xE9 meno je u\u017E obsaden\xE9. Sk\xFAste pros\xEDm in\xE9.",
    form_identifier_not_found: "\xDA\u010Det s t\xFDmto identifik\xE1torom nebol n\xE1jden\xFD. Skontrolujte \xFAdaje a sk\xFAste to znova.",
    form_param_format_invalid: "Zadan\xE1 hodnota nie je v spr\xE1vnom form\xE1te. Skontrolujte a opravte.",
    form_param_format_invalid__email_address: "Emailov\xE1 adresa mus\xED by\u0165 platn\xE1.",
    form_param_format_invalid__phone_number: "Telef\xF3nne \u010D\xEDslo mus\xED by\u0165 v platnom medzin\xE1rodnom form\xE1te.",
    form_param_max_length_exceeded__first_name: "Krstn\xE9 meno nesmie by\u0165 dlh\u0161ie ako 256 znakov.",
    form_param_max_length_exceeded__last_name: "Priezvisko nesmie by\u0165 dlh\u0161ie ako 256 znakov.",
    form_param_max_length_exceeded__name: "Meno nesmie by\u0165 dlh\u0161ie ako 256 znakov.",
    form_param_nil: "Pole je povinn\xE9 a nesmie by\u0165 pr\xE1zdne.",
    form_param_type_invalid: void 0,
    form_param_type_invalid__email_address: void 0,
    form_param_type_invalid__phone_number: void 0,
    form_param_value_invalid: "Hodnota je neplatn\xE1. Skontrolujte a opravte.",
    form_password_incorrect: "Heslo je nespr\xE1vne. Skontrolujte a sk\xFAste to znova.",
    form_password_length_too_short: "Heslo mus\xED ma\u0165 aspo\u0148 8 znakov.",
    form_password_not_strong_enough: "Va\u0161e heslo nie je dostato\u010Dne siln\xE9.",
    form_password_pwned: "Toto heslo bolo n\xE1jden\xE9 v r\xE1mci \xFAniku d\xE1t a nem\xF4\u017Ee by\u0165 pou\u017Eit\xE9, pros\xEDm zvo\u013Ete in\xE9 heslo.",
    form_password_pwned__sign_in: "Toto heslo bolo n\xE1jden\xE9 v r\xE1mci \xFAniku d\xE1t a nem\xF4\u017Ee by\u0165 pou\u017Eit\xE9, pros\xEDm zvo\u013Ete in\xE9 heslo.",
    form_password_size_in_bytes_exceeded: "Va\u0161e heslo prekro\u010Dilo maxim\xE1lny povolen\xFD po\u010Det bytov, pros\xEDm skr\xE1\u0165te ho alebo odstr\xE1\u0148te niektor\xE9 \u0161peci\xE1lne znaky.",
    form_password_validation_failed: "Nespr\xE1vne heslo",
    form_username_invalid_character: "Pou\u017E\xEDvate\u013Esk\xE9 meno m\xF4\u017Ee obsahova\u0165 len p\xEDsmen\xE1, \u010D\xEDslice, poml\u010Dky a pod\u010Diarkovn\xEDky.",
    form_username_invalid_length: "Pou\u017E\xEDvate\u013Esk\xE9 meno mus\xED ma\u0165 aspo\u0148 {{min_length}} znaky a maxim\xE1lne {{max_length}} znakov.",
    identification_deletion_failed: "Nem\xF4\u017Eete odstr\xE1ni\u0165 v\xE1\u0161 posledn\xFD identifik\xE1tor.",
    not_allowed_access: "Nem\xE1te povolen\xFD pr\xEDstup k tejto str\xE1nke. Ak si mysl\xEDte, \u017Ee je to chyba, kontaktujte podporu.",
    organization_domain_blocked: void 0,
    organization_domain_common: void 0,
    organization_domain_exists_for_enterprise_connection: void 0,
    organization_membership_quota_exceeded: void 0,
    organization_minimum_permissions_needed: void 0,
    organization_not_found_or_unauthorized: void 0,
    organization_not_found_or_unauthorized_with_create_organization_disabled: void 0,
    passkey_already_exists: "Na tomto zariaden\xED u\u017E existuje passkey pre tento \xFA\u010Det.",
    passkey_not_supported: "V\xE1\u0161 prehliada\u010D alebo zariadenie nepodporuje passkey.",
    passkey_pa_not_supported: "Registr\xE1cia vy\u017Eaduje autentifik\xE1tor ale toto zariadenie ho nepodporuje.",
    passkey_registration_cancelled: "Registr\xE1cia passkey bola zru\u0161en\xE1 alebo vypr\u0161ala.",
    passkey_retrieval_cancelled: "Overenie passkey bolo zru\u0161en\xE9 alebo vypr\u0161alo.",
    passwordComplexity: {
      maximumLength: "menej ako {{length}} znakov",
      minimumLength: "{{length}} alebo viac znakov",
      requireLowercase: "mal\xE9 p\xEDsmeno",
      requireNumbers: "\u010D\xEDslicu",
      requireSpecialCharacter: "\u0161peci\xE1lny znak",
      requireUppercase: "ve\u013Ek\xE9 p\xEDsmeno",
      sentencePrefix: "Va\u0161e heslo mus\xED obsahova\u0165"
    },
    phone_number_exists: "Toto telef\xF3nne \u010D\xEDslo je u\u017E obsaden\xE9. Sk\xFAste pros\xEDm in\xE9.",
    session_exists: "Jste u\u017E p\u0159ihl\xE1\u0161en.",
    web3_missing_identifier: "Roz\u0161\xEDrenie Web3 Pe\u0148a\u017Eenky nebolo n\xE1jden\xE9. Je potrebn\xE9 ho nain\u0161talova\u0165.",
    zxcvbn: {
      couldBeStronger: "Va\u0161e heslo funguje, ale mohlo by by\u0165 silnej\u0161ie. Sk\xFAste prida\u0165 viac znakov.",
      goodPassword: "Dobr\xE1 pr\xE1ca. Toto je vynikaj\xFAce heslo.",
      notEnough: "Va\u0161e heslo nen\xED dostato\u010Dne siln\xE9.",
      suggestions: {
        allUppercase: "Pou\u017Eite ve\u013Ek\xE9 p\xEDsmena len u niektor\xFDch, nie v\u0161etk\xFDch p\xEDsmen.",
        anotherWord: "Pridajte viac slov, ktor\xE9 nie s\xFA tak be\u017En\xE9.",
        associatedYears: "Vyhnite sa rokom, ktor\xE9 s\xFA s vami spojen\xE9.",
        capitalization: "P\xEDsmen\xE1 p\xED\u0161te s ve\u013Ek\xFDm po\u010Diato\u010Dn\xFDm p\xEDsmenom a viac ako len prv\xFDm p\xEDsmenom.",
        dates: "Vyhnite sa d\xE1tumom a rokom, ktor\xE9 s\xFA s vami spojen\xE9.",
        l33t: "Vyhnite sa predv\xEDdate\u013En\xFDm n\xE1hrad\xE1m p\xEDsmen, napr\xEDklad '@' miesto 'a'.",
        longerKeyboardPattern: "Pou\u017Eite dlh\u0161ie vzory na kl\xE1vesnici a menite smer p\xEDsania viackr\xE1t.",
        noNeed: "M\xF4\u017Eete vytv\xE1ra\u0165 siln\xE9 hesl\xE1 aj bez pou\u017Eitia symbolov, \u010D\xEDsel alebo ve\u013Ek\xFDch p\xEDsmen.",
        pwned: "Ak pou\u017E\xEDvate toto heslo aj niekde inde, mali by ste ho zmeni\u0165.",
        recentYears: "Vyhnite sa ned\xE1vnym rokom.",
        repeated: "Vyhnite sa opakuj\xFAcim sa slov\xE1m a znakom.",
        reverseWords: "Vyhnite sa obr\xE1ten\xFDm pravopisom be\u017En\xFDch slov.",
        sequences: "Vyhnite sa be\u017En\xFDm sekvenci\xE1m znakov.",
        useWords: "Pou\u017Eite viac slov, ale vyhnite sa be\u017En\xFDm fr\xE1zam."
      },
      warnings: {
        common: "Toto je be\u017Ene pou\u017E\xEDvan\xE9 heslo.",
        commonNames: "Be\u017En\xE9 men\xE1 a priezvisk\xE1 s\xFA \u013Eahko uh\xE1dnute\u013En\xE9.",
        dates: "D\xE1tum je \u013Eahko uh\xE1dnute\u013En\xFD.",
        extendedRepeat: 'Opakuj\xFAce sa vzory znakov ako "abcabcabc" s\xFA \u013Eahko uh\xE1dnute\u013En\xE9.',
        keyPattern: "Kr\xE1tke vzory na kl\xE1vesnici s\xFA \u013Eahko uh\xE1dnute\u013En\xE9.",
        namesByThemselves: "Samostatn\xE9 men\xE1 alebo priezvisk\xE1 s\xFA \u013Eahko uh\xE1dnute\u013En\xE9.",
        pwned: "Va\u0161e heslo bolo odhalen\xE9 pri \xFAniku \xFAdajov na internete.",
        recentYears: "Ned\xE1vne roky s\xFA \u013Eahko uh\xE1dnute\u013En\xE9.",
        sequences: 'Be\u017En\xE9 sekvencie znakov ako "abc" s\xFA \u013Eahko uh\xE1dnute\u013En\xE9.',
        similarToCommon: "Toto je podobn\xE9 be\u017Ene pou\u017E\xEDvan\xE9mu heslu.",
        simpleRepeat: 'Opakuj\xFAce sa znaky ako "aaa" s\xFA \u013Eahko uh\xE1dnute\u013En\xE9.',
        straightRow: "Rady kl\xE1vesnice s\xFA \u013Eahko uh\xE1dnute\u013En\xE9.",
        topHundred: "Toto je \u010Dasto pou\u017E\xEDvan\xE9 heslo.",
        topTen: "Toto je \u010Dasto pou\u017E\xEDvan\xE9 heslo.",
        userInputs: "Heslo by nemalo obsahova\u0165 osobn\xE9 alebo str\xE1nkou s\xFAvisiace \xFAdaje.",
        wordByItself: "Samostatn\xE9 slov\xE1 s\xFA \u013Eahko uh\xE1dnute\u013En\xE9."
      }
    }
  },
  userButton: {
    action__addAccount: "Prida\u0165 \xFA\u010Det",
    action__closeUserMenu: void 0,
    action__manageAccount: "Spravova\u0165 \xFA\u010Det",
    action__openUserMenu: void 0,
    action__signOut: "Odhl\xE1si\u0165 sa",
    action__signOutAll: "Odhl\xE1si\u0165 sa zo v\u0161etk\xFDch \xFA\u010Dtov"
  },
  userProfile: {
    apiKeysPage: {
      title: "K\u013E\xFA\u010De API"
    },
    backupCodePage: {
      actionLabel__copied: "Skop\xEDrovan\xE9!",
      actionLabel__copy: "Kop\xEDrova\u0165 v\u0161etko",
      actionLabel__download: "Stiahnu\u0165 .txt",
      actionLabel__print: "Vytla\u010Di\u0165",
      infoText1: "Pre tento \xFA\u010Det bud\xFA povolen\xE9 z\xE1lo\u017En\xE9 k\xF3dy.",
      infoText2: "Z\xE1lo\u017En\xE9 k\xF3dy uchov\xE1vajte tajne a bezpe\u010Dne. M\xF4\u017Eete vygenerova\u0165 nov\xE9 z\xE1lo\u017En\xE9 k\xF3dy, ak m\xE1te podozrenie, \u017Ee boli skompromitovan\xE9.",
      subtitle__codelist: "Uchov\xE1vajte ich bezpe\u010Dne a tajne.",
      successMessage: "Z\xE1lo\u017En\xE9 k\xF3dy s\xFA teraz povolen\xE9. Ak strat\xEDte pr\xEDstup k v\xE1\u0161mu overovaciemu zariadeniu, m\xF4\u017Eete pou\u017Ei\u0165 jeden z t\xFDchto k\xF3dov na prihl\xE1senie do v\xE1\u0161ho \xFA\u010Dtu. Ka\u017Ed\xFD k\xF3d mo\u017Eno pou\u017Ei\u0165 iba raz.",
      successSubtitle: "Pou\u017Eite jeden z t\xFDchto k\xF3dov na prihl\xE1senie do v\xE1\u0161ho \xFA\u010Dtu, ak strat\xEDte pr\xEDstup k v\xE1\u0161mu overovaciemu zariadeniu.",
      title: "Prida\u0165 overovanie pomocou z\xE1lo\u017En\xFDch k\xF3dov",
      title__codelist: "Z\xE1lo\u017En\xE9 k\xF3dy"
    },
    billingPage: {
      paymentHistorySection: {
        empty: void 0,
        notFound: void 0,
        tableHeader__amount: void 0,
        tableHeader__date: void 0,
        tableHeader__status: void 0
      },
      paymentMethodsSection: {
        actionLabel__default: void 0,
        actionLabel__remove: void 0,
        add: void 0,
        addSubtitle: void 0,
        cancelButton: void 0,
        formButtonPrimary__add: void 0,
        formButtonPrimary__pay: void 0,
        payWithTestCardButton: void 0,
        removeMethod: {
          messageLine1: void 0,
          messageLine2: void 0,
          successMessage: void 0,
          title: void 0
        },
        title: void 0
      },
      start: {
        headerTitle__payments: void 0,
        headerTitle__plans: "Bal\xEDky",
        headerTitle__statements: "Fakt\xFAry",
        headerTitle__subscriptions: void 0
      },
      statementsSection: {
        empty: void 0,
        itemCaption__paidForPlan: void 0,
        itemCaption__proratedCredit: void 0,
        itemCaption__subscribedAndPaidForPlan: void 0,
        notFound: void 0,
        tableHeader__amount: void 0,
        tableHeader__date: void 0,
        title: void 0,
        totalPaid: void 0
      },
      subscriptionsListSection: {
        actionLabel__manageSubscription: void 0,
        actionLabel__newSubscription: void 0,
        actionLabel__switchPlan: void 0,
        tableHeader__edit: void 0,
        tableHeader__plan: void 0,
        tableHeader__startDate: void 0,
        title: void 0
      },
      subscriptionsSection: {
        actionLabel__default: void 0
      },
      switchPlansSection: {
        title: void 0
      },
      title: "Fatur\xE1cia a platby"
    },
    connectedAccountPage: {
      formHint: "Vyberte poskytovate\u013Ea pre pripojenie v\xE1\u0161ho \xFA\u010Dtu.",
      formHint__noAccounts: "Nie s\xFA k dispoz\xEDcii \u017Eiadni dostupn\xED extern\xED poskytovatelia \xFA\u010Dtov.",
      removeResource: {
        messageLine1: "{{identifier}} bude odobran\xFD z tohto \xFA\u010Dtu.",
        messageLine2: "Nebudete u\u017E m\xF4c\u0165 pou\u017E\xEDva\u0165 tento pripojen\xFD \xFA\u010Det a ak\xE9ko\u013Evek z\xE1visl\xE9 funkcie prestan\xFA fungova\u0165.",
        successMessage: "{{connectedAccount}} bol odstr\xE1nen\xFD z v\xE1\u0161ho \xFA\u010Dtu.",
        title: "Odstr\xE1ni\u0165 pripojen\xFD \xFA\u010Det"
      },
      socialButtonsBlockButton: "Pripoji\u0165 \xFA\u010Det {{provider|titleize}}",
      successMessage: "Poskytovate\u013E bol pridan\xFD k v\xE1\u0161mu \xFA\u010Dtu.",
      title: "Prida\u0165 pripojen\xFD \xFA\u010Det"
    },
    deletePage: {
      actionDescription: 'Pre pokra\u010Dovanie nap\xED\u0161te "Zmaza\u0165 \xFA\u010Det".',
      confirm: "Zmaza\u0165 \xFA\u010Det",
      messageLine1: "Naozaj chcete odstr\xE1ni\u0165 svoj \xFA\u010Det? Niektor\xE9 s\xFAvisiace \xFAdaje m\xF4\u017Eu zosta\u0165 zachovan\xE9. Ak chcete po\u017Eiada\u0165 o \xFApln\xE9 vymazanie \xFAdajov, obr\xE1\u0165te sa na podporu.",
      messageLine2: "T\xE1to akcia je trval\xE1 a nem\xF4\u017Ee by\u0165 vr\xE1ten\xE1 sp\xE4\u0165.",
      title: "Zmaza\u0165 \xFA\u010Det"
    },
    emailAddressPage: {
      emailCode: {
        formHint: "Na t\xFAto e-mailov\xFA adresu bude odoslan\xFD overovac\xED k\xF3d.",
        formSubtitle: "Zadajte overovac\xED k\xF3d zaslan\xFD na adresu {{identifier}}",
        formTitle: "Overovac\xED k\xF3d",
        resendButton: "Znovu odosla\u0165 k\xF3d",
        successMessage: "E-mailov\xE1 adresa {{identifier}} bola pridan\xE1 k v\xE1\u0161mu \xFA\u010Dtu."
      },
      emailLink: {
        formHint: "Na t\xFAto e-mailov\xFA adresu bude odoslan\xFD overovac\xED odkaz.",
        formSubtitle: "Kliknite na overovac\xED odkaz v e-maile zaslanom na adresu {{identifier}}",
        formTitle: "Overovac\xED odkaz",
        resendButton: "Znovu odosla\u0165 odkaz",
        successMessage: "E-mailov\xE1 adresa {{identifier}} bola pridan\xE1 k v\xE1\u0161mu \xFA\u010Dtu."
      },
      enterpriseSSOLink: {
        formButton: "Pre prihl\xE1senie kliknite",
        formSubtitle: "Dokon\u010Di\u0165 prihl\xE1senie pomocou {{identifier}}"
      },
      formHint: "Pred pridan\xEDm tejto e-mailovej adresy k v\xE1\u0161mu \xFA\u010Dtu je potrebn\xE9 jej overenie.",
      removeResource: {
        messageLine1: "{{identifier}} bude odstr\xE1nen\xE1 z tohto \xFA\u010Dtu.",
        messageLine2: "Nebudete sa m\xF4c\u0165 prihl\xE1si\u0165 pomocou tejto e-mailovej adresy.",
        successMessage: "{{emailAddress}} bola odobran\xE1 z v\xE1\u0161ho \xFA\u010Dtu.",
        title: "Odstr\xE1ni\u0165 e-mailov\xFA adresu"
      },
      title: "Prida\u0165 e-mailov\xFA adresu",
      verifyTitle: "Overi\u0165 e-mailov\xFA adresu"
    },
    formButtonPrimary__add: "Prida\u0165",
    formButtonPrimary__continue: "Pokra\u010Dova\u0165",
    formButtonPrimary__finish: "Dokon\u010Di\u0165",
    formButtonPrimary__remove: "Odstr\xE1ni\u0165",
    formButtonPrimary__save: "Ulo\u017Ei\u0165",
    formButtonReset: "Zru\u0161i\u0165",
    mfaPage: {
      formHint: "Vyberte sp\xF4sob pridania.",
      title: "Prida\u0165 dvojfaktorov\xE9 overenie"
    },
    mfaPhoneCodePage: {
      backButton: "Pou\u017Ei\u0165 existuj\xFAce \u010D\xEDslo",
      primaryButton__addPhoneNumber: "Prida\u0165 telef\xF3nne \u010D\xEDslo",
      removeResource: {
        messageLine1: "{{identifier}} u\u017E nebude dost\xE1va\u0165 overovacie k\xF3dy pri prihlasovan\xED.",
        messageLine2: "V\xE1\u0161 \xFA\u010Det nemus\xED by\u0165 tak bezpe\u010Dn\xFD. Naozaj chcete pokra\u010Dova\u0165?",
        successMessage: "Dvojfaktorov\xE9 overovanie pomocou SMS k\xF3du bolo odstr\xE1nen\xE9 pre {{mfaPhoneCode}}",
        title: "Odstr\xE1ni\u0165 dvojfaktorov\xE9 overovanie"
      },
      subtitle__availablePhoneNumbers: "Vyberte telef\xF3nne \u010D\xEDslo pre registr\xE1ciu dvojfaktorov\xE9ho overovania pomocou SMS k\xF3du.",
      subtitle__unavailablePhoneNumbers: "Nie s\xFA k dispoz\xEDcii \u017Eiadne dostupn\xE9 telef\xF3nne \u010D\xEDsla pre registr\xE1ciu dvojfaktorov\xE9ho overovania pomocou SMS k\xF3du.",
      successMessage1: "Ako dodato\u010Dn\xFD krok, pri prihl\xE1sen\xED budete musie\u0165 zada\u0165 overovac\xED k\xF3d odoslan\xFD na toto telef\xF3nne \u010D\xEDslo.",
      successMessage2: "Ulo\u017Ete si tieto z\xE1lo\u017En\xE9 k\xF3dy a uchovajte ich na bezpe\u010Dnom mieste. Ak strat\xEDte pr\xEDstup k svojmu autentifika\u010Dn\xE9mu zariadeniu, m\xF4\u017Eete sa prihl\xE1si\u0165 pomocou z\xE1lo\u017En\xFDch k\xF3dov.",
      successTitle: "Dvojfaktorov\xE9 overenie pomocou SMS k\xF3du je teraz povolen\xE9",
      title: "Prida\u0165 overovanie pomocou SMS k\xF3du"
    },
    mfaTOTPPage: {
      authenticatorApp: {
        buttonAbleToScan__nonPrimary: "Namiesto toho naskenujte QR k\xF3d",
        buttonUnableToScan__nonPrimary: "Nem\xF4\u017Eete naskenova\u0165 QR k\xF3d?",
        infoText__ableToScan: "Nastavte nov\xFA met\xF3du prihl\xE1senia vo va\u0161ej aplik\xE1cii pre overovanie a naskenujte nasleduj\xFAci QR k\xF3d, aby ste ju spojili so svoj\xEDm \xFA\u010Dtom.",
        infoText__unableToScan: "Nastavte nov\xFA met\xF3du prihl\xE1senia vo va\u0161ej aplik\xE1cii pre overovanie a zadajte ni\u017E\u0161ie uveden\xFD k\u013E\xFA\u010D.",
        inputLabel__unableToScan1: "Uistite sa, \u017Ee je povolen\xE9 \u010Dasovo z\xE1visl\xE9 alebo jednor\xE1zov\xE9 heslo a dokon\u010Dite spojenie v\xE1\u0161ho \xFA\u010Dtu.",
        inputLabel__unableToScan2: "Alternat\xEDvne, ak va\u0161a aplik\xE1cia pre overovanie podporuje TOTP URI, m\xF4\u017Eete tie\u017E skop\xEDrova\u0165 cel\xFD URI."
      },
      removeResource: {
        messageLine1: "Pri prihlasovan\xED u\u017E nebud\xFA vy\u017Eadovan\xE9 overovacie k\xF3dy z tejto aplik\xE1cie pre overovanie.",
        messageLine2: "V\xE1\u0161 \xFA\u010Det nemus\xED by\u0165 tak bezpe\u010Dn\xFD. Naozaj chcete pokra\u010Dova\u0165?",
        successMessage: "Dvojfaktorov\xE9 overovanie pomocou aplik\xE1cie pre overovanie bolo odstr\xE1nen\xE9.",
        title: "Odstr\xE1ni\u0165 dvojfaktorov\xE9 overovanie"
      },
      successMessage: "Dvojfaktorov\xE9 overovanie je teraz povolen\xE9. Pri prihl\xE1sen\xED budete musie\u0165 zada\u0165 overovac\xED k\xF3d z tejto aplik\xE1cie pre overovanie ako \u010Fal\u0161\xED krok.",
      title: "Prida\u0165 aplik\xE1ciu pre overovanie",
      verifySubtitle: "Zadajte overovac\xED k\xF3d generovan\xFD va\u0161ou aplik\xE1ciou pre overovanie.",
      verifyTitle: "Overovac\xED k\xF3d"
    },
    mobileButton__menu: "Menu",
    navbar: {
      account: "Profil",
      apiKeys: "K\u013E\xFA\u010De API",
      billing: "Faktur\xE1cia",
      description: "Spravujte \xFAdaje svojho \xFA\u010Dtu a zabezpe\u010Denie",
      security: "Bezpe\u010Dnos\u0165",
      title: "\xDA\u010Det"
    },
    passkeyScreen: {
      removeResource: {
        messageLine1: "{{name}} bude odstr\xE1nen\xFD z tohto \xFA\u010Dtu.",
        title: "Odstr\xE1ni\u0165 Passkey"
      },
      subtitle__rename: "Zme\u0148te n\xE1zov Passkey na nie\u010Do, \u010Do v\xE1m pom\xF4\u017Ee si ho zapam\xE4ta\u0165.",
      title__rename: "Premenova\u0165 Passkey"
    },
    passwordPage: {
      checkboxInfoText__signOutOfOtherSessions: "Odpor\xFA\u010Da sa odhl\xE1si\u0165 zo v\u0161etk\xFDch ostatn\xFDch zariaden\xED, ktor\xE9 mohli pou\u017E\xEDva\u0165 va\u0161e star\xE9 heslo.",
      readonly: "Va\u0161e heslo moment\xE1lne nie je mo\u017En\xE9 upravi\u0165, preto\u017Ee sa m\xF4\u017Eete prihl\xE1si\u0165 iba cez firemn\xE9 prihl\xE1senie.",
      successMessage__set: "Va\u0161e heslo bolo nastaven\xE9.",
      successMessage__signOutOfOtherSessions: "V\u0161etky ostatn\xE9 zariadenia boli odhl\xE1sen\xE9.",
      successMessage__update: "Va\u0161e heslo bolo aktualizovan\xE9.",
      title__set: "Nastavi\u0165 heslo",
      title__update: "Zmeni\u0165 heslo"
    },
    phoneNumberPage: {
      infoText: "Na toto telef\xF3nne \u010D\xEDslo bude odoslan\xE1 textov\xE1 spr\xE1va obsahuj\xFAca overovac\xED odkaz.",
      removeResource: {
        messageLine1: "{{identifier}} bude odobran\xE9 z tohto \xFA\u010Dtu.",
        messageLine2: "Nebudete sa m\xF4c\u0165 prihl\xE1si\u0165 pomocou tohto telef\xF3nneho \u010D\xEDsla.",
        successMessage: "{{phoneNumber}} bolo odstr\xE1nen\xE9 z v\xE1\u0161ho \xFA\u010Dtu.",
        title: "Odstr\xE1ni\u0165 telef\xF3nne \u010D\xEDslo"
      },
      successMessage: "{{identifier}} bolo pridan\xE9 k v\xE1\u0161mu \xFA\u010Dtu.",
      title: "Prida\u0165 telef\xF3nne \u010D\xEDslo",
      verifySubtitle: "Zadajte overovac\xED k\xF3d odoslan\xFD na {{identifier}}",
      verifyTitle: "Overte telef\xF3nne \u010D\xEDslo"
    },
    plansPage: {
      title: void 0
    },
    profilePage: {
      fileDropAreaHint: "Nahrajte obr\xE1zok vo form\xE1toch JPG, PNG, GIF alebo WEBP s ve\u013Ekos\u0165ou men\u0161ou ne\u017E 10 MB",
      imageFormDestructiveActionSubtitle: "Odstr\xE1ni\u0165 obr\xE1zok",
      imageFormSubtitle: "Nahra\u0165 obr\xE1zok",
      imageFormTitle: "Profilov\xFD obr\xE1zok",
      readonly: "Inform\xE1cie vo va\u0161om profile boli poskytnut\xE9 firemn\xFDm prihl\xE1sen\xEDm a nie je mo\u017En\xE9 ich upravi\u0165.",
      successMessage: "V\xE1\u0161 profil bol aktualizovan\xFD.",
      title: "Aktualizova\u0165 profil"
    },
    start: {
      activeDevicesSection: {
        destructiveAction: "Odhl\xE1si\u0165 sa zo zariadenia",
        title: "Akt\xEDvne zariadenia"
      },
      connectedAccountsSection: {
        actionLabel__connectionFailed: "Sk\xFAsi\u0165 znovu",
        actionLabel__reauthorize: "Autorizova\u0165 teraz",
        destructiveActionTitle: "Odstr\xE1ni\u0165",
        primaryButton: "Pripoji\u0165 \xFA\u010Det",
        subtitle__disconnected: "Tento \xFA\u010Det bol odpojen\xFD.",
        subtitle__reauthorize: "Po\u017Eadovan\xE9 opr\xE1vnenia boli aktualizovan\xE9, a preto m\xF4\u017Eete zaznamena\u0165 obmedzen\xFA funkcionalitu. Znovu autorizujte t\xFAto aplik\xE1ciu, aby ste predi\u0161li probl\xE9mom",
        title: "Pripojen\xE9 \xFA\u010Dty"
      },
      dangerSection: {
        deleteAccountButton: "Zmaza\u0165 \xFA\u010Det",
        title: "Zmazanie \xFA\u010Dtu"
      },
      emailAddressesSection: {
        destructiveAction: "Odstr\xE1ni\u0165 emailov\xFA adresu",
        detailsAction__nonPrimary: "Nastavi\u0165 ako hlavn\xFA",
        detailsAction__primary: "Dokon\u010Di\u0165 overenie",
        detailsAction__unverified: "Dokon\u010Di\u0165 overenie",
        primaryButton: "Prida\u0165 emailov\xFA adresu",
        title: "Emailov\xE9 adresy"
      },
      enterpriseAccountsSection: {
        title: "Firemn\xE9 \xFA\u010Dty"
      },
      headerTitle__account: "\xDA\u010Det",
      headerTitle__security: "Bezpe\u010Dnos\u0165",
      mfaSection: {
        backupCodes: {
          actionLabel__regenerate: "Obnovi\u0165 k\xF3dy",
          headerTitle: "Z\xE1lo\u017En\xE9 k\xF3dy",
          subtitle__regenerate: "Z\xEDskajte nov\xFA sadu zabezpe\u010Den\xFDch z\xE1lo\u017En\xFDch k\xF3dov. Predch\xE1dzaj\xFAce z\xE1lo\u017En\xE9 k\xF3dy bud\xFA zmazan\xE9 a nebud\xFA pou\u017Eite\u013En\xE9.",
          title__regenerate: "Obnovi\u0165 z\xE1lo\u017En\xE9 k\xF3dy"
        },
        phoneCode: {
          actionLabel__setDefault: "Nastavi\u0165 ako predvolen\xE9",
          destructiveActionLabel: "Odstr\xE1ni\u0165 telef\xF3nne \u010D\xEDslo"
        },
        primaryButton: "Prida\u0165 dvojfaktorov\xFA autentifik\xE1ciu",
        title: "Dvojfaktorov\xE1 autentifik\xE1cia",
        totp: {
          destructiveActionTitle: "Odstr\xE1ni\u0165",
          headerTitle: "Aplik\xE1cia Authenticator"
        }
      },
      passkeysSection: {
        menuAction__destructive: "Odstr\xE1ni\u0165",
        menuAction__rename: "Premenova\u0165",
        primaryButton: "Prida\u0165 Passkey",
        title: "Passkey"
      },
      passwordSection: {
        primaryButton__setPassword: "Nastavi\u0165 heslo",
        primaryButton__updatePassword: "Zmeni\u0165 heslo",
        title: "Heslo"
      },
      phoneNumbersSection: {
        destructiveAction: "Odstr\xE1ni\u0165 telef\xF3nne \u010D\xEDslo",
        detailsAction__nonPrimary: "Nastavi\u0165 ako hlavn\xE9",
        detailsAction__primary: "Dokon\u010Di\u0165 overenie",
        detailsAction__unverified: "Dokon\u010Di\u0165 overenie",
        primaryButton: "Prida\u0165 telef\xF3nne \u010D\xEDslo",
        title: "Telef\xF3nne \u010D\xEDsla"
      },
      profileSection: {
        primaryButton: "Aktualizova\u0165 profil",
        title: "Profil"
      },
      usernameSection: {
        primaryButton__setUsername: "Nastavi\u0165 u\u017E\xEDvate\u013Esk\xE9 meno",
        primaryButton__updateUsername: "Zmeni\u0165 u\u017E\xEDvate\u013Esk\xE9 meno",
        title: "U\u017E\xEDvate\u013Esk\xE9 meno"
      },
      web3WalletsSection: {
        destructiveAction: "Odstr\xE1ni\u0165 pe\u0148a\u017Eenku",
        detailsAction__nonPrimary: "Nastai\u0165 ako hlavn\xFA",
        primaryButton: "Web3 pe\u0148a\u017Eenky",
        title: "Web3 pe\u0148a\u017Eenky"
      }
    },
    usernamePage: {
      successMessage: "Va\u0161e u\u017E\xEDvate\u013Esk\xE9 meno bolo aktualizovan\xE9.",
      title__set: "Aktualizova\u0165 u\u017E\xEDvate\u013Esk\xE9 meno",
      title__update: "Aktualizova\u0165 u\u017E\xEDvate\u013Esk\xE9 meno"
    },
    web3WalletPage: {
      removeResource: {
        messageLine1: "{{identifier}} bude odobran\xE1 z tohto \xFA\u010Dtu.",
        messageLine2: "Nebudete sa u\u017E m\xF4c\u0165 prihl\xE1si\u0165 pomocou tejto web3 pe\u0148a\u017Eenky.",
        successMessage: "{{web3Wallet}} bola odstr\xE1nen\xE1 z v\xE1\u0161ho \xFA\u010Dtu.",
        title: "Odstr\xE1ni\u0165 web3 pe\u0148a\u017Eenku"
      },
      subtitle__availableWallets: "Vyberte web3 pe\u0148a\u017Eenku na pripojenie k v\xE1\u0161mu \xFA\u010Dtu.",
      subtitle__unavailableWallets: "Nie s\xFA k dispoz\xEDcii \u017Eiadne dostupn\xE9 web3 pe\u0148a\u017Eenky.",
      successMessage: "Pe\u0148a\u017Eenka bola pridan\xE1 k v\xE1\u0161mu \xFA\u010Dtu.",
      title: "Prida\u0165 web3 pe\u0148a\u017Eenku",
      web3WalletButtonsBlockButton: void 0
    }
  },
  waitlist: {
    start: {
      actionLink: "Prihl\xE1si\u0165 sa",
      actionText: "M\xE1te pr\xEDstup?",
      formButton: "Prida\u0165 sa na waitlist",
      subtitle: "Zadajte svoj email a my v\xE1s budeme informova\u0165, ke\u010F bude pr\xEDstup povolen\xFD.",
      title: "Prida\u0165 sa na waitlist"
    },
    success: {
      message: "\u010Coskoro budete presmerovan\xED...",
      subtitle: "Budeme v\xE1s informova\u0165, ke\u010F bude pr\xEDstup povolen\xFD.",
      title: "\u010Eakujeme, \u017Ee ste sa pridali na waitlist!"
    }
  }
};
// Annotate the CommonJS export names for ESM import in node:
0 && (module.exports = {
  skSK
});
