// @clerk/localizations/dist/zh-TW.js
"use strict";
var __defProp = Object.defineProperty;
var __getOwnPropDesc = Object.getOwnPropertyDescriptor;
var __getOwnPropNames = Object.getOwnPropertyNames;
var __hasOwnProp = Object.prototype.hasOwnProperty;
var __export = (target, all) => {
  for (var name in all)
    __defProp(target, name, { get: all[name], enumerable: true });
};
var __copyProps = (to, from, except, desc) => {
  if (from && typeof from === "object" || typeof from === "function") {
    for (let key of __getOwnPropNames(from))
      if (!__hasOwnProp.call(to, key) && key !== except)
        __defProp(to, key, { get: () => from[key], enumerable: !(desc = __getOwnPropDesc(from, key)) || desc.enumerable });
  }
  return to;
};
var __toCommonJS = (mod) => __copyProps(__defProp({}, "__esModule", { value: true }), mod);

// src/zh-TW.ts
var zh_TW_exports = {};
__export(zh_TW_exports, {
  zhTW: () => zhTW
});
module.exports = __toCommonJS(zh_TW_exports);
var zhTW = {
  locale: "zh-TW",
  apiKeys: {
    action__add: void 0,
    action__search: void 0,
    createdAndExpirationStatus__expiresOn: void 0,
    createdAndExpirationStatus__never: void 0,
    detailsTitle__emptyRow: void 0,
    formButtonPrimary__add: void 0,
    formFieldCaption__expiration__expiresOn: void 0,
    formFieldCaption__expiration__never: void 0,
    formFieldOption__expiration__180d: void 0,
    formFieldOption__expiration__1d: void 0,
    formFieldOption__expiration__1y: void 0,
    formFieldOption__expiration__30d: void 0,
    formFieldOption__expiration__60d: void 0,
    formFieldOption__expiration__7d: void 0,
    formFieldOption__expiration__90d: void 0,
    formFieldOption__expiration__never: void 0,
    formHint: void 0,
    formTitle: void 0,
    lastUsed__days: void 0,
    lastUsed__hours: void 0,
    lastUsed__minutes: void 0,
    lastUsed__months: void 0,
    lastUsed__seconds: void 0,
    lastUsed__years: void 0,
    menuAction__revoke: void 0,
    revokeConfirmation: {
      confirmationText: void 0,
      formButtonPrimary__revoke: void 0,
      formHint: void 0,
      formTitle: void 0
    }
  },
  backButton: "\u8FD4\u56DE",
  badge__activePlan: void 0,
  badge__canceledEndsAt: void 0,
  badge__currentPlan: void 0,
  badge__default: "\u9810\u8A2D",
  badge__endsAt: void 0,
  badge__expired: void 0,
  badge__freeTrial: void 0,
  badge__otherImpersonatorDevice: "\u5176\u4ED6\u6A21\u64EC\u88DD\u7F6E",
  badge__pastDueAt: void 0,
  badge__pastDuePlan: void 0,
  badge__primary: "\u4E3B\u8981",
  badge__renewsAt: void 0,
  badge__requiresAction: "\u9700\u8981\u64CD\u4F5C",
  badge__startsAt: void 0,
  badge__thisDevice: "\u6B64\u8A2D\u5099",
  badge__trialEndsAt: void 0,
  badge__unverified: "\u672A\u9A57\u8B49",
  badge__upcomingPlan: void 0,
  badge__userDevice: "\u7528\u6236\u8A2D\u5099",
  badge__you: "\u60A8",
  billing: {
    addPaymentMethod__label: void 0,
    alwaysFree: void 0,
    annually: void 0,
    availableFeatures: void 0,
    billedAnnually: void 0,
    billedMonthlyOnly: void 0,
    cancelFreeTrial: void 0,
    cancelFreeTrialAccessUntil: void 0,
    cancelFreeTrialTitle: void 0,
    cancelSubscription: void 0,
    cancelSubscriptionAccessUntil: void 0,
    cancelSubscriptionNoCharge: void 0,
    cancelSubscriptionPastDue: void 0,
    cancelSubscriptionTitle: void 0,
    cannotSubscribeMonthly: void 0,
    cannotSubscribeUnrecoverable: void 0,
    checkout: {
      description__paymentSuccessful: void 0,
      description__subscriptionSuccessful: void 0,
      downgradeNotice: void 0,
      emailForm: {
        subtitle: void 0,
        title: void 0
      },
      lineItems: {
        title__freeTrialEndsAt: void 0,
        title__paymentMethod: void 0,
        title__statementId: void 0,
        title__subscriptionBegins: void 0,
        title__totalPaid: void 0
      },
      pastDueNotice: void 0,
      perMonth: void 0,
      title: void 0,
      title__paymentSuccessful: void 0,
      title__subscriptionSuccessful: void 0,
      title__trialSuccess: void 0,
      totalDueAfterTrial: void 0
    },
    credit: void 0,
    creditRemainder: void 0,
    defaultFreePlanActive: void 0,
    free: void 0,
    getStarted: void 0,
    keepFreeTrial: void 0,
    keepSubscription: void 0,
    manage: void 0,
    manageSubscription: void 0,
    month: void 0,
    monthly: void 0,
    pastDue: void 0,
    pay: void 0,
    paymentMethod: {
      applePayDescription: {
        annual: void 0,
        monthly: void 0
      },
      dev: {
        anyNumbers: void 0,
        cardNumber: void 0,
        cvcZip: void 0,
        developmentMode: void 0,
        expirationDate: void 0,
        testCardInfo: void 0
      }
    },
    paymentMethods__label: void 0,
    popular: void 0,
    pricingTable: {
      billingCycle: void 0,
      included: void 0
    },
    reSubscribe: void 0,
    seeAllFeatures: void 0,
    startFreeTrial: void 0,
    startFreeTrial__days: void 0,
    subscribe: void 0,
    subscriptionDetails: {
      beginsOn: void 0,
      currentBillingCycle: void 0,
      endsOn: void 0,
      firstPaymentAmount: void 0,
      firstPaymentOn: void 0,
      nextPaymentAmount: void 0,
      nextPaymentOn: void 0,
      pastDueAt: void 0,
      renewsAt: void 0,
      subscribedOn: void 0,
      title: void 0,
      trialEndsOn: void 0,
      trialStartedOn: void 0
    },
    subtotal: void 0,
    switchPlan: void 0,
    switchToAnnual: void 0,
    switchToAnnualWithAnnualPrice: void 0,
    switchToMonthly: void 0,
    switchToMonthlyWithPrice: void 0,
    totalDue: void 0,
    totalDueToday: void 0,
    viewFeatures: void 0,
    viewPayment: void 0,
    year: void 0
  },
  createOrganization: {
    formButtonSubmit: "\u5275\u5EFA\u7D44\u7E54",
    invitePage: {
      formButtonReset: "\u8DF3\u904E"
    },
    title: "\u5275\u5EFA\u7D44\u7E54"
  },
  dates: {
    lastDay: "\u6628\u5929{{ date | timeString('zh-TW') }}",
    next6Days: "{{ date | weekday('zh-TW','long') }} {{ date | timeString('zh-TW') }}",
    nextDay: "\u660E\u5929{{ date | timeString('zh-TW') }}",
    numeric: "{{ date | numeric('zh-TW') }}",
    previous6Days: "\u4E0A\u9031{{ date | weekday('zh-TW','long') }} {{ date | timeString('zh-TW') }}",
    sameDay: "\u4ECA\u5929{{ date | timeString('zh-TW') }}"
  },
  dividerText: "\u6216\u8005",
  footerActionLink__alternativePhoneCodeProvider: void 0,
  footerActionLink__useAnotherMethod: "\u4F7F\u7528\u53E6\u4E00\u7A2E\u65B9\u6CD5",
  footerPageLink__help: "\u5E6B\u52A9",
  footerPageLink__privacy: "\u96B1\u79C1",
  footerPageLink__terms: "\u689D\u6B3E",
  formButtonPrimary: "\u7E7C\u7E8C",
  formButtonPrimary__verify: "\u9A57\u8B49",
  formFieldAction__forgotPassword: "\u5FD8\u8A18\u5BC6\u78BC\uFF1F",
  formFieldError__matchingPasswords: "\u5BC6\u78BC\u5339\u914D\u3002",
  formFieldError__notMatchingPasswords: "\u5BC6\u78BC\u4E0D\u5339\u914D\u3002",
  formFieldError__verificationLinkExpired: "\u9A57\u8B49\u9023\u7D50\u5DF2\u904E\u671F\uFF0C\u8ACB\u8ACB\u6C42\u65B0\u7684\u9023\u7D50\u3002",
  formFieldHintText__optional: "\u9078\u586B",
  formFieldHintText__slug: "slug \u662F\u4E00\u500B\u6613\u65BC\u4EBA\u985E\u95B1\u8B80\u4E14\u5FC5\u9808\u552F\u4E00\u7684 ID\uFF0C\u901A\u5E38\u7528\u65BC URL\u3002",
  formFieldInputPlaceholder__apiKeyDescription: void 0,
  formFieldInputPlaceholder__apiKeyExpirationDate: void 0,
  formFieldInputPlaceholder__apiKeyName: void 0,
  formFieldInputPlaceholder__backupCode: void 0,
  formFieldInputPlaceholder__confirmDeletionUserAccount: "\u522A\u9664\u5E33\u6236",
  formFieldInputPlaceholder__emailAddress: void 0,
  formFieldInputPlaceholder__emailAddress_username: void 0,
  formFieldInputPlaceholder__emailAddresses: "\u8F38\u5165\u6216\u8CBC\u4E0A\u4E00\u500B\u6216\u591A\u500B\u96FB\u5B50\u90F5\u4EF6\u5730\u5740\uFF0C\u4EE5\u7A7A\u683C\u6216\u9017\u865F\u5206\u9694",
  formFieldInputPlaceholder__firstName: void 0,
  formFieldInputPlaceholder__lastName: void 0,
  formFieldInputPlaceholder__organizationDomain: void 0,
  formFieldInputPlaceholder__organizationDomainEmailAddress: void 0,
  formFieldInputPlaceholder__organizationName: void 0,
  formFieldInputPlaceholder__organizationSlug: void 0,
  formFieldInputPlaceholder__password: void 0,
  formFieldInputPlaceholder__phoneNumber: void 0,
  formFieldInputPlaceholder__username: void 0,
  formFieldInput__emailAddress_format: void 0,
  formFieldLabel__apiKeyDescription: void 0,
  formFieldLabel__apiKeyExpiration: void 0,
  formFieldLabel__apiKeyName: void 0,
  formFieldLabel__automaticInvitations: "\u70BA\u6B64\u7DB2\u57DF\u555F\u7528\u81EA\u52D5\u9080\u8ACB",
  formFieldLabel__backupCode: "\u5099\u7528\u78BC",
  formFieldLabel__confirmDeletion: "\u78BA\u5B9A",
  formFieldLabel__confirmPassword: "\u78BA\u8A8D\u5BC6\u78BC",
  formFieldLabel__currentPassword: "\u76EE\u524D\u5BC6\u78BC",
  formFieldLabel__emailAddress: "\u96FB\u5B50\u90F5\u4EF6\u5730\u5740",
  formFieldLabel__emailAddress_username: "\u96FB\u5B50\u90F5\u4EF6\u5730\u5740\u6216\u4F7F\u7528\u8005\u540D\u7A31",
  formFieldLabel__emailAddresses: "\u96FB\u5B50\u90F5\u4EF6\u5730\u5740",
  formFieldLabel__firstName: "\u540D\u5B57",
  formFieldLabel__lastName: "\u59D3\u6C0F",
  formFieldLabel__newPassword: "\u65B0\u5BC6\u78BC",
  formFieldLabel__organizationDomain: "\u7DB2\u57DF",
  formFieldLabel__organizationDomainDeletePending: "\u522A\u9664\u5F85\u8655\u7406\u7684\u9080\u8ACB\u548C\u5EFA\u8B70",
  formFieldLabel__organizationDomainEmailAddress: "\u9A57\u8B49\u7528\u96FB\u5B50\u90F5\u4EF6\u5730\u5740",
  formFieldLabel__organizationDomainEmailAddressDescription: "\u8F38\u5165\u6B64\u7DB2\u57DF\u4E0B\u7684\u4E00\u500B\u96FB\u5B50\u90F5\u4EF6\u5730\u5740\u4EE5\u63A5\u6536\u4EE3\u78BC\u4E26\u9A57\u8B49\u6B64\u7DB2\u57DF\u3002",
  formFieldLabel__organizationName: "\u7D44\u7E54\u540D\u7A31",
  formFieldLabel__organizationSlug: "URL \u4EE3\u7A31",
  formFieldLabel__passkeyName: void 0,
  formFieldLabel__password: "\u5BC6\u78BC",
  formFieldLabel__phoneNumber: "\u96FB\u8A71\u865F\u78BC",
  formFieldLabel__role: "\u89D2\u8272",
  formFieldLabel__signOutOfOtherSessions: "\u767B\u51FA\u6240\u6709\u5176\u4ED6\u88DD\u7F6E",
  formFieldLabel__username: "\u4F7F\u7528\u8005\u540D\u7A31",
  impersonationFab: {
    action__signOut: "\u767B\u51FA",
    title: "\u4EE5 {{identifier}} \u8EAB\u4EFD\u767B\u5165"
  },
  lastAuthenticationStrategy: "\u4E0A\u6B21\u4F7F\u7528",
  maintenanceMode: void 0,
  membershipRole__admin: "\u7BA1\u7406\u54E1",
  membershipRole__basicMember: "\u6210\u54E1",
  membershipRole__guestMember: "\u8A2A\u5BA2",
  organizationList: {
    action__createOrganization: "\u5EFA\u7ACB\u7D44\u7E54",
    action__invitationAccept: "\u52A0\u5165",
    action__suggestionsAccept: "\u8ACB\u6C42\u52A0\u5165",
    createOrganization: "\u5EFA\u7ACB\u7D44\u7E54",
    invitationAcceptedLabel: "\u5DF2\u52A0\u5165",
    subtitle: "\u4EE5\u7E7C\u7E8C\u524D\u5F80 {{applicationName}}",
    suggestionsAcceptedLabel: "\u5F85\u6838\u51C6",
    title: "\u9078\u64C7\u4E00\u500B\u5E33\u6236",
    titleWithoutPersonal: "\u9078\u64C7\u4E00\u500B\u7D44\u7E54"
  },
  organizationProfile: {
    apiKeysPage: {
      title: void 0
    },
    badge__automaticInvitation: "\u81EA\u52D5\u9080\u8ACB",
    badge__automaticSuggestion: "\u81EA\u52D5\u5EFA\u8B70",
    badge__manualInvitation: "\u672A\u555F\u7528\u81EA\u52D5\u8A3B\u518A",
    badge__unverified: "\u672A\u9A57\u8B49",
    billingPage: {
      paymentHistorySection: {
        empty: void 0,
        notFound: void 0,
        tableHeader__amount: void 0,
        tableHeader__date: void 0,
        tableHeader__status: void 0
      },
      paymentMethodsSection: {
        actionLabel__default: void 0,
        actionLabel__remove: void 0,
        add: void 0,
        addSubtitle: void 0,
        cancelButton: void 0,
        formButtonPrimary__add: void 0,
        formButtonPrimary__pay: void 0,
        payWithTestCardButton: void 0,
        removeMethod: {
          messageLine1: void 0,
          messageLine2: void 0,
          successMessage: void 0,
          title: void 0
        },
        title: void 0
      },
      start: {
        headerTitle__payments: void 0,
        headerTitle__plans: void 0,
        headerTitle__statements: void 0,
        headerTitle__subscriptions: void 0
      },
      statementsSection: {
        empty: void 0,
        itemCaption__paidForPlan: void 0,
        itemCaption__proratedCredit: void 0,
        itemCaption__subscribedAndPaidForPlan: void 0,
        notFound: void 0,
        tableHeader__amount: void 0,
        tableHeader__date: void 0,
        title: void 0,
        totalPaid: void 0
      },
      subscriptionsListSection: {
        actionLabel__manageSubscription: void 0,
        actionLabel__newSubscription: void 0,
        actionLabel__switchPlan: void 0,
        tableHeader__edit: void 0,
        tableHeader__plan: void 0,
        tableHeader__startDate: void 0,
        title: void 0
      },
      subscriptionsSection: {
        actionLabel__default: void 0
      },
      switchPlansSection: {
        title: void 0
      },
      title: void 0
    },
    createDomainPage: {
      subtitle: "\u65B0\u589E\u8981\u9A57\u8B49\u7684\u7DB2\u57DF\u3002\u5728\u6B64\u7DB2\u57DF\u4E0B\u64C1\u6709\u96FB\u5B50\u90F5\u4EF6\u5730\u5740\u7684\u4F7F\u7528\u8005\u53EF\u4EE5\u81EA\u52D5\u52A0\u5165\u7D44\u7E54\u6216\u8ACB\u6C42\u52A0\u5165\u3002",
      title: "\u65B0\u589E\u7DB2\u57DF"
    },
    invitePage: {
      detailsTitle__inviteFailed: "\u9080\u8ACB\u7121\u6CD5\u50B3\u9001\u3002\u8ACB\u4FEE\u6B63\u4EE5\u4E0B\u554F\u984C\u5F8C\u518D\u8A66\u4E00\u6B21\uFF1A",
      formButtonPrimary__continue: "\u50B3\u9001\u9080\u8ACB",
      selectDropdown__role: "\u9078\u53D6\u89D2\u8272",
      subtitle: "\u9080\u8ACB\u65B0\u6210\u54E1\u52A0\u5165\u6B64\u7D44\u7E54",
      successMessage: "\u9080\u8ACB\u5DF2\u6210\u529F\u50B3\u9001",
      title: "\u9080\u8ACB\u6210\u54E1"
    },
    membersPage: {
      action__invite: "\u9080\u8ACB",
      action__search: void 0,
      activeMembersTab: {
        menuAction__remove: "\u79FB\u9664\u6210\u54E1",
        tableHeader__actions: void 0,
        tableHeader__joined: "\u52A0\u5165",
        tableHeader__role: "\u89D2\u8272",
        tableHeader__user: "\u4F7F\u7528\u8005"
      },
      detailsTitle__emptyRow: "\u6C92\u6709\u53EF\u986F\u793A\u7684\u6210\u54E1",
      invitationsTab: {
        autoInvitations: {
          headerSubtitle: "\u900F\u904E\u5C07\u96FB\u5B50\u90F5\u4EF6\u7DB2\u57DF\u8207\u60A8\u7684\u7D44\u7E54\u9023\u7D50\u4F86\u9080\u8ACB\u4F7F\u7528\u8005\u3002\u4EFB\u4F55\u4F7F\u7528\u5339\u914D\u7684\u96FB\u5B50\u90F5\u4EF6\u7DB2\u57DF\u8A3B\u518A\u7684\u4EBA\u90FD\u53EF\u4EE5\u96A8\u6642\u52A0\u5165\u8A72\u7D44\u7E54\u3002",
          headerTitle: "\u81EA\u52D5\u9080\u8ACB",
          primaryButton: "\u7BA1\u7406\u5DF2\u9A57\u8B49\u7DB2\u57DF"
        },
        table__emptyRow: "\u6C92\u6709\u53EF\u986F\u793A\u7684\u9080\u8ACB"
      },
      invitedMembersTab: {
        menuAction__revoke: "\u64A4\u92B7\u9080\u8ACB",
        tableHeader__invited: "\u5DF2\u9080\u8ACB"
      },
      requestsTab: {
        autoSuggestions: {
          headerSubtitle: "\u4F7F\u7528\u5339\u914D\u7684\u96FB\u5B50\u90F5\u4EF6\u7DB2\u57DF\u8A3B\u518A\u7684\u4F7F\u7528\u8005\u5C07\u80FD\u5920\u770B\u5230\u8ACB\u6C42\u52A0\u5165\u60A8\u7D44\u7E54\u7684\u5EFA\u8B70\u3002",
          headerTitle: "\u81EA\u52D5\u5EFA\u8B70",
          primaryButton: "\u7BA1\u7406\u5DF2\u9A57\u8B49\u7DB2\u57DF"
        },
        menuAction__approve: "\u6838\u51C6",
        menuAction__reject: "\u62D2\u7D55",
        tableHeader__requested: "\u8ACB\u6C42\u5B58\u53D6",
        table__emptyRow: "\u6C92\u6709\u53EF\u986F\u793A\u7684\u8ACB\u6C42"
      },
      start: {
        headerTitle__invitations: "\u9080\u8ACB",
        headerTitle__members: "\u6210\u54E1",
        headerTitle__requests: "\u8ACB\u6C42"
      }
    },
    navbar: {
      apiKeys: void 0,
      billing: void 0,
      description: "\u7BA1\u7406\u60A8\u7684\u7D44\u7E54\u8A2D\u5B9A\u3002",
      general: "\u4E00\u822C",
      members: "\u6210\u54E1",
      title: "\u7D44\u7E54"
    },
    plansPage: {
      alerts: {
        noPermissionsToManageBilling: void 0
      },
      title: void 0
    },
    profilePage: {
      dangerSection: {
        deleteOrganization: {
          actionDescription: "\u5728\u4E0B\u65B9\u8F38\u5165\u300C{{organizationName}}\u300D\u4EE5\u7E7C\u7E8C\u3002",
          messageLine1: "\u60A8\u78BA\u5B9A\u8981\u522A\u9664\u6B64\u7D44\u7E54\u55CE\uFF1F",
          messageLine2: "\u6B64\u52D5\u4F5C\u5C07\u6C38\u4E45\u751F\u6548\u4E14\u7121\u6CD5\u5FA9\u539F\u3002",
          successMessage: "\u60A8\u5DF2\u522A\u9664\u8A72\u7D44\u7E54\u3002",
          title: "\u522A\u9664\u7D44\u7E54"
        },
        leaveOrganization: {
          actionDescription: "\u5728\u4E0B\u65B9\u8F38\u5165\u300C{{organizationName}}\u300D\u4EE5\u7E7C\u7E8C\u3002",
          messageLine1: "\u60A8\u78BA\u5B9A\u8981\u96E2\u958B\u6B64\u7D44\u7E54\u55CE\uFF1F\u60A8\u5C07\u5931\u53BB\u5C0D\u6B64\u7D44\u7E54\u53CA\u5176\u61C9\u7528\u7A0B\u5F0F\u7684\u5B58\u53D6\u6B0A\u9650\u3002",
          messageLine2: "\u6B64\u52D5\u4F5C\u5C07\u6C38\u4E45\u751F\u6548\u4E14\u7121\u6CD5\u5FA9\u539F\u3002",
          successMessage: "\u60A8\u5DF2\u96E2\u958B\u8A72\u7D44\u7E54\u3002",
          title: "\u96E2\u958B\u7D44\u7E54"
        },
        title: "\u5371\u96AA"
      },
      domainSection: {
        menuAction__manage: "\u7BA1\u7406",
        menuAction__remove: "\u522A\u9664",
        menuAction__verify: "\u9A57\u8B49",
        primaryButton: "\u65B0\u589E\u7DB2\u57DF",
        subtitle: "\u5141\u8A31\u4F7F\u7528\u8005\u6839\u64DA\u5DF2\u9A57\u8B49\u7684\u96FB\u5B50\u90F5\u4EF6\u7DB2\u57DF\u81EA\u52D5\u52A0\u5165\u7D44\u7E54\u6216\u8ACB\u6C42\u52A0\u5165\u3002",
        title: "\u5DF2\u9A57\u8B49\u7DB2\u57DF"
      },
      successMessage: "\u7D44\u7E54\u8CC7\u8A0A\u5DF2\u66F4\u65B0\u3002",
      title: "\u7D44\u7E54\u8A2D\u5B9A"
    },
    removeDomainPage: {
      messageLine1: "\u96FB\u5B50\u90F5\u4EF6\u7DB2\u57DF {{domain}} \u5C07\u88AB\u79FB\u9664\u3002",
      messageLine2: "\u6B64\u5F8C\uFF0C\u4F7F\u7528\u8005\u5C07\u7121\u6CD5\u900F\u904E\u6B64\u7DB2\u57DF\u81EA\u52D5\u52A0\u5165\u7D44\u7E54\u3002",
      successMessage: "\u7DB2\u57DF {{domain}} \u5DF2\u88AB\u79FB\u9664\u3002",
      title: "\u79FB\u9664\u7DB2\u57DF"
    },
    start: {
      headerTitle__general: "\u4E00\u822C",
      headerTitle__members: "\u6210\u54E1",
      profileSection: {
        primaryButton: void 0,
        title: "\u7D44\u7E54\u8CC7\u6599",
        uploadAction__title: "\u6A19\u8A8C"
      }
    },
    verifiedDomainPage: {
      dangerTab: {
        calloutInfoLabel: "\u79FB\u9664\u6B64\u7DB2\u57DF\u5C07\u5F71\u97FF\u5DF2\u9080\u8ACB\u7684\u4F7F\u7528\u8005\u3002",
        removeDomainActionLabel__remove: "\u79FB\u9664\u7DB2\u57DF",
        removeDomainSubtitle: "\u5F9E\u60A8\u5DF2\u9A57\u8B49\u7684\u7DB2\u57DF\u4E2D\u79FB\u9664\u6B64\u7DB2\u57DF",
        removeDomainTitle: "\u79FB\u9664\u7DB2\u57DF"
      },
      enrollmentTab: {
        automaticInvitationOption__description: "\u4F7F\u7528\u8005\u8A3B\u518A\u6642\u6703\u81EA\u52D5\u53D7\u9080\u52A0\u5165\u7D44\u7E54\uFF0C\u4E26\u53EF\u96A8\u6642\u52A0\u5165\u3002",
        automaticInvitationOption__label: "\u81EA\u52D5\u9080\u8ACB",
        automaticSuggestionOption__description: "\u4F7F\u7528\u8005\u6703\u6536\u5230\u8ACB\u6C42\u52A0\u5165\u7684\u5EFA\u8B70\uFF0C\u4F46\u5FC5\u9808\u7531\u7BA1\u7406\u54E1\u6838\u51C6\u5F8C\u624D\u80FD\u52A0\u5165\u7D44\u7E54\u3002",
        automaticSuggestionOption__label: "\u81EA\u52D5\u5EFA\u8B70",
        calloutInfoLabel: "\u8B8A\u66F4\u8A3B\u518A\u6A21\u5F0F\u53EA\u6703\u5F71\u97FF\u65B0\u4F7F\u7528\u8005\u3002",
        calloutInvitationCountLabel: "\u5DF2\u50B3\u9001\u7D66\u4F7F\u7528\u8005\u7684\u5F85\u8655\u7406\u9080\u8ACB\uFF1A{{count}}",
        calloutSuggestionCountLabel: "\u5DF2\u50B3\u9001\u7D66\u4F7F\u7528\u8005\u7684\u5F85\u8655\u7406\u5EFA\u8B70\uFF1A{{count}}",
        manualInvitationOption__description: "\u4F7F\u7528\u8005\u53EA\u80FD\u900F\u904E\u624B\u52D5\u9080\u8ACB\u52A0\u5165\u7D44\u7E54\u3002",
        manualInvitationOption__label: "\u672A\u555F\u7528\u81EA\u52D5\u8A3B\u518A",
        subtitle: "\u9078\u64C7\u6B64\u7DB2\u57DF\u7684\u4F7F\u7528\u8005\u5982\u4F55\u52A0\u5165\u7D44\u7E54\u3002"
      },
      start: {
        headerTitle__danger: "\u5371\u96AA",
        headerTitle__enrollment: "\u8A3B\u518A\u9078\u9805"
      },
      subtitle: "\u7DB2\u57DF {{domain}} \u76EE\u524D\u5DF2\u9A57\u8B49\u3002\u8ACB\u7E7C\u7E8C\u9078\u64C7\u8A3B\u518A\u6A21\u5F0F\u3002",
      title: "\u66F4\u65B0 {{domain}}"
    },
    verifyDomainPage: {
      formSubtitle: "\u8ACB\u8F38\u5165\u50B3\u9001\u81F3\u60A8\u96FB\u5B50\u90F5\u4EF6\u5730\u5740\u7684\u9A57\u8B49\u78BC",
      formTitle: "\u9A57\u8B49\u78BC",
      resendButton: "\u6C92\u6536\u5230\u4EE3\u78BC\uFF1F\u91CD\u65B0\u50B3\u9001",
      subtitle: "\u7DB2\u57DF {{domainName}} \u9700\u8981\u900F\u904E\u96FB\u5B50\u90F5\u4EF6\u9032\u884C\u9A57\u8B49\u3002",
      subtitleVerificationCodeScreen: "\u9A57\u8B49\u78BC\u5DF2\u50B3\u9001\u81F3 {{emailAddress}}\u3002\u8ACB\u8F38\u5165\u4EE3\u78BC\u4EE5\u7E7C\u7E8C\u3002",
      title: "\u9A57\u8B49\u7DB2\u57DF"
    }
  },
  organizationSwitcher: {
    action__closeOrganizationSwitcher: void 0,
    action__createOrganization: "\u5EFA\u7ACB\u7D44\u7E54",
    action__invitationAccept: "\u52A0\u5165",
    action__manageOrganization: "\u7BA1\u7406\u7D44\u7E54",
    action__openOrganizationSwitcher: void 0,
    action__suggestionsAccept: "\u8ACB\u6C42\u52A0\u5165",
    notSelected: "\u672A\u9078\u53D6\u7D44\u7E54",
    personalWorkspace: "\u500B\u4EBA\u5DE5\u4F5C\u5340",
    suggestionsAcceptedLabel: "\u5F85\u6838\u51C6"
  },
  paginationButton__next: "\u4E0B\u4E00\u9801",
  paginationButton__previous: "\u4E0A\u4E00\u9801",
  paginationRowText__displaying: "\u986F\u793A",
  paginationRowText__of: "\u7684",
  reverification: {
    alternativeMethods: {
      actionLink: void 0,
      actionText: void 0,
      blockButton__backupCode: void 0,
      blockButton__emailCode: void 0,
      blockButton__passkey: void 0,
      blockButton__password: void 0,
      blockButton__phoneCode: void 0,
      blockButton__totp: void 0,
      getHelp: {
        blockButton__emailSupport: void 0,
        content: void 0,
        title: void 0
      },
      subtitle: void 0,
      title: void 0
    },
    backupCodeMfa: {
      subtitle: void 0,
      title: void 0
    },
    emailCode: {
      formTitle: void 0,
      resendButton: void 0,
      subtitle: void 0,
      title: void 0
    },
    noAvailableMethods: {
      message: void 0,
      subtitle: void 0,
      title: void 0
    },
    passkey: {
      blockButton__passkey: void 0,
      subtitle: void 0,
      title: void 0
    },
    password: {
      actionLink: void 0,
      subtitle: void 0,
      title: void 0
    },
    phoneCode: {
      formTitle: void 0,
      resendButton: void 0,
      subtitle: void 0,
      title: void 0
    },
    phoneCodeMfa: {
      formTitle: void 0,
      resendButton: void 0,
      subtitle: void 0,
      title: void 0
    },
    totpMfa: {
      formTitle: void 0,
      subtitle: void 0,
      title: void 0
    }
  },
  signIn: {
    accountSwitcher: {
      action__addAccount: "\u65B0\u589E\u5E33\u6236",
      action__signOutAll: "\u767B\u51FA\u6240\u6709\u5E33\u6236",
      subtitle: "\u9078\u64C7\u60A8\u60F3\u8981\u7E7C\u7E8C\u4F7F\u7528\u7684\u5E33\u6236\u3002",
      title: "\u9078\u64C7\u4E00\u500B\u5E33\u6236"
    },
    alternativeMethods: {
      actionLink: "\u53D6\u5F97\u5354\u52A9",
      actionText: "\u6C92\u6709\u4EE5\u4E0A\u4EFB\u4F55\u4E00\u7A2E\u65B9\u5F0F\u55CE\uFF1F",
      blockButton__backupCode: "\u4F7F\u7528\u5099\u7528\u78BC",
      blockButton__emailCode: "\u50B3\u9001\u9A57\u8B49\u78BC\u81F3 {{identifier}}",
      blockButton__emailLink: "\u50B3\u9001\u96FB\u5B50\u90F5\u4EF6\u9023\u7D50\u81F3 {{identifier}}",
      blockButton__passkey: void 0,
      blockButton__password: "\u4F7F\u7528\u60A8\u7684\u5BC6\u78BC\u767B\u5165",
      blockButton__phoneCode: "\u50B3\u9001\u7C21\u8A0A\u4EE3\u78BC\u81F3 {{identifier}}",
      blockButton__totp: "\u4F7F\u7528\u60A8\u7684\u9A57\u8B49\u5668\u61C9\u7528\u7A0B\u5F0F",
      getHelp: {
        blockButton__emailSupport: "\u96FB\u5B50\u90F5\u4EF6\u652F\u63F4",
        content: "\u5982\u679C\u60A8\u767B\u5165\u5E33\u6236\u6642\u9047\u5230\u554F\u984C\uFF0C\u8ACB\u50B3\u9001\u96FB\u5B50\u90F5\u4EF6\u7D66\u6211\u5011\uFF0C\u6211\u5011\u5C07\u76E1\u5FEB\u5354\u52A9\u60A8\u6062\u5FA9\u5B58\u53D6\u6B0A\u9650\u3002",
        title: "\u53D6\u5F97\u5354\u52A9"
      },
      subtitle: "\u9047\u5230\u554F\u984C\uFF1F\u60A8\u53EF\u4EE5\u4F7F\u7528\u4E0B\u5217\u4EFB\u4E00\u65B9\u6CD5\u767B\u5165\u3002",
      title: "\u4F7F\u7528\u5176\u4ED6\u65B9\u5F0F"
    },
    alternativePhoneCodeProvider: {
      formTitle: void 0,
      resendButton: void 0,
      subtitle: void 0,
      title: void 0
    },
    backupCodeMfa: {
      subtitle: "\u4EE5\u7E7C\u7E8C\u524D\u5F80 {{applicationName}}",
      title: "\u8F38\u5165\u5099\u7528\u78BC"
    },
    emailCode: {
      formTitle: "\u9A57\u8B49\u78BC",
      resendButton: "\u91CD\u65B0\u50B3\u9001\u9A57\u8B49\u78BC",
      subtitle: "\u4EE5\u7E7C\u7E8C\u524D\u5F80 {{applicationName}}",
      title: "\u6AA2\u67E5\u60A8\u7684\u96FB\u5B50\u90F5\u4EF6"
    },
    emailLink: {
      clientMismatch: {
        subtitle: void 0,
        title: void 0
      },
      expired: {
        subtitle: "\u8FD4\u56DE\u539F\u5148\u7684\u5206\u9801\u4EE5\u7E7C\u7E8C\u3002",
        title: "\u6B64\u9A57\u8B49\u9023\u7D50\u5DF2\u904E\u671F"
      },
      failed: {
        subtitle: "\u8FD4\u56DE\u539F\u5148\u7684\u5206\u9801\u4EE5\u7E7C\u7E8C\u3002",
        title: "\u6B64\u9A57\u8B49\u9023\u7D50\u7121\u6548"
      },
      formSubtitle: "\u4F7F\u7528\u50B3\u9001\u81F3\u60A8\u96FB\u5B50\u90F5\u4EF6\u7684\u9A57\u8B49\u9023\u7D50",
      formTitle: "\u9A57\u8B49\u9023\u7D50",
      loading: {
        subtitle: "\u5373\u5C07\u70BA\u60A8\u91CD\u65B0\u5C0E\u5411",
        title: "\u6B63\u5728\u767B\u5165\u2026"
      },
      resendButton: "\u91CD\u65B0\u50B3\u9001\u9023\u7D50",
      subtitle: "\u4EE5\u7E7C\u7E8C\u524D\u5F80 {{applicationName}}",
      title: "\u6AA2\u67E5\u60A8\u7684\u96FB\u5B50\u90F5\u4EF6",
      unusedTab: {
        title: "\u60A8\u53EF\u4EE5\u95DC\u9589\u6B64\u5206\u9801"
      },
      verified: {
        subtitle: "\u5373\u5C07\u70BA\u60A8\u91CD\u65B0\u5C0E\u5411",
        title: "\u6210\u529F\u767B\u5165"
      },
      verifiedSwitchTab: {
        subtitle: "\u8FD4\u56DE\u539F\u5148\u7684\u5206\u9801\u4EE5\u7E7C\u7E8C",
        subtitleNewTab: "\u8FD4\u56DE\u65B0\u958B\u555F\u7684\u5206\u9801\u4EE5\u7E7C\u7E8C",
        titleNewTab: "\u5DF2\u5728\u5176\u4ED6\u5206\u9801\u767B\u5165"
      }
    },
    enterpriseConnections: {
      subtitle: void 0,
      title: void 0
    },
    forgotPassword: {
      formTitle: "\u91CD\u8A2D\u5BC6\u78BC\u4EE3\u78BC",
      resendButton: "\u91CD\u65B0\u50B3\u9001\u4EE3\u78BC",
      subtitle: "\u4EE5\u91CD\u8A2D\u60A8\u7684\u5BC6\u78BC",
      subtitle_email: "\u9996\u5148\uFF0C\u8ACB\u8F38\u5165\u50B3\u9001\u81F3\u60A8\u96FB\u5B50\u90F5\u4EF6\u7684\u4EE3\u78BC",
      subtitle_phone: "\u9996\u5148\uFF0C\u8ACB\u8F38\u5165\u50B3\u9001\u81F3\u60A8\u624B\u6A5F\u7684\u4EE3\u78BC",
      title: "\u91CD\u8A2D\u5BC6\u78BC"
    },
    forgotPasswordAlternativeMethods: {
      blockButton__resetPassword: "\u91CD\u8A2D\u5BC6\u78BC",
      label__alternativeMethods: "\u6216\u8005\uFF0C\u4F7F\u7528\u5176\u4ED6\u65B9\u5F0F\u767B\u5165\u3002",
      title: "\u5FD8\u8A18\u5BC6\u78BC\uFF1F"
    },
    noAvailableMethods: {
      message: "\u7121\u6CD5\u7E7C\u7E8C\u767B\u5165\u3002\u6C92\u6709\u53EF\u7528\u7684\u9A57\u8B49\u65B9\u5F0F\u3002",
      subtitle: "\u767C\u751F\u932F\u8AA4",
      title: "\u7121\u6CD5\u767B\u5165"
    },
    passkey: {
      subtitle: void 0,
      title: void 0
    },
    password: {
      actionLink: "\u4F7F\u7528\u5176\u4ED6\u65B9\u5F0F",
      subtitle: "\u4EE5\u7E7C\u7E8C\u524D\u5F80 {{applicationName}}",
      title: "\u8F38\u5165\u60A8\u7684\u5BC6\u78BC"
    },
    passwordPwned: {
      title: void 0
    },
    phoneCode: {
      formTitle: "\u9A57\u8B49\u78BC",
      resendButton: "\u91CD\u65B0\u50B3\u9001\u9A57\u8B49\u78BC",
      subtitle: "\u4EE5\u7E7C\u7E8C\u524D\u5F80 {{applicationName}}",
      title: "\u6AA2\u67E5\u624B\u6A5F\u7C21\u8A0A"
    },
    phoneCodeMfa: {
      formTitle: "\u9A57\u8B49\u78BC",
      resendButton: "\u91CD\u65B0\u50B3\u9001\u9A57\u8B49\u78BC",
      subtitle: void 0,
      title: "\u6AA2\u67E5\u624B\u6A5F\u7C21\u8A0A"
    },
    resetPassword: {
      formButtonPrimary: "\u91CD\u8A2D\u5BC6\u78BC",
      requiredMessage: "\u57FA\u65BC\u5B89\u5168\u8003\u91CF\uFF0C\u60A8\u5FC5\u9808\u91CD\u8A2D\u5BC6\u78BC\u3002",
      successMessage: "\u60A8\u7684\u5BC6\u78BC\u5DF2\u6210\u529F\u8B8A\u66F4\u3002\u6B63\u5728\u70BA\u60A8\u767B\u5165\uFF0C\u8ACB\u7A0D\u5019\u3002",
      title: "\u91CD\u8A2D\u5BC6\u78BC"
    },
    resetPasswordMfa: {
      detailsLabel: "\u6211\u5011\u9700\u8981\u9A57\u8B49\u60A8\u7684\u8EAB\u4EFD\u624D\u80FD\u91CD\u8A2D\u60A8\u7684\u5BC6\u78BC\u3002"
    },
    start: {
      actionLink: "\u8A3B\u518A",
      actionLink__join_waitlist: void 0,
      actionLink__use_email: "\u4F7F\u7528\u96FB\u5B50\u90F5\u4EF6",
      actionLink__use_email_username: "\u4F7F\u7528\u96FB\u5B50\u90F5\u4EF6\u6216\u4F7F\u7528\u8005\u540D\u7A31",
      actionLink__use_passkey: void 0,
      actionLink__use_phone: "\u4F7F\u7528\u96FB\u8A71\u865F\u78BC",
      actionLink__use_username: "\u4F7F\u7528\u4F7F\u7528\u8005\u540D\u7A31",
      actionText: "\u9084\u6C92\u6709\u5E33\u6236\u55CE\uFF1F",
      actionText__join_waitlist: void 0,
      alternativePhoneCodeProvider: {
        actionLink: void 0,
        label: void 0,
        subtitle: void 0,
        title: void 0
      },
      subtitle: "\u4EE5\u7E7C\u7E8C\u524D\u5F80 {{applicationName}}",
      subtitleCombined: void 0,
      title: "\u767B\u5165",
      titleCombined: "\u7E7C\u7E8C\u524D\u5F80 {{applicationName}}"
    },
    totpMfa: {
      formTitle: "\u9A57\u8B49\u78BC",
      subtitle: void 0,
      title: "\u5169\u6B65\u9A5F\u9A57\u8B49"
    }
  },
  signInEnterPasswordTitle: "\u8F38\u5165\u60A8\u7684\u5BC6\u78BC",
  signUp: {
    alternativePhoneCodeProvider: {
      resendButton: void 0,
      subtitle: void 0,
      title: void 0
    },
    continue: {
      actionLink: "\u767B\u5165",
      actionText: "\u5DF2\u7D93\u6709\u5E33\u6236\u4E86\u55CE\uFF1F",
      subtitle: "\u4EE5\u7E7C\u7E8C\u524D\u5F80 {{applicationName}}",
      title: "\u586B\u5BEB\u7F3A\u5C11\u7684\u6B04\u4F4D"
    },
    emailCode: {
      formSubtitle: "\u8ACB\u8F38\u5165\u50B3\u9001\u81F3\u60A8\u96FB\u5B50\u90F5\u4EF6\u5730\u5740\u7684\u9A57\u8B49\u78BC",
      formTitle: "\u9A57\u8B49\u78BC",
      resendButton: "\u91CD\u65B0\u50B3\u9001\u9A57\u8B49\u78BC",
      subtitle: "\u4EE5\u7E7C\u7E8C\u524D\u5F80 {{applicationName}}",
      title: "\u9A57\u8B49\u60A8\u7684\u96FB\u5B50\u90F5\u4EF6"
    },
    emailLink: {
      clientMismatch: {
        subtitle: void 0,
        title: void 0
      },
      formSubtitle: "\u4F7F\u7528\u50B3\u9001\u81F3\u60A8\u96FB\u5B50\u90F5\u4EF6\u5730\u5740\u7684\u9A57\u8B49\u9023\u7D50",
      formTitle: "\u9A57\u8B49\u9023\u7D50",
      loading: {
        title: "\u6B63\u5728\u8A3B\u518A\u2026"
      },
      resendButton: "\u91CD\u65B0\u50B3\u9001\u9023\u7D50",
      subtitle: "\u4EE5\u7E7C\u7E8C\u524D\u5F80 {{applicationName}}",
      title: "\u9A57\u8B49\u60A8\u7684\u96FB\u5B50\u90F5\u4EF6",
      verified: {
        title: "\u8A3B\u518A\u6210\u529F"
      },
      verifiedSwitchTab: {
        subtitle: "\u8FD4\u56DE\u65B0\u958B\u555F\u7684\u5206\u9801\u4EE5\u7E7C\u7E8C",
        subtitleNewTab: "\u8FD4\u56DE\u4E0A\u4E00\u500B\u5206\u9801\u4EE5\u7E7C\u7E8C",
        title: "\u96FB\u5B50\u90F5\u4EF6\u9A57\u8B49\u6210\u529F"
      }
    },
    enterpriseConnections: {
      subtitle: void 0,
      title: void 0
    },
    legalConsent: {
      checkbox: {
        label__onlyPrivacyPolicy: '\u6211\u540C\u610F {{ privacyPolicyLink || link("\u96B1\u79C1\u6B0A\u653F\u7B56") }}',
        label__onlyTermsOfService: '\u6211\u540C\u610F {{ termsOfServiceLink || link("\u670D\u52D9\u689D\u6B3E") }}',
        label__termsOfServiceAndPrivacyPolicy: '\u6211\u540C\u610F {{ termsOfServiceLink || link("\u670D\u52D9\u689D\u6B3E") }} \u8207 {{ privacyPolicyLink || link("\u96B1\u79C1\u6B0A\u653F\u7B56") }}'
      },
      continue: {
        subtitle: "\u8ACB\u95B1\u8B80\u4E26\u63A5\u53D7\u689D\u6B3E\u4EE5\u7E7C\u7E8C",
        title: "\u540C\u610F\u689D\u6B3E"
      }
    },
    phoneCode: {
      formSubtitle: "\u8F38\u5165\u50B3\u9001\u5230\u60A8\u7684\u96FB\u8A71\u865F\u78BC\u7684\u9A57\u8B49\u78BC",
      formTitle: "\u9A57\u8B49\u78BC",
      resendButton: "\u91CD\u65B0\u50B3\u9001\u9A57\u8B49\u78BC",
      subtitle: "\u4EE5\u7E7C\u7E8C\u524D\u5F80 {{applicationName}}",
      title: "\u9A57\u8B49\u60A8\u7684\u96FB\u8A71\u865F\u78BC"
    },
    restrictedAccess: {
      actionLink: void 0,
      actionText: void 0,
      blockButton__emailSupport: void 0,
      blockButton__joinWaitlist: void 0,
      subtitle: void 0,
      subtitleWaitlist: void 0,
      title: void 0
    },
    start: {
      actionLink: "\u767B\u5165",
      actionLink__use_email: void 0,
      actionLink__use_phone: void 0,
      actionText: "\u5DF2\u7D93\u6709\u5E33\u6236\u4E86\u55CE\uFF1F",
      alternativePhoneCodeProvider: {
        actionLink: void 0,
        label: void 0,
        subtitle: void 0,
        title: void 0
      },
      subtitle: "\u4EE5\u7E7C\u7E8C\u524D\u5F80 {{applicationName}}",
      subtitleCombined: "\u4EE5\u7E7C\u7E8C\u524D\u5F80 {{applicationName}}",
      title: "\u5EFA\u7ACB\u60A8\u7684\u5E33\u6236",
      titleCombined: "\u5EFA\u7ACB\u60A8\u7684\u5E33\u6236"
    }
  },
  socialButtonsBlockButton: "\u4EE5 {{provider|titleize}} \u5E33\u6236\u767B\u5165",
  socialButtonsBlockButtonManyInView: void 0,
  taskChooseOrganization: {
    chooseOrganization: {
      action__createOrganization: void 0,
      action__invitationAccept: void 0,
      action__suggestionsAccept: void 0,
      subtitle: void 0,
      suggestionsAcceptedLabel: void 0,
      title: void 0
    },
    createOrganization: {
      formButtonReset: void 0,
      formButtonSubmit: void 0,
      formFieldInputPlaceholder__name: void 0,
      formFieldInputPlaceholder__slug: void 0,
      formFieldLabel__name: void 0,
      formFieldLabel__slug: void 0,
      subtitle: void 0,
      title: void 0
    },
    signOut: {
      actionLink: void 0,
      actionText: void 0
    }
  },
  unstable__errors: {
    already_a_member_in_organization: void 0,
    captcha_invalid: "\u7531\u65BC\u5B89\u5168\u9A57\u8B49\u5931\u6557\uFF0C\u8A3B\u518A\u672A\u6210\u529F\u3002\u8ACB\u91CD\u65B0\u6574\u7406\u9801\u9762\u518D\u8A66\u4E00\u6B21\uFF0C\u6216\u806F\u7D61\u652F\u63F4\u4EE5\u53D6\u5F97\u5354\u52A9\u3002",
    captcha_unavailable: "\u7531\u65BC\u6A5F\u5668\u4EBA\u9A57\u8B49\u5931\u6557\u5C0E\u81F4\u8A3B\u518A\u672A\u6210\u529F\u3002\u8ACB\u91CD\u65B0\u6574\u7406\u9801\u9762\u518D\u8A66\u4E00\u6B21\uFF0C\u6216\u806F\u7D61\u652F\u63F4\u4EE5\u53D6\u5F97\u5354\u52A9\u3002",
    form_code_incorrect: void 0,
    form_identifier_exists__email_address: void 0,
    form_identifier_exists__phone_number: void 0,
    form_identifier_exists__username: void 0,
    form_identifier_not_found: "\u6211\u5011\u7121\u6CD5\u627E\u5230\u5177\u6709\u9019\u4E9B\u4FE1\u606F\u7684\u5E33\u6236\u3002",
    form_param_format_invalid: void 0,
    form_param_format_invalid__email_address: "\u96FB\u5B50\u90F5\u4EF6\u5730\u5740\u5FC5\u9808\u662F\u6709\u6548\u7684\u683C\u5F0F\u3002",
    form_param_format_invalid__phone_number: "\u96FB\u8A71\u865F\u78BC\u5FC5\u9808\u662F\u6709\u6548\u7684\u570B\u969B\u683C\u5F0F\u3002",
    form_param_max_length_exceeded__first_name: "\u540D\u5B57\u4E0D\u61C9\u8D85\u904E 256 \u500B\u5B57\u5143\u3002",
    form_param_max_length_exceeded__last_name: "\u59D3\u6C0F\u4E0D\u61C9\u8D85\u904E 256 \u500B\u5B57\u5143\u3002",
    form_param_max_length_exceeded__name: "\u540D\u7A31\u4E0D\u61C9\u8D85\u904E 256 \u500B\u5B57\u5143\u3002",
    form_param_nil: void 0,
    form_param_type_invalid: void 0,
    form_param_type_invalid__email_address: void 0,
    form_param_type_invalid__phone_number: void 0,
    form_param_value_invalid: void 0,
    form_password_incorrect: void 0,
    form_password_length_too_short: void 0,
    form_password_not_strong_enough: "\u60A8\u7684\u5BC6\u78BC\u5F37\u5EA6\u4E0D\u8DB3\u3002",
    form_password_pwned: "\u6B64\u5BC6\u78BC\u5DF2\u5728\u5DF2\u77E5\u7684\u8CC7\u6599\u5916\u6D29\u4E8B\u4EF6\u4E2D\u51FA\u73FE\uFF0C\u8ACB\u6539\u7528\u5176\u4ED6\u5BC6\u78BC\u3002",
    form_password_pwned__sign_in: void 0,
    form_password_size_in_bytes_exceeded: "\u60A8\u7684\u5BC6\u78BC\u8D85\u904E\u5141\u8A31\u7684\u5927\u5C0F\u4E0A\u9650\uFF0C\u8ACB\u7E2E\u77ED\u6216\u79FB\u9664\u90E8\u5206\u7279\u6B8A\u5B57\u5143\u3002",
    form_password_validation_failed: "\u5BC6\u78BC\u9A57\u8B49\u5931\u6557",
    form_username_invalid_character: void 0,
    form_username_invalid_length: void 0,
    identification_deletion_failed: "\u60A8\u7121\u6CD5\u522A\u9664\u6700\u5F8C\u4E00\u500B\u9A57\u8B49\u65B9\u5F0F\u3002",
    not_allowed_access: void 0,
    organization_domain_blocked: void 0,
    organization_domain_common: void 0,
    organization_domain_exists_for_enterprise_connection: void 0,
    organization_membership_quota_exceeded: void 0,
    organization_minimum_permissions_needed: void 0,
    organization_not_found_or_unauthorized: void 0,
    organization_not_found_or_unauthorized_with_create_organization_disabled: void 0,
    passkey_already_exists: void 0,
    passkey_not_supported: void 0,
    passkey_pa_not_supported: void 0,
    passkey_registration_cancelled: void 0,
    passkey_retrieval_cancelled: void 0,
    passwordComplexity: {
      maximumLength: "\u5C11\u65BC {{length}} \u500B\u5B57\u5143",
      minimumLength: "{{length}} \u500B\u6216\u66F4\u591A\u5B57\u5143",
      requireLowercase: "\u4E00\u500B\u5C0F\u5BEB\u5B57\u6BCD",
      requireNumbers: "\u4E00\u500B\u6578\u5B57",
      requireSpecialCharacter: "\u4E00\u500B\u7279\u6B8A\u5B57\u5143",
      requireUppercase: "\u4E00\u500B\u5927\u5BEB\u5B57\u6BCD",
      sentencePrefix: "\u60A8\u7684\u5BC6\u78BC\u5FC5\u9808\u5305\u542B\uFF1A"
    },
    phone_number_exists: "\u6B64\u96FB\u8A71\u865F\u78BC\u5DF2\u88AB\u4F7F\u7528\uFF0C\u8ACB\u5617\u8A66\u5176\u4ED6\u865F\u78BC\u3002",
    session_exists: "\u60A8\u5DF2\u7D93\u767B\u9304\u3002",
    web3_missing_identifier: void 0,
    zxcvbn: {
      couldBeStronger: "\u60A8\u7684\u5BC6\u78BC\u5F37\u5EA6\u5C1A\u53EF\uFF0C\u4F46\u53EF\u4EE5\u66F4\u5B89\u5168\u3002\u8ACB\u5617\u8A66\u589E\u52A0\u9577\u5EA6\u6216\u8907\u96DC\u5EA6\u3002",
      goodPassword: "\u5BC6\u78BC\u5F37\u5EA6\u826F\u597D\u3002",
      notEnough: "\u60A8\u7684\u5BC6\u78BC\u5F37\u5EA6\u4E0D\u8DB3\u3002",
      suggestions: {
        allUppercase: "\u90E8\u5206\u5B57\u6BCD\u5927\u5BEB\uFF0C\u4F46\u975E\u5168\u90E8\u3002",
        anotherWord: "\u589E\u52A0\u66F4\u591A\u3001\u66F4\u5C11\u898B\u7684\u55AE\u5B57\u3002",
        associatedYears: "\u907F\u514D\u4F7F\u7528\u8207\u60A8\u76F8\u95DC\u7684\u5E74\u4EFD\u3002",
        capitalization: "\u4E0D\u50C5\u9996\u5B57\u6BCD\u5927\u5BEB\u3002",
        dates: "\u907F\u514D\u4F7F\u7528\u8207\u60A8\u76F8\u95DC\u7684\u65E5\u671F\u548C\u5E74\u4EFD\u3002",
        l33t: '\u907F\u514D\u53EF\u9810\u6E2C\u7684\u5B57\u6BCD\u66FF\u63DB\uFF08\u4F8B\u5982\u7528 "@" \u53D6\u4EE3 "a"\uFF09\u3002',
        longerKeyboardPattern: "\u4F7F\u7528\u66F4\u9577\u7684\u9375\u76E4\u6392\u5217\uFF0C\u4E26\u591A\u6B21\u6539\u8B8A\u65B9\u5411\u3002",
        noNeed: "\u60A8\u7121\u9700\u4F7F\u7528\u7B26\u865F\u3001\u6578\u5B57\u6216\u5927\u5BEB\u5B57\u6BCD\u4E5F\u80FD\u5EFA\u7ACB\u9AD8\u5F37\u5EA6\u5BC6\u78BC\u3002",
        pwned: "\u82E5\u60A8\u5728\u5176\u4ED6\u5730\u65B9\u4F7F\u7528\u904E\u6B64\u5BC6\u78BC\uFF0C\u5EFA\u8B70\u7ACB\u5373\u66F4\u63DB\u3002",
        recentYears: "\u907F\u514D\u4F7F\u7528\u8FD1\u5E74\u7684\u5E74\u4EFD\u3002",
        repeated: void 0,
        reverseWords: "\u907F\u514D\u5E38\u7528\u8A5E\u8A9E\u7684\u53CD\u5411\u62FC\u5BEB\u3002",
        sequences: "\u907F\u514D\u5E38\u898B\u7684\u5B57\u5143\u5E8F\u5217\u3002",
        useWords: "\u4F7F\u7528\u591A\u500B\u55AE\u5B57\uFF0C\u4F46\u907F\u514D\u5E38\u7528\u7247\u8A9E\u3002"
      },
      warnings: {
        common: "\u9019\u662F\u4E00\u500B\u5E38\u7528\u7684\u5BC6\u78BC\u3002",
        commonNames: "\u5E38\u898B\u7684\u59D3\u540D\u5BB9\u6613\u88AB\u731C\u5230\u3002",
        dates: "\u65E5\u671F\u6613\u88AB\u731C\u5230\u3002",
        extendedRepeat: '\u91CD\u8907\u7684\u5B57\u5143\u6A21\u5F0F\uFF08\u4F8B\u5982 "abcabcabc"\uFF09\u5BB9\u6613\u88AB\u731C\u5230\u3002',
        keyPattern: "\u7C21\u77ED\u7684\u9375\u76E4\u6392\u5217\u5BB9\u6613\u88AB\u731C\u5230\u3002",
        namesByThemselves: "\u55AE\u7368\u4F7F\u7528\u59D3\u540D\u5BB9\u6613\u88AB\u731C\u5230\u3002",
        pwned: "\u60A8\u7684\u5BC6\u78BC\u5DF2\u5728\u516C\u958B\u7684\u8CC7\u6599\u5916\u6D29\u4E8B\u4EF6\u4E2D\u51FA\u73FE\u3002",
        recentYears: "\u8FD1\u5E74\u7684\u5E74\u4EFD\u5BB9\u6613\u88AB\u731C\u5230\u3002",
        sequences: '\u5E38\u898B\u7684\u5B57\u5143\u5E8F\u5217\uFF08\u4F8B\u5982 "abc"\uFF09\u5BB9\u6613\u88AB\u731C\u5230\u3002',
        similarToCommon: "\u6B64\u5BC6\u78BC\u8207\u5E38\u898B\u5BC6\u78BC\u76F8\u4F3C\u3002",
        simpleRepeat: '\u91CD\u8907\u7684\u5B57\u5143\uFF08\u4F8B\u5982 "aaa"\uFF09\u5BB9\u6613\u88AB\u731C\u5230\u3002',
        straightRow: "\u9375\u76E4\u4E0A\u6210\u76F4\u7DDA\u7684\u6309\u9375\u5BB9\u6613\u88AB\u731C\u5230\u3002",
        topHundred: "\u9019\u662F\u975E\u5E38\u5E38\u7528\u7684\u5BC6\u78BC\u3002",
        topTen: "\u9019\u662F\u6975\u5E38\u7528\u7684\u5BC6\u78BC\u3002",
        userInputs: "\u5BC6\u78BC\u4E0D\u61C9\u5305\u542B\u500B\u4EBA\u6216\u9801\u9762\u76F8\u95DC\u8CC7\u8A0A\u3002",
        wordByItself: "\u55AE\u7368\u4F7F\u7528\u55AE\u5B57\u5BB9\u6613\u88AB\u731C\u5230\u3002"
      }
    }
  },
  userButton: {
    action__addAccount: "\u65B0\u589E\u5E33\u6236",
    action__closeUserMenu: "\u95DC\u9589\u4F7F\u7528\u8005\u9078\u55AE",
    action__manageAccount: "\u7BA1\u7406\u5E33\u6236",
    action__openUserMenu: "\u958B\u555F\u4F7F\u7528\u8005\u9078\u55AE",
    action__signOut: "\u767B\u51FA",
    action__signOutAll: "\u767B\u51FA\u6240\u6709\u5E33\u6236"
  },
  userProfile: {
    apiKeysPage: {
      title: void 0
    },
    backupCodePage: {
      actionLabel__copied: "\u5DF2\u8907\u88FD\uFF01",
      actionLabel__copy: "\u5168\u90E8\u8907\u88FD",
      actionLabel__download: "\u4E0B\u8F09 .txt",
      actionLabel__print: "\u5217\u5370",
      infoText1: "\u5C07\u70BA\u6B64\u5E33\u6236\u555F\u7528\u5099\u7528\u78BC\u3002",
      infoText2: "\u8ACB\u59A5\u5584\u4FDD\u7BA1\u60A8\u7684\u5099\u7528\u78BC\u3002\u82E5\u61F7\u7591\u5DF2\u5916\u6D29\uFF0C\u60A8\u53EF\u4EE5\u91CD\u65B0\u7522\u751F\u3002",
      subtitle__codelist: "\u8ACB\u59A5\u5584\u4FDD\u7BA1\u3002",
      successMessage: "\u5099\u7528\u78BC\u73FE\u5DF2\u555F\u7528\u3002\u82E5\u60A8\u7121\u6CD5\u5B58\u53D6\u9A57\u8B49\u88DD\u7F6E\uFF0C\u53EF\u4F7F\u7528\u5176\u4E2D\u4E00\u7D44\u5099\u7528\u78BC\u767B\u5165\u3002\u6BCF\u7D44\u4EE3\u78BC\u50C5\u80FD\u4F7F\u7528\u4E00\u6B21\u3002",
      successSubtitle: "\u82E5\u60A8\u7121\u6CD5\u5B58\u53D6\u9A57\u8B49\u88DD\u7F6E\uFF0C\u53EF\u4F7F\u7528\u5176\u4E2D\u4E00\u7D44\u5099\u7528\u78BC\u767B\u5165\u60A8\u7684\u5E33\u6236\u3002",
      title: "\u65B0\u589E\u5099\u7528\u78BC\u9A57\u8B49",
      title__codelist: "\u5099\u7528\u78BC"
    },
    billingPage: {
      paymentHistorySection: {
        empty: void 0,
        notFound: void 0,
        tableHeader__amount: void 0,
        tableHeader__date: void 0,
        tableHeader__status: void 0
      },
      paymentMethodsSection: {
        actionLabel__default: void 0,
        actionLabel__remove: void 0,
        add: void 0,
        addSubtitle: void 0,
        cancelButton: void 0,
        formButtonPrimary__add: void 0,
        formButtonPrimary__pay: void 0,
        payWithTestCardButton: void 0,
        removeMethod: {
          messageLine1: void 0,
          messageLine2: void 0,
          successMessage: void 0,
          title: void 0
        },
        title: void 0
      },
      start: {
        headerTitle__payments: void 0,
        headerTitle__plans: void 0,
        headerTitle__statements: void 0,
        headerTitle__subscriptions: void 0
      },
      statementsSection: {
        empty: void 0,
        itemCaption__paidForPlan: void 0,
        itemCaption__proratedCredit: void 0,
        itemCaption__subscribedAndPaidForPlan: void 0,
        notFound: void 0,
        tableHeader__amount: void 0,
        tableHeader__date: void 0,
        title: void 0,
        totalPaid: void 0
      },
      subscriptionsListSection: {
        actionLabel__manageSubscription: void 0,
        actionLabel__newSubscription: void 0,
        actionLabel__switchPlan: void 0,
        tableHeader__edit: void 0,
        tableHeader__plan: void 0,
        tableHeader__startDate: void 0,
        title: void 0
      },
      subscriptionsSection: {
        actionLabel__default: void 0
      },
      switchPlansSection: {
        title: void 0
      },
      title: void 0
    },
    connectedAccountPage: {
      formHint: "\u9078\u64C7\u4E00\u500B\u4F9B\u61C9\u5546\u4EE5\u9023\u7D50\u60A8\u7684\u5E33\u6236\u3002",
      formHint__noAccounts: "\u76EE\u524D\u7121\u53EF\u7528\u7684\u5916\u90E8\u5E33\u6236\u4F9B\u61C9\u5546\u3002",
      removeResource: {
        messageLine1: "{{identifier}} \u5C07\u5F9E\u6B64\u5E33\u6236\u4E2D\u88AB\u79FB\u9664\u3002",
        messageLine2: "\u60A8\u5C07\u7121\u6CD5\u518D\u4F7F\u7528\u6B64\u5DF2\u9023\u7D50\u5E33\u6236\uFF0C\u4EFB\u4F55\u4F9D\u8CF4\u6B64\u9023\u7D50\u7684\u529F\u80FD\u4E5F\u5C07\u5931\u6548\u3002",
        successMessage: "{{connectedAccount}} \u5DF2\u5F9E\u60A8\u7684\u5E33\u6236\u4E2D\u79FB\u9664\u3002",
        title: "\u79FB\u9664\u5DF2\u9023\u7D50\u7684\u5E33\u6236"
      },
      socialButtonsBlockButton: "\u9023\u7D50 {{provider|titleize}} \u5E33\u6236",
      successMessage: "\u4F9B\u61C9\u5546\u5DF2\u65B0\u589E\u81F3\u60A8\u7684\u5E33\u6236",
      title: "\u65B0\u589E\u5DF2\u9023\u7D50\u7684\u5E33\u6236"
    },
    deletePage: {
      actionDescription: "\u8ACB\u5728\u4E0B\u65B9\u8F38\u5165\u300CDelete account\u300D\u4EE5\u7E7C\u7E8C\u3002",
      confirm: "\u522A\u9664\u5E33\u6236",
      messageLine1: "\u60A8\u78BA\u5B9A\u8981\u522A\u9664\u60A8\u7684\u5E33\u6236\u55CE\uFF1F\u67D0\u4E9B\u76F8\u95DC\u8CC7\u6599\u53EF\u80FD\u6703\u88AB\u4FDD\u7559\u3002\u5982\u9700\u7533\u8ACB\u5B8C\u5168\u522A\u9664\u8CC7\u6599\uFF0C\u8ACB\u806F\u7D61\u652F\u63F4\u5718\u968A\u3002",
      messageLine2: "\u6B64\u52D5\u4F5C\u5C07\u6C38\u4E45\u751F\u6548\u4E14\u7121\u6CD5\u5FA9\u539F\u3002",
      title: "\u522A\u9664\u5E33\u6236"
    },
    emailAddressPage: {
      emailCode: {
        formHint: "\u7CFB\u7D71\u5C07\u6703\u50B3\u9001\u4E00\u5C01\u5305\u542B\u9A57\u8B49\u78BC\u7684\u90F5\u4EF6\u81F3\u6B64\u96FB\u5B50\u90F5\u4EF6\u5730\u5740\u3002",
        formSubtitle: "\u8ACB\u8F38\u5165\u50B3\u9001\u81F3 {{identifier}} \u7684\u9A57\u8B49\u78BC",
        formTitle: "\u9A57\u8B49\u78BC",
        resendButton: "\u91CD\u65B0\u50B3\u9001\u9A57\u8B49\u78BC",
        successMessage: "\u96FB\u5B50\u90F5\u4EF6 {{identifier}} \u5DF2\u65B0\u589E\u81F3\u60A8\u7684\u5E33\u6236\u3002"
      },
      emailLink: {
        formHint: "\u7CFB\u7D71\u5C07\u6703\u50B3\u9001\u4E00\u5C01\u5305\u542B\u9A57\u8B49\u9023\u7D50\u7684\u90F5\u4EF6\u81F3\u6B64\u96FB\u5B50\u90F5\u4EF6\u5730\u5740\u3002",
        formSubtitle: "\u8ACB\u9EDE\u64CA\u50B3\u9001\u81F3 {{identifier}} \u7684\u90F5\u4EF6\u4E2D\u7684\u9A57\u8B49\u9023\u7D50",
        formTitle: "\u9A57\u8B49\u9023\u7D50",
        resendButton: "\u91CD\u65B0\u50B3\u9001\u9023\u7D50",
        successMessage: "\u96FB\u5B50\u90F5\u4EF6 {{identifier}} \u5DF2\u65B0\u589E\u81F3\u60A8\u7684\u5E33\u6236\u3002"
      },
      enterpriseSSOLink: {
        formButton: void 0,
        formSubtitle: void 0
      },
      formHint: void 0,
      removeResource: {
        messageLine1: "{{identifier}} \u5C07\u5F9E\u6B64\u5E33\u6236\u4E2D\u88AB\u79FB\u9664\u3002",
        messageLine2: "\u60A8\u5C07\u7121\u6CD5\u518D\u4F7F\u7528\u6B64\u96FB\u5B50\u90F5\u4EF6\u5730\u5740\u767B\u5165\u3002",
        successMessage: "\u96FB\u5B50\u90F5\u4EF6 {{emailAddress}} \u5DF2\u5F9E\u60A8\u7684\u5E33\u6236\u4E2D\u79FB\u9664\u3002",
        title: "\u79FB\u9664\u96FB\u5B50\u90F5\u4EF6\u5730\u5740"
      },
      title: "\u65B0\u589E\u96FB\u5B50\u90F5\u4EF6\u5730\u5740",
      verifyTitle: "\u9A57\u8B49\u96FB\u5B50\u90F5\u4EF6\u5730\u5740"
    },
    formButtonPrimary__add: "\u65B0\u589E",
    formButtonPrimary__continue: "\u7E7C\u7E8C",
    formButtonPrimary__finish: "\u5B8C\u6210",
    formButtonPrimary__remove: "\u79FB\u9664",
    formButtonPrimary__save: "\u5132\u5B58",
    formButtonReset: "\u53D6\u6D88",
    mfaPage: {
      formHint: "\u9078\u64C7\u4E00\u7A2E\u65B0\u589E\u65B9\u5F0F\u3002",
      title: "\u65B0\u589E\u5169\u6B65\u9A5F\u9A57\u8B49"
    },
    mfaPhoneCodePage: {
      backButton: "\u4F7F\u7528\u73FE\u6709\u865F\u78BC",
      primaryButton__addPhoneNumber: "\u65B0\u589E\u96FB\u8A71\u865F\u78BC",
      removeResource: {
        messageLine1: "{{identifier}} \u5C07\u4E0D\u518D\u7528\u65BC\u63A5\u6536\u767B\u5165\u9A57\u8B49\u78BC\u3002",
        messageLine2: "\u60A8\u7684\u5E33\u6236\u5B89\u5168\u6027\u53EF\u80FD\u6703\u964D\u4F4E\uFF0C\u78BA\u5B9A\u8981\u7E7C\u7E8C\u55CE\uFF1F",
        successMessage: "\u5DF2\u79FB\u9664 {{mfaPhoneCode}} \u7684\u7C21\u8A0A\u5169\u6B65\u9A5F\u9A57\u8B49",
        title: "\u79FB\u9664\u5169\u6B65\u9A5F\u9A57\u8B49"
      },
      subtitle__availablePhoneNumbers: "\u9078\u64C7\u4E00\u500B\u96FB\u8A71\u865F\u78BC\u4EE5\u8A3B\u518A\u7C21\u8A0A\u5169\u6B65\u9A5F\u9A57\u8B49\u3002",
      subtitle__unavailablePhoneNumbers: "\u76EE\u524D\u6C92\u6709\u53EF\u7528\u7684\u96FB\u8A71\u865F\u78BC\u4EE5\u8A3B\u518A\u7C21\u8A0A\u5169\u6B65\u9A5F\u9A57\u8B49\u3002",
      successMessage1: "\u767B\u5165\u6642\uFF0C\u60A8\u9700\u8981\u8F38\u5165\u50B3\u9001\u5230\u6B64\u96FB\u8A71\u865F\u78BC\u7684\u9A57\u8B49\u78BC\uFF0C\u4F5C\u70BA\u984D\u5916\u7684\u9A57\u8B49\u6B65\u9A5F\u3002",
      successMessage2: "\u8ACB\u59A5\u5584\u5132\u5B58\u9019\u4E9B\u5099\u7528\u78BC\u3002\u82E5\u60A8\u7121\u6CD5\u5B58\u53D6\u9A57\u8B49\u88DD\u7F6E\uFF0C\u53EF\u4EE5\u4F7F\u7528\u5099\u7528\u78BC\u767B\u5165\u3002",
      successTitle: "\u7C21\u8A0A\u9A57\u8B49\u5DF2\u555F\u7528",
      title: "\u65B0\u589E\u7C21\u8A0A\u9A57\u8B49"
    },
    mfaTOTPPage: {
      authenticatorApp: {
        buttonAbleToScan__nonPrimary: "\u6383\u63CF QR Code",
        buttonUnableToScan__nonPrimary: "\u7121\u6CD5\u6383\u63CF QR Code\uFF1F",
        infoText__ableToScan: "\u5728\u60A8\u7684\u9A57\u8B49\u5668\u61C9\u7528\u7A0B\u5F0F\u4E2D\u8A2D\u5B9A\u65B0\u7684\u767B\u5165\u65B9\u5F0F\uFF0C\u4E26\u6383\u63CF\u4E0B\u65B9\u7684 QR Code \u4EE5\u9023\u7D50\u60A8\u7684\u5E33\u6236\u3002",
        infoText__unableToScan: "\u5728\u9A57\u8B49\u5668\u4E2D\u8A2D\u5B9A\u65B0\u7684\u767B\u5165\u65B9\u5F0F\uFF0C\u4E26\u8F38\u5165\u4E0B\u65B9\u63D0\u4F9B\u7684\u91D1\u9470 (Key)\u3002",
        inputLabel__unableToScan1: "\u8ACB\u78BA\u8A8D\u5DF2\u555F\u7528 TOTP (\u57FA\u65BC\u6642\u9593\u7684\u4E00\u6B21\u6027\u5BC6\u78BC)\uFF0C\u7136\u5F8C\u5B8C\u6210\u5E33\u6236\u9023\u7D50\u3002",
        inputLabel__unableToScan2: "\u6216\u8005\uFF0C\u82E5\u60A8\u7684\u9A57\u8B49\u5668\u652F\u63F4 TOTP URI\uFF0C\u60A8\u4E5F\u53EF\u4EE5\u8907\u88FD\u5B8C\u6574\u7684 URI\u3002"
      },
      removeResource: {
        messageLine1: "\u767B\u5165\u6642\u5C07\u4E0D\u518D\u9700\u8981\u8F38\u5165\u6B64\u9A57\u8B49\u5668\u7684\u9A57\u8B49\u78BC\u3002",
        messageLine2: "\u60A8\u7684\u5E33\u6236\u5B89\u5168\u6027\u53EF\u80FD\u6703\u964D\u4F4E\uFF0C\u78BA\u5B9A\u8981\u7E7C\u7E8C\u55CE\uFF1F",
        successMessage: "\u5DF2\u79FB\u9664\u900F\u904E\u9A57\u8B49\u5668\u61C9\u7528\u7A0B\u5F0F\u7684\u5169\u6B65\u9A5F\u9A57\u8B49\u3002",
        title: "\u79FB\u9664\u5169\u6B65\u9A5F\u9A57\u8B49"
      },
      successMessage: "\u5169\u6B65\u9A5F\u9A57\u8B49\u73FE\u5DF2\u555F\u7528\u3002\u767B\u5165\u6642\uFF0C\u60A8\u9700\u8981\u8F38\u5165\u6B64\u9A57\u8B49\u5668\u7522\u751F\u7684\u9A57\u8B49\u78BC\u4F5C\u70BA\u984D\u5916\u6B65\u9A5F\u3002",
      title: "\u65B0\u589E\u9A57\u8B49\u5668\u61C9\u7528\u7A0B\u5F0F",
      verifySubtitle: "\u8ACB\u8F38\u5165\u60A8\u7684\u9A57\u8B49\u5668\u7522\u751F\u7684\u9A57\u8B49\u78BC",
      verifyTitle: "\u9A57\u8B49\u78BC"
    },
    mobileButton__menu: "\u9078\u55AE",
    navbar: {
      account: "\u500B\u4EBA\u8CC7\u6599",
      apiKeys: void 0,
      billing: void 0,
      description: "\u7BA1\u7406\u60A8\u7684\u5E33\u6236\u8CC7\u8A0A\u3002",
      security: "\u5B89\u5168\u6027",
      title: "\u5E33\u6236"
    },
    passkeyScreen: {
      removeResource: {
        messageLine1: void 0,
        title: void 0
      },
      subtitle__rename: void 0,
      title__rename: void 0
    },
    passwordPage: {
      checkboxInfoText__signOutOfOtherSessions: "\u5EFA\u8B70\u60A8\u767B\u51FA\u6240\u6709\u53EF\u80FD\u4F7F\u7528\u904E\u820A\u5BC6\u78BC\u7684\u5176\u4ED6\u88DD\u7F6E\u3002",
      readonly: "\u60A8\u76EE\u524D\u7121\u6CD5\u7DE8\u8F2F\u5BC6\u78BC\uFF0C\u56E0\u70BA\u60A8\u53EA\u80FD\u900F\u904E\u4F01\u696D\u9023\u7DDA\u767B\u5165\u3002",
      successMessage__set: "\u60A8\u7684\u5BC6\u78BC\u5DF2\u8A2D\u5B9A\u3002",
      successMessage__signOutOfOtherSessions: "\u5DF2\u767B\u51FA\u6240\u6709\u5176\u4ED6\u88DD\u7F6E\u3002",
      successMessage__update: "\u60A8\u7684\u5BC6\u78BC\u5DF2\u66F4\u65B0\u3002",
      title__set: "\u8A2D\u5B9A\u5BC6\u78BC",
      title__update: "\u8B8A\u66F4\u5BC6\u78BC"
    },
    phoneNumberPage: {
      infoText: "\u7CFB\u7D71\u5C07\u6703\u50B3\u9001\u4E00\u5C01\u5305\u542B\u9A57\u8B49\u9023\u7D50\u7684\u7C21\u8A0A\u81F3\u6B64\u96FB\u8A71\u865F\u78BC\u3002",
      removeResource: {
        messageLine1: "{{identifier}} \u5C07\u5F9E\u6B64\u5E33\u6236\u4E2D\u88AB\u79FB\u9664\u3002",
        messageLine2: "\u60A8\u5C07\u7121\u6CD5\u518D\u4F7F\u7528\u6B64\u96FB\u8A71\u865F\u78BC\u767B\u5165\u3002",
        successMessage: "\u96FB\u8A71\u865F\u78BC {{phoneNumber}} \u5DF2\u5F9E\u60A8\u7684\u5E33\u6236\u4E2D\u79FB\u9664\u3002",
        title: "\u79FB\u9664\u96FB\u8A71\u865F\u78BC"
      },
      successMessage: "{{identifier}} \u5DF2\u65B0\u589E\u81F3\u60A8\u7684\u5E33\u6236\u3002",
      title: "\u65B0\u589E\u96FB\u8A71\u865F\u78BC",
      verifySubtitle: "\u8ACB\u8F38\u5165\u50B3\u9001\u81F3 {{identifier}} \u7684\u9A57\u8B49\u78BC",
      verifyTitle: "\u9A57\u8B49\u96FB\u8A71\u865F\u78BC"
    },
    plansPage: {
      title: void 0
    },
    profilePage: {
      fileDropAreaHint: "\u4E0A\u50B3 JPG, PNG, GIF, \u6216 WEBP \u683C\u5F0F\u5716\u7247\uFF08\u6A94\u6848\u5927\u5C0F\u4E0A\u9650 10MB\uFF09",
      imageFormDestructiveActionSubtitle: "\u79FB\u9664\u5716\u7247",
      imageFormSubtitle: "\u4E0A\u50B3\u5716\u7247",
      imageFormTitle: "\u500B\u4EBA\u8CC7\u6599\u5716\u7247",
      readonly: "\u60A8\u7684\u500B\u4EBA\u8CC7\u6599\u7531\u4F01\u696D\u9023\u7DDA\u63D0\u4F9B\uFF0C\u7121\u6CD5\u7DE8\u8F2F\u3002",
      successMessage: "\u60A8\u7684\u500B\u4EBA\u8CC7\u6599\u5DF2\u66F4\u65B0\u3002",
      title: "\u66F4\u65B0\u500B\u4EBA\u8CC7\u6599"
    },
    start: {
      activeDevicesSection: {
        destructiveAction: "\u767B\u51FA\u88DD\u7F6E",
        title: "\u5DF2\u767B\u5165\u7684\u88DD\u7F6E"
      },
      connectedAccountsSection: {
        actionLabel__connectionFailed: "\u518D\u8A66\u4E00\u6B21",
        actionLabel__reauthorize: "\u7ACB\u5373\u91CD\u65B0\u6388\u6B0A",
        destructiveActionTitle: "\u79FB\u9664",
        primaryButton: "\u9023\u7D50\u5E33\u6236",
        subtitle__disconnected: void 0,
        subtitle__reauthorize: "\u5FC5\u8981\u7684\u6B0A\u9650\u7BC4\u570D\u5DF2\u66F4\u65B0\uFF0C\u60A8\u53EF\u80FD\u6703\u9047\u5230\u529F\u80FD\u53D7\u9650\u7684\u60C5\u6CC1\u3002\u8ACB\u91CD\u65B0\u6388\u6B0A\u6B64\u61C9\u7528\u7A0B\u5F0F\u4EE5\u907F\u514D\u4EFB\u4F55\u554F\u984C\u3002",
        title: "\u5DF2\u9023\u7D50\u7684\u5E33\u6236"
      },
      dangerSection: {
        deleteAccountButton: "\u522A\u9664\u5E33\u6236",
        title: "\u5371\u96AA"
      },
      emailAddressesSection: {
        destructiveAction: "\u79FB\u9664\u96FB\u5B50\u90F5\u4EF6\u5730\u5740",
        detailsAction__nonPrimary: "\u8A2D\u70BA\u4E3B\u8981",
        detailsAction__primary: "\u5B8C\u6210\u9A57\u8B49",
        detailsAction__unverified: "\u5B8C\u6210\u9A57\u8B49",
        primaryButton: "\u65B0\u589E\u96FB\u5B50\u90F5\u4EF6\u5730\u5740",
        title: "\u96FB\u5B50\u90F5\u4EF6\u5730\u5740"
      },
      enterpriseAccountsSection: {
        title: "\u4F01\u696D\u5E33\u6236"
      },
      headerTitle__account: "\u5E33\u6236",
      headerTitle__security: "\u5B89\u5168\u6027",
      mfaSection: {
        backupCodes: {
          actionLabel__regenerate: "\u91CD\u65B0\u7522\u751F\u4EE3\u78BC",
          headerTitle: "\u5099\u7528\u78BC",
          subtitle__regenerate: "\u53D6\u5F97\u4E00\u7D44\u65B0\u7684\u5B89\u5168\u5099\u7528\u78BC\u3002\u5148\u524D\u7522\u751F\u7684\u5099\u7528\u78BC\u5C07\u5931\u6548\u3002",
          title__regenerate: "\u91CD\u65B0\u7522\u751F\u5099\u7528\u78BC"
        },
        phoneCode: {
          actionLabel__setDefault: "\u8A2D\u70BA\u9810\u8A2D",
          destructiveActionLabel: "\u79FB\u9664\u96FB\u8A71\u865F\u78BC"
        },
        primaryButton: "\u65B0\u589E\u5169\u6B65\u9A5F\u9A57\u8B49",
        title: "\u5169\u6B65\u9A5F\u9A57\u8B49",
        totp: {
          destructiveActionTitle: "\u79FB\u9664",
          headerTitle: "\u9A57\u8B49\u5668\u61C9\u7528\u7A0B\u5F0F"
        }
      },
      passkeysSection: {
        menuAction__destructive: void 0,
        menuAction__rename: void 0,
        primaryButton: void 0,
        title: void 0
      },
      passwordSection: {
        primaryButton__setPassword: "\u8A2D\u5B9A\u5BC6\u78BC",
        primaryButton__updatePassword: "\u8B8A\u66F4\u5BC6\u78BC",
        title: "\u5BC6\u78BC"
      },
      phoneNumbersSection: {
        destructiveAction: "\u79FB\u9664\u96FB\u8A71\u865F\u78BC",
        detailsAction__nonPrimary: "\u8A2D\u70BA\u4E3B\u8981",
        detailsAction__primary: "\u5B8C\u6210\u9A57\u8B49",
        detailsAction__unverified: "\u5B8C\u6210\u9A57\u8B49",
        primaryButton: "\u65B0\u589E\u96FB\u8A71\u865F\u78BC",
        title: "\u96FB\u8A71\u865F\u78BC"
      },
      profileSection: {
        primaryButton: "\u66F4\u65B0\u500B\u4EBA\u8CC7\u6599",
        title: "\u500B\u4EBA\u8CC7\u6599"
      },
      usernameSection: {
        primaryButton__setUsername: "\u8A2D\u5B9A\u4F7F\u7528\u8005\u540D\u7A31",
        primaryButton__updateUsername: "\u8B8A\u66F4\u4F7F\u7528\u8005\u540D\u7A31",
        title: "\u4F7F\u7528\u8005\u540D\u7A31"
      },
      web3WalletsSection: {
        destructiveAction: "\u79FB\u9664\u9322\u5305",
        detailsAction__nonPrimary: void 0,
        primaryButton: "\u65B0\u589E Web3 \u9322\u5305",
        title: "Web3 \u9322\u5305"
      }
    },
    usernamePage: {
      successMessage: "\u60A8\u7684\u4F7F\u7528\u8005\u540D\u7A31\u5DF2\u66F4\u65B0\u3002",
      title__set: "\u8A2D\u5B9A\u4F7F\u7528\u8005\u540D\u7A31",
      title__update: "\u8B8A\u66F4\u4F7F\u7528\u8005\u540D\u7A31"
    },
    web3WalletPage: {
      removeResource: {
        messageLine1: "{{identifier}} \u5C07\u5F9E\u6B64\u5E33\u6236\u4E2D\u88AB\u79FB\u9664\u3002",
        messageLine2: "\u60A8\u5C07\u7121\u6CD5\u518D\u4F7F\u7528\u6B64 web3 \u9322\u5305\u767B\u5165\u3002",
        successMessage: "{{web3Wallet}} \u5DF2\u5F9E\u60A8\u7684\u5E33\u6236\u4E2D\u79FB\u9664\u3002",
        title: "\u79FB\u9664 Web3 \u9322\u5305"
      },
      subtitle__availableWallets: "\u9078\u64C7\u4E00\u500B web3 \u9322\u5305\u4EE5\u9023\u7D50\u60A8\u7684\u5E33\u6236\u3002",
      subtitle__unavailableWallets: "\u76EE\u524D\u6C92\u6709\u53EF\u7528\u7684 web3 \u9322\u5305\u3002",
      successMessage: "\u9322\u5305\u5DF2\u65B0\u589E\u81F3\u60A8\u7684\u5E33\u6236\u3002",
      title: "\u65B0\u589E web3 \u9322\u5305",
      web3WalletButtonsBlockButton: void 0
    }
  },
  waitlist: {
    start: {
      actionLink: void 0,
      actionText: void 0,
      formButton: void 0,
      subtitle: void 0,
      title: void 0
    },
    success: {
      message: void 0,
      subtitle: void 0,
      title: void 0
    }
  }
};
// Annotate the CommonJS export names for ESM import in node:
0 && (module.exports = {
  zhTW
});
